{
 "cells": [
  {
   "cell_type": "markdown",
   "id": "21eb6c08-fe92-470d-8ec9-f28ea33bf099",
   "metadata": {},
   "source": [
    "Purpose:\n",
    "A deep mutational scanning experiment was performed on the Hrd1 ubiquitin ligase to determine residues required for wild-type function and find mutations specifically required for ERAD-L degradation.\n",
    "The notebook below shows the pre-processing of sequencing reads from an Illumina 2x300bp MiSeq run. The libraries were prepared using amplicon sequencing across 5 regions (~110 amino acids each) of Hrd1. Nextera adaptors used and, on each amplicon primer is a 8 “N” nucleotide sequence on the 3’ side of the Nextera adaptor.\n",
    "\n",
    "0)Input is demultiplexed fastq files provided by Azenta (formally Genewiz). \n",
    "1)Cutadapt to trim Nextera adaptors and remove the 8 “N” nucleotides on each read\n",
    "2)Fastp to merge reads into single contig\n",
    "3)Bowtie to align to reference sequence\n",
    "4)Samtools to convert into bam and sort on name"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "id": "452b7a9c-fa94-420d-8f3a-334b7d111862",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "cutadapt version:\n",
      "1.18\n",
      "\n",
      "fastp version:\n",
      "fastp 0.20.0\n",
      "\n",
      "bowtie2 version:\n",
      "/usr/bin/bowtie2-align-s version 2.3.5.1\n",
      "64-bit\n",
      "\n",
      "samtools version:\n",
      "samtools 1.10\n",
      "Using htslib 1.10.2-3\n",
      "Copyright (C) 2019 Genome Research Ltd.\n"
     ]
    }
   ],
   "source": [
    "## Output versions of software used\n",
    "print('cutadapt version:')\n",
    "!cutadapt --version\n",
    "print('\\nfastp version:')\n",
    "!fastp --version\n",
    "print('\\nbowtie2 version:')\n",
    "!bowtie2 --version\n",
    "print('\\nsamtools version:')\n",
    "!samtools --version"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "id": "779ff594-e35c-453c-9070-429a449c9c5f",
   "metadata": {
    "tags": []
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 1.18 with Python 3.7.16\n",
      "Command line parameters: -a CTGTCTCTTATACACATCT -A CTGTCTCTTATACACATCT -u 8 -U 8 -q 30 -o 1_cutadapt/Hrd1_DMS_Rep1_Region1_Input_R1_cut.fastq.gz -p 1_cutadapt/Hrd1_DMS_Rep1_Region1_Input_R2_cut.fastq.gz --cores=12 0_fastq_files/Hrd1_DMS_Rep1_Region1_Input_R1_001.fastq.gz 0_fastq_files/Hrd1_DMS_Rep1_Region1_Input_R2_001.fastq.gz\n",
      "Processing reads on 12 cores in paired-end mode ...\n",
      "Finished in 1.55 s (9 us/read; 6.78 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            174,667\n",
      "  Read 1 with adapter:                   9,614 (5.5%)\n",
      "  Read 2 with adapter:                  10,838 (6.2%)\n",
      "Pairs written (passing filters):       174,667 (100.0%)\n",
      "\n",
      "Total basepairs processed:   104,775,635 bp\n",
      "  Read 1:    52,388,959 bp\n",
      "  Read 2:    52,386,676 bp\n",
      "Quality-trimmed:               5,640,445 bp (5.4%)\n",
      "  Read 1:     1,706,578 bp\n",
      "  Read 2:     3,933,867 bp\n",
      "Total written (filtered):     95,001,727 bp (90.7%)\n",
      "  Read 1:    48,565,374 bp\n",
      "  Read 2:    46,436,353 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: CTGTCTCTTATACACATCT; Type: regular 3'; Length: 19; Trimmed: 9614 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 32.2%\n",
      "  C: 25.8%\n",
      "  G: 17.2%\n",
      "  T: 24.7%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t77\t2729.2\t0\t77\n",
      "4\t3\t682.3\t0\t3\n",
      "9\t2\t0.7\t0\t1 1\n",
      "10\t1\t0.2\t1\t1\n",
      "11\t2\t0.0\t1\t2\n",
      "12\t2\t0.0\t1\t1 1\n",
      "13\t3\t0.0\t1\t3\n",
      "14\t1\t0.0\t1\t1\n",
      "17\t4\t0.0\t1\t4\n",
      "18\t2\t0.0\t1\t2\n",
      "19\t10\t0.0\t1\t10\n",
      "20\t7\t0.0\t1\t6 1\n",
      "21\t4\t0.0\t1\t4\n",
      "22\t2\t0.0\t1\t2\n",
      "23\t2\t0.0\t1\t2\n",
      "25\t1\t0.0\t1\t1\n",
      "27\t4\t0.0\t1\t4\n",
      "28\t3\t0.0\t1\t3\n",
      "29\t2\t0.0\t1\t2\n",
      "30\t4\t0.0\t1\t4\n",
      "31\t1\t0.0\t1\t1\n",
      "33\t2\t0.0\t1\t2\n",
      "34\t4\t0.0\t1\t3 1\n",
      "35\t5\t0.0\t1\t4 1\n",
      "36\t14\t0.0\t1\t13 1\n",
      "37\t9\t0.0\t1\t8 1\n",
      "38\t21\t0.0\t1\t20 1\n",
      "39\t15\t0.0\t1\t15\n",
      "40\t31\t0.0\t1\t29 2\n",
      "41\t18\t0.0\t1\t18\n",
      "42\t15\t0.0\t1\t12 3\n",
      "43\t89\t0.0\t1\t88 1\n",
      "44\t4\t0.0\t1\t3 1\n",
      "45\t1\t0.0\t1\t1\n",
      "47\t1\t0.0\t1\t0 1\n",
      "48\t2\t0.0\t1\t1 1\n",
      "49\t3\t0.0\t1\t3\n",
      "50\t4\t0.0\t1\t3 1\n",
      "51\t4\t0.0\t1\t4\n",
      "52\t4\t0.0\t1\t3 1\n",
      "53\t7\t0.0\t1\t7\n",
      "54\t5\t0.0\t1\t4 1\n",
      "55\t4\t0.0\t1\t3 1\n",
      "56\t13\t0.0\t1\t13\n",
      "57\t9\t0.0\t1\t9\n",
      "58\t10\t0.0\t1\t10\n",
      "59\t16\t0.0\t1\t16\n",
      "60\t15\t0.0\t1\t13 2\n",
      "61\t9\t0.0\t1\t8 1\n",
      "62\t11\t0.0\t1\t11\n",
      "63\t13\t0.0\t1\t12 1\n",
      "64\t17\t0.0\t1\t16 1\n",
      "65\t26\t0.0\t1\t23 3\n",
      "66\t41\t0.0\t1\t40 1\n",
      "67\t25\t0.0\t1\t24 1\n",
      "68\t29\t0.0\t1\t28 1\n",
      "69\t24\t0.0\t1\t23 1\n",
      "70\t40\t0.0\t1\t39 1\n",
      "71\t94\t0.0\t1\t92 2\n",
      "72\t114\t0.0\t1\t112 2\n",
      "73\t105\t0.0\t1\t101 4\n",
      "74\t189\t0.0\t1\t188 1\n",
      "75\t259\t0.0\t1\t235 24\n",
      "76\t5836\t0.0\t1\t5764 72\n",
      "77\t1898\t0.0\t1\t1867 31\n",
      "78\t262\t0.0\t1\t255 7\n",
      "79\t42\t0.0\t1\t42\n",
      "80\t14\t0.0\t1\t14\n",
      "81\t13\t0.0\t1\t13\n",
      "82\t4\t0.0\t1\t4\n",
      "83\t4\t0.0\t1\t3 1\n",
      "84\t2\t0.0\t1\t2\n",
      "88\t1\t0.0\t1\t1\n",
      "90\t1\t0.0\t1\t1\n",
      "108\t1\t0.0\t1\t1\n",
      "109\t1\t0.0\t1\t1\n",
      "119\t1\t0.0\t1\t1\n",
      "129\t2\t0.0\t1\t2\n",
      "130\t2\t0.0\t1\t2\n",
      "142\t1\t0.0\t1\t1\n",
      "148\t1\t0.0\t1\t1\n",
      "150\t1\t0.0\t1\t1\n",
      "152\t3\t0.0\t1\t2 1\n",
      "155\t1\t0.0\t1\t1\n",
      "158\t1\t0.0\t1\t1\n",
      "159\t3\t0.0\t1\t3\n",
      "160\t1\t0.0\t1\t1\n",
      "161\t1\t0.0\t1\t1\n",
      "174\t2\t0.0\t1\t2\n",
      "177\t1\t0.0\t1\t1\n",
      "179\t1\t0.0\t1\t0 1\n",
      "183\t1\t0.0\t1\t1\n",
      "185\t1\t0.0\t1\t1\n",
      "188\t1\t0.0\t1\t1\n",
      "201\t1\t0.0\t1\t0 1\n",
      "207\t1\t0.0\t1\t1\n",
      "208\t1\t0.0\t1\t1\n",
      "213\t2\t0.0\t1\t2\n",
      "214\t1\t0.0\t1\t1\n",
      "216\t4\t0.0\t1\t3 1\n",
      "217\t23\t0.0\t1\t22 1\n",
      "218\t17\t0.0\t1\t14 3\n",
      "219\t2\t0.0\t1\t2\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CTGTCTCTTATACACATCT; Type: regular 3'; Length: 19; Trimmed: 10838 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 28.3%\n",
      "  C: 22.5%\n",
      "  G: 27.1%\n",
      "  T: 22.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1305\t2729.2\t0\t1305\n",
      "4\t9\t682.3\t0\t9\n",
      "5\t3\t170.6\t0\t3\n",
      "6\t4\t42.6\t0\t4\n",
      "8\t2\t2.7\t0\t2\n",
      "9\t1\t0.7\t0\t1\n",
      "10\t1\t0.2\t1\t1\n",
      "11\t2\t0.0\t1\t2\n",
      "12\t2\t0.0\t1\t2\n",
      "13\t1\t0.0\t1\t0 1\n",
      "14\t1\t0.0\t1\t1\n",
      "15\t4\t0.0\t1\t4\n",
      "16\t1\t0.0\t1\t0 1\n",
      "17\t1\t0.0\t1\t1\n",
      "18\t1\t0.0\t1\t1\n",
      "19\t18\t0.0\t1\t17 1\n",
      "20\t10\t0.0\t1\t10\n",
      "21\t7\t0.0\t1\t7\n",
      "22\t16\t0.0\t1\t16\n",
      "23\t4\t0.0\t1\t3 1\n",
      "24\t5\t0.0\t1\t5\n",
      "25\t11\t0.0\t1\t10 1\n",
      "26\t8\t0.0\t1\t8\n",
      "27\t7\t0.0\t1\t7\n",
      "28\t19\t0.0\t1\t17 2\n",
      "29\t32\t0.0\t1\t30 2\n",
      "30\t31\t0.0\t1\t30 1\n",
      "31\t37\t0.0\t1\t37\n",
      "32\t44\t0.0\t1\t43 1\n",
      "33\t33\t0.0\t1\t29 4\n",
      "34\t197\t0.0\t1\t190 7\n",
      "35\t8\t0.0\t1\t7 1\n",
      "36\t8\t0.0\t1\t7 1\n",
      "37\t28\t0.0\t1\t26 2\n",
      "38\t35\t0.0\t1\t34 1\n",
      "39\t39\t0.0\t1\t38 1\n",
      "40\t9\t0.0\t1\t7 2\n",
      "41\t32\t0.0\t1\t31 1\n",
      "42\t46\t0.0\t1\t44 2\n",
      "43\t196\t0.0\t1\t192 4\n",
      "44\t85\t0.0\t1\t82 3\n",
      "45\t183\t0.0\t1\t181 2\n",
      "46\t277\t0.0\t1\t273 4\n",
      "47\t213\t0.0\t1\t209 4\n",
      "48\t212\t0.0\t1\t211 1\n",
      "49\t189\t0.0\t1\t188 1\n",
      "50\t117\t0.0\t1\t115 2\n",
      "51\t137\t0.0\t1\t132 5\n",
      "52\t220\t0.0\t1\t220\n",
      "53\t123\t0.0\t1\t121 2\n",
      "54\t300\t0.0\t1\t296 4\n",
      "55\t203\t0.0\t1\t198 5\n",
      "56\t128\t0.0\t1\t122 6\n",
      "57\t368\t0.0\t1\t362 6\n",
      "58\t310\t0.0\t1\t308 2\n",
      "59\t323\t0.0\t1\t320 3\n",
      "60\t145\t0.0\t1\t145\n",
      "61\t43\t0.0\t1\t42 1\n",
      "62\t119\t0.0\t1\t113 6\n",
      "63\t317\t0.0\t1\t313 4\n",
      "64\t60\t0.0\t1\t58 2\n",
      "65\t20\t0.0\t1\t20\n",
      "66\t8\t0.0\t1\t8\n",
      "67\t27\t0.0\t1\t27\n",
      "68\t6\t0.0\t1\t6\n",
      "69\t11\t0.0\t1\t11\n",
      "70\t18\t0.0\t1\t17 1\n",
      "71\t7\t0.0\t1\t6 1\n",
      "72\t7\t0.0\t1\t7\n",
      "73\t7\t0.0\t1\t7\n",
      "74\t2\t0.0\t1\t2\n",
      "75\t50\t0.0\t1\t50\n",
      "76\t31\t0.0\t1\t30 1\n",
      "77\t85\t0.0\t1\t84 1\n",
      "78\t291\t0.0\t1\t283 8\n",
      "79\t405\t0.0\t1\t392 13\n",
      "80\t1819\t0.0\t1\t1798 21\n",
      "81\t1225\t0.0\t1\t1216 9\n",
      "82\t403\t0.0\t1\t396 7\n",
      "83\t102\t0.0\t1\t100 2\n",
      "84\t17\t0.0\t1\t16 1\n",
      "85\t3\t0.0\t1\t2 1\n",
      "86\t1\t0.0\t1\t1\n",
      "218\t3\t0.0\t1\t3\n",
      "\n",
      "Read1 before filtering:\n",
      "total reads: 174667\n",
      "total bases: 48565374\n",
      "Q20 bases: 48334068(99.5237%)\n",
      "Q30 bases: 47694872(98.2076%)\n",
      "\n",
      "Read2 before filtering:\n",
      "total reads: 174667\n",
      "total bases: 46436353\n",
      "Q20 bases: 46088875(99.2517%)\n",
      "Q30 bases: 44977664(96.8587%)\n",
      "\n",
      "Merged and filtered:\n",
      "total reads: 171197\n",
      "total bases: 63520314\n",
      "Q20 bases: 63323374(99.69%)\n",
      "Q30 bases: 62672603(98.6654%)\n",
      "\n",
      "Filtering result:\n",
      "reads passed filter: 347536\n",
      "reads failed due to low quality: 240\n",
      "reads failed due to too many N: 38\n",
      "reads failed due to too short: 1520\n",
      "reads with adapter trimmed: 19092\n",
      "bases trimmed due to adapters: 156436\n",
      "reads corrected by overlap analysis: 14337\n",
      "bases corrected by overlap analysis: 20130\n",
      "\n",
      "Duplication rate: 86.7958%\n",
      "\n",
      "Insert size peak (evaluated by paired-end reads): 390\n",
      "\n",
      "Read pairs merged: 171197\n",
      "% of original read pairs: 98.0134%\n",
      "% in reads after filtering: 100%\n",
      "\n",
      "\n",
      "JSON report: 2_fastp/Hrd1_DMS_Rep1_Region1_Input_report.json\n",
      "HTML report: 2_fastp/Hrd1_DMS_Rep1_Region1_Input_report.html\n",
      "\n",
      "fastp -i 1_cutadapt/Hrd1_DMS_Rep1_Region1_Input_R1_cut.fastq.gz -I 1_cutadapt/Hrd1_DMS_Rep1_Region1_Input_R2_cut.fastq.gz -m --merged_out 2_fastp/Hrd1_DMS_Rep1_Region1_Input_merge.fastq.gz -o 2_fastp/Hrd1_DMS_Rep1_Region1_Input_R1_cut.fastq.gz -O 2_fastp/Hrd1_DMS_Rep1_Region1_Input_R2_cut.fastq.gz -w 12 -h 2_fastp/Hrd1_DMS_Rep1_Region1_Input_report.html -j 2_fastp/Hrd1_DMS_Rep1_Region1_Input_report.json \n",
      "fastp v0.20.0, time used: 2 seconds\n",
      "171197 reads; of these:\n",
      "  171197 (100.00%) were unpaired; of these:\n",
      "    9539 (5.57%) aligned 0 times\n",
      "    161658 (94.43%) aligned exactly 1 time\n",
      "    0 (0.00%) aligned >1 times\n",
      "94.43% overall alignment rate\n",
      "[bam_sort_core] merging from 0 files and 12 in-memory blocks...\n",
      "This is cutadapt 1.18 with Python 3.7.16\n",
      "Command line parameters: -a CTGTCTCTTATACACATCT -A CTGTCTCTTATACACATCT -u 8 -U 8 -q 30 -o 1_cutadapt/Hrd1_DMS_Rep2_Region1_Input_R1_cut.fastq.gz -p 1_cutadapt/Hrd1_DMS_Rep2_Region1_Input_R2_cut.fastq.gz --cores=12 0_fastq_files/Hrd1_DMS_Rep2_Region1_Input_R1_001.fastq.gz 0_fastq_files/Hrd1_DMS_Rep2_Region1_Input_R2_001.fastq.gz\n",
      "Processing reads on 12 cores in paired-end mode ...\n",
      "Finished in 1.45 s (10 us/read; 6.30 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            151,730\n",
      "  Read 1 with adapter:                   4,709 (3.1%)\n",
      "  Read 2 with adapter:                   4,874 (3.2%)\n",
      "Pairs written (passing filters):       151,730 (100.0%)\n",
      "\n",
      "Total basepairs processed:    91,022,165 bp\n",
      "  Read 1:    45,510,769 bp\n",
      "  Read 2:    45,511,396 bp\n",
      "Quality-trimmed:               3,907,515 bp (4.3%)\n",
      "  Read 1:     1,070,830 bp\n",
      "  Read 2:     2,836,685 bp\n",
      "Total written (filtered):     84,074,877 bp (92.4%)\n",
      "  Read 1:    42,888,618 bp\n",
      "  Read 2:    41,186,259 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: CTGTCTCTTATACACATCT; Type: regular 3'; Length: 19; Trimmed: 4709 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 34.3%\n",
      "  C: 21.6%\n",
      "  G: 21.8%\n",
      "  T: 22.3%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t305\t2370.8\t0\t305\n",
      "4\t1\t592.7\t0\t1\n",
      "5\t2\t148.2\t0\t2\n",
      "10\t1\t0.1\t1\t1\n",
      "11\t1\t0.0\t1\t1\n",
      "12\t1\t0.0\t1\t1\n",
      "13\t1\t0.0\t1\t1\n",
      "14\t1\t0.0\t1\t1\n",
      "16\t2\t0.0\t1\t2\n",
      "17\t1\t0.0\t1\t1\n",
      "18\t1\t0.0\t1\t1\n",
      "19\t2\t0.0\t1\t1 1\n",
      "20\t3\t0.0\t1\t2 1\n",
      "21\t1\t0.0\t1\t0 1\n",
      "22\t1\t0.0\t1\t1\n",
      "23\t1\t0.0\t1\t1\n",
      "24\t1\t0.0\t1\t1\n",
      "26\t1\t0.0\t1\t1\n",
      "27\t2\t0.0\t1\t2\n",
      "28\t1\t0.0\t1\t1\n",
      "29\t1\t0.0\t1\t1\n",
      "30\t1\t0.0\t1\t1\n",
      "31\t2\t0.0\t1\t2\n",
      "33\t1\t0.0\t1\t1\n",
      "34\t1\t0.0\t1\t1\n",
      "35\t1\t0.0\t1\t1\n",
      "36\t2\t0.0\t1\t1 1\n",
      "37\t1\t0.0\t1\t0 1\n",
      "38\t1\t0.0\t1\t1\n",
      "39\t5\t0.0\t1\t5\n",
      "40\t1\t0.0\t1\t1\n",
      "41\t5\t0.0\t1\t5\n",
      "42\t7\t0.0\t1\t7\n",
      "43\t29\t0.0\t1\t28 1\n",
      "44\t1\t0.0\t1\t1\n",
      "45\t1\t0.0\t1\t1\n",
      "47\t1\t0.0\t1\t1\n",
      "49\t1\t0.0\t1\t1\n",
      "50\t2\t0.0\t1\t2\n",
      "51\t1\t0.0\t1\t1\n",
      "53\t1\t0.0\t1\t1\n",
      "54\t7\t0.0\t1\t7\n",
      "55\t3\t0.0\t1\t3\n",
      "56\t3\t0.0\t1\t3\n",
      "57\t2\t0.0\t1\t1 1\n",
      "58\t7\t0.0\t1\t7\n",
      "59\t9\t0.0\t1\t9\n",
      "60\t7\t0.0\t1\t6 1\n",
      "61\t1\t0.0\t1\t1\n",
      "62\t7\t0.0\t1\t6 1\n",
      "63\t5\t0.0\t1\t5\n",
      "64\t2\t0.0\t1\t1 1\n",
      "65\t13\t0.0\t1\t13\n",
      "66\t24\t0.0\t1\t22 2\n",
      "67\t9\t0.0\t1\t9\n",
      "68\t13\t0.0\t1\t12 1\n",
      "69\t6\t0.0\t1\t6\n",
      "70\t21\t0.0\t1\t21\n",
      "71\t39\t0.0\t1\t39\n",
      "72\t49\t0.0\t1\t49\n",
      "73\t52\t0.0\t1\t52\n",
      "74\t93\t0.0\t1\t91 2\n",
      "75\t138\t0.0\t1\t122 16\n",
      "76\t2828\t0.0\t1\t2782 46\n",
      "77\t783\t0.0\t1\t761 22\n",
      "78\t102\t0.0\t1\t99 3\n",
      "79\t15\t0.0\t1\t14 1\n",
      "80\t5\t0.0\t1\t5\n",
      "81\t7\t0.0\t1\t7\n",
      "82\t2\t0.0\t1\t2\n",
      "84\t1\t0.0\t1\t1\n",
      "85\t1\t0.0\t1\t1\n",
      "90\t1\t0.0\t1\t1\n",
      "97\t1\t0.0\t1\t1\n",
      "123\t1\t0.0\t1\t1\n",
      "126\t1\t0.0\t1\t1\n",
      "129\t1\t0.0\t1\t1\n",
      "140\t1\t0.0\t1\t1\n",
      "142\t1\t0.0\t1\t0 1\n",
      "144\t1\t0.0\t1\t1\n",
      "146\t1\t0.0\t1\t1\n",
      "147\t1\t0.0\t1\t1\n",
      "150\t1\t0.0\t1\t1\n",
      "156\t1\t0.0\t1\t1\n",
      "166\t1\t0.0\t1\t1\n",
      "167\t1\t0.0\t1\t1\n",
      "170\t1\t0.0\t1\t1\n",
      "173\t1\t0.0\t1\t1\n",
      "183\t2\t0.0\t1\t2\n",
      "184\t1\t0.0\t1\t1\n",
      "193\t1\t0.0\t1\t0 1\n",
      "200\t1\t0.0\t1\t1\n",
      "215\t1\t0.0\t1\t1\n",
      "216\t2\t0.0\t1\t2\n",
      "217\t16\t0.0\t1\t16\n",
      "218\t18\t0.0\t1\t16 2\n",
      "232\t1\t0.0\t1\t1\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CTGTCTCTTATACACATCT; Type: regular 3'; Length: 19; Trimmed: 4874 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 39.9%\n",
      "  C: 25.0%\n",
      "  G: 15.8%\n",
      "  T: 19.3%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t489\t2370.8\t0\t489\n",
      "4\t7\t592.7\t0\t7\n",
      "5\t1\t148.2\t0\t1\n",
      "8\t3\t2.3\t0\t3\n",
      "10\t2\t0.1\t1\t2\n",
      "11\t4\t0.0\t1\t4\n",
      "12\t3\t0.0\t1\t3\n",
      "14\t2\t0.0\t1\t2\n",
      "15\t1\t0.0\t1\t1\n",
      "16\t1\t0.0\t1\t1\n",
      "17\t1\t0.0\t1\t1\n",
      "18\t1\t0.0\t1\t1\n",
      "19\t14\t0.0\t1\t12 2\n",
      "20\t1\t0.0\t1\t1\n",
      "21\t4\t0.0\t1\t3 1\n",
      "22\t8\t0.0\t1\t7 1\n",
      "23\t9\t0.0\t1\t8 1\n",
      "24\t6\t0.0\t1\t6\n",
      "25\t6\t0.0\t1\t5 1\n",
      "26\t6\t0.0\t1\t6\n",
      "27\t4\t0.0\t1\t4\n",
      "28\t5\t0.0\t1\t5\n",
      "29\t10\t0.0\t1\t8 2\n",
      "30\t15\t0.0\t1\t14 1\n",
      "31\t27\t0.0\t1\t26 1\n",
      "32\t18\t0.0\t1\t18\n",
      "33\t32\t0.0\t1\t31 1\n",
      "34\t100\t0.0\t1\t97 3\n",
      "35\t4\t0.0\t1\t4\n",
      "36\t5\t0.0\t1\t5\n",
      "37\t10\t0.0\t1\t10\n",
      "38\t23\t0.0\t1\t23\n",
      "39\t19\t0.0\t1\t19\n",
      "40\t6\t0.0\t1\t6\n",
      "41\t8\t0.0\t1\t8\n",
      "42\t28\t0.0\t1\t27 1\n",
      "43\t87\t0.0\t1\t84 3\n",
      "44\t57\t0.0\t1\t57\n",
      "45\t103\t0.0\t1\t101 2\n",
      "46\t156\t0.0\t1\t154 2\n",
      "47\t98\t0.0\t1\t96 2\n",
      "48\t96\t0.0\t1\t94 2\n",
      "49\t103\t0.0\t1\t101 2\n",
      "50\t56\t0.0\t1\t53 3\n",
      "51\t86\t0.0\t1\t86\n",
      "52\t98\t0.0\t1\t98\n",
      "53\t38\t0.0\t1\t38\n",
      "54\t158\t0.0\t1\t157 1\n",
      "55\t127\t0.0\t1\t127\n",
      "56\t76\t0.0\t1\t72 4\n",
      "57\t196\t0.0\t1\t195 1\n",
      "58\t153\t0.0\t1\t153\n",
      "59\t137\t0.0\t1\t137\n",
      "60\t77\t0.0\t1\t77\n",
      "61\t34\t0.0\t1\t34\n",
      "62\t60\t0.0\t1\t60\n",
      "63\t174\t0.0\t1\t172 2\n",
      "64\t40\t0.0\t1\t39 1\n",
      "65\t11\t0.0\t1\t11\n",
      "66\t8\t0.0\t1\t7 1\n",
      "67\t8\t0.0\t1\t8\n",
      "68\t3\t0.0\t1\t3\n",
      "69\t3\t0.0\t1\t3\n",
      "70\t4\t0.0\t1\t4\n",
      "71\t2\t0.0\t1\t2\n",
      "72\t5\t0.0\t1\t5\n",
      "73\t2\t0.0\t1\t2\n",
      "74\t6\t0.0\t1\t6\n",
      "75\t15\t0.0\t1\t15\n",
      "76\t14\t0.0\t1\t14\n",
      "77\t36\t0.0\t1\t36\n",
      "78\t126\t0.0\t1\t125 1\n",
      "79\t161\t0.0\t1\t157 4\n",
      "80\t682\t0.0\t1\t674 8\n",
      "81\t487\t0.0\t1\t481 6\n",
      "82\t166\t0.0\t1\t164 2\n",
      "83\t34\t0.0\t1\t34\n",
      "84\t5\t0.0\t1\t4 1\n",
      "85\t1\t0.0\t1\t0 1\n",
      "96\t1\t0.0\t1\t1\n",
      "218\t1\t0.0\t1\t1\n",
      "\n",
      "Read1 before filtering:\n",
      "total reads: 151730\n",
      "total bases: 42888618\n",
      "Q20 bases: 42741923(99.658%)\n",
      "Q30 bases: 42272214(98.5628%)\n",
      "\n",
      "Read2 before filtering:\n",
      "total reads: 151730\n",
      "total bases: 41186259\n",
      "Q20 bases: 40529091(98.4044%)\n",
      "Q30 bases: 38960216(94.5952%)\n",
      "\n",
      "Merged and filtered:\n",
      "total reads: 146550\n",
      "total bases: 55691214\n",
      "Q20 bases: 55555937(99.7571%)\n",
      "Q30 bases: 55066051(98.8774%)\n",
      "\n",
      "Filtering result:\n",
      "reads passed filter: 299888\n",
      "reads failed due to low quality: 368\n",
      "reads failed due to too many N: 28\n",
      "reads failed due to too short: 3176\n",
      "reads with adapter trimmed: 8778\n",
      "bases trimmed due to adapters: 71817\n",
      "reads corrected by overlap analysis: 20974\n",
      "bases corrected by overlap analysis: 30506\n",
      "\n",
      "Duplication rate: 69.6395%\n",
      "\n",
      "Insert size peak (evaluated by paired-end reads): 390\n",
      "\n",
      "Read pairs merged: 146550\n",
      "% of original read pairs: 96.586%\n",
      "% in reads after filtering: 100%\n",
      "\n",
      "\n",
      "JSON report: 2_fastp/Hrd1_DMS_Rep2_Region1_Input_report.json\n",
      "HTML report: 2_fastp/Hrd1_DMS_Rep2_Region1_Input_report.html\n",
      "\n",
      "fastp -i 1_cutadapt/Hrd1_DMS_Rep2_Region1_Input_R1_cut.fastq.gz -I 1_cutadapt/Hrd1_DMS_Rep2_Region1_Input_R2_cut.fastq.gz -m --merged_out 2_fastp/Hrd1_DMS_Rep2_Region1_Input_merge.fastq.gz -o 2_fastp/Hrd1_DMS_Rep2_Region1_Input_R1_cut.fastq.gz -O 2_fastp/Hrd1_DMS_Rep2_Region1_Input_R2_cut.fastq.gz -w 12 -h 2_fastp/Hrd1_DMS_Rep2_Region1_Input_report.html -j 2_fastp/Hrd1_DMS_Rep2_Region1_Input_report.json \n",
      "fastp v0.20.0, time used: 2 seconds\n",
      "146550 reads; of these:\n",
      "  146550 (100.00%) were unpaired; of these:\n",
      "    4396 (3.00%) aligned 0 times\n",
      "    142154 (97.00%) aligned exactly 1 time\n",
      "    0 (0.00%) aligned >1 times\n",
      "97.00% overall alignment rate\n",
      "[bam_sort_core] merging from 0 files and 12 in-memory blocks...\n",
      "This is cutadapt 1.18 with Python 3.7.16\n",
      "Command line parameters: -a CTGTCTCTTATACACATCT -A CTGTCTCTTATACACATCT -u 8 -U 8 -q 30 -o 1_cutadapt/Hrd1_DMS_Rep3_Region1_Input_R1_cut.fastq.gz -p 1_cutadapt/Hrd1_DMS_Rep3_Region1_Input_R2_cut.fastq.gz --cores=12 0_fastq_files/Hrd1_DMS_Rep3_Region1_Input_R1_001.fastq.gz 0_fastq_files/Hrd1_DMS_Rep3_Region1_Input_R2_001.fastq.gz\n",
      "Processing reads on 12 cores in paired-end mode ...\n",
      "Finished in 1.67 s (9 us/read; 6.49 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            181,107\n",
      "  Read 1 with adapter:                   1,989 (1.1%)\n",
      "  Read 2 with adapter:                   4,366 (2.4%)\n",
      "Pairs written (passing filters):       181,107 (100.0%)\n",
      "\n",
      "Total basepairs processed:   108,651,112 bp\n",
      "  Read 1:    54,326,023 bp\n",
      "  Read 2:    54,325,089 bp\n",
      "Quality-trimmed:               3,361,633 bp (3.1%)\n",
      "  Read 1:       654,196 bp\n",
      "  Read 2:     2,707,437 bp\n",
      "Total written (filtered):    102,104,315 bp (94.0%)\n",
      "  Read 1:    52,078,288 bp\n",
      "  Read 2:    50,026,027 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: CTGTCTCTTATACACATCT; Type: regular 3'; Length: 19; Trimmed: 1989 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 35.1%\n",
      "  C: 24.3%\n",
      "  G: 18.5%\n",
      "  T: 22.1%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t67\t2829.8\t0\t67\n",
      "4\t1\t707.4\t0\t1\n",
      "5\t1\t176.9\t0\t1\n",
      "9\t1\t0.7\t0\t1\n",
      "11\t1\t0.0\t1\t1\n",
      "12\t1\t0.0\t1\t1\n",
      "17\t1\t0.0\t1\t0 1\n",
      "18\t1\t0.0\t1\t1\n",
      "20\t2\t0.0\t1\t1 1\n",
      "21\t4\t0.0\t1\t4\n",
      "24\t1\t0.0\t1\t1\n",
      "29\t1\t0.0\t1\t1\n",
      "30\t2\t0.0\t1\t2\n",
      "31\t1\t0.0\t1\t1\n",
      "33\t1\t0.0\t1\t1\n",
      "36\t4\t0.0\t1\t3 1\n",
      "37\t13\t0.0\t1\t13\n",
      "38\t6\t0.0\t1\t6\n",
      "39\t2\t0.0\t1\t2\n",
      "40\t4\t0.0\t1\t4\n",
      "41\t4\t0.0\t1\t4\n",
      "42\t7\t0.0\t1\t6 1\n",
      "43\t14\t0.0\t1\t14\n",
      "45\t1\t0.0\t1\t1\n",
      "47\t2\t0.0\t1\t2\n",
      "48\t2\t0.0\t1\t2\n",
      "49\t1\t0.0\t1\t1\n",
      "51\t1\t0.0\t1\t1\n",
      "52\t3\t0.0\t1\t3\n",
      "53\t5\t0.0\t1\t4 1\n",
      "54\t3\t0.0\t1\t3\n",
      "55\t3\t0.0\t1\t3\n",
      "56\t1\t0.0\t1\t1\n",
      "57\t2\t0.0\t1\t2\n",
      "58\t2\t0.0\t1\t2\n",
      "59\t2\t0.0\t1\t2\n",
      "61\t6\t0.0\t1\t6\n",
      "63\t3\t0.0\t1\t3\n",
      "64\t1\t0.0\t1\t1\n",
      "65\t5\t0.0\t1\t5\n",
      "66\t6\t0.0\t1\t6\n",
      "67\t3\t0.0\t1\t3\n",
      "68\t7\t0.0\t1\t7\n",
      "69\t12\t0.0\t1\t12\n",
      "70\t10\t0.0\t1\t9 1\n",
      "71\t10\t0.0\t1\t9 1\n",
      "72\t13\t0.0\t1\t13\n",
      "73\t15\t0.0\t1\t15\n",
      "74\t43\t0.0\t1\t42 1\n",
      "75\t77\t0.0\t1\t68 9\n",
      "76\t1040\t0.0\t1\t1030 10\n",
      "77\t478\t0.0\t1\t472 6\n",
      "78\t45\t0.0\t1\t45\n",
      "79\t20\t0.0\t1\t20\n",
      "80\t3\t0.0\t1\t3\n",
      "81\t4\t0.0\t1\t4\n",
      "82\t1\t0.0\t1\t1\n",
      "83\t1\t0.0\t1\t1\n",
      "103\t1\t0.0\t1\t1\n",
      "118\t1\t0.0\t1\t1\n",
      "129\t1\t0.0\t1\t1\n",
      "201\t1\t0.0\t1\t1\n",
      "211\t2\t0.0\t1\t1 1\n",
      "214\t1\t0.0\t1\t1\n",
      "215\t4\t0.0\t1\t4\n",
      "216\t7\t0.0\t1\t6 1\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CTGTCTCTTATACACATCT; Type: regular 3'; Length: 19; Trimmed: 4366 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 14.8%\n",
      "  C: 11.7%\n",
      "  G: 62.6%\n",
      "  T: 10.8%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2442\t2829.8\t0\t2442\n",
      "4\t16\t707.4\t0\t16\n",
      "5\t1\t176.9\t0\t1\n",
      "7\t1\t11.1\t0\t1\n",
      "8\t1\t2.8\t0\t1\n",
      "10\t1\t0.2\t1\t1\n",
      "11\t1\t0.0\t1\t1\n",
      "12\t1\t0.0\t1\t1\n",
      "16\t1\t0.0\t1\t1\n",
      "17\t2\t0.0\t1\t2\n",
      "19\t2\t0.0\t1\t2\n",
      "20\t1\t0.0\t1\t1\n",
      "22\t2\t0.0\t1\t2\n",
      "23\t2\t0.0\t1\t2\n",
      "24\t2\t0.0\t1\t2\n",
      "26\t1\t0.0\t1\t1\n",
      "28\t6\t0.0\t1\t6\n",
      "29\t3\t0.0\t1\t3\n",
      "30\t2\t0.0\t1\t2\n",
      "31\t1\t0.0\t1\t1\n",
      "32\t8\t0.0\t1\t8\n",
      "33\t5\t0.0\t1\t5\n",
      "34\t6\t0.0\t1\t6\n",
      "35\t13\t0.0\t1\t12 1\n",
      "36\t6\t0.0\t1\t5 1\n",
      "37\t1\t0.0\t1\t1\n",
      "38\t5\t0.0\t1\t5\n",
      "39\t3\t0.0\t1\t3\n",
      "40\t9\t0.0\t1\t8 1\n",
      "41\t13\t0.0\t1\t13\n",
      "42\t10\t0.0\t1\t10\n",
      "43\t12\t0.0\t1\t12\n",
      "44\t11\t0.0\t1\t11\n",
      "45\t42\t0.0\t1\t41 1\n",
      "46\t36\t0.0\t1\t36\n",
      "47\t18\t0.0\t1\t18\n",
      "48\t12\t0.0\t1\t12\n",
      "49\t17\t0.0\t1\t16 1\n",
      "50\t17\t0.0\t1\t17\n",
      "51\t11\t0.0\t1\t11\n",
      "52\t20\t0.0\t1\t20\n",
      "53\t13\t0.0\t1\t13\n",
      "54\t27\t0.0\t1\t27\n",
      "55\t25\t0.0\t1\t24 1\n",
      "56\t9\t0.0\t1\t9\n",
      "57\t48\t0.0\t1\t48\n",
      "58\t26\t0.0\t1\t26\n",
      "59\t20\t0.0\t1\t19 1\n",
      "60\t16\t0.0\t1\t16\n",
      "61\t9\t0.0\t1\t9\n",
      "62\t14\t0.0\t1\t14\n",
      "63\t47\t0.0\t1\t47\n",
      "64\t10\t0.0\t1\t8 2\n",
      "65\t3\t0.0\t1\t3\n",
      "66\t3\t0.0\t1\t3\n",
      "67\t3\t0.0\t1\t3\n",
      "69\t2\t0.0\t1\t2\n",
      "70\t8\t0.0\t1\t8\n",
      "73\t2\t0.0\t1\t2\n",
      "74\t1\t0.0\t1\t1\n",
      "75\t6\t0.0\t1\t5 1\n",
      "76\t13\t0.0\t1\t13\n",
      "77\t25\t0.0\t1\t24 1\n",
      "78\t46\t0.0\t1\t46\n",
      "79\t137\t0.0\t1\t134 3\n",
      "80\t544\t0.0\t1\t539 5\n",
      "81\t432\t0.0\t1\t425 7\n",
      "82\t104\t0.0\t1\t103 1\n",
      "83\t8\t0.0\t1\t8\n",
      "85\t1\t0.0\t1\t1\n",
      "\n",
      "Read1 before filtering:\n",
      "total reads: 181107\n",
      "total bases: 52078288\n",
      "Q20 bases: 51804010(99.4733%)\n",
      "Q30 bases: 51045670(98.0172%)\n",
      "\n",
      "Read2 before filtering:\n",
      "total reads: 181107\n",
      "total bases: 50026027\n",
      "Q20 bases: 49542616(99.0337%)\n",
      "Q30 bases: 48210296(96.3704%)\n",
      "\n",
      "Merged and filtered:\n",
      "total reads: 177226\n",
      "total bases: 68684376\n",
      "Q20 bases: 68443033(99.6486%)\n",
      "Q30 bases: 67656525(98.5035%)\n",
      "\n",
      "Filtering result:\n",
      "reads passed filter: 359886\n",
      "reads failed due to low quality: 256\n",
      "reads failed due to too many N: 36\n",
      "reads failed due to too short: 2036\n",
      "reads with adapter trimmed: 3830\n",
      "bases trimmed due to adapters: 31701\n",
      "reads corrected by overlap analysis: 20153\n",
      "bases corrected by overlap analysis: 27775\n",
      "\n",
      "Duplication rate: 71.9745%\n",
      "\n",
      "Insert size peak (evaluated by paired-end reads): 392\n",
      "\n",
      "Read pairs merged: 177226\n",
      "% of original read pairs: 97.8571%\n",
      "% in reads after filtering: 100%\n",
      "\n",
      "\n",
      "JSON report: 2_fastp/Hrd1_DMS_Rep3_Region1_Input_report.json\n",
      "HTML report: 2_fastp/Hrd1_DMS_Rep3_Region1_Input_report.html\n",
      "\n",
      "fastp -i 1_cutadapt/Hrd1_DMS_Rep3_Region1_Input_R1_cut.fastq.gz -I 1_cutadapt/Hrd1_DMS_Rep3_Region1_Input_R2_cut.fastq.gz -m --merged_out 2_fastp/Hrd1_DMS_Rep3_Region1_Input_merge.fastq.gz -o 2_fastp/Hrd1_DMS_Rep3_Region1_Input_R1_cut.fastq.gz -O 2_fastp/Hrd1_DMS_Rep3_Region1_Input_R2_cut.fastq.gz -w 12 -h 2_fastp/Hrd1_DMS_Rep3_Region1_Input_report.html -j 2_fastp/Hrd1_DMS_Rep3_Region1_Input_report.json \n",
      "fastp v0.20.0, time used: 1 seconds\n",
      "177226 reads; of these:\n",
      "  177226 (100.00%) were unpaired; of these:\n",
      "    1922 (1.08%) aligned 0 times\n",
      "    175304 (98.92%) aligned exactly 1 time\n",
      "    0 (0.00%) aligned >1 times\n",
      "98.92% overall alignment rate\n",
      "[bam_sort_core] merging from 0 files and 12 in-memory blocks...\n",
      "This is cutadapt 1.18 with Python 3.7.16\n",
      "Command line parameters: -a CTGTCTCTTATACACATCT -A CTGTCTCTTATACACATCT -u 8 -U 8 -q 30 -o 1_cutadapt/Hrd1_DMS_Rep4_Region1_Input_R1_cut.fastq.gz -p 1_cutadapt/Hrd1_DMS_Rep4_Region1_Input_R2_cut.fastq.gz --cores=12 0_fastq_files/Hrd1_DMS_Rep4_Region1_Input_R1_001.fastq.gz 0_fastq_files/Hrd1_DMS_Rep4_Region1_Input_R2_001.fastq.gz\n",
      "Processing reads on 12 cores in paired-end mode ...\n",
      "Finished in 1.01 s (9 us/read; 6.32 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            106,484\n",
      "  Read 1 with adapter:                     767 (0.7%)\n",
      "  Read 2 with adapter:                     854 (0.8%)\n",
      "Pairs written (passing filters):       106,484 (100.0%)\n",
      "\n",
      "Total basepairs processed:    63,883,595 bp\n",
      "  Read 1:    31,941,778 bp\n",
      "  Read 2:    31,941,817 bp\n",
      "Quality-trimmed:               1,686,642 bp (2.6%)\n",
      "  Read 1:       404,018 bp\n",
      "  Read 2:     1,282,624 bp\n",
      "Total written (filtered):     60,407,464 bp (94.6%)\n",
      "  Read 1:    30,641,602 bp\n",
      "  Read 2:    29,765,862 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: CTGTCTCTTATACACATCT; Type: regular 3'; Length: 19; Trimmed: 767 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 34.0%\n",
      "  C: 15.9%\n",
      "  G: 32.9%\n",
      "  T: 17.2%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t191\t1663.8\t0\t191\n",
      "11\t1\t0.0\t1\t1\n",
      "25\t1\t0.0\t1\t1\n",
      "34\t1\t0.0\t1\t1\n",
      "37\t1\t0.0\t1\t1\n",
      "38\t1\t0.0\t1\t1\n",
      "39\t1\t0.0\t1\t1\n",
      "41\t2\t0.0\t1\t1 1\n",
      "42\t2\t0.0\t1\t2\n",
      "43\t3\t0.0\t1\t2 1\n",
      "50\t1\t0.0\t1\t1\n",
      "54\t2\t0.0\t1\t2\n",
      "59\t3\t0.0\t1\t3\n",
      "60\t1\t0.0\t1\t1\n",
      "62\t1\t0.0\t1\t0 1\n",
      "63\t1\t0.0\t1\t0 1\n",
      "64\t1\t0.0\t1\t1\n",
      "65\t1\t0.0\t1\t1\n",
      "66\t5\t0.0\t1\t5\n",
      "67\t2\t0.0\t1\t2\n",
      "68\t1\t0.0\t1\t1\n",
      "69\t1\t0.0\t1\t1\n",
      "70\t2\t0.0\t1\t2\n",
      "72\t12\t0.0\t1\t12\n",
      "73\t9\t0.0\t1\t9\n",
      "74\t8\t0.0\t1\t8\n",
      "75\t21\t0.0\t1\t19 2\n",
      "76\t367\t0.0\t1\t361 6\n",
      "77\t102\t0.0\t1\t97 5\n",
      "78\t12\t0.0\t1\t12\n",
      "79\t3\t0.0\t1\t3\n",
      "82\t1\t0.0\t1\t1\n",
      "149\t1\t0.0\t1\t0 1\n",
      "175\t1\t0.0\t1\t1\n",
      "181\t1\t0.0\t1\t1\n",
      "214\t1\t0.0\t1\t1\n",
      "217\t1\t0.0\t1\t1\n",
      "219\t1\t0.0\t1\t1\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CTGTCTCTTATACACATCT; Type: regular 3'; Length: 19; Trimmed: 854 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 55.3%\n",
      "  C: 20.5%\n",
      "  G: 10.0%\n",
      "  T: 14.3%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t280\t1663.8\t0\t280\n",
      "5\t1\t104.0\t0\t1\n",
      "19\t2\t0.0\t1\t2\n",
      "21\t2\t0.0\t1\t2\n",
      "22\t2\t0.0\t1\t2\n",
      "23\t1\t0.0\t1\t1\n",
      "25\t1\t0.0\t1\t1\n",
      "27\t1\t0.0\t1\t1\n",
      "30\t2\t0.0\t1\t2\n",
      "32\t1\t0.0\t1\t1\n",
      "34\t2\t0.0\t1\t1 1\n",
      "35\t4\t0.0\t1\t4\n",
      "36\t6\t0.0\t1\t6\n",
      "39\t5\t0.0\t1\t5\n",
      "40\t2\t0.0\t1\t2\n",
      "41\t1\t0.0\t1\t1\n",
      "42\t1\t0.0\t1\t1\n",
      "43\t5\t0.0\t1\t5\n",
      "44\t6\t0.0\t1\t6\n",
      "45\t4\t0.0\t1\t2 2\n",
      "46\t8\t0.0\t1\t8\n",
      "47\t5\t0.0\t1\t5\n",
      "48\t7\t0.0\t1\t7\n",
      "49\t5\t0.0\t1\t5\n",
      "50\t4\t0.0\t1\t4\n",
      "51\t10\t0.0\t1\t10\n",
      "52\t13\t0.0\t1\t13\n",
      "53\t3\t0.0\t1\t2 1\n",
      "54\t14\t0.0\t1\t14\n",
      "55\t15\t0.0\t1\t14 1\n",
      "56\t6\t0.0\t1\t6\n",
      "57\t5\t0.0\t1\t5\n",
      "58\t5\t0.0\t1\t5\n",
      "59\t5\t0.0\t1\t5\n",
      "60\t5\t0.0\t1\t5\n",
      "62\t2\t0.0\t1\t2\n",
      "63\t9\t0.0\t1\t9\n",
      "64\t2\t0.0\t1\t2\n",
      "65\t2\t0.0\t1\t1 1\n",
      "66\t1\t0.0\t1\t1\n",
      "70\t1\t0.0\t1\t1\n",
      "73\t1\t0.0\t1\t1\n",
      "74\t1\t0.0\t1\t1\n",
      "75\t3\t0.0\t1\t2 1\n",
      "76\t1\t0.0\t1\t1\n",
      "77\t5\t0.0\t1\t5\n",
      "78\t17\t0.0\t1\t17\n",
      "79\t24\t0.0\t1\t24\n",
      "80\t147\t0.0\t1\t146 1\n",
      "81\t160\t0.0\t1\t160\n",
      "82\t32\t0.0\t1\t32\n",
      "83\t6\t0.0\t1\t6\n",
      "84\t1\t0.0\t1\t1\n",
      "\n",
      "Read1 before filtering:\n",
      "total reads: 106484\n",
      "total bases: 30641602\n",
      "Q20 bases: 30537331(99.6597%)\n",
      "Q30 bases: 30196491(98.5474%)\n",
      "\n",
      "Read2 before filtering:\n",
      "total reads: 106484\n",
      "total bases: 29765862\n",
      "Q20 bases: 29359265(98.634%)\n",
      "Q30 bases: 28370636(95.3127%)\n",
      "\n",
      "Merged and filtered:\n",
      "total reads: 103415\n",
      "total bases: 40125412\n",
      "Q20 bases: 40025462(99.7509%)\n",
      "Q30 bases: 39663877(98.8498%)\n",
      "\n",
      "Filtering result:\n",
      "reads passed filter: 210824\n",
      "reads failed due to low quality: 270\n",
      "reads failed due to too many N: 26\n",
      "reads failed due to too short: 1848\n",
      "reads with adapter trimmed: 1152\n",
      "bases trimmed due to adapters: 9612\n",
      "reads corrected by overlap analysis: 12727\n",
      "bases corrected by overlap analysis: 18499\n",
      "\n",
      "Duplication rate: 79.2647%\n",
      "\n",
      "Insert size peak (evaluated by paired-end reads): 390\n",
      "\n",
      "Read pairs merged: 103415\n",
      "% of original read pairs: 97.1179%\n",
      "% in reads after filtering: 100%\n",
      "\n",
      "\n",
      "JSON report: 2_fastp/Hrd1_DMS_Rep4_Region1_Input_report.json\n",
      "HTML report: 2_fastp/Hrd1_DMS_Rep4_Region1_Input_report.html\n",
      "\n",
      "fastp -i 1_cutadapt/Hrd1_DMS_Rep4_Region1_Input_R1_cut.fastq.gz -I 1_cutadapt/Hrd1_DMS_Rep4_Region1_Input_R2_cut.fastq.gz -m --merged_out 2_fastp/Hrd1_DMS_Rep4_Region1_Input_merge.fastq.gz -o 2_fastp/Hrd1_DMS_Rep4_Region1_Input_R1_cut.fastq.gz -O 2_fastp/Hrd1_DMS_Rep4_Region1_Input_R2_cut.fastq.gz -w 12 -h 2_fastp/Hrd1_DMS_Rep4_Region1_Input_report.html -j 2_fastp/Hrd1_DMS_Rep4_Region1_Input_report.json \n",
      "fastp v0.20.0, time used: 1 seconds\n",
      "103415 reads; of these:\n",
      "  103415 (100.00%) were unpaired; of these:\n",
      "    577 (0.56%) aligned 0 times\n",
      "    102838 (99.44%) aligned exactly 1 time\n",
      "    0 (0.00%) aligned >1 times\n",
      "99.44% overall alignment rate\n",
      "[bam_sort_core] merging from 0 files and 12 in-memory blocks...\n",
      "This is cutadapt 1.18 with Python 3.7.16\n",
      "Command line parameters: -a CTGTCTCTTATACACATCT -A CTGTCTCTTATACACATCT -u 8 -U 8 -q 30 -o 1_cutadapt/Hrd1_DMS_Rep1_Region1_WT_R1_cut.fastq.gz -p 1_cutadapt/Hrd1_DMS_Rep1_Region1_WT_R2_cut.fastq.gz --cores=12 0_fastq_files/Hrd1_DMS_Rep1_Region1_WT_R1_001.fastq.gz 0_fastq_files/Hrd1_DMS_Rep1_Region1_WT_R2_001.fastq.gz\n",
      "Processing reads on 12 cores in paired-end mode ...\n",
      "Finished in 1.69 s (9 us/read; 6.88 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            194,044\n",
      "  Read 1 with adapter:                     134 (0.1%)\n",
      "  Read 2 with adapter:                   1,662 (0.9%)\n",
      "Pairs written (passing filters):       194,044 (100.0%)\n",
      "\n",
      "Total basepairs processed:   116,416,269 bp\n",
      "  Read 1:    58,209,109 bp\n",
      "  Read 2:    58,207,160 bp\n",
      "Quality-trimmed:               2,970,179 bp (2.6%)\n",
      "  Read 1:       398,793 bp\n",
      "  Read 2:     2,571,386 bp\n",
      "Total written (filtered):    110,333,300 bp (94.8%)\n",
      "  Read 1:    56,256,152 bp\n",
      "  Read 2:    54,077,148 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: CTGTCTCTTATACACATCT; Type: regular 3'; Length: 19; Trimmed: 134 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 82.8%\n",
      "  C: 6.7%\n",
      "  G: 2.2%\n",
      "  T: 8.2%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by \"A\" extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 3' end.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t113\t3031.9\t0\t113\n",
      "4\t1\t758.0\t0\t1\n",
      "25\t1\t0.0\t1\t1\n",
      "69\t1\t0.0\t1\t1\n",
      "75\t1\t0.0\t1\t0 1\n",
      "76\t9\t0.0\t1\t9\n",
      "77\t8\t0.0\t1\t7 1\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CTGTCTCTTATACACATCT; Type: regular 3'; Length: 19; Trimmed: 1662 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1632\t3031.9\t0\t1632\n",
      "4\t7\t758.0\t0\t7\n",
      "6\t3\t47.4\t0\t3\n",
      "43\t1\t0.0\t1\t1\n",
      "44\t1\t0.0\t1\t1\n",
      "45\t1\t0.0\t1\t1\n",
      "47\t2\t0.0\t1\t2\n",
      "58\t2\t0.0\t1\t2\n",
      "63\t1\t0.0\t1\t1\n",
      "64\t1\t0.0\t1\t1\n",
      "66\t1\t0.0\t1\t1\n",
      "77\t1\t0.0\t1\t1\n",
      "79\t2\t0.0\t1\t2\n",
      "80\t5\t0.0\t1\t5\n",
      "81\t2\t0.0\t1\t2\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "Read1 before filtering:\n",
      "total reads: 194044\n",
      "total bases: 56256152\n",
      "Q20 bases: 55971574(99.4941%)\n",
      "Q30 bases: 55178154(98.0838%)\n",
      "\n",
      "Read2 before filtering:\n",
      "total reads: 194044\n",
      "total bases: 54077148\n",
      "Q20 bases: 53685941(99.2766%)\n",
      "Q30 bases: 52445478(96.9827%)\n",
      "\n",
      "Merged and filtered:\n",
      "total reads: 190338\n",
      "total bases: 74070092\n",
      "Q20 bases: 73820551(99.6631%)\n",
      "Q30 bases: 73007005(98.5648%)\n",
      "\n",
      "Filtering result:\n",
      "reads passed filter: 386234\n",
      "reads failed due to low quality: 258\n",
      "reads failed due to too many N: 34\n",
      "reads failed due to too short: 1562\n",
      "reads with adapter trimmed: 46\n",
      "bases trimmed due to adapters: 694\n",
      "reads corrected by overlap analysis: 16558\n",
      "bases corrected by overlap analysis: 23428\n",
      "\n",
      "Duplication rate: 85.0042%\n",
      "\n",
      "Insert size peak (evaluated by paired-end reads): 390\n",
      "\n",
      "Read pairs merged: 190338\n",
      "% of original read pairs: 98.0901%\n",
      "% in reads after filtering: 100%\n",
      "\n",
      "\n",
      "JSON report: 2_fastp/Hrd1_DMS_Rep1_Region1_WT_report.json\n",
      "HTML report: 2_fastp/Hrd1_DMS_Rep1_Region1_WT_report.html\n",
      "\n",
      "fastp -i 1_cutadapt/Hrd1_DMS_Rep1_Region1_WT_R1_cut.fastq.gz -I 1_cutadapt/Hrd1_DMS_Rep1_Region1_WT_R2_cut.fastq.gz -m --merged_out 2_fastp/Hrd1_DMS_Rep1_Region1_WT_merge.fastq.gz -o 2_fastp/Hrd1_DMS_Rep1_Region1_WT_R1_cut.fastq.gz -O 2_fastp/Hrd1_DMS_Rep1_Region1_WT_R2_cut.fastq.gz -w 12 -h 2_fastp/Hrd1_DMS_Rep1_Region1_WT_report.html -j 2_fastp/Hrd1_DMS_Rep1_Region1_WT_report.json \n",
      "fastp v0.20.0, time used: 2 seconds\n",
      "190338 reads; of these:\n",
      "  190338 (100.00%) were unpaired; of these:\n",
      "    22 (0.01%) aligned 0 times\n",
      "    190316 (99.99%) aligned exactly 1 time\n",
      "    0 (0.00%) aligned >1 times\n",
      "99.99% overall alignment rate\n",
      "[bam_sort_core] merging from 0 files and 12 in-memory blocks...\n",
      "This is cutadapt 1.18 with Python 3.7.16\n",
      "Command line parameters: -a CTGTCTCTTATACACATCT -A CTGTCTCTTATACACATCT -u 8 -U 8 -q 30 -o 1_cutadapt/Hrd1_DMS_Rep2_Region1_WT_R1_cut.fastq.gz -p 1_cutadapt/Hrd1_DMS_Rep2_Region1_WT_R2_cut.fastq.gz --cores=12 0_fastq_files/Hrd1_DMS_Rep2_Region1_WT_R1_001.fastq.gz 0_fastq_files/Hrd1_DMS_Rep2_Region1_WT_R2_001.fastq.gz\n",
      "Processing reads on 12 cores in paired-end mode ...\n",
      "Finished in 1.42 s (9 us/read; 6.80 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            161,112\n",
      "  Read 1 with adapter:                     294 (0.2%)\n",
      "  Read 2 with adapter:                     411 (0.3%)\n",
      "Pairs written (passing filters):       161,112 (100.0%)\n",
      "\n",
      "Total basepairs processed:    96,660,640 bp\n",
      "  Read 1:    48,330,658 bp\n",
      "  Read 2:    48,329,982 bp\n",
      "Quality-trimmed:               2,231,961 bp (2.3%)\n",
      "  Read 1:       439,272 bp\n",
      "  Read 2:     1,792,689 bp\n",
      "Total written (filtered):     91,846,799 bp (95.0%)\n",
      "  Read 1:    46,600,589 bp\n",
      "  Read 2:    45,246,210 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: CTGTCTCTTATACACATCT; Type: regular 3'; Length: 19; Trimmed: 294 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 19.0%\n",
      "  C: 2.4%\n",
      "  G: 76.5%\n",
      "  T: 2.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t280\t2517.4\t0\t280\n",
      "72\t1\t0.0\t1\t1\n",
      "76\t12\t0.0\t1\t12\n",
      "77\t1\t0.0\t1\t1\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CTGTCTCTTATACACATCT; Type: regular 3'; Length: 19; Trimmed: 411 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 95.6%\n",
      "  C: 1.0%\n",
      "  G: 1.5%\n",
      "  T: 1.9%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by \"A\" extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 3' end.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t395\t2517.4\t0\t395\n",
      "4\t2\t629.3\t0\t2\n",
      "48\t1\t0.0\t1\t1\n",
      "49\t1\t0.0\t1\t1\n",
      "54\t1\t0.0\t1\t1\n",
      "57\t1\t0.0\t1\t1\n",
      "62\t1\t0.0\t1\t1\n",
      "79\t2\t0.0\t1\t2\n",
      "80\t2\t0.0\t1\t2\n",
      "81\t4\t0.0\t1\t4\n",
      "82\t1\t0.0\t1\t1\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "Read1 before filtering:\n",
      "total reads: 161112\n",
      "total bases: 46600589\n",
      "Q20 bases: 46452489(99.6822%)\n",
      "Q30 bases: 45971102(98.6492%)\n",
      "\n",
      "Read2 before filtering:\n",
      "total reads: 161112\n",
      "total bases: 45246210\n",
      "Q20 bases: 44647901(98.6777%)\n",
      "Q30 bases: 43196143(95.4691%)\n",
      "\n",
      "Merged and filtered:\n",
      "total reads: 156412\n",
      "total bases: 60961875\n",
      "Q20 bases: 60820988(99.7689%)\n",
      "Q30 bases: 60312448(98.9347%)\n",
      "\n",
      "Filtering result:\n",
      "reads passed filter: 318758\n",
      "reads failed due to low quality: 434\n",
      "reads failed due to too many N: 36\n",
      "reads failed due to too short: 2996\n",
      "reads with adapter trimmed: 30\n",
      "bases trimmed due to adapters: 277\n",
      "reads corrected by overlap analysis: 18093\n",
      "bases corrected by overlap analysis: 26769\n",
      "\n",
      "Duplication rate: 79.0869%\n",
      "\n",
      "Insert size peak (evaluated by paired-end reads): 390\n",
      "\n",
      "Read pairs merged: 156412\n",
      "% of original read pairs: 97.0828%\n",
      "% in reads after filtering: 100%\n",
      "\n",
      "\n",
      "JSON report: 2_fastp/Hrd1_DMS_Rep2_Region1_WT_report.json\n",
      "HTML report: 2_fastp/Hrd1_DMS_Rep2_Region1_WT_report.html\n",
      "\n",
      "fastp -i 1_cutadapt/Hrd1_DMS_Rep2_Region1_WT_R1_cut.fastq.gz -I 1_cutadapt/Hrd1_DMS_Rep2_Region1_WT_R2_cut.fastq.gz -m --merged_out 2_fastp/Hrd1_DMS_Rep2_Region1_WT_merge.fastq.gz -o 2_fastp/Hrd1_DMS_Rep2_Region1_WT_R1_cut.fastq.gz -O 2_fastp/Hrd1_DMS_Rep2_Region1_WT_R2_cut.fastq.gz -w 12 -h 2_fastp/Hrd1_DMS_Rep2_Region1_WT_report.html -j 2_fastp/Hrd1_DMS_Rep2_Region1_WT_report.json \n",
      "fastp v0.20.0, time used: 1 seconds\n",
      "156412 reads; of these:\n",
      "  156412 (100.00%) were unpaired; of these:\n",
      "    15 (0.01%) aligned 0 times\n",
      "    156397 (99.99%) aligned exactly 1 time\n",
      "    0 (0.00%) aligned >1 times\n",
      "99.99% overall alignment rate\n",
      "[bam_sort_core] merging from 0 files and 12 in-memory blocks...\n",
      "This is cutadapt 1.18 with Python 3.7.16\n",
      "Command line parameters: -a CTGTCTCTTATACACATCT -A CTGTCTCTTATACACATCT -u 8 -U 8 -q 30 -o 1_cutadapt/Hrd1_DMS_Rep3_Region1_WT_R1_cut.fastq.gz -p 1_cutadapt/Hrd1_DMS_Rep3_Region1_WT_R2_cut.fastq.gz --cores=12 0_fastq_files/Hrd1_DMS_Rep3_Region1_WT_R1_001.fastq.gz 0_fastq_files/Hrd1_DMS_Rep3_Region1_WT_R2_001.fastq.gz\n",
      "Processing reads on 12 cores in paired-end mode ...\n",
      "Finished in 1.23 s (10 us/read; 6.12 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            125,693\n",
      "  Read 1 with adapter:                     243 (0.2%)\n",
      "  Read 2 with adapter:                   1,614 (1.3%)\n",
      "Pairs written (passing filters):       125,693 (100.0%)\n",
      "\n",
      "Total basepairs processed:    75,409,572 bp\n",
      "  Read 1:    37,705,220 bp\n",
      "  Read 2:    37,704,352 bp\n",
      "Quality-trimmed:               2,507,971 bp (3.3%)\n",
      "  Read 1:       875,163 bp\n",
      "  Read 2:     1,632,808 bp\n",
      "Total written (filtered):     70,884,349 bp (94.0%)\n",
      "  Read 1:    35,823,527 bp\n",
      "  Read 2:    35,060,822 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: CTGTCTCTTATACACATCT; Type: regular 3'; Length: 19; Trimmed: 243 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 96.7%\n",
      "  C: 0.8%\n",
      "  G: 1.2%\n",
      "  T: 1.2%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by \"A\" extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 3' end.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t239\t1964.0\t0\t239\n",
      "43\t1\t0.0\t1\t1\n",
      "72\t1\t0.0\t1\t1\n",
      "76\t1\t0.0\t1\t1\n",
      "78\t1\t0.0\t1\t1\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CTGTCTCTTATACACATCT; Type: regular 3'; Length: 19; Trimmed: 1614 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1599\t1964.0\t0\t1599\n",
      "4\t6\t491.0\t0\t6\n",
      "5\t1\t122.7\t0\t1\n",
      "6\t3\t30.7\t0\t3\n",
      "46\t1\t0.0\t1\t1\n",
      "49\t1\t0.0\t1\t1\n",
      "79\t2\t0.0\t1\t2\n",
      "81\t1\t0.0\t1\t1\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "Read1 before filtering:\n",
      "total reads: 125693\n",
      "total bases: 35823527\n",
      "Q20 bases: 35379301(98.76%)\n",
      "Q30 bases: 34216130(95.513%)\n",
      "\n",
      "Read2 before filtering:\n",
      "total reads: 125693\n",
      "total bases: 35060822\n",
      "Q20 bases: 34704023(98.9823%)\n",
      "Q30 bases: 33712822(96.1553%)\n",
      "\n",
      "Merged and filtered:\n",
      "total reads: 121993\n",
      "total bases: 47745219\n",
      "Q20 bases: 47341538(99.1545%)\n",
      "Q30 bases: 46171372(96.7037%)\n",
      "\n",
      "Filtering result:\n",
      "reads passed filter: 249884\n",
      "reads failed due to low quality: 178\n",
      "reads failed due to too many N: 22\n",
      "reads failed due to too short: 1302\n",
      "reads with adapter trimmed: 24\n",
      "bases trimmed due to adapters: 1786\n",
      "reads corrected by overlap analysis: 23727\n",
      "bases corrected by overlap analysis: 34327\n",
      "\n",
      "Duplication rate: 81.0277%\n",
      "\n",
      "Insert size peak (evaluated by paired-end reads): 392\n",
      "\n",
      "Read pairs merged: 121993\n",
      "% of original read pairs: 97.0563%\n",
      "% in reads after filtering: 100%\n",
      "\n",
      "\n",
      "JSON report: 2_fastp/Hrd1_DMS_Rep3_Region1_WT_report.json\n",
      "HTML report: 2_fastp/Hrd1_DMS_Rep3_Region1_WT_report.html\n",
      "\n",
      "fastp -i 1_cutadapt/Hrd1_DMS_Rep3_Region1_WT_R1_cut.fastq.gz -I 1_cutadapt/Hrd1_DMS_Rep3_Region1_WT_R2_cut.fastq.gz -m --merged_out 2_fastp/Hrd1_DMS_Rep3_Region1_WT_merge.fastq.gz -o 2_fastp/Hrd1_DMS_Rep3_Region1_WT_R1_cut.fastq.gz -O 2_fastp/Hrd1_DMS_Rep3_Region1_WT_R2_cut.fastq.gz -w 12 -h 2_fastp/Hrd1_DMS_Rep3_Region1_WT_report.html -j 2_fastp/Hrd1_DMS_Rep3_Region1_WT_report.json \n",
      "fastp v0.20.0, time used: 1 seconds\n",
      "121993 reads; of these:\n",
      "  121993 (100.00%) were unpaired; of these:\n",
      "    11 (0.01%) aligned 0 times\n",
      "    121982 (99.99%) aligned exactly 1 time\n",
      "    0 (0.00%) aligned >1 times\n",
      "99.99% overall alignment rate\n",
      "[bam_sort_core] merging from 0 files and 12 in-memory blocks...\n",
      "This is cutadapt 1.18 with Python 3.7.16\n",
      "Command line parameters: -a CTGTCTCTTATACACATCT -A CTGTCTCTTATACACATCT -u 8 -U 8 -q 30 -o 1_cutadapt/Hrd1_DMS_Rep4_Region1_WT_R1_cut.fastq.gz -p 1_cutadapt/Hrd1_DMS_Rep4_Region1_WT_R2_cut.fastq.gz --cores=12 0_fastq_files/Hrd1_DMS_Rep4_Region1_WT_R1_001.fastq.gz 0_fastq_files/Hrd1_DMS_Rep4_Region1_WT_R2_001.fastq.gz\n",
      "Processing reads on 12 cores in paired-end mode ...\n",
      "Finished in 1.32 s (10 us/read; 6.02 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            132,579\n",
      "  Read 1 with adapter:                     337 (0.3%)\n",
      "  Read 2 with adapter:                     316 (0.2%)\n",
      "Pairs written (passing filters):       132,579 (100.0%)\n",
      "\n",
      "Total basepairs processed:    79,541,940 bp\n",
      "  Read 1:    39,771,142 bp\n",
      "  Read 2:    39,770,798 bp\n",
      "Quality-trimmed:               1,900,788 bp (2.4%)\n",
      "  Read 1:       620,872 bp\n",
      "  Read 2:     1,279,916 bp\n",
      "Total written (filtered):     75,517,360 bp (94.9%)\n",
      "  Read 1:    38,088,336 bp\n",
      "  Read 2:    37,429,024 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: CTGTCTCTTATACACATCT; Type: regular 3'; Length: 19; Trimmed: 337 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 10.1%\n",
      "  C: 0.0%\n",
      "  G: 89.0%\n",
      "  T: 0.9%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by \"G\" extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 3' end.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t329\t2071.5\t0\t329\n",
      "4\t4\t517.9\t0\t4\n",
      "71\t1\t0.0\t1\t1\n",
      "76\t3\t0.0\t1\t3\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CTGTCTCTTATACACATCT; Type: regular 3'; Length: 19; Trimmed: 316 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t312\t2071.5\t0\t312\n",
      "49\t1\t0.0\t1\t1\n",
      "80\t2\t0.0\t1\t2\n",
      "81\t1\t0.0\t1\t1\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "Read1 before filtering:\n",
      "total reads: 132579\n",
      "total bases: 38088336\n",
      "Q20 bases: 37885244(99.4668%)\n",
      "Q30 bases: 37252959(97.8067%)\n",
      "\n",
      "Read2 before filtering:\n",
      "total reads: 132579\n",
      "total bases: 37429024\n",
      "Q20 bases: 36938104(98.6884%)\n",
      "Q30 bases: 35744882(95.5004%)\n",
      "\n",
      "Merged and filtered:\n",
      "total reads: 128810\n",
      "total bases: 50218660\n",
      "Q20 bases: 50026525(99.6174%)\n",
      "Q30 bases: 49367800(98.3057%)\n",
      "\n",
      "Filtering result:\n",
      "reads passed filter: 262746\n",
      "reads failed due to low quality: 386\n",
      "reads failed due to too many N: 20\n",
      "reads failed due to too short: 2006\n",
      "reads with adapter trimmed: 30\n",
      "bases trimmed due to adapters: 2279\n",
      "reads corrected by overlap analysis: 17792\n",
      "bases corrected by overlap analysis: 25611\n",
      "\n",
      "Duplication rate: 77.3463%\n",
      "\n",
      "Insert size peak (evaluated by paired-end reads): 390\n",
      "\n",
      "Read pairs merged: 128810\n",
      "% of original read pairs: 97.1572%\n",
      "% in reads after filtering: 100%\n",
      "\n",
      "\n",
      "JSON report: 2_fastp/Hrd1_DMS_Rep4_Region1_WT_report.json\n",
      "HTML report: 2_fastp/Hrd1_DMS_Rep4_Region1_WT_report.html\n",
      "\n",
      "fastp -i 1_cutadapt/Hrd1_DMS_Rep4_Region1_WT_R1_cut.fastq.gz -I 1_cutadapt/Hrd1_DMS_Rep4_Region1_WT_R2_cut.fastq.gz -m --merged_out 2_fastp/Hrd1_DMS_Rep4_Region1_WT_merge.fastq.gz -o 2_fastp/Hrd1_DMS_Rep4_Region1_WT_R1_cut.fastq.gz -O 2_fastp/Hrd1_DMS_Rep4_Region1_WT_R2_cut.fastq.gz -w 12 -h 2_fastp/Hrd1_DMS_Rep4_Region1_WT_report.html -j 2_fastp/Hrd1_DMS_Rep4_Region1_WT_report.json \n",
      "fastp v0.20.0, time used: 1 seconds\n",
      "128810 reads; of these:\n",
      "  128810 (100.00%) were unpaired; of these:\n",
      "    10 (0.01%) aligned 0 times\n",
      "    128800 (99.99%) aligned exactly 1 time\n",
      "    0 (0.00%) aligned >1 times\n",
      "99.99% overall alignment rate\n",
      "[bam_sort_core] merging from 0 files and 12 in-memory blocks...\n",
      "This is cutadapt 1.18 with Python 3.7.16\n",
      "Command line parameters: -a CTGTCTCTTATACACATCT -A CTGTCTCTTATACACATCT -u 8 -U 8 -q 30 -o 1_cutadapt/Hrd1_DMS_Rep1_Region1_LDead_R1_cut.fastq.gz -p 1_cutadapt/Hrd1_DMS_Rep1_Region1_LDead_R2_cut.fastq.gz --cores=12 0_fastq_files/Hrd1_DMS_Rep1_Region1_LDead_R1_001.fastq.gz 0_fastq_files/Hrd1_DMS_Rep1_Region1_LDead_R2_001.fastq.gz\n",
      "Processing reads on 12 cores in paired-end mode ...\n",
      "Finished in 0.76 s (10 us/read; 6.00 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             76,444\n",
      "  Read 1 with adapter:                      37 (0.0%)\n",
      "  Read 2 with adapter:                     570 (0.7%)\n",
      "Pairs written (passing filters):        76,444 (100.0%)\n",
      "\n",
      "Total basepairs processed:    45,862,264 bp\n",
      "  Read 1:    22,931,712 bp\n",
      "  Read 2:    22,930,552 bp\n",
      "Quality-trimmed:               1,368,065 bp (3.0%)\n",
      "  Read 1:       167,186 bp\n",
      "  Read 2:     1,200,879 bp\n",
      "Total written (filtered):     43,268,707 bp (94.3%)\n",
      "  Read 1:    22,152,550 bp\n",
      "  Read 2:    21,116,157 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: CTGTCTCTTATACACATCT; Type: regular 3'; Length: 19; Trimmed: 37 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 86.5%\n",
      "  C: 2.7%\n",
      "  G: 5.4%\n",
      "  T: 5.4%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by \"A\" extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 3' end.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t32\t1194.4\t0\t32\n",
      "43\t1\t0.0\t1\t1\n",
      "62\t1\t0.0\t1\t1\n",
      "71\t1\t0.0\t1\t1\n",
      "75\t1\t0.0\t1\t1\n",
      "77\t1\t0.0\t1\t1\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CTGTCTCTTATACACATCT; Type: regular 3'; Length: 19; Trimmed: 570 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t530\t1194.4\t0\t530\n",
      "4\t12\t298.6\t0\t12\n",
      "5\t6\t74.7\t0\t6\n",
      "6\t18\t18.7\t0\t18\n",
      "34\t1\t0.0\t1\t1\n",
      "45\t1\t0.0\t1\t1\n",
      "51\t1\t0.0\t1\t1\n",
      "58\t1\t0.0\t1\t1\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "Read1 before filtering:\n",
      "total reads: 76444\n",
      "total bases: 22152550\n",
      "Q20 bases: 22050166(99.5378%)\n",
      "Q30 bases: 21764760(98.2495%)\n",
      "\n",
      "Read2 before filtering:\n",
      "total reads: 76444\n",
      "total bases: 21116157\n",
      "Q20 bases: 20948080(99.204%)\n",
      "Q30 bases: 20418412(96.6957%)\n",
      "\n",
      "Merged and filtered:\n",
      "total reads: 74759\n",
      "total bases: 29086685\n",
      "Q20 bases: 28999052(99.6987%)\n",
      "Q30 bases: 28709307(98.7026%)\n",
      "\n",
      "Filtering result:\n",
      "reads passed filter: 151978\n",
      "reads failed due to low quality: 106\n",
      "reads failed due to too many N: 10\n",
      "reads failed due to too short: 794\n",
      "reads with adapter trimmed: 38\n",
      "bases trimmed due to adapters: 2115\n",
      "reads corrected by overlap analysis: 6520\n",
      "bases corrected by overlap analysis: 9149\n",
      "\n",
      "Duplication rate: 82.8439%\n",
      "\n",
      "Insert size peak (evaluated by paired-end reads): 390\n",
      "\n",
      "Read pairs merged: 74759\n",
      "% of original read pairs: 97.7958%\n",
      "% in reads after filtering: 100%\n",
      "\n",
      "\n",
      "JSON report: 2_fastp/Hrd1_DMS_Rep1_Region1_LDead_report.json\n",
      "HTML report: 2_fastp/Hrd1_DMS_Rep1_Region1_LDead_report.html\n",
      "\n",
      "fastp -i 1_cutadapt/Hrd1_DMS_Rep1_Region1_LDead_R1_cut.fastq.gz -I 1_cutadapt/Hrd1_DMS_Rep1_Region1_LDead_R2_cut.fastq.gz -m --merged_out 2_fastp/Hrd1_DMS_Rep1_Region1_LDead_merge.fastq.gz -o 2_fastp/Hrd1_DMS_Rep1_Region1_LDead_R1_cut.fastq.gz -O 2_fastp/Hrd1_DMS_Rep1_Region1_LDead_R2_cut.fastq.gz -w 12 -h 2_fastp/Hrd1_DMS_Rep1_Region1_LDead_report.html -j 2_fastp/Hrd1_DMS_Rep1_Region1_LDead_report.json \n",
      "fastp v0.20.0, time used: 1 seconds\n",
      "74759 reads; of these:\n",
      "  74759 (100.00%) were unpaired; of these:\n",
      "    6 (0.01%) aligned 0 times\n",
      "    74753 (99.99%) aligned exactly 1 time\n",
      "    0 (0.00%) aligned >1 times\n",
      "99.99% overall alignment rate\n",
      "[bam_sort_core] merging from 0 files and 12 in-memory blocks...\n",
      "This is cutadapt 1.18 with Python 3.7.16\n",
      "Command line parameters: -a CTGTCTCTTATACACATCT -A CTGTCTCTTATACACATCT -u 8 -U 8 -q 30 -o 1_cutadapt/Hrd1_DMS_Rep2_Region1_LDead_R1_cut.fastq.gz -p 1_cutadapt/Hrd1_DMS_Rep2_Region1_LDead_R2_cut.fastq.gz --cores=12 0_fastq_files/Hrd1_DMS_Rep2_Region1_LDead_R1_001.fastq.gz 0_fastq_files/Hrd1_DMS_Rep2_Region1_LDead_R2_001.fastq.gz\n",
      "Processing reads on 12 cores in paired-end mode ...\n",
      "Finished in 0.80 s (10 us/read; 6.12 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             81,834\n",
      "  Read 1 with adapter:                     118 (0.1%)\n",
      "  Read 2 with adapter:                     284 (0.3%)\n",
      "Pairs written (passing filters):        81,834 (100.0%)\n",
      "\n",
      "Total basepairs processed:    49,096,952 bp\n",
      "  Read 1:    24,548,525 bp\n",
      "  Read 2:    24,548,427 bp\n",
      "Quality-trimmed:               1,238,959 bp (2.5%)\n",
      "  Read 1:       237,682 bp\n",
      "  Read 2:     1,001,277 bp\n",
      "Total written (filtered):     46,546,523 bp (94.8%)\n",
      "  Read 1:    23,655,246 bp\n",
      "  Read 2:    22,891,277 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: CTGTCTCTTATACACATCT; Type: regular 3'; Length: 19; Trimmed: 118 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 34.7%\n",
      "  C: 2.5%\n",
      "  G: 61.0%\n",
      "  T: 1.7%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t109\t1278.7\t0\t109\n",
      "24\t1\t0.0\t1\t0 1\n",
      "59\t1\t0.0\t1\t1\n",
      "64\t1\t0.0\t1\t0 1\n",
      "72\t1\t0.0\t1\t1\n",
      "75\t1\t0.0\t1\t1\n",
      "76\t4\t0.0\t1\t4\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CTGTCTCTTATACACATCT; Type: regular 3'; Length: 19; Trimmed: 284 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 97.9%\n",
      "  C: 1.1%\n",
      "  G: 0.7%\n",
      "  T: 0.4%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by \"A\" extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 3' end.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t274\t1278.7\t0\t274\n",
      "4\t3\t319.7\t0\t3\n",
      "33\t1\t0.0\t1\t1\n",
      "50\t1\t0.0\t1\t1\n",
      "54\t2\t0.0\t1\t2\n",
      "55\t1\t0.0\t1\t1\n",
      "59\t1\t0.0\t1\t1\n",
      "62\t1\t0.0\t1\t1\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "Read1 before filtering:\n",
      "total reads: 81834\n",
      "total bases: 23655246\n",
      "Q20 bases: 23575587(99.6633%)\n",
      "Q30 bases: 23315713(98.5647%)\n",
      "\n",
      "Read2 before filtering:\n",
      "total reads: 81834\n",
      "total bases: 22891277\n",
      "Q20 bases: 22559847(98.5522%)\n",
      "Q30 bases: 21768834(95.0966%)\n",
      "\n",
      "Merged and filtered:\n",
      "total reads: 79288\n",
      "total bases: 30898927\n",
      "Q20 bases: 30825194(99.7614%)\n",
      "Q30 bases: 30555538(98.8887%)\n",
      "\n",
      "Filtering result:\n",
      "reads passed filter: 161934\n",
      "reads failed due to low quality: 224\n",
      "reads failed due to too many N: 18\n",
      "reads failed due to too short: 1492\n",
      "reads with adapter trimmed: 20\n",
      "bases trimmed due to adapters: 464\n",
      "reads corrected by overlap analysis: 10226\n",
      "bases corrected by overlap analysis: 15624\n",
      "\n",
      "Duplication rate: 68.2051%\n",
      "\n",
      "Insert size peak (evaluated by paired-end reads): 390\n",
      "\n",
      "Read pairs merged: 79288\n",
      "% of original read pairs: 96.8888%\n",
      "% in reads after filtering: 100%\n",
      "\n",
      "\n",
      "JSON report: 2_fastp/Hrd1_DMS_Rep2_Region1_LDead_report.json\n",
      "HTML report: 2_fastp/Hrd1_DMS_Rep2_Region1_LDead_report.html\n",
      "\n",
      "fastp -i 1_cutadapt/Hrd1_DMS_Rep2_Region1_LDead_R1_cut.fastq.gz -I 1_cutadapt/Hrd1_DMS_Rep2_Region1_LDead_R2_cut.fastq.gz -m --merged_out 2_fastp/Hrd1_DMS_Rep2_Region1_LDead_merge.fastq.gz -o 2_fastp/Hrd1_DMS_Rep2_Region1_LDead_R1_cut.fastq.gz -O 2_fastp/Hrd1_DMS_Rep2_Region1_LDead_R2_cut.fastq.gz -w 12 -h 2_fastp/Hrd1_DMS_Rep2_Region1_LDead_report.html -j 2_fastp/Hrd1_DMS_Rep2_Region1_LDead_report.json \n",
      "fastp v0.20.0, time used: 1 seconds\n",
      "79288 reads; of these:\n",
      "  79288 (100.00%) were unpaired; of these:\n",
      "    10 (0.01%) aligned 0 times\n",
      "    79278 (99.99%) aligned exactly 1 time\n",
      "    0 (0.00%) aligned >1 times\n",
      "99.99% overall alignment rate\n",
      "[bam_sort_core] merging from 0 files and 12 in-memory blocks...\n",
      "This is cutadapt 1.18 with Python 3.7.16\n",
      "Command line parameters: -a CTGTCTCTTATACACATCT -A CTGTCTCTTATACACATCT -u 8 -U 8 -q 30 -o 1_cutadapt/Hrd1_DMS_Rep3_Region1_LDead_R1_cut.fastq.gz -p 1_cutadapt/Hrd1_DMS_Rep3_Region1_LDead_R2_cut.fastq.gz --cores=12 0_fastq_files/Hrd1_DMS_Rep3_Region1_LDead_R1_001.fastq.gz 0_fastq_files/Hrd1_DMS_Rep3_Region1_LDead_R2_001.fastq.gz\n",
      "Processing reads on 12 cores in paired-end mode ...\n",
      "Finished in 0.25 s (23 us/read; 2.57 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             10,683\n",
      "  Read 1 with adapter:                       8 (0.1%)\n",
      "  Read 2 with adapter:                     131 (1.2%)\n",
      "Pairs written (passing filters):        10,683 (100.0%)\n",
      "\n",
      "Total basepairs processed:     6,409,289 bp\n",
      "  Read 1:     3,204,696 bp\n",
      "  Read 2:     3,204,593 bp\n",
      "Quality-trimmed:                 152,663 bp (2.4%)\n",
      "  Read 1:        22,926 bp\n",
      "  Read 2:       129,737 bp\n",
      "Total written (filtered):      6,085,078 bp (94.9%)\n",
      "  Read 1:     3,096,138 bp\n",
      "  Read 2:     2,988,940 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: CTGTCTCTTATACACATCT; Type: regular 3'; Length: 19; Trimmed: 8 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 75.0%\n",
      "  C: 12.5%\n",
      "  G: 12.5%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t6\t166.9\t0\t6\n",
      "74\t1\t0.0\t1\t1\n",
      "76\t1\t0.0\t1\t0 1\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CTGTCTCTTATACACATCT; Type: regular 3'; Length: 19; Trimmed: 131 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 2.3%\n",
      "  C: 0.0%\n",
      "  G: 97.7%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by \"G\" extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 3' end.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t130\t166.9\t0\t130\n",
      "62\t1\t0.0\t1\t1\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "Read1 before filtering:\n",
      "total reads: 10683\n",
      "total bases: 3096138\n",
      "Q20 bases: 3080202(99.4853%)\n",
      "Q30 bases: 3035018(98.0259%)\n",
      "\n",
      "Read2 before filtering:\n",
      "total reads: 10683\n",
      "total bases: 2988940\n",
      "Q20 bases: 2959428(99.0126%)\n",
      "Q30 bases: 2881387(96.4016%)\n",
      "\n",
      "Merged and filtered:\n",
      "total reads: 10472\n",
      "total bases: 4093225\n",
      "Q20 bases: 4078716(99.6455%)\n",
      "Q30 bases: 4031469(98.4913%)\n",
      "\n",
      "Filtering result:\n",
      "reads passed filter: 21246\n",
      "reads failed due to low quality: 26\n",
      "reads failed due to too many N: 0\n",
      "reads failed due to too short: 94\n",
      "reads with adapter trimmed: 4\n",
      "bases trimmed due to adapters: 235\n",
      "reads corrected by overlap analysis: 1225\n",
      "bases corrected by overlap analysis: 1693\n",
      "\n",
      "Duplication rate: 63.4483%\n",
      "\n",
      "Insert size peak (evaluated by paired-end reads): 392\n",
      "\n",
      "Read pairs merged: 10472\n",
      "% of original read pairs: 98.0249%\n",
      "% in reads after filtering: 100%\n",
      "\n",
      "\n",
      "JSON report: 2_fastp/Hrd1_DMS_Rep3_Region1_LDead_report.json\n",
      "HTML report: 2_fastp/Hrd1_DMS_Rep3_Region1_LDead_report.html\n",
      "\n",
      "fastp -i 1_cutadapt/Hrd1_DMS_Rep3_Region1_LDead_R1_cut.fastq.gz -I 1_cutadapt/Hrd1_DMS_Rep3_Region1_LDead_R2_cut.fastq.gz -m --merged_out 2_fastp/Hrd1_DMS_Rep3_Region1_LDead_merge.fastq.gz -o 2_fastp/Hrd1_DMS_Rep3_Region1_LDead_R1_cut.fastq.gz -O 2_fastp/Hrd1_DMS_Rep3_Region1_LDead_R2_cut.fastq.gz -w 12 -h 2_fastp/Hrd1_DMS_Rep3_Region1_LDead_report.html -j 2_fastp/Hrd1_DMS_Rep3_Region1_LDead_report.json \n",
      "fastp v0.20.0, time used: 0 seconds\n",
      "10472 reads; of these:\n",
      "  10472 (100.00%) were unpaired; of these:\n",
      "    3 (0.03%) aligned 0 times\n",
      "    10469 (99.97%) aligned exactly 1 time\n",
      "    0 (0.00%) aligned >1 times\n",
      "99.97% overall alignment rate\n",
      "[bam_sort_core] merging from 0 files and 12 in-memory blocks...\n",
      "This is cutadapt 1.18 with Python 3.7.16\n",
      "Command line parameters: -a CTGTCTCTTATACACATCT -A CTGTCTCTTATACACATCT -u 8 -U 8 -q 30 -o 1_cutadapt/Hrd1_DMS_Rep4_Region1_LDead_R1_cut.fastq.gz -p 1_cutadapt/Hrd1_DMS_Rep4_Region1_LDead_R2_cut.fastq.gz --cores=12 0_fastq_files/Hrd1_DMS_Rep4_Region1_LDead_R1_001.fastq.gz 0_fastq_files/Hrd1_DMS_Rep4_Region1_LDead_R2_001.fastq.gz\n",
      "Processing reads on 12 cores in paired-end mode ...\n",
      "Finished in 0.25 s (25 us/read; 2.44 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             10,208\n",
      "  Read 1 with adapter:                      20 (0.2%)\n",
      "  Read 2 with adapter:                      35 (0.3%)\n",
      "Pairs written (passing filters):        10,208 (100.0%)\n",
      "\n",
      "Total basepairs processed:     6,124,362 bp\n",
      "  Read 1:     3,062,216 bp\n",
      "  Read 2:     3,062,146 bp\n",
      "Quality-trimmed:                 144,562 bp (2.4%)\n",
      "  Read 1:        33,253 bp\n",
      "  Read 2:       111,309 bp\n",
      "Total written (filtered):      5,816,211 bp (95.0%)\n",
      "  Read 1:     2,947,169 bp\n",
      "  Read 2:     2,869,042 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: CTGTCTCTTATACACATCT; Type: regular 3'; Length: 19; Trimmed: 20 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 20.0%\n",
      "  C: 0.0%\n",
      "  G: 70.0%\n",
      "  T: 10.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t19\t159.5\t0\t19\n",
      "73\t1\t0.0\t1\t1\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CTGTCTCTTATACACATCT; Type: regular 3'; Length: 19; Trimmed: 35 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 97.1%\n",
      "  C: 0.0%\n",
      "  G: 2.9%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by \"A\" extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 3' end.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t34\t159.5\t0\t34\n",
      "29\t1\t0.0\t1\t1\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "Read1 before filtering:\n",
      "total reads: 10208\n",
      "total bases: 2947169\n",
      "Q20 bases: 2935909(99.6179%)\n",
      "Q30 bases: 2900006(98.3997%)\n",
      "\n",
      "Read2 before filtering:\n",
      "total reads: 10208\n",
      "total bases: 2869042\n",
      "Q20 bases: 2831477(98.6907%)\n",
      "Q30 bases: 2738294(95.4428%)\n",
      "\n",
      "Merged and filtered:\n",
      "total reads: 9921\n",
      "total bases: 3866900\n",
      "Q20 bases: 3856079(99.7202%)\n",
      "Q30 bases: 3818252(98.7419%)\n",
      "\n",
      "Filtering result:\n",
      "reads passed filter: 20224\n",
      "reads failed due to low quality: 22\n",
      "reads failed due to too many N: 0\n",
      "reads failed due to too short: 170\n",
      "reads with adapter trimmed: 2\n",
      "bases trimmed due to adapters: 16\n",
      "reads corrected by overlap analysis: 1232\n",
      "bases corrected by overlap analysis: 1805\n",
      "\n",
      "Duplication rate: 50.5495%\n",
      "\n",
      "Insert size peak (evaluated by paired-end reads): 390\n",
      "\n",
      "Read pairs merged: 9921\n",
      "% of original read pairs: 97.1885%\n",
      "% in reads after filtering: 100%\n",
      "\n",
      "\n",
      "JSON report: 2_fastp/Hrd1_DMS_Rep4_Region1_LDead_report.json\n",
      "HTML report: 2_fastp/Hrd1_DMS_Rep4_Region1_LDead_report.html\n",
      "\n",
      "fastp -i 1_cutadapt/Hrd1_DMS_Rep4_Region1_LDead_R1_cut.fastq.gz -I 1_cutadapt/Hrd1_DMS_Rep4_Region1_LDead_R2_cut.fastq.gz -m --merged_out 2_fastp/Hrd1_DMS_Rep4_Region1_LDead_merge.fastq.gz -o 2_fastp/Hrd1_DMS_Rep4_Region1_LDead_R1_cut.fastq.gz -O 2_fastp/Hrd1_DMS_Rep4_Region1_LDead_R2_cut.fastq.gz -w 12 -h 2_fastp/Hrd1_DMS_Rep4_Region1_LDead_report.html -j 2_fastp/Hrd1_DMS_Rep4_Region1_LDead_report.json \n",
      "fastp v0.20.0, time used: 1 seconds\n",
      "9921 reads; of these:\n",
      "  9921 (100.00%) were unpaired; of these:\n",
      "    1 (0.01%) aligned 0 times\n",
      "    9920 (99.99%) aligned exactly 1 time\n",
      "    0 (0.00%) aligned >1 times\n",
      "99.99% overall alignment rate\n",
      "[bam_sort_core] merging from 0 files and 12 in-memory blocks...\n",
      "This is cutadapt 1.18 with Python 3.7.16\n",
      "Command line parameters: -a CTGTCTCTTATACACATCT -A CTGTCTCTTATACACATCT -u 8 -U 8 -q 30 -o 1_cutadapt/Hrd1_DMS_Rep1_Region2_Input_R1_cut.fastq.gz -p 1_cutadapt/Hrd1_DMS_Rep1_Region2_Input_R2_cut.fastq.gz --cores=12 0_fastq_files/Hrd1_DMS_Rep1_Region2_Input_R1_001.fastq.gz 0_fastq_files/Hrd1_DMS_Rep1_Region2_Input_R2_001.fastq.gz\n",
      "Processing reads on 12 cores in paired-end mode ...\n",
      "Finished in 1.91 s (9 us/read; 6.75 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            214,883\n",
      "  Read 1 with adapter:                  26,494 (12.3%)\n",
      "  Read 2 with adapter:                  30,826 (14.3%)\n",
      "Pairs written (passing filters):       214,883 (100.0%)\n",
      "\n",
      "Total basepairs processed:   128,712,463 bp\n",
      "  Read 1:    64,448,983 bp\n",
      "  Read 2:    64,263,480 bp\n",
      "Quality-trimmed:              11,429,796 bp (8.9%)\n",
      "  Read 1:     4,777,570 bp\n",
      "  Read 2:     6,652,226 bp\n",
      "Total written (filtered):    110,139,599 bp (85.6%)\n",
      "  Read 1:    55,946,693 bp\n",
      "  Read 2:    54,192,906 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: CTGTCTCTTATACACATCT; Type: regular 3'; Length: 19; Trimmed: 26494 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 32.0%\n",
      "  C: 26.4%\n",
      "  G: 17.6%\n",
      "  T: 24.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t123\t3357.5\t0\t123\n",
      "4\t46\t839.4\t0\t46\n",
      "5\t5\t209.8\t0\t5\n",
      "6\t5\t52.5\t0\t5\n",
      "7\t1\t13.1\t0\t1\n",
      "8\t1\t3.3\t0\t1\n",
      "9\t1\t0.8\t0\t1\n",
      "10\t2\t0.2\t1\t2\n",
      "11\t10\t0.1\t1\t10\n",
      "12\t6\t0.0\t1\t4 2\n",
      "13\t6\t0.0\t1\t6\n",
      "14\t6\t0.0\t1\t6\n",
      "15\t1\t0.0\t1\t1\n",
      "16\t4\t0.0\t1\t4\n",
      "17\t8\t0.0\t1\t8\n",
      "18\t12\t0.0\t1\t10 2\n",
      "19\t14\t0.0\t1\t14\n",
      "20\t5\t0.0\t1\t5\n",
      "21\t26\t0.0\t1\t23 3\n",
      "22\t9\t0.0\t1\t9\n",
      "23\t8\t0.0\t1\t8\n",
      "24\t2\t0.0\t1\t2\n",
      "25\t4\t0.0\t1\t4\n",
      "26\t8\t0.0\t1\t7 1\n",
      "27\t3\t0.0\t1\t2 1\n",
      "28\t13\t0.0\t1\t12 1\n",
      "29\t18\t0.0\t1\t14 4\n",
      "30\t45\t0.0\t1\t42 3\n",
      "31\t11\t0.0\t1\t11\n",
      "32\t8\t0.0\t1\t7 1\n",
      "33\t3\t0.0\t1\t2 1\n",
      "34\t13\t0.0\t1\t13\n",
      "35\t8\t0.0\t1\t8\n",
      "36\t13\t0.0\t1\t12 1\n",
      "37\t22\t0.0\t1\t20 2\n",
      "38\t14\t0.0\t1\t14\n",
      "39\t20\t0.0\t1\t19 1\n",
      "40\t16\t0.0\t1\t15 1\n",
      "41\t15\t0.0\t1\t14 1\n",
      "42\t30\t0.0\t1\t27 3\n",
      "43\t98\t0.0\t1\t90 8\n",
      "44\t29\t0.0\t1\t27 2\n",
      "45\t16\t0.0\t1\t15 1\n",
      "46\t2\t0.0\t1\t2\n",
      "47\t16\t0.0\t1\t16\n",
      "48\t14\t0.0\t1\t14\n",
      "49\t13\t0.0\t1\t12 1\n",
      "50\t25\t0.0\t1\t24 1\n",
      "51\t27\t0.0\t1\t25 2\n",
      "52\t18\t0.0\t1\t16 2\n",
      "53\t27\t0.0\t1\t26 1\n",
      "54\t56\t0.0\t1\t55 1\n",
      "55\t61\t0.0\t1\t59 2\n",
      "56\t60\t0.0\t1\t55 5\n",
      "57\t51\t0.0\t1\t49 2\n",
      "58\t50\t0.0\t1\t50\n",
      "59\t47\t0.0\t1\t46 1\n",
      "60\t98\t0.0\t1\t96 2\n",
      "61\t103\t0.0\t1\t103\n",
      "62\t36\t0.0\t1\t33 3\n",
      "63\t90\t0.0\t1\t87 3\n",
      "64\t96\t0.0\t1\t94 2\n",
      "65\t221\t0.0\t1\t219 2\n",
      "66\t58\t0.0\t1\t58\n",
      "67\t24\t0.0\t1\t23 1\n",
      "68\t28\t0.0\t1\t28\n",
      "69\t39\t0.0\t1\t38 1\n",
      "70\t39\t0.0\t1\t38 1\n",
      "71\t82\t0.0\t1\t78 4\n",
      "72\t230\t0.0\t1\t224 6\n",
      "73\t207\t0.0\t1\t205 2\n",
      "74\t689\t0.0\t1\t671 18\n",
      "75\t762\t0.0\t1\t683 79\n",
      "76\t13745\t0.0\t1\t13620 125\n",
      "77\t7208\t0.0\t1\t7144 64\n",
      "78\t879\t0.0\t1\t869 10\n",
      "79\t170\t0.0\t1\t169 1\n",
      "80\t89\t0.0\t1\t88 1\n",
      "81\t25\t0.0\t1\t25\n",
      "82\t21\t0.0\t1\t18 3\n",
      "83\t13\t0.0\t1\t13\n",
      "84\t5\t0.0\t1\t5\n",
      "85\t7\t0.0\t1\t7\n",
      "86\t2\t0.0\t1\t2\n",
      "87\t11\t0.0\t1\t11\n",
      "88\t3\t0.0\t1\t3\n",
      "90\t1\t0.0\t1\t1\n",
      "91\t2\t0.0\t1\t2\n",
      "92\t1\t0.0\t1\t1\n",
      "93\t2\t0.0\t1\t2\n",
      "94\t2\t0.0\t1\t2\n",
      "95\t2\t0.0\t1\t2\n",
      "96\t1\t0.0\t1\t1\n",
      "98\t1\t0.0\t1\t1\n",
      "99\t2\t0.0\t1\t2\n",
      "102\t1\t0.0\t1\t1\n",
      "104\t1\t0.0\t1\t1\n",
      "105\t2\t0.0\t1\t2\n",
      "106\t1\t0.0\t1\t1\n",
      "112\t1\t0.0\t1\t1\n",
      "115\t1\t0.0\t1\t1\n",
      "117\t1\t0.0\t1\t1\n",
      "118\t2\t0.0\t1\t2\n",
      "119\t3\t0.0\t1\t3\n",
      "120\t2\t0.0\t1\t2\n",
      "121\t1\t0.0\t1\t1\n",
      "122\t1\t0.0\t1\t1\n",
      "123\t1\t0.0\t1\t1\n",
      "125\t1\t0.0\t1\t1\n",
      "127\t1\t0.0\t1\t1\n",
      "129\t1\t0.0\t1\t1\n",
      "130\t3\t0.0\t1\t3\n",
      "133\t2\t0.0\t1\t2\n",
      "134\t2\t0.0\t1\t2\n",
      "135\t1\t0.0\t1\t1\n",
      "136\t2\t0.0\t1\t2\n",
      "139\t1\t0.0\t1\t1\n",
      "140\t1\t0.0\t1\t1\n",
      "146\t2\t0.0\t1\t2\n",
      "147\t2\t0.0\t1\t2\n",
      "148\t2\t0.0\t1\t2\n",
      "149\t3\t0.0\t1\t3\n",
      "150\t2\t0.0\t1\t2\n",
      "151\t1\t0.0\t1\t1\n",
      "152\t2\t0.0\t1\t2\n",
      "154\t1\t0.0\t1\t1\n",
      "156\t1\t0.0\t1\t1\n",
      "157\t2\t0.0\t1\t2\n",
      "159\t3\t0.0\t1\t3\n",
      "160\t3\t0.0\t1\t2 1\n",
      "161\t1\t0.0\t1\t1\n",
      "162\t2\t0.0\t1\t2\n",
      "163\t2\t0.0\t1\t2\n",
      "164\t1\t0.0\t1\t1\n",
      "165\t3\t0.0\t1\t3\n",
      "166\t2\t0.0\t1\t2\n",
      "169\t2\t0.0\t1\t2\n",
      "170\t1\t0.0\t1\t1\n",
      "174\t2\t0.0\t1\t2\n",
      "175\t3\t0.0\t1\t3\n",
      "177\t1\t0.0\t1\t1\n",
      "178\t3\t0.0\t1\t3\n",
      "179\t3\t0.0\t1\t3\n",
      "180\t3\t0.0\t1\t3\n",
      "181\t1\t0.0\t1\t1\n",
      "182\t1\t0.0\t1\t1\n",
      "184\t2\t0.0\t1\t2\n",
      "186\t4\t0.0\t1\t2 2\n",
      "188\t1\t0.0\t1\t1\n",
      "189\t1\t0.0\t1\t1\n",
      "190\t2\t0.0\t1\t2\n",
      "191\t1\t0.0\t1\t1\n",
      "192\t2\t0.0\t1\t2\n",
      "193\t2\t0.0\t1\t2\n",
      "195\t1\t0.0\t1\t1\n",
      "196\t4\t0.0\t1\t4\n",
      "199\t2\t0.0\t1\t2\n",
      "200\t1\t0.0\t1\t1\n",
      "203\t1\t0.0\t1\t1\n",
      "204\t1\t0.0\t1\t1\n",
      "205\t1\t0.0\t1\t1\n",
      "206\t2\t0.0\t1\t1 1\n",
      "207\t1\t0.0\t1\t1\n",
      "208\t3\t0.0\t1\t3\n",
      "209\t1\t0.0\t1\t1\n",
      "212\t1\t0.0\t1\t1\n",
      "214\t1\t0.0\t1\t1\n",
      "215\t1\t0.0\t1\t1\n",
      "216\t1\t0.0\t1\t1\n",
      "219\t1\t0.0\t1\t1\n",
      "221\t3\t0.0\t1\t3\n",
      "223\t2\t0.0\t1\t1 1\n",
      "224\t5\t0.0\t1\t5\n",
      "225\t8\t0.0\t1\t7 1\n",
      "226\t17\t0.0\t1\t15 2\n",
      "227\t82\t0.0\t1\t79 3\n",
      "228\t79\t0.0\t1\t74 5\n",
      "229\t1\t0.0\t1\t1\n",
      "231\t1\t0.0\t1\t1\n",
      "234\t1\t0.0\t1\t1\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CTGTCTCTTATACACATCT; Type: regular 3'; Length: 19; Trimmed: 30826 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 40.7%\n",
      "  C: 22.7%\n",
      "  G: 15.3%\n",
      "  T: 21.2%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2959\t3357.5\t0\t2959\n",
      "4\t1696\t839.4\t0\t1696\n",
      "5\t10\t209.8\t0\t10\n",
      "6\t8\t52.5\t0\t8\n",
      "7\t2\t13.1\t0\t2\n",
      "8\t1\t3.3\t0\t1\n",
      "9\t9\t0.8\t0\t9\n",
      "10\t6\t0.2\t1\t6\n",
      "11\t2\t0.1\t1\t2\n",
      "12\t24\t0.0\t1\t24\n",
      "13\t10\t0.0\t1\t10\n",
      "14\t4\t0.0\t1\t3 1\n",
      "15\t3\t0.0\t1\t3\n",
      "16\t15\t0.0\t1\t14 1\n",
      "17\t25\t0.0\t1\t22 3\n",
      "18\t8\t0.0\t1\t7 1\n",
      "19\t14\t0.0\t1\t12 2\n",
      "20\t16\t0.0\t1\t14 2\n",
      "21\t16\t0.0\t1\t15 1\n",
      "22\t15\t0.0\t1\t14 1\n",
      "23\t29\t0.0\t1\t28 1\n",
      "24\t38\t0.0\t1\t35 3\n",
      "25\t50\t0.0\t1\t47 3\n",
      "26\t60\t0.0\t1\t51 9\n",
      "27\t35\t0.0\t1\t34 1\n",
      "28\t119\t0.0\t1\t110 9\n",
      "29\t119\t0.0\t1\t117 2\n",
      "30\t92\t0.0\t1\t90 2\n",
      "31\t98\t0.0\t1\t95 3\n",
      "32\t218\t0.0\t1\t213 5\n",
      "33\t150\t0.0\t1\t141 9\n",
      "34\t865\t0.0\t1\t844 21\n",
      "35\t17\t0.0\t1\t13 4\n",
      "36\t21\t0.0\t1\t21\n",
      "37\t105\t0.0\t1\t104 1\n",
      "38\t93\t0.0\t1\t90 3\n",
      "39\t71\t0.0\t1\t71\n",
      "40\t55\t0.0\t1\t55\n",
      "41\t78\t0.0\t1\t76 2\n",
      "42\t131\t0.0\t1\t125 6\n",
      "43\t648\t0.0\t1\t643 5\n",
      "44\t412\t0.0\t1\t401 11\n",
      "45\t633\t0.0\t1\t616 17\n",
      "46\t741\t0.0\t1\t727 14\n",
      "47\t386\t0.0\t1\t384 2\n",
      "48\t347\t0.0\t1\t341 6\n",
      "49\t503\t0.0\t1\t499 4\n",
      "50\t429\t0.0\t1\t427 2\n",
      "51\t186\t0.0\t1\t183 3\n",
      "52\t451\t0.0\t1\t441 10\n",
      "53\t181\t0.0\t1\t169 12\n",
      "54\t1212\t0.0\t1\t1202 10\n",
      "55\t352\t0.0\t1\t350 2\n",
      "56\t213\t0.0\t1\t206 7\n",
      "57\t1315\t0.0\t1\t1305 10\n",
      "58\t691\t0.0\t1\t681 10\n",
      "59\t395\t0.0\t1\t391 4\n",
      "60\t196\t0.0\t1\t195 1\n",
      "61\t80\t0.0\t1\t77 3\n",
      "62\t440\t0.0\t1\t437 3\n",
      "63\t684\t0.0\t1\t680 4\n",
      "64\t116\t0.0\t1\t115 1\n",
      "65\t80\t0.0\t1\t79 1\n",
      "66\t49\t0.0\t1\t49\n",
      "67\t53\t0.0\t1\t52 1\n",
      "68\t15\t0.0\t1\t15\n",
      "69\t22\t0.0\t1\t21 1\n",
      "70\t27\t0.0\t1\t27\n",
      "71\t5\t0.0\t1\t5\n",
      "72\t8\t0.0\t1\t8\n",
      "73\t7\t0.0\t1\t7\n",
      "74\t15\t0.0\t1\t14 1\n",
      "75\t26\t0.0\t1\t26\n",
      "76\t77\t0.0\t1\t76 1\n",
      "77\t245\t0.0\t1\t240 5\n",
      "78\t549\t0.0\t1\t540 9\n",
      "79\t587\t0.0\t1\t560 27\n",
      "80\t4770\t0.0\t1\t4718 52\n",
      "81\t4305\t0.0\t1\t4266 39\n",
      "82\t1715\t0.0\t1\t1702 13\n",
      "83\t279\t0.0\t1\t276 3\n",
      "84\t69\t0.0\t1\t67 2\n",
      "85\t14\t0.0\t1\t14\n",
      "86\t4\t0.0\t1\t4\n",
      "92\t1\t0.0\t1\t1\n",
      "95\t1\t0.0\t1\t1\n",
      "224\t1\t0.0\t1\t1\n",
      "227\t1\t0.0\t1\t1\n",
      "228\t3\t0.0\t1\t3\n",
      "\n",
      "Read1 before filtering:\n",
      "total reads: 214883\n",
      "total bases: 55946693\n",
      "Q20 bases: 55552372(99.2952%)\n",
      "Q30 bases: 54492948(97.4016%)\n",
      "\n",
      "Read2 before filtering:\n",
      "total reads: 214883\n",
      "total bases: 54192906\n",
      "Q20 bases: 53544566(98.8036%)\n",
      "Q30 bases: 51834770(95.6486%)\n",
      "\n",
      "Merged and filtered:\n",
      "total reads: 209572\n",
      "total bases: 71090208\n",
      "Q20 bases: 70737983(99.5045%)\n",
      "Q30 bases: 69665370(97.9957%)\n",
      "\n",
      "Filtering result:\n",
      "reads passed filter: 425974\n",
      "reads failed due to low quality: 286\n",
      "reads failed due to too many N: 20\n",
      "reads failed due to too short: 3486\n",
      "reads with adapter trimmed: 52572\n",
      "bases trimmed due to adapters: 425596\n",
      "reads corrected by overlap analysis: 30704\n",
      "bases corrected by overlap analysis: 47761\n",
      "\n",
      "Duplication rate: 79.4808%\n",
      "\n",
      "Insert size peak (evaluated by paired-end reads): 380\n",
      "\n",
      "Read pairs merged: 209572\n",
      "% of original read pairs: 97.5284%\n",
      "% in reads after filtering: 100%\n",
      "\n",
      "\n",
      "JSON report: 2_fastp/Hrd1_DMS_Rep1_Region2_Input_report.json\n",
      "HTML report: 2_fastp/Hrd1_DMS_Rep1_Region2_Input_report.html\n",
      "\n",
      "fastp -i 1_cutadapt/Hrd1_DMS_Rep1_Region2_Input_R1_cut.fastq.gz -I 1_cutadapt/Hrd1_DMS_Rep1_Region2_Input_R2_cut.fastq.gz -m --merged_out 2_fastp/Hrd1_DMS_Rep1_Region2_Input_merge.fastq.gz -o 2_fastp/Hrd1_DMS_Rep1_Region2_Input_R1_cut.fastq.gz -O 2_fastp/Hrd1_DMS_Rep1_Region2_Input_R2_cut.fastq.gz -w 12 -h 2_fastp/Hrd1_DMS_Rep1_Region2_Input_report.html -j 2_fastp/Hrd1_DMS_Rep1_Region2_Input_report.json \n",
      "fastp v0.20.0, time used: 2 seconds\n",
      "209572 reads; of these:\n",
      "  209572 (100.00%) were unpaired; of these:\n",
      "    26225 (12.51%) aligned 0 times\n",
      "    183347 (87.49%) aligned exactly 1 time\n",
      "    0 (0.00%) aligned >1 times\n",
      "87.49% overall alignment rate\n",
      "[bam_sort_core] merging from 0 files and 12 in-memory blocks...\n",
      "This is cutadapt 1.18 with Python 3.7.16\n",
      "Command line parameters: -a CTGTCTCTTATACACATCT -A CTGTCTCTTATACACATCT -u 8 -U 8 -q 30 -o 1_cutadapt/Hrd1_DMS_Rep2_Region2_Input_R1_cut.fastq.gz -p 1_cutadapt/Hrd1_DMS_Rep2_Region2_Input_R2_cut.fastq.gz --cores=12 0_fastq_files/Hrd1_DMS_Rep2_Region2_Input_R1_001.fastq.gz 0_fastq_files/Hrd1_DMS_Rep2_Region2_Input_R2_001.fastq.gz\n",
      "Processing reads on 12 cores in paired-end mode ...\n",
      "Finished in 1.20 s (11 us/read; 5.56 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            110,880\n",
      "  Read 1 with adapter:                   5,628 (5.1%)\n",
      "  Read 2 with adapter:                   4,070 (3.7%)\n",
      "Pairs written (passing filters):       110,880 (100.0%)\n",
      "\n",
      "Total basepairs processed:    66,417,041 bp\n",
      "  Read 1:    33,155,467 bp\n",
      "  Read 2:    33,261,574 bp\n",
      "Quality-trimmed:               7,292,018 bp (11.0%)\n",
      "  Read 1:     1,242,635 bp\n",
      "  Read 2:     6,049,383 bp\n",
      "Total written (filtered):     56,918,486 bp (85.7%)\n",
      "  Read 1:    30,766,592 bp\n",
      "  Read 2:    26,151,894 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: CTGTCTCTTATACACATCT; Type: regular 3'; Length: 19; Trimmed: 5628 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 58.8%\n",
      "  C: 14.9%\n",
      "  G: 11.2%\n",
      "  T: 15.1%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t950\t1732.5\t0\t950\n",
      "4\t1159\t433.1\t0\t1159\n",
      "5\t4\t108.3\t0\t4\n",
      "6\t1\t27.1\t0\t1\n",
      "8\t1\t1.7\t0\t1\n",
      "10\t2\t0.1\t1\t2\n",
      "11\t2\t0.0\t1\t2\n",
      "12\t1\t0.0\t1\t1\n",
      "13\t2\t0.0\t1\t1 1\n",
      "14\t5\t0.0\t1\t3 2\n",
      "15\t3\t0.0\t1\t3\n",
      "16\t2\t0.0\t1\t2\n",
      "17\t8\t0.0\t1\t8\n",
      "18\t2\t0.0\t1\t2\n",
      "19\t12\t0.0\t1\t12\n",
      "20\t6\t0.0\t1\t6\n",
      "21\t13\t0.0\t1\t13\n",
      "22\t2\t0.0\t1\t2\n",
      "23\t4\t0.0\t1\t4\n",
      "24\t2\t0.0\t1\t2\n",
      "25\t4\t0.0\t1\t4\n",
      "26\t4\t0.0\t1\t4\n",
      "27\t3\t0.0\t1\t2 1\n",
      "28\t6\t0.0\t1\t5 1\n",
      "29\t12\t0.0\t1\t9 3\n",
      "30\t30\t0.0\t1\t27 3\n",
      "31\t11\t0.0\t1\t9 2\n",
      "32\t1\t0.0\t1\t1\n",
      "33\t1\t0.0\t1\t1\n",
      "34\t8\t0.0\t1\t7 1\n",
      "35\t1\t0.0\t1\t1\n",
      "36\t2\t0.0\t1\t2\n",
      "37\t5\t0.0\t1\t5\n",
      "38\t6\t0.0\t1\t4 2\n",
      "39\t5\t0.0\t1\t5\n",
      "40\t11\t0.0\t1\t10 1\n",
      "41\t8\t0.0\t1\t7 1\n",
      "42\t12\t0.0\t1\t11 1\n",
      "43\t30\t0.0\t1\t28 2\n",
      "44\t3\t0.0\t1\t1 2\n",
      "45\t2\t0.0\t1\t2\n",
      "47\t2\t0.0\t1\t1 1\n",
      "48\t3\t0.0\t1\t3\n",
      "49\t4\t0.0\t1\t4\n",
      "50\t6\t0.0\t1\t6\n",
      "51\t10\t0.0\t1\t9 1\n",
      "52\t2\t0.0\t1\t2\n",
      "53\t9\t0.0\t1\t9\n",
      "54\t8\t0.0\t1\t8\n",
      "55\t16\t0.0\t1\t16\n",
      "56\t21\t0.0\t1\t20 1\n",
      "57\t13\t0.0\t1\t13\n",
      "58\t15\t0.0\t1\t14 1\n",
      "59\t20\t0.0\t1\t19 1\n",
      "60\t23\t0.0\t1\t23\n",
      "61\t23\t0.0\t1\t23\n",
      "62\t13\t0.0\t1\t13\n",
      "63\t24\t0.0\t1\t23 1\n",
      "64\t37\t0.0\t1\t36 1\n",
      "65\t46\t0.0\t1\t45 1\n",
      "66\t28\t0.0\t1\t28\n",
      "67\t12\t0.0\t1\t12\n",
      "68\t16\t0.0\t1\t14 2\n",
      "69\t18\t0.0\t1\t18\n",
      "70\t23\t0.0\t1\t21 2\n",
      "71\t36\t0.0\t1\t36\n",
      "72\t82\t0.0\t1\t76 6\n",
      "73\t81\t0.0\t1\t79 2\n",
      "74\t242\t0.0\t1\t239 3\n",
      "75\t239\t0.0\t1\t224 15\n",
      "76\t1634\t0.0\t1\t1617 17\n",
      "77\t487\t0.0\t1\t481 6\n",
      "78\t54\t0.0\t1\t51 3\n",
      "79\t11\t0.0\t1\t11\n",
      "80\t5\t0.0\t1\t5\n",
      "81\t1\t0.0\t1\t1\n",
      "83\t2\t0.0\t1\t2\n",
      "84\t1\t0.0\t1\t1\n",
      "85\t1\t0.0\t1\t1\n",
      "88\t2\t0.0\t1\t2\n",
      "106\t1\t0.0\t1\t1\n",
      "146\t1\t0.0\t1\t1\n",
      "189\t1\t0.0\t1\t1\n",
      "224\t1\t0.0\t1\t0 1\n",
      "225\t1\t0.0\t1\t1\n",
      "226\t1\t0.0\t1\t0 1\n",
      "227\t3\t0.0\t1\t2 1\n",
      "228\t3\t0.0\t1\t3\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CTGTCTCTTATACACATCT; Type: regular 3'; Length: 19; Trimmed: 4070 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 28.9%\n",
      "  C: 35.0%\n",
      "  G: 16.9%\n",
      "  T: 19.2%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t595\t1732.5\t0\t595\n",
      "4\t6\t433.1\t0\t6\n",
      "5\t3\t108.3\t0\t3\n",
      "6\t3\t27.1\t0\t3\n",
      "8\t3\t1.7\t0\t3\n",
      "9\t4\t0.4\t0\t4\n",
      "10\t4\t0.1\t1\t4\n",
      "11\t8\t0.0\t1\t8\n",
      "12\t2\t0.0\t1\t2\n",
      "13\t5\t0.0\t1\t5\n",
      "14\t2\t0.0\t1\t2\n",
      "16\t3\t0.0\t1\t3\n",
      "17\t7\t0.0\t1\t7\n",
      "18\t3\t0.0\t1\t2 1\n",
      "19\t3\t0.0\t1\t3\n",
      "20\t3\t0.0\t1\t3\n",
      "21\t5\t0.0\t1\t5\n",
      "22\t8\t0.0\t1\t8\n",
      "23\t5\t0.0\t1\t5\n",
      "24\t10\t0.0\t1\t8 2\n",
      "25\t12\t0.0\t1\t12\n",
      "26\t14\t0.0\t1\t12 2\n",
      "27\t7\t0.0\t1\t6 1\n",
      "28\t37\t0.0\t1\t37\n",
      "29\t32\t0.0\t1\t30 2\n",
      "30\t31\t0.0\t1\t30 1\n",
      "31\t33\t0.0\t1\t33\n",
      "32\t74\t0.0\t1\t71 3\n",
      "33\t43\t0.0\t1\t41 2\n",
      "34\t147\t0.0\t1\t143 4\n",
      "35\t5\t0.0\t1\t4 1\n",
      "36\t8\t0.0\t1\t6 2\n",
      "37\t30\t0.0\t1\t29 1\n",
      "38\t25\t0.0\t1\t25\n",
      "39\t32\t0.0\t1\t31 1\n",
      "40\t34\t0.0\t1\t33 1\n",
      "41\t48\t0.0\t1\t46 2\n",
      "42\t54\t0.0\t1\t51 3\n",
      "43\t323\t0.0\t1\t321 2\n",
      "44\t105\t0.0\t1\t104 1\n",
      "45\t237\t0.0\t1\t232 5\n",
      "46\t234\t0.0\t1\t231 3\n",
      "47\t124\t0.0\t1\t124\n",
      "48\t63\t0.0\t1\t62 1\n",
      "49\t144\t0.0\t1\t142 2\n",
      "50\t57\t0.0\t1\t57\n",
      "51\t73\t0.0\t1\t73\n",
      "52\t130\t0.0\t1\t129 1\n",
      "53\t46\t0.0\t1\t44 2\n",
      "54\t184\t0.0\t1\t182 2\n",
      "55\t83\t0.0\t1\t81 2\n",
      "56\t51\t0.0\t1\t49 2\n",
      "57\t172\t0.0\t1\t168 4\n",
      "58\t128\t0.0\t1\t126 2\n",
      "59\t67\t0.0\t1\t67\n",
      "60\t33\t0.0\t1\t33\n",
      "61\t9\t0.0\t1\t9\n",
      "62\t39\t0.0\t1\t39\n",
      "63\t59\t0.0\t1\t59\n",
      "64\t14\t0.0\t1\t14\n",
      "65\t9\t0.0\t1\t9\n",
      "66\t11\t0.0\t1\t11\n",
      "67\t9\t0.0\t1\t9\n",
      "68\t1\t0.0\t1\t1\n",
      "69\t3\t0.0\t1\t3\n",
      "70\t5\t0.0\t1\t5\n",
      "71\t1\t0.0\t1\t1\n",
      "72\t1\t0.0\t1\t0 1\n",
      "74\t1\t0.0\t1\t1\n",
      "75\t7\t0.0\t1\t7\n",
      "76\t3\t0.0\t1\t3\n",
      "77\t19\t0.0\t1\t18 1\n",
      "78\t31\t0.0\t1\t31\n",
      "79\t31\t0.0\t1\t31\n",
      "80\t99\t0.0\t1\t99\n",
      "81\t70\t0.0\t1\t70\n",
      "82\t39\t0.0\t1\t37 2\n",
      "83\t4\t0.0\t1\t4\n",
      "84\t1\t0.0\t1\t1\n",
      "228\t2\t0.0\t1\t2\n",
      "\n",
      "Read1 before filtering:\n",
      "total reads: 110880\n",
      "total bases: 30766592\n",
      "Q20 bases: 30334167(98.5945%)\n",
      "Q30 bases: 29182698(94.8519%)\n",
      "\n",
      "Read2 before filtering:\n",
      "total reads: 110880\n",
      "total bases: 26151894\n",
      "Q20 bases: 25348609(96.9284%)\n",
      "Q30 bases: 23609887(90.2798%)\n",
      "\n",
      "Merged and filtered:\n",
      "total reads: 104272\n",
      "total bases: 38454440\n",
      "Q20 bases: 38080658(99.028%)\n",
      "Q30 bases: 36941657(96.066%)\n",
      "\n",
      "Filtering result:\n",
      "reads passed filter: 219246\n",
      "reads failed due to low quality: 100\n",
      "reads failed due to too many N: 6\n",
      "reads failed due to too short: 2408\n",
      "reads with adapter trimmed: 6990\n",
      "bases trimmed due to adapters: 56858\n",
      "reads corrected by overlap analysis: 37832\n",
      "bases corrected by overlap analysis: 59569\n",
      "\n",
      "Duplication rate: 20.8092%\n",
      "\n",
      "Insert size peak (evaluated by paired-end reads): 380\n",
      "\n",
      "Read pairs merged: 104272\n",
      "% of original read pairs: 94.0404%\n",
      "% in reads after filtering: 100%\n",
      "\n",
      "\n",
      "JSON report: 2_fastp/Hrd1_DMS_Rep2_Region2_Input_report.json\n",
      "HTML report: 2_fastp/Hrd1_DMS_Rep2_Region2_Input_report.html\n",
      "\n",
      "fastp -i 1_cutadapt/Hrd1_DMS_Rep2_Region2_Input_R1_cut.fastq.gz -I 1_cutadapt/Hrd1_DMS_Rep2_Region2_Input_R2_cut.fastq.gz -m --merged_out 2_fastp/Hrd1_DMS_Rep2_Region2_Input_merge.fastq.gz -o 2_fastp/Hrd1_DMS_Rep2_Region2_Input_R1_cut.fastq.gz -O 2_fastp/Hrd1_DMS_Rep2_Region2_Input_R2_cut.fastq.gz -w 12 -h 2_fastp/Hrd1_DMS_Rep2_Region2_Input_report.html -j 2_fastp/Hrd1_DMS_Rep2_Region2_Input_report.json \n",
      "fastp v0.20.0, time used: 2 seconds\n",
      "104272 reads; of these:\n",
      "  104272 (100.00%) were unpaired; of these:\n",
      "    3482 (3.34%) aligned 0 times\n",
      "    100790 (96.66%) aligned exactly 1 time\n",
      "    0 (0.00%) aligned >1 times\n",
      "96.66% overall alignment rate\n",
      "[bam_sort_core] merging from 0 files and 12 in-memory blocks...\n",
      "This is cutadapt 1.18 with Python 3.7.16\n",
      "Command line parameters: -a CTGTCTCTTATACACATCT -A CTGTCTCTTATACACATCT -u 8 -U 8 -q 30 -o 1_cutadapt/Hrd1_DMS_Rep3_Region2_Input_R1_cut.fastq.gz -p 1_cutadapt/Hrd1_DMS_Rep3_Region2_Input_R2_cut.fastq.gz --cores=12 0_fastq_files/Hrd1_DMS_Rep3_Region2_Input_R1_001.fastq.gz 0_fastq_files/Hrd1_DMS_Rep3_Region2_Input_R2_001.fastq.gz\n",
      "Processing reads on 12 cores in paired-end mode ...\n",
      "Finished in 2.35 s (9 us/read; 6.59 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            258,350\n",
      "  Read 1 with adapter:                   4,021 (1.6%)\n",
      "  Read 2 with adapter:                   6,184 (2.4%)\n",
      "Pairs written (passing filters):       258,350 (100.0%)\n",
      "\n",
      "Total basepairs processed:   154,997,553 bp\n",
      "  Read 1:    77,496,687 bp\n",
      "  Read 2:    77,500,866 bp\n",
      "Quality-trimmed:               6,505,399 bp (4.2%)\n",
      "  Read 1:     1,704,928 bp\n",
      "  Read 2:     4,800,471 bp\n",
      "Total written (filtered):    143,810,216 bp (92.8%)\n",
      "  Read 1:    73,441,933 bp\n",
      "  Read 2:    70,368,283 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: CTGTCTCTTATACACATCT; Type: regular 3'; Length: 19; Trimmed: 4021 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 32.1%\n",
      "  C: 30.3%\n",
      "  G: 17.6%\n",
      "  T: 20.1%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t264\t4036.7\t0\t264\n",
      "4\t9\t1009.2\t0\t9\n",
      "5\t3\t252.3\t0\t3\n",
      "8\t1\t3.9\t0\t1\n",
      "11\t2\t0.1\t1\t2\n",
      "13\t2\t0.0\t1\t2\n",
      "14\t2\t0.0\t1\t2\n",
      "15\t1\t0.0\t1\t1\n",
      "16\t2\t0.0\t1\t2\n",
      "17\t1\t0.0\t1\t1\n",
      "18\t1\t0.0\t1\t0 1\n",
      "19\t2\t0.0\t1\t2\n",
      "20\t2\t0.0\t1\t2\n",
      "21\t2\t0.0\t1\t1 1\n",
      "22\t2\t0.0\t1\t2\n",
      "24\t2\t0.0\t1\t2\n",
      "28\t1\t0.0\t1\t1\n",
      "29\t2\t0.0\t1\t2\n",
      "30\t5\t0.0\t1\t2 3\n",
      "31\t4\t0.0\t1\t2 2\n",
      "33\t1\t0.0\t1\t1\n",
      "34\t8\t0.0\t1\t7 1\n",
      "35\t3\t0.0\t1\t3\n",
      "36\t5\t0.0\t1\t4 1\n",
      "37\t1\t0.0\t1\t1\n",
      "38\t7\t0.0\t1\t7\n",
      "39\t12\t0.0\t1\t9 3\n",
      "40\t7\t0.0\t1\t5 2\n",
      "41\t3\t0.0\t1\t3\n",
      "42\t6\t0.0\t1\t6\n",
      "43\t49\t0.0\t1\t43 6\n",
      "44\t4\t0.0\t1\t4\n",
      "45\t1\t0.0\t1\t1\n",
      "47\t2\t0.0\t1\t1 1\n",
      "49\t2\t0.0\t1\t2\n",
      "50\t1\t0.0\t1\t1\n",
      "51\t6\t0.0\t1\t5 1\n",
      "53\t1\t0.0\t1\t1\n",
      "54\t4\t0.0\t1\t4\n",
      "55\t12\t0.0\t1\t9 3\n",
      "56\t1\t0.0\t1\t1\n",
      "57\t3\t0.0\t1\t3\n",
      "58\t4\t0.0\t1\t3 1\n",
      "59\t9\t0.0\t1\t8 1\n",
      "60\t5\t0.0\t1\t4 1\n",
      "61\t14\t0.0\t1\t9 5\n",
      "62\t9\t0.0\t1\t9\n",
      "63\t11\t0.0\t1\t10 1\n",
      "64\t8\t0.0\t1\t8\n",
      "65\t14\t0.0\t1\t10 4\n",
      "66\t14\t0.0\t1\t13 1\n",
      "67\t8\t0.0\t1\t7 1\n",
      "68\t5\t0.0\t1\t5\n",
      "69\t11\t0.0\t1\t10 1\n",
      "70\t7\t0.0\t1\t7\n",
      "71\t19\t0.0\t1\t18 1\n",
      "72\t34\t0.0\t1\t32 2\n",
      "73\t48\t0.0\t1\t38 10\n",
      "74\t67\t0.0\t1\t56 11\n",
      "75\t138\t0.0\t1\t107 31\n",
      "76\t1582\t0.0\t1\t1384 198\n",
      "77\t1294\t0.0\t1\t1111 183\n",
      "78\t178\t0.0\t1\t151 27\n",
      "79\t38\t0.0\t1\t31 7\n",
      "80\t13\t0.0\t1\t10 3\n",
      "81\t3\t0.0\t1\t3\n",
      "82\t2\t0.0\t1\t2\n",
      "83\t1\t0.0\t1\t1\n",
      "84\t1\t0.0\t1\t1\n",
      "85\t1\t0.0\t1\t1\n",
      "90\t1\t0.0\t1\t1\n",
      "105\t1\t0.0\t1\t0 1\n",
      "117\t1\t0.0\t1\t1\n",
      "152\t1\t0.0\t1\t0 1\n",
      "157\t1\t0.0\t1\t1\n",
      "158\t1\t0.0\t1\t1\n",
      "168\t1\t0.0\t1\t1\n",
      "173\t1\t0.0\t1\t1\n",
      "177\t1\t0.0\t1\t1\n",
      "187\t2\t0.0\t1\t2\n",
      "213\t1\t0.0\t1\t1\n",
      "218\t8\t0.0\t1\t6 2\n",
      "219\t14\t0.0\t1\t9 5\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CTGTCTCTTATACACATCT; Type: regular 3'; Length: 19; Trimmed: 6184 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 51.9%\n",
      "  C: 15.8%\n",
      "  G: 13.4%\n",
      "  T: 18.9%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2235\t4036.7\t0\t2235\n",
      "4\t225\t1009.2\t0\t225\n",
      "8\t1\t3.9\t0\t1\n",
      "9\t1\t1.0\t0\t1\n",
      "10\t1\t0.2\t1\t1\n",
      "12\t1\t0.0\t1\t1\n",
      "13\t1\t0.0\t1\t1\n",
      "14\t3\t0.0\t1\t3\n",
      "15\t5\t0.0\t1\t4 1\n",
      "16\t3\t0.0\t1\t3\n",
      "18\t1\t0.0\t1\t1\n",
      "19\t6\t0.0\t1\t6\n",
      "20\t3\t0.0\t1\t3\n",
      "21\t8\t0.0\t1\t7 1\n",
      "22\t2\t0.0\t1\t2\n",
      "23\t1\t0.0\t1\t1\n",
      "24\t6\t0.0\t1\t5 1\n",
      "25\t2\t0.0\t1\t2\n",
      "26\t1\t0.0\t1\t1\n",
      "27\t1\t0.0\t1\t1\n",
      "28\t2\t0.0\t1\t2\n",
      "29\t7\t0.0\t1\t6 1\n",
      "30\t2\t0.0\t1\t2\n",
      "31\t8\t0.0\t1\t8\n",
      "32\t8\t0.0\t1\t8\n",
      "33\t34\t0.0\t1\t33 1\n",
      "34\t9\t0.0\t1\t9\n",
      "35\t51\t0.0\t1\t50 1\n",
      "36\t23\t0.0\t1\t22 1\n",
      "37\t12\t0.0\t1\t12\n",
      "38\t9\t0.0\t1\t8 1\n",
      "39\t9\t0.0\t1\t9\n",
      "40\t21\t0.0\t1\t21\n",
      "41\t43\t0.0\t1\t41 2\n",
      "42\t30\t0.0\t1\t30\n",
      "43\t60\t0.0\t1\t59 1\n",
      "44\t57\t0.0\t1\t56 1\n",
      "45\t40\t0.0\t1\t39 1\n",
      "46\t48\t0.0\t1\t47 1\n",
      "47\t44\t0.0\t1\t44\n",
      "48\t41\t0.0\t1\t40 1\n",
      "49\t41\t0.0\t1\t41\n",
      "50\t18\t0.0\t1\t18\n",
      "51\t30\t0.0\t1\t30\n",
      "52\t28\t0.0\t1\t26 2\n",
      "53\t24\t0.0\t1\t24\n",
      "54\t71\t0.0\t1\t69 2\n",
      "55\t37\t0.0\t1\t37\n",
      "56\t43\t0.0\t1\t43\n",
      "57\t93\t0.0\t1\t93\n",
      "58\t54\t0.0\t1\t53 1\n",
      "59\t33\t0.0\t1\t33\n",
      "60\t41\t0.0\t1\t41\n",
      "61\t16\t0.0\t1\t16\n",
      "62\t34\t0.0\t1\t34\n",
      "63\t106\t0.0\t1\t106\n",
      "64\t33\t0.0\t1\t33\n",
      "65\t10\t0.0\t1\t10\n",
      "66\t6\t0.0\t1\t6\n",
      "67\t3\t0.0\t1\t3\n",
      "68\t4\t0.0\t1\t4\n",
      "69\t1\t0.0\t1\t1\n",
      "70\t4\t0.0\t1\t4\n",
      "71\t1\t0.0\t1\t1\n",
      "72\t1\t0.0\t1\t1\n",
      "73\t3\t0.0\t1\t3\n",
      "74\t8\t0.0\t1\t7 1\n",
      "75\t12\t0.0\t1\t12\n",
      "76\t21\t0.0\t1\t20 1\n",
      "77\t29\t0.0\t1\t28 1\n",
      "78\t75\t0.0\t1\t75\n",
      "79\t225\t0.0\t1\t222 3\n",
      "80\t886\t0.0\t1\t879 7\n",
      "81\t918\t0.0\t1\t907 11\n",
      "82\t187\t0.0\t1\t184 3\n",
      "83\t19\t0.0\t1\t19\n",
      "84\t3\t0.0\t1\t3\n",
      "85\t1\t0.0\t1\t1\n",
      "\n",
      "Read1 before filtering:\n",
      "total reads: 258350\n",
      "total bases: 73441933\n",
      "Q20 bases: 72950459(99.3308%)\n",
      "Q30 bases: 71542558(97.4138%)\n",
      "\n",
      "Read2 before filtering:\n",
      "total reads: 258350\n",
      "total bases: 70368283\n",
      "Q20 bases: 69480633(98.7386%)\n",
      "Q30 bases: 67126850(95.3936%)\n",
      "\n",
      "Merged and filtered:\n",
      "total reads: 249276\n",
      "total bases: 95577322\n",
      "Q20 bases: 95152666(99.5557%)\n",
      "Q30 bases: 93747339(98.0853%)\n",
      "\n",
      "Filtering result:\n",
      "reads passed filter: 510312\n",
      "reads failed due to low quality: 420\n",
      "reads failed due to too many N: 36\n",
      "reads failed due to too short: 5932\n",
      "reads with adapter trimmed: 7482\n",
      "bases trimmed due to adapters: 61704\n",
      "reads corrected by overlap analysis: 44112\n",
      "bases corrected by overlap analysis: 60711\n",
      "\n",
      "Duplication rate: 81.9048%\n",
      "\n",
      "Insert size peak (evaluated by paired-end reads): 389\n",
      "\n",
      "Read pairs merged: 249276\n",
      "% of original read pairs: 96.4877%\n",
      "% in reads after filtering: 100%\n",
      "\n",
      "\n",
      "JSON report: 2_fastp/Hrd1_DMS_Rep3_Region2_Input_report.json\n",
      "HTML report: 2_fastp/Hrd1_DMS_Rep3_Region2_Input_report.html\n",
      "\n",
      "fastp -i 1_cutadapt/Hrd1_DMS_Rep3_Region2_Input_R1_cut.fastq.gz -I 1_cutadapt/Hrd1_DMS_Rep3_Region2_Input_R2_cut.fastq.gz -m --merged_out 2_fastp/Hrd1_DMS_Rep3_Region2_Input_merge.fastq.gz -o 2_fastp/Hrd1_DMS_Rep3_Region2_Input_R1_cut.fastq.gz -O 2_fastp/Hrd1_DMS_Rep3_Region2_Input_R2_cut.fastq.gz -w 12 -h 2_fastp/Hrd1_DMS_Rep3_Region2_Input_report.html -j 2_fastp/Hrd1_DMS_Rep3_Region2_Input_report.json \n",
      "fastp v0.20.0, time used: 2 seconds\n",
      "249276 reads; of these:\n",
      "  249276 (100.00%) were unpaired; of these:\n",
      "    3749 (1.50%) aligned 0 times\n",
      "    245527 (98.50%) aligned exactly 1 time\n",
      "    0 (0.00%) aligned >1 times\n",
      "98.50% overall alignment rate\n",
      "[bam_sort_core] merging from 0 files and 12 in-memory blocks...\n",
      "This is cutadapt 1.18 with Python 3.7.16\n",
      "Command line parameters: -a CTGTCTCTTATACACATCT -A CTGTCTCTTATACACATCT -u 8 -U 8 -q 30 -o 1_cutadapt/Hrd1_DMS_Rep4_Region2_Input_R1_cut.fastq.gz -p 1_cutadapt/Hrd1_DMS_Rep4_Region2_Input_R2_cut.fastq.gz --cores=12 0_fastq_files/Hrd1_DMS_Rep4_Region2_Input_R1_001.fastq.gz 0_fastq_files/Hrd1_DMS_Rep4_Region2_Input_R2_001.fastq.gz\n",
      "Processing reads on 12 cores in paired-end mode ...\n",
      "Finished in 1.92 s (9 us/read; 6.51 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            208,471\n",
      "  Read 1 with adapter:                   3,840 (1.8%)\n",
      "  Read 2 with adapter:                   5,663 (2.7%)\n",
      "Pairs written (passing filters):       208,471 (100.0%)\n",
      "\n",
      "Total basepairs processed:   125,072,561 bp\n",
      "  Read 1:    62,534,552 bp\n",
      "  Read 2:    62,538,009 bp\n",
      "Quality-trimmed:               4,573,062 bp (3.7%)\n",
      "  Read 1:     1,625,778 bp\n",
      "  Read 2:     2,947,284 bp\n",
      "Total written (filtered):    116,624,405 bp (93.2%)\n",
      "  Read 1:    58,967,736 bp\n",
      "  Read 2:    57,656,669 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: CTGTCTCTTATACACATCT; Type: regular 3'; Length: 19; Trimmed: 3840 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 31.5%\n",
      "  C: 29.6%\n",
      "  G: 19.6%\n",
      "  T: 19.4%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t235\t3257.4\t0\t235\n",
      "4\t4\t814.3\t0\t4\n",
      "5\t2\t203.6\t0\t2\n",
      "6\t1\t50.9\t0\t1\n",
      "8\t2\t3.2\t0\t2\n",
      "10\t1\t0.2\t1\t1\n",
      "11\t3\t0.0\t1\t3\n",
      "13\t3\t0.0\t1\t3\n",
      "17\t2\t0.0\t1\t2\n",
      "19\t6\t0.0\t1\t4 2\n",
      "20\t4\t0.0\t1\t4\n",
      "21\t3\t0.0\t1\t2 1\n",
      "23\t3\t0.0\t1\t2 1\n",
      "27\t2\t0.0\t1\t2\n",
      "28\t3\t0.0\t1\t2 1\n",
      "29\t2\t0.0\t1\t1 1\n",
      "30\t4\t0.0\t1\t4\n",
      "31\t4\t0.0\t1\t3 1\n",
      "32\t1\t0.0\t1\t1\n",
      "36\t5\t0.0\t1\t5\n",
      "37\t2\t0.0\t1\t1 1\n",
      "38\t2\t0.0\t1\t2\n",
      "39\t8\t0.0\t1\t7 1\n",
      "40\t5\t0.0\t1\t4 1\n",
      "42\t4\t0.0\t1\t3 1\n",
      "43\t28\t0.0\t1\t23 5\n",
      "44\t2\t0.0\t1\t2\n",
      "45\t2\t0.0\t1\t2\n",
      "47\t3\t0.0\t1\t2 1\n",
      "48\t1\t0.0\t1\t1\n",
      "52\t1\t0.0\t1\t1\n",
      "53\t1\t0.0\t1\t1\n",
      "54\t9\t0.0\t1\t8 1\n",
      "55\t7\t0.0\t1\t7\n",
      "56\t3\t0.0\t1\t2 1\n",
      "57\t6\t0.0\t1\t5 1\n",
      "58\t4\t0.0\t1\t2 2\n",
      "59\t10\t0.0\t1\t9 1\n",
      "60\t13\t0.0\t1\t12 1\n",
      "61\t8\t0.0\t1\t7 1\n",
      "62\t5\t0.0\t1\t5\n",
      "63\t11\t0.0\t1\t8 3\n",
      "64\t6\t0.0\t1\t4 2\n",
      "65\t17\t0.0\t1\t15 2\n",
      "66\t13\t0.0\t1\t12 1\n",
      "67\t7\t0.0\t1\t6 1\n",
      "68\t1\t0.0\t1\t1\n",
      "69\t3\t0.0\t1\t2 1\n",
      "70\t5\t0.0\t1\t3 2\n",
      "71\t35\t0.0\t1\t33 2\n",
      "72\t44\t0.0\t1\t34 10\n",
      "73\t62\t0.0\t1\t48 14\n",
      "74\t75\t0.0\t1\t66 9\n",
      "75\t148\t0.0\t1\t121 27\n",
      "76\t1737\t0.0\t1\t1523 214\n",
      "77\t1070\t0.0\t1\t929 141\n",
      "78\t126\t0.0\t1\t105 21\n",
      "79\t18\t0.0\t1\t16 2\n",
      "80\t4\t0.0\t1\t4\n",
      "81\t1\t0.0\t1\t1\n",
      "82\t1\t0.0\t1\t1\n",
      "84\t1\t0.0\t1\t1\n",
      "85\t2\t0.0\t1\t2\n",
      "89\t1\t0.0\t1\t1\n",
      "95\t1\t0.0\t1\t0 1\n",
      "119\t1\t0.0\t1\t0 1\n",
      "141\t1\t0.0\t1\t1\n",
      "153\t1\t0.0\t1\t0 1\n",
      "154\t1\t0.0\t1\t1\n",
      "164\t1\t0.0\t1\t1\n",
      "172\t1\t0.0\t1\t1\n",
      "199\t1\t0.0\t1\t1\n",
      "201\t1\t0.0\t1\t1\n",
      "202\t1\t0.0\t1\t0 1\n",
      "204\t1\t0.0\t1\t0 1\n",
      "213\t1\t0.0\t1\t1\n",
      "215\t1\t0.0\t1\t1\n",
      "218\t12\t0.0\t1\t9 3\n",
      "219\t13\t0.0\t1\t9 4\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CTGTCTCTTATACACATCT; Type: regular 3'; Length: 19; Trimmed: 5663 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 49.7%\n",
      "  C: 17.1%\n",
      "  G: 15.1%\n",
      "  T: 18.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1825\t3257.4\t0\t1825\n",
      "4\t189\t814.3\t0\t189\n",
      "5\t1\t203.6\t0\t1\n",
      "8\t1\t3.2\t0\t1\n",
      "12\t2\t0.0\t1\t2\n",
      "13\t2\t0.0\t1\t2\n",
      "14\t2\t0.0\t1\t2\n",
      "15\t6\t0.0\t1\t6\n",
      "16\t4\t0.0\t1\t3 1\n",
      "17\t4\t0.0\t1\t4\n",
      "18\t2\t0.0\t1\t1 1\n",
      "19\t4\t0.0\t1\t3 1\n",
      "20\t2\t0.0\t1\t2\n",
      "21\t3\t0.0\t1\t3\n",
      "23\t5\t0.0\t1\t4 1\n",
      "24\t2\t0.0\t1\t2\n",
      "25\t8\t0.0\t1\t7 1\n",
      "26\t3\t0.0\t1\t3\n",
      "27\t2\t0.0\t1\t2\n",
      "28\t4\t0.0\t1\t4\n",
      "29\t3\t0.0\t1\t3\n",
      "30\t2\t0.0\t1\t1 1\n",
      "31\t6\t0.0\t1\t6\n",
      "32\t8\t0.0\t1\t7 1\n",
      "33\t23\t0.0\t1\t21 2\n",
      "34\t8\t0.0\t1\t8\n",
      "35\t32\t0.0\t1\t32\n",
      "36\t12\t0.0\t1\t12\n",
      "37\t8\t0.0\t1\t8\n",
      "38\t15\t0.0\t1\t15\n",
      "39\t10\t0.0\t1\t10\n",
      "40\t22\t0.0\t1\t22\n",
      "41\t43\t0.0\t1\t42 1\n",
      "42\t18\t0.0\t1\t18\n",
      "43\t56\t0.0\t1\t56\n",
      "44\t42\t0.0\t1\t42\n",
      "45\t23\t0.0\t1\t23\n",
      "46\t44\t0.0\t1\t43 1\n",
      "47\t25\t0.0\t1\t24 1\n",
      "48\t38\t0.0\t1\t38\n",
      "49\t42\t0.0\t1\t42\n",
      "50\t18\t0.0\t1\t18\n",
      "51\t15\t0.0\t1\t14 1\n",
      "52\t17\t0.0\t1\t17\n",
      "53\t29\t0.0\t1\t28 1\n",
      "54\t51\t0.0\t1\t51\n",
      "55\t34\t0.0\t1\t33 1\n",
      "56\t28\t0.0\t1\t28\n",
      "57\t61\t0.0\t1\t61\n",
      "58\t40\t0.0\t1\t40\n",
      "59\t21\t0.0\t1\t21\n",
      "60\t36\t0.0\t1\t36\n",
      "61\t11\t0.0\t1\t11\n",
      "62\t39\t0.0\t1\t38 1\n",
      "63\t84\t0.0\t1\t84\n",
      "64\t11\t0.0\t1\t11\n",
      "65\t15\t0.0\t1\t15\n",
      "66\t2\t0.0\t1\t2\n",
      "67\t4\t0.0\t1\t4\n",
      "68\t3\t0.0\t1\t3\n",
      "69\t2\t0.0\t1\t2\n",
      "70\t3\t0.0\t1\t3\n",
      "71\t2\t0.0\t1\t2\n",
      "72\t3\t0.0\t1\t3\n",
      "73\t6\t0.0\t1\t6\n",
      "74\t6\t0.0\t1\t6\n",
      "75\t9\t0.0\t1\t9\n",
      "76\t28\t0.0\t1\t27 1\n",
      "77\t38\t0.0\t1\t37 1\n",
      "78\t53\t0.0\t1\t52 1\n",
      "79\t230\t0.0\t1\t223 7\n",
      "80\t903\t0.0\t1\t892 11\n",
      "81\t1010\t0.0\t1\t1003 7\n",
      "82\t264\t0.0\t1\t262 2\n",
      "83\t29\t0.0\t1\t29\n",
      "84\t4\t0.0\t1\t4\n",
      "86\t1\t0.0\t1\t1\n",
      "218\t1\t0.0\t1\t1\n",
      "219\t1\t0.0\t1\t1\n",
      "\n",
      "Read1 before filtering:\n",
      "total reads: 208471\n",
      "total bases: 58967736\n",
      "Q20 bases: 58486989(99.1847%)\n",
      "Q30 bases: 57127756(96.8797%)\n",
      "\n",
      "Read2 before filtering:\n",
      "total reads: 208471\n",
      "total bases: 57656669\n",
      "Q20 bases: 57048982(98.946%)\n",
      "Q30 bases: 55433813(96.1447%)\n",
      "\n",
      "Merged and filtered:\n",
      "total reads: 202217\n",
      "total bases: 77399283\n",
      "Q20 bases: 76969161(99.4443%)\n",
      "Q30 bases: 75599335(97.6745%)\n",
      "\n",
      "Filtering result:\n",
      "reads passed filter: 413130\n",
      "reads failed due to low quality: 434\n",
      "reads failed due to too many N: 28\n",
      "reads failed due to too short: 3350\n",
      "reads with adapter trimmed: 7314\n",
      "bases trimmed due to adapters: 60794\n",
      "reads corrected by overlap analysis: 35520\n",
      "bases corrected by overlap analysis: 49864\n",
      "\n",
      "Duplication rate: 85.4144%\n",
      "\n",
      "Insert size peak (evaluated by paired-end reads): 389\n",
      "\n",
      "Read pairs merged: 202217\n",
      "% of original read pairs: 97.0001%\n",
      "% in reads after filtering: 100%\n",
      "\n",
      "\n",
      "JSON report: 2_fastp/Hrd1_DMS_Rep4_Region2_Input_report.json\n",
      "HTML report: 2_fastp/Hrd1_DMS_Rep4_Region2_Input_report.html\n",
      "\n",
      "fastp -i 1_cutadapt/Hrd1_DMS_Rep4_Region2_Input_R1_cut.fastq.gz -I 1_cutadapt/Hrd1_DMS_Rep4_Region2_Input_R2_cut.fastq.gz -m --merged_out 2_fastp/Hrd1_DMS_Rep4_Region2_Input_merge.fastq.gz -o 2_fastp/Hrd1_DMS_Rep4_Region2_Input_R1_cut.fastq.gz -O 2_fastp/Hrd1_DMS_Rep4_Region2_Input_R2_cut.fastq.gz -w 12 -h 2_fastp/Hrd1_DMS_Rep4_Region2_Input_report.html -j 2_fastp/Hrd1_DMS_Rep4_Region2_Input_report.json \n",
      "fastp v0.20.0, time used: 2 seconds\n",
      "202217 reads; of these:\n",
      "  202217 (100.00%) were unpaired; of these:\n",
      "    3647 (1.80%) aligned 0 times\n",
      "    198570 (98.20%) aligned exactly 1 time\n",
      "    0 (0.00%) aligned >1 times\n",
      "98.20% overall alignment rate\n",
      "[bam_sort_core] merging from 0 files and 12 in-memory blocks...\n",
      "This is cutadapt 1.18 with Python 3.7.16\n",
      "Command line parameters: -a CTGTCTCTTATACACATCT -A CTGTCTCTTATACACATCT -u 8 -U 8 -q 30 -o 1_cutadapt/Hrd1_DMS_Rep1_Region2_WT_R1_cut.fastq.gz -p 1_cutadapt/Hrd1_DMS_Rep1_Region2_WT_R2_cut.fastq.gz --cores=12 0_fastq_files/Hrd1_DMS_Rep1_Region2_WT_R1_001.fastq.gz 0_fastq_files/Hrd1_DMS_Rep1_Region2_WT_R2_001.fastq.gz\n",
      "Processing reads on 12 cores in paired-end mode ...\n",
      "Finished in 1.82 s (9 us/read; 6.70 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            203,106\n",
      "  Read 1 with adapter:                     146 (0.1%)\n",
      "  Read 2 with adapter:                   4,792 (2.4%)\n",
      "Pairs written (passing filters):       203,106 (100.0%)\n",
      "\n",
      "Total basepairs processed:   121,660,211 bp\n",
      "  Read 1:    60,931,226 bp\n",
      "  Read 2:    60,728,985 bp\n",
      "Quality-trimmed:               3,130,563 bp (2.6%)\n",
      "  Read 1:       856,904 bp\n",
      "  Read 2:     2,273,659 bp\n",
      "Total written (filtered):    115,262,727 bp (94.7%)\n",
      "  Read 1:    58,448,743 bp\n",
      "  Read 2:    56,813,984 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: CTGTCTCTTATACACATCT; Type: regular 3'; Length: 19; Trimmed: 146 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 1.4%\n",
      "  C: 79.5%\n",
      "  G: 10.3%\n",
      "  T: 8.9%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t142\t3173.5\t0\t142\n",
      "76\t3\t0.0\t1\t3\n",
      "77\t1\t0.0\t1\t1\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CTGTCTCTTATACACATCT; Type: regular 3'; Length: 19; Trimmed: 4792 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 97.3%\n",
      "  C: 0.1%\n",
      "  G: 0.3%\n",
      "  T: 2.3%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by \"A\" extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 3' end.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2986\t3173.5\t0\t2986\n",
      "4\t1799\t793.4\t0\t1799\n",
      "5\t1\t198.3\t0\t1\n",
      "6\t2\t49.6\t0\t2\n",
      "80\t2\t0.0\t1\t2\n",
      "81\t1\t0.0\t1\t1\n",
      "82\t1\t0.0\t1\t1\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "Read1 before filtering:\n",
      "total reads: 203106\n",
      "total bases: 58448743\n",
      "Q20 bases: 58001974(99.2356%)\n",
      "Q30 bases: 56795519(97.1715%)\n",
      "\n",
      "Read2 before filtering:\n",
      "total reads: 203106\n",
      "total bases: 56813984\n",
      "Q20 bases: 56161287(98.8512%)\n",
      "Q30 bases: 54406046(95.7617%)\n",
      "\n",
      "Merged and filtered:\n",
      "total reads: 197806\n",
      "total bases: 75125621\n",
      "Q20 bases: 74720476(99.4607%)\n",
      "Q30 bases: 73495598(97.8303%)\n",
      "\n",
      "Filtering result:\n",
      "reads passed filter: 402900\n",
      "reads failed due to low quality: 248\n",
      "reads failed due to too many N: 26\n",
      "reads failed due to too short: 3038\n",
      "reads with adapter trimmed: 16\n",
      "bases trimmed due to adapters: 1082\n",
      "reads corrected by overlap analysis: 32098\n",
      "bases corrected by overlap analysis: 51170\n",
      "\n",
      "Duplication rate: 78.5615%\n",
      "\n",
      "Insert size peak (evaluated by paired-end reads): 380\n",
      "\n",
      "Read pairs merged: 197806\n",
      "% of original read pairs: 97.3905%\n",
      "% in reads after filtering: 100%\n",
      "\n",
      "\n",
      "JSON report: 2_fastp/Hrd1_DMS_Rep1_Region2_WT_report.json\n",
      "HTML report: 2_fastp/Hrd1_DMS_Rep1_Region2_WT_report.html\n",
      "\n",
      "fastp -i 1_cutadapt/Hrd1_DMS_Rep1_Region2_WT_R1_cut.fastq.gz -I 1_cutadapt/Hrd1_DMS_Rep1_Region2_WT_R2_cut.fastq.gz -m --merged_out 2_fastp/Hrd1_DMS_Rep1_Region2_WT_merge.fastq.gz -o 2_fastp/Hrd1_DMS_Rep1_Region2_WT_R1_cut.fastq.gz -O 2_fastp/Hrd1_DMS_Rep1_Region2_WT_R2_cut.fastq.gz -w 12 -h 2_fastp/Hrd1_DMS_Rep1_Region2_WT_report.html -j 2_fastp/Hrd1_DMS_Rep1_Region2_WT_report.json \n",
      "fastp v0.20.0, time used: 2 seconds\n",
      "197806 reads; of these:\n",
      "  197806 (100.00%) were unpaired; of these:\n",
      "    14 (0.01%) aligned 0 times\n",
      "    197792 (99.99%) aligned exactly 1 time\n",
      "    0 (0.00%) aligned >1 times\n",
      "99.99% overall alignment rate\n",
      "[bam_sort_core] merging from 0 files and 12 in-memory blocks...\n",
      "This is cutadapt 1.18 with Python 3.7.16\n",
      "Command line parameters: -a CTGTCTCTTATACACATCT -A CTGTCTCTTATACACATCT -u 8 -U 8 -q 30 -o 1_cutadapt/Hrd1_DMS_Rep2_Region2_WT_R1_cut.fastq.gz -p 1_cutadapt/Hrd1_DMS_Rep2_Region2_WT_R2_cut.fastq.gz --cores=12 0_fastq_files/Hrd1_DMS_Rep2_Region2_WT_R1_001.fastq.gz 0_fastq_files/Hrd1_DMS_Rep2_Region2_WT_R2_001.fastq.gz\n",
      "Processing reads on 12 cores in paired-end mode ...\n",
      "Finished in 2.01 s (9 us/read; 6.51 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            218,556\n",
      "  Read 1 with adapter:                   2,065 (0.9%)\n",
      "  Read 2 with adapter:                   1,057 (0.5%)\n",
      "Pairs written (passing filters):       218,556 (100.0%)\n",
      "\n",
      "Total basepairs processed:   130,915,530 bp\n",
      "  Read 1:    65,349,312 bp\n",
      "  Read 2:    65,566,218 bp\n",
      "Quality-trimmed:               5,054,776 bp (3.9%)\n",
      "  Read 1:       412,296 bp\n",
      "  Read 2:     4,642,480 bp\n",
      "Total written (filtered):    122,351,292 bp (93.5%)\n",
      "  Read 1:    63,180,014 bp\n",
      "  Read 2:    59,171,278 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: CTGTCTCTTATACACATCT; Type: regular 3'; Length: 19; Trimmed: 2065 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 96.5%\n",
      "  C: 0.3%\n",
      "  G: 0.9%\n",
      "  T: 2.3%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by \"A\" extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 3' end.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t871\t3414.9\t0\t871\n",
      "4\t1179\t853.7\t0\t1179\n",
      "5\t1\t213.4\t0\t1\n",
      "76\t9\t0.0\t1\t9\n",
      "77\t4\t0.0\t1\t4\n",
      "228\t1\t0.0\t1\t1\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CTGTCTCTTATACACATCT; Type: regular 3'; Length: 19; Trimmed: 1057 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1040\t3414.9\t0\t1040\n",
      "4\t2\t853.7\t0\t2\n",
      "5\t1\t213.4\t0\t1\n",
      "31\t1\t0.0\t1\t1\n",
      "41\t1\t0.0\t1\t1\n",
      "42\t1\t0.0\t1\t1\n",
      "46\t1\t0.0\t1\t1\n",
      "55\t1\t0.0\t1\t1\n",
      "57\t2\t0.0\t1\t2\n",
      "63\t1\t0.0\t1\t1\n",
      "80\t2\t0.0\t1\t2\n",
      "81\t3\t0.0\t1\t3\n",
      "84\t1\t0.0\t1\t1\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "Read1 before filtering:\n",
      "total reads: 218556\n",
      "total bases: 63180014\n",
      "Q20 bases: 62873523(99.5149%)\n",
      "Q30 bases: 61950220(98.0535%)\n",
      "\n",
      "Read2 before filtering:\n",
      "total reads: 218556\n",
      "total bases: 59171278\n",
      "Q20 bases: 57986225(97.9972%)\n",
      "Q30 bases: 55208080(93.3022%)\n",
      "\n",
      "Merged and filtered:\n",
      "total reads: 212249\n",
      "total bases: 80576518\n",
      "Q20 bases: 80314208(99.6745%)\n",
      "Q30 bases: 79389258(98.5265%)\n",
      "\n",
      "Filtering result:\n",
      "reads passed filter: 432762\n",
      "reads failed due to low quality: 452\n",
      "reads failed due to too many N: 30\n",
      "reads failed due to too short: 3868\n",
      "reads with adapter trimmed: 34\n",
      "bases trimmed due to adapters: 900\n",
      "reads corrected by overlap analysis: 42592\n",
      "bases corrected by overlap analysis: 62647\n",
      "\n",
      "Duplication rate: 25.431%\n",
      "\n",
      "Insert size peak (evaluated by paired-end reads): 380\n",
      "\n",
      "Read pairs merged: 212249\n",
      "% of original read pairs: 97.1142%\n",
      "% in reads after filtering: 100%\n",
      "\n",
      "\n",
      "JSON report: 2_fastp/Hrd1_DMS_Rep2_Region2_WT_report.json\n",
      "HTML report: 2_fastp/Hrd1_DMS_Rep2_Region2_WT_report.html\n",
      "\n",
      "fastp -i 1_cutadapt/Hrd1_DMS_Rep2_Region2_WT_R1_cut.fastq.gz -I 1_cutadapt/Hrd1_DMS_Rep2_Region2_WT_R2_cut.fastq.gz -m --merged_out 2_fastp/Hrd1_DMS_Rep2_Region2_WT_merge.fastq.gz -o 2_fastp/Hrd1_DMS_Rep2_Region2_WT_R1_cut.fastq.gz -O 2_fastp/Hrd1_DMS_Rep2_Region2_WT_R2_cut.fastq.gz -w 12 -h 2_fastp/Hrd1_DMS_Rep2_Region2_WT_report.html -j 2_fastp/Hrd1_DMS_Rep2_Region2_WT_report.json \n",
      "fastp v0.20.0, time used: 2 seconds\n",
      "212249 reads; of these:\n",
      "  212249 (100.00%) were unpaired; of these:\n",
      "    28 (0.01%) aligned 0 times\n",
      "    212221 (99.99%) aligned exactly 1 time\n",
      "    0 (0.00%) aligned >1 times\n",
      "99.99% overall alignment rate\n",
      "[bam_sort_core] merging from 0 files and 12 in-memory blocks...\n",
      "This is cutadapt 1.18 with Python 3.7.16\n",
      "Command line parameters: -a CTGTCTCTTATACACATCT -A CTGTCTCTTATACACATCT -u 8 -U 8 -q 30 -o 1_cutadapt/Hrd1_DMS_Rep3_Region2_WT_R1_cut.fastq.gz -p 1_cutadapt/Hrd1_DMS_Rep3_Region2_WT_R2_cut.fastq.gz --cores=12 0_fastq_files/Hrd1_DMS_Rep3_Region2_WT_R1_001.fastq.gz 0_fastq_files/Hrd1_DMS_Rep3_Region2_WT_R2_001.fastq.gz\n",
      "Processing reads on 12 cores in paired-end mode ...\n",
      "Finished in 1.47 s (9 us/read; 6.40 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            157,123\n",
      "  Read 1 with adapter:                     193 (0.1%)\n",
      "  Read 2 with adapter:                     999 (0.6%)\n",
      "Pairs written (passing filters):       157,123 (100.0%)\n",
      "\n",
      "Total basepairs processed:    94,270,623 bp\n",
      "  Read 1:    47,134,715 bp\n",
      "  Read 2:    47,135,908 bp\n",
      "Quality-trimmed:               3,391,633 bp (3.6%)\n",
      "  Read 1:       945,442 bp\n",
      "  Read 2:     2,446,191 bp\n",
      "Total written (filtered):     88,360,163 bp (93.7%)\n",
      "  Read 1:    44,931,063 bp\n",
      "  Read 2:    43,429,100 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: CTGTCTCTTATACACATCT; Type: regular 3'; Length: 19; Trimmed: 193 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 12.4%\n",
      "  C: 78.8%\n",
      "  G: 6.2%\n",
      "  T: 2.6%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t183\t2455.0\t0\t183\n",
      "5\t1\t153.4\t0\t1\n",
      "67\t1\t0.0\t1\t1\n",
      "69\t1\t0.0\t1\t1\n",
      "75\t1\t0.0\t1\t1\n",
      "76\t2\t0.0\t1\t2\n",
      "77\t3\t0.0\t1\t3\n",
      "78\t1\t0.0\t1\t1\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CTGTCTCTTATACACATCT; Type: regular 3'; Length: 19; Trimmed: 999 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 80.7%\n",
      "  C: 0.6%\n",
      "  G: 0.6%\n",
      "  T: 18.1%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by \"A\" extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 3' end.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t898\t2455.0\t0\t898\n",
      "4\t93\t613.8\t0\t93\n",
      "47\t1\t0.0\t1\t0 1\n",
      "62\t1\t0.0\t1\t1\n",
      "64\t1\t0.0\t1\t1\n",
      "76\t1\t0.0\t1\t1\n",
      "77\t1\t0.0\t1\t1\n",
      "80\t2\t0.0\t1\t2\n",
      "81\t1\t0.0\t1\t1\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "Read1 before filtering:\n",
      "total reads: 157123\n",
      "total bases: 44931063\n",
      "Q20 bases: 44547213(99.1457%)\n",
      "Q30 bases: 43461140(96.7285%)\n",
      "\n",
      "Read2 before filtering:\n",
      "total reads: 157123\n",
      "total bases: 43429100\n",
      "Q20 bases: 42888068(98.7542%)\n",
      "Q30 bases: 41442621(95.4259%)\n",
      "\n",
      "Merged and filtered:\n",
      "total reads: 151481\n",
      "total bases: 58839760\n",
      "Q20 bases: 58501342(99.4248%)\n",
      "Q30 bases: 57414644(97.578%)\n",
      "\n",
      "Filtering result:\n",
      "reads passed filter: 310966\n",
      "reads failed due to low quality: 284\n",
      "reads failed due to too many N: 14\n",
      "reads failed due to too short: 2982\n",
      "reads with adapter trimmed: 26\n",
      "bases trimmed due to adapters: 1224\n",
      "reads corrected by overlap analysis: 30032\n",
      "bases corrected by overlap analysis: 41219\n",
      "\n",
      "Duplication rate: 77.8612%\n",
      "\n",
      "Insert size peak (evaluated by paired-end reads): 389\n",
      "\n",
      "Read pairs merged: 151481\n",
      "% of original read pairs: 96.4092%\n",
      "% in reads after filtering: 100%\n",
      "\n",
      "\n",
      "JSON report: 2_fastp/Hrd1_DMS_Rep3_Region2_WT_report.json\n",
      "HTML report: 2_fastp/Hrd1_DMS_Rep3_Region2_WT_report.html\n",
      "\n",
      "fastp -i 1_cutadapt/Hrd1_DMS_Rep3_Region2_WT_R1_cut.fastq.gz -I 1_cutadapt/Hrd1_DMS_Rep3_Region2_WT_R2_cut.fastq.gz -m --merged_out 2_fastp/Hrd1_DMS_Rep3_Region2_WT_merge.fastq.gz -o 2_fastp/Hrd1_DMS_Rep3_Region2_WT_R1_cut.fastq.gz -O 2_fastp/Hrd1_DMS_Rep3_Region2_WT_R2_cut.fastq.gz -w 12 -h 2_fastp/Hrd1_DMS_Rep3_Region2_WT_report.html -j 2_fastp/Hrd1_DMS_Rep3_Region2_WT_report.json \n",
      "fastp v0.20.0, time used: 1 seconds\n",
      "151481 reads; of these:\n",
      "  151481 (100.00%) were unpaired; of these:\n",
      "    16 (0.01%) aligned 0 times\n",
      "    151465 (99.99%) aligned exactly 1 time\n",
      "    0 (0.00%) aligned >1 times\n",
      "99.99% overall alignment rate\n",
      "[bam_sort_core] merging from 0 files and 12 in-memory blocks...\n",
      "This is cutadapt 1.18 with Python 3.7.16\n",
      "Command line parameters: -a CTGTCTCTTATACACATCT -A CTGTCTCTTATACACATCT -u 8 -U 8 -q 30 -o 1_cutadapt/Hrd1_DMS_Rep4_Region2_WT_R1_cut.fastq.gz -p 1_cutadapt/Hrd1_DMS_Rep4_Region2_WT_R2_cut.fastq.gz --cores=12 0_fastq_files/Hrd1_DMS_Rep4_Region2_WT_R1_001.fastq.gz 0_fastq_files/Hrd1_DMS_Rep4_Region2_WT_R2_001.fastq.gz\n",
      "Processing reads on 12 cores in paired-end mode ...\n",
      "Finished in 1.63 s (9 us/read; 6.33 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            172,213\n",
      "  Read 1 with adapter:                     356 (0.2%)\n",
      "  Read 2 with adapter:                   1,191 (0.7%)\n",
      "Pairs written (passing filters):       172,213 (100.0%)\n",
      "\n",
      "Total basepairs processed:   103,324,557 bp\n",
      "  Read 1:    51,661,416 bp\n",
      "  Read 2:    51,663,141 bp\n",
      "Quality-trimmed:               3,314,809 bp (3.2%)\n",
      "  Read 1:     1,606,915 bp\n",
      "  Read 2:     1,707,894 bp\n",
      "Total written (filtered):     97,248,771 bp (94.1%)\n",
      "  Read 1:    48,675,330 bp\n",
      "  Read 2:    48,573,441 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: CTGTCTCTTATACACATCT; Type: regular 3'; Length: 19; Trimmed: 356 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 5.9%\n",
      "  C: 84.8%\n",
      "  G: 7.0%\n",
      "  T: 2.2%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by \"C\" extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 3' end.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t346\t2690.8\t0\t346\n",
      "4\t4\t672.7\t0\t4\n",
      "32\t1\t0.0\t1\t1\n",
      "75\t1\t0.0\t1\t1\n",
      "76\t2\t0.0\t1\t2\n",
      "77\t2\t0.0\t1\t1 1\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CTGTCTCTTATACACATCT; Type: regular 3'; Length: 19; Trimmed: 1191 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1085\t2690.8\t0\t1085\n",
      "4\t100\t672.7\t0\t100\n",
      "47\t1\t0.0\t1\t1\n",
      "78\t1\t0.0\t1\t1\n",
      "80\t2\t0.0\t1\t2\n",
      "81\t2\t0.0\t1\t2\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "Read1 before filtering:\n",
      "total reads: 172213\n",
      "total bases: 48675330\n",
      "Q20 bases: 48076140(98.769%)\n",
      "Q30 bases: 46469517(95.4683%)\n",
      "\n",
      "Read2 before filtering:\n",
      "total reads: 172213\n",
      "total bases: 48573441\n",
      "Q20 bases: 48103842(99.0332%)\n",
      "Q30 bases: 46822449(96.3952%)\n",
      "\n",
      "Merged and filtered:\n",
      "total reads: 166331\n",
      "total bases: 64656950\n",
      "Q20 bases: 64119274(99.1684%)\n",
      "Q30 bases: 62516922(96.6902%)\n",
      "\n",
      "Filtering result:\n",
      "reads passed filter: 341810\n",
      "reads failed due to low quality: 332\n",
      "reads failed due to too many N: 30\n",
      "reads failed due to too short: 2254\n",
      "reads with adapter trimmed: 40\n",
      "bases trimmed due to adapters: 2383\n",
      "reads corrected by overlap analysis: 36094\n",
      "bases corrected by overlap analysis: 53295\n",
      "\n",
      "Duplication rate: 87.9765%\n",
      "\n",
      "Insert size peak (evaluated by paired-end reads): 389\n",
      "\n",
      "Read pairs merged: 166331\n",
      "% of original read pairs: 96.5845%\n",
      "% in reads after filtering: 100%\n",
      "\n",
      "\n",
      "JSON report: 2_fastp/Hrd1_DMS_Rep4_Region2_WT_report.json\n",
      "HTML report: 2_fastp/Hrd1_DMS_Rep4_Region2_WT_report.html\n",
      "\n",
      "fastp -i 1_cutadapt/Hrd1_DMS_Rep4_Region2_WT_R1_cut.fastq.gz -I 1_cutadapt/Hrd1_DMS_Rep4_Region2_WT_R2_cut.fastq.gz -m --merged_out 2_fastp/Hrd1_DMS_Rep4_Region2_WT_merge.fastq.gz -o 2_fastp/Hrd1_DMS_Rep4_Region2_WT_R1_cut.fastq.gz -O 2_fastp/Hrd1_DMS_Rep4_Region2_WT_R2_cut.fastq.gz -w 12 -h 2_fastp/Hrd1_DMS_Rep4_Region2_WT_report.html -j 2_fastp/Hrd1_DMS_Rep4_Region2_WT_report.json \n",
      "fastp v0.20.0, time used: 2 seconds\n",
      "166331 reads; of these:\n",
      "  166331 (100.00%) were unpaired; of these:\n",
      "    18 (0.01%) aligned 0 times\n",
      "    166313 (99.99%) aligned exactly 1 time\n",
      "    0 (0.00%) aligned >1 times\n",
      "99.99% overall alignment rate\n",
      "[bam_sort_core] merging from 0 files and 12 in-memory blocks...\n",
      "This is cutadapt 1.18 with Python 3.7.16\n",
      "Command line parameters: -a CTGTCTCTTATACACATCT -A CTGTCTCTTATACACATCT -u 8 -U 8 -q 30 -o 1_cutadapt/Hrd1_DMS_Rep2_Region2_LDead_R1_cut.fastq.gz -p 1_cutadapt/Hrd1_DMS_Rep2_Region2_LDead_R2_cut.fastq.gz --cores=12 0_fastq_files/Hrd1_DMS_Rep2_Region2_LDead_R1_001.fastq.gz 0_fastq_files/Hrd1_DMS_Rep2_Region2_LDead_R2_001.fastq.gz\n",
      "Processing reads on 12 cores in paired-end mode ...\n",
      "Finished in 0.33 s (17 us/read; 3.44 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             19,112\n",
      "  Read 1 with adapter:                     201 (1.1%)\n",
      "  Read 2 with adapter:                      89 (0.5%)\n",
      "Pairs written (passing filters):        19,112 (100.0%)\n",
      "\n",
      "Total basepairs processed:    11,448,122 bp\n",
      "  Read 1:     5,714,572 bp\n",
      "  Read 2:     5,733,550 bp\n",
      "Quality-trimmed:                 502,685 bp (4.4%)\n",
      "  Read 1:        37,189 bp\n",
      "  Read 2:       465,496 bp\n",
      "Total written (filtered):     10,637,382 bp (92.9%)\n",
      "  Read 1:     5,522,984 bp\n",
      "  Read 2:     5,114,398 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: CTGTCTCTTATACACATCT; Type: regular 3'; Length: 19; Trimmed: 201 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 94.5%\n",
      "  C: 0.5%\n",
      "  G: 0.5%\n",
      "  T: 4.5%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by \"A\" extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 3' end.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t79\t298.6\t0\t79\n",
      "4\t113\t74.7\t0\t113\n",
      "23\t1\t0.0\t1\t1\n",
      "28\t1\t0.0\t1\t1\n",
      "41\t1\t0.0\t1\t1\n",
      "64\t1\t0.0\t1\t1\n",
      "66\t1\t0.0\t1\t1\n",
      "67\t1\t0.0\t1\t1\n",
      "74\t1\t0.0\t1\t0 1\n",
      "219\t1\t0.0\t1\t1\n",
      "232\t1\t0.0\t1\t1\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CTGTCTCTTATACACATCT; Type: regular 3'; Length: 19; Trimmed: 89 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t80\t298.6\t0\t80\n",
      "19\t1\t0.0\t1\t1\n",
      "30\t1\t0.0\t1\t1\n",
      "32\t1\t0.0\t1\t1\n",
      "35\t1\t0.0\t1\t1\n",
      "44\t1\t0.0\t1\t1\n",
      "48\t1\t0.0\t1\t1\n",
      "49\t1\t0.0\t1\t1\n",
      "80\t1\t0.0\t1\t1\n",
      "183\t1\t0.0\t1\t0 1\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "Read1 before filtering:\n",
      "total reads: 19112\n",
      "total bases: 5522984\n",
      "Q20 bases: 5496434(99.5193%)\n",
      "Q30 bases: 5416290(98.0682%)\n",
      "\n",
      "Read2 before filtering:\n",
      "total reads: 19112\n",
      "total bases: 5114398\n",
      "Q20 bases: 5002161(97.8055%)\n",
      "Q30 bases: 4745302(92.7832%)\n",
      "\n",
      "Merged and filtered:\n",
      "total reads: 18516\n",
      "total bases: 7027467\n",
      "Q20 bases: 7005252(99.6839%)\n",
      "Q30 bases: 6926727(98.5665%)\n",
      "\n",
      "Filtering result:\n",
      "reads passed filter: 37858\n",
      "reads failed due to low quality: 34\n",
      "reads failed due to too many N: 0\n",
      "reads failed due to too short: 332\n",
      "reads with adapter trimmed: 18\n",
      "bases trimmed due to adapters: 144\n",
      "reads corrected by overlap analysis: 4086\n",
      "bases corrected by overlap analysis: 6103\n",
      "\n",
      "Duplication rate: 56.6327%\n",
      "\n",
      "Insert size peak (evaluated by paired-end reads): 380\n",
      "\n",
      "Read pairs merged: 18516\n",
      "% of original read pairs: 96.8815%\n",
      "% in reads after filtering: 100%\n",
      "\n",
      "\n",
      "JSON report: 2_fastp/Hrd1_DMS_Rep2_Region2_LDead_report.json\n",
      "HTML report: 2_fastp/Hrd1_DMS_Rep2_Region2_LDead_report.html\n",
      "\n",
      "fastp -i 1_cutadapt/Hrd1_DMS_Rep2_Region2_LDead_R1_cut.fastq.gz -I 1_cutadapt/Hrd1_DMS_Rep2_Region2_LDead_R2_cut.fastq.gz -m --merged_out 2_fastp/Hrd1_DMS_Rep2_Region2_LDead_merge.fastq.gz -o 2_fastp/Hrd1_DMS_Rep2_Region2_LDead_R1_cut.fastq.gz -O 2_fastp/Hrd1_DMS_Rep2_Region2_LDead_R2_cut.fastq.gz -w 12 -h 2_fastp/Hrd1_DMS_Rep2_Region2_LDead_report.html -j 2_fastp/Hrd1_DMS_Rep2_Region2_LDead_report.json \n",
      "fastp v0.20.0, time used: 1 seconds\n",
      "18516 reads; of these:\n",
      "  18516 (100.00%) were unpaired; of these:\n",
      "    10 (0.05%) aligned 0 times\n",
      "    18506 (99.95%) aligned exactly 1 time\n",
      "    0 (0.00%) aligned >1 times\n",
      "99.95% overall alignment rate\n",
      "[bam_sort_core] merging from 0 files and 12 in-memory blocks...\n",
      "This is cutadapt 1.18 with Python 3.7.16\n",
      "Command line parameters: -a CTGTCTCTTATACACATCT -A CTGTCTCTTATACACATCT -u 8 -U 8 -q 30 -o 1_cutadapt/Hrd1_DMS_Rep3_Region2_LDead_R1_cut.fastq.gz -p 1_cutadapt/Hrd1_DMS_Rep3_Region2_LDead_R2_cut.fastq.gz --cores=12 0_fastq_files/Hrd1_DMS_Rep3_Region2_LDead_R1_001.fastq.gz 0_fastq_files/Hrd1_DMS_Rep3_Region2_LDead_R2_001.fastq.gz\n",
      "Processing reads on 12 cores in paired-end mode ...\n",
      "Finished in 0.27 s (22 us/read; 2.79 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             12,457\n",
      "  Read 1 with adapter:                      13 (0.1%)\n",
      "  Read 2 with adapter:                      79 (0.6%)\n",
      "Pairs written (passing filters):        12,457 (100.0%)\n",
      "\n",
      "Total basepairs processed:     7,473,954 bp\n",
      "  Read 1:     3,736,933 bp\n",
      "  Read 2:     3,737,021 bp\n",
      "Quality-trimmed:                 243,354 bp (3.3%)\n",
      "  Read 1:        58,020 bp\n",
      "  Read 2:       185,334 bp\n",
      "Total written (filtered):      7,030,863 bp (94.1%)\n",
      "  Read 1:     3,579,110 bp\n",
      "  Read 2:     3,451,753 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: CTGTCTCTTATACACATCT; Type: regular 3'; Length: 19; Trimmed: 13 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 15.4%\n",
      "  C: 61.5%\n",
      "  G: 15.4%\n",
      "  T: 7.7%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t11\t194.6\t0\t11\n",
      "40\t1\t0.0\t1\t1\n",
      "74\t1\t0.0\t1\t1\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CTGTCTCTTATACACATCT; Type: regular 3'; Length: 19; Trimmed: 79 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 79.7%\n",
      "  C: 0.0%\n",
      "  G: 0.0%\n",
      "  T: 20.3%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t70\t194.6\t0\t70\n",
      "4\t8\t48.7\t0\t8\n",
      "36\t1\t0.0\t1\t1\n",
      "\n",
      "Read1 before filtering:\n",
      "total reads: 12457\n",
      "total bases: 3579110\n",
      "Q20 bases: 3554980(99.3258%)\n",
      "Q30 bases: 3485983(97.398%)\n",
      "\n",
      "Read2 before filtering:\n",
      "total reads: 12457\n",
      "total bases: 3451753\n",
      "Q20 bases: 3410401(98.802%)\n",
      "Q30 bases: 3303591(95.7076%)\n",
      "\n",
      "Merged and filtered:\n",
      "total reads: 12016\n",
      "total bases: 4667116\n",
      "Q20 bases: 4646550(99.5593%)\n",
      "Q30 bases: 4578480(98.1008%)\n",
      "\n",
      "Filtering result:\n",
      "reads passed filter: 24576\n",
      "reads failed due to low quality: 30\n",
      "reads failed due to too many N: 0\n",
      "reads failed due to too short: 308\n",
      "reads with adapter trimmed: 4\n",
      "bases trimmed due to adapters: 305\n",
      "reads corrected by overlap analysis: 2069\n",
      "bases corrected by overlap analysis: 2893\n",
      "\n",
      "Duplication rate: 53.3333%\n",
      "\n",
      "Insert size peak (evaluated by paired-end reads): 0\n",
      "\n",
      "Read pairs merged: 12016\n",
      "% of original read pairs: 96.4598%\n",
      "% in reads after filtering: 100%\n",
      "\n",
      "\n",
      "JSON report: 2_fastp/Hrd1_DMS_Rep3_Region2_LDead_report.json\n",
      "HTML report: 2_fastp/Hrd1_DMS_Rep3_Region2_LDead_report.html\n",
      "\n",
      "fastp -i 1_cutadapt/Hrd1_DMS_Rep3_Region2_LDead_R1_cut.fastq.gz -I 1_cutadapt/Hrd1_DMS_Rep3_Region2_LDead_R2_cut.fastq.gz -m --merged_out 2_fastp/Hrd1_DMS_Rep3_Region2_LDead_merge.fastq.gz -o 2_fastp/Hrd1_DMS_Rep3_Region2_LDead_R1_cut.fastq.gz -O 2_fastp/Hrd1_DMS_Rep3_Region2_LDead_R2_cut.fastq.gz -w 12 -h 2_fastp/Hrd1_DMS_Rep3_Region2_LDead_report.html -j 2_fastp/Hrd1_DMS_Rep3_Region2_LDead_report.json \n",
      "fastp v0.20.0, time used: 1 seconds\n",
      "12016 reads; of these:\n",
      "  12016 (100.00%) were unpaired; of these:\n",
      "    2 (0.02%) aligned 0 times\n",
      "    12014 (99.98%) aligned exactly 1 time\n",
      "    0 (0.00%) aligned >1 times\n",
      "99.98% overall alignment rate\n",
      "[bam_sort_core] merging from 0 files and 12 in-memory blocks...\n",
      "This is cutadapt 1.18 with Python 3.7.16\n",
      "Command line parameters: -a CTGTCTCTTATACACATCT -A CTGTCTCTTATACACATCT -u 8 -U 8 -q 30 -o 1_cutadapt/Hrd1_DMS_Rep1_Region3_Input_R1_cut.fastq.gz -p 1_cutadapt/Hrd1_DMS_Rep1_Region3_Input_R2_cut.fastq.gz --cores=12 0_fastq_files/Hrd1_DMS_Rep1_Region3_Input_R1_001.fastq.gz 0_fastq_files/Hrd1_DMS_Rep1_Region3_Input_R2_001.fastq.gz\n",
      "Processing reads on 12 cores in paired-end mode ...\n",
      "Finished in 1.91 s (9 us/read; 6.44 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            205,077\n",
      "  Read 1 with adapter:                   8,947 (4.4%)\n",
      "  Read 2 with adapter:                  13,504 (6.6%)\n",
      "Pairs written (passing filters):       205,077 (100.0%)\n",
      "\n",
      "Total basepairs processed:   122,333,445 bp\n",
      "  Read 1:    61,335,138 bp\n",
      "  Read 2:    60,998,307 bp\n",
      "Quality-trimmed:               8,415,185 bp (6.9%)\n",
      "  Read 1:     2,267,075 bp\n",
      "  Read 2:     6,148,110 bp\n",
      "Total written (filtered):    109,416,514 bp (89.4%)\n",
      "  Read 1:    56,766,278 bp\n",
      "  Read 2:    52,650,236 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: CTGTCTCTTATACACATCT; Type: regular 3'; Length: 19; Trimmed: 8947 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 30.9%\n",
      "  C: 25.0%\n",
      "  G: 18.6%\n",
      "  T: 25.5%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t146\t3204.3\t0\t146\n",
      "4\t8\t801.1\t0\t8\n",
      "5\t5\t200.3\t0\t5\n",
      "6\t4\t50.1\t0\t4\n",
      "9\t2\t0.8\t0\t2\n",
      "10\t2\t0.2\t1\t2\n",
      "11\t8\t0.0\t1\t8\n",
      "12\t3\t0.0\t1\t3\n",
      "13\t1\t0.0\t1\t0 1\n",
      "15\t2\t0.0\t1\t2\n",
      "16\t6\t0.0\t1\t6\n",
      "17\t5\t0.0\t1\t4 1\n",
      "18\t6\t0.0\t1\t5 1\n",
      "19\t8\t0.0\t1\t6 2\n",
      "20\t3\t0.0\t1\t3\n",
      "21\t6\t0.0\t1\t5 1\n",
      "22\t6\t0.0\t1\t5 1\n",
      "23\t1\t0.0\t1\t1\n",
      "24\t5\t0.0\t1\t4 1\n",
      "25\t3\t0.0\t1\t2 1\n",
      "26\t6\t0.0\t1\t5 1\n",
      "27\t5\t0.0\t1\t4 1\n",
      "28\t7\t0.0\t1\t6 1\n",
      "29\t19\t0.0\t1\t18 1\n",
      "30\t16\t0.0\t1\t14 2\n",
      "31\t19\t0.0\t1\t18 1\n",
      "32\t1\t0.0\t1\t1\n",
      "33\t7\t0.0\t1\t5 2\n",
      "34\t4\t0.0\t1\t4\n",
      "35\t7\t0.0\t1\t7\n",
      "36\t9\t0.0\t1\t9\n",
      "37\t13\t0.0\t1\t11 2\n",
      "38\t4\t0.0\t1\t3 1\n",
      "39\t3\t0.0\t1\t2 1\n",
      "40\t8\t0.0\t1\t8\n",
      "41\t33\t0.0\t1\t32 1\n",
      "42\t14\t0.0\t1\t13 1\n",
      "43\t26\t0.0\t1\t23 3\n",
      "44\t6\t0.0\t1\t6\n",
      "45\t1\t0.0\t1\t1\n",
      "46\t2\t0.0\t1\t2\n",
      "47\t1\t0.0\t1\t1\n",
      "48\t1\t0.0\t1\t1\n",
      "49\t3\t0.0\t1\t3\n",
      "50\t2\t0.0\t1\t2\n",
      "51\t16\t0.0\t1\t14 2\n",
      "52\t3\t0.0\t1\t2 1\n",
      "53\t5\t0.0\t1\t5\n",
      "54\t12\t0.0\t1\t11 1\n",
      "55\t45\t0.0\t1\t45\n",
      "56\t15\t0.0\t1\t12 3\n",
      "57\t24\t0.0\t1\t22 2\n",
      "58\t18\t0.0\t1\t17 1\n",
      "59\t14\t0.0\t1\t13 1\n",
      "60\t18\t0.0\t1\t17 1\n",
      "61\t18\t0.0\t1\t18\n",
      "62\t19\t0.0\t1\t19\n",
      "63\t52\t0.0\t1\t50 2\n",
      "64\t21\t0.0\t1\t21\n",
      "65\t69\t0.0\t1\t66 3\n",
      "66\t51\t0.0\t1\t48 3\n",
      "67\t61\t0.0\t1\t61\n",
      "68\t39\t0.0\t1\t39\n",
      "69\t29\t0.0\t1\t29\n",
      "70\t27\t0.0\t1\t27\n",
      "71\t58\t0.0\t1\t57 1\n",
      "72\t64\t0.0\t1\t64\n",
      "73\t78\t0.0\t1\t76 2\n",
      "74\t228\t0.0\t1\t216 12\n",
      "75\t595\t0.0\t1\t559 36\n",
      "76\t4450\t0.0\t1\t4410 40\n",
      "77\t2168\t0.0\t1\t2137 31\n",
      "78\t160\t0.0\t1\t158 2\n",
      "79\t22\t0.0\t1\t22\n",
      "80\t1\t0.0\t1\t1\n",
      "81\t3\t0.0\t1\t3\n",
      "82\t3\t0.0\t1\t2 1\n",
      "83\t4\t0.0\t1\t4\n",
      "84\t3\t0.0\t1\t3\n",
      "85\t1\t0.0\t1\t1\n",
      "86\t3\t0.0\t1\t3\n",
      "87\t1\t0.0\t1\t1\n",
      "89\t1\t0.0\t1\t1\n",
      "90\t1\t0.0\t1\t1\n",
      "91\t1\t0.0\t1\t1\n",
      "94\t2\t0.0\t1\t2\n",
      "96\t1\t0.0\t1\t1\n",
      "98\t1\t0.0\t1\t1\n",
      "101\t1\t0.0\t1\t1\n",
      "102\t1\t0.0\t1\t1\n",
      "104\t1\t0.0\t1\t1\n",
      "105\t1\t0.0\t1\t1\n",
      "110\t2\t0.0\t1\t2\n",
      "119\t1\t0.0\t1\t1\n",
      "121\t1\t0.0\t1\t1\n",
      "123\t1\t0.0\t1\t1\n",
      "124\t1\t0.0\t1\t1\n",
      "128\t1\t0.0\t1\t1\n",
      "130\t1\t0.0\t1\t1\n",
      "132\t3\t0.0\t1\t3\n",
      "138\t1\t0.0\t1\t1\n",
      "154\t1\t0.0\t1\t1\n",
      "160\t1\t0.0\t1\t0 1\n",
      "161\t1\t0.0\t1\t1\n",
      "162\t2\t0.0\t1\t2\n",
      "164\t1\t0.0\t1\t1\n",
      "165\t1\t0.0\t1\t1\n",
      "171\t1\t0.0\t1\t1\n",
      "173\t1\t0.0\t1\t1\n",
      "177\t1\t0.0\t1\t1\n",
      "179\t1\t0.0\t1\t1\n",
      "183\t1\t0.0\t1\t1\n",
      "185\t2\t0.0\t1\t2\n",
      "188\t1\t0.0\t1\t1\n",
      "196\t1\t0.0\t1\t1\n",
      "200\t1\t0.0\t1\t1\n",
      "204\t1\t0.0\t1\t1\n",
      "218\t1\t0.0\t1\t1\n",
      "226\t1\t0.0\t1\t1\n",
      "227\t1\t0.0\t1\t1\n",
      "229\t5\t0.0\t1\t5\n",
      "230\t24\t0.0\t1\t24\n",
      "231\t26\t0.0\t1\t24 2\n",
      "232\t2\t0.0\t1\t1 1\n",
      "234\t1\t0.0\t1\t1\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CTGTCTCTTATACACATCT; Type: regular 3'; Length: 19; Trimmed: 13504 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 31.0%\n",
      "  C: 17.3%\n",
      "  G: 34.0%\n",
      "  T: 17.6%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t4621\t3204.3\t0\t4621\n",
      "4\t126\t801.1\t0\t126\n",
      "5\t28\t200.3\t0\t28\n",
      "6\t9\t50.1\t0\t9\n",
      "8\t2\t3.1\t0\t2\n",
      "9\t1\t0.8\t0\t1\n",
      "10\t4\t0.2\t1\t4\n",
      "11\t6\t0.0\t1\t5 1\n",
      "12\t2\t0.0\t1\t2\n",
      "13\t3\t0.0\t1\t3\n",
      "14\t2\t0.0\t1\t2\n",
      "15\t1\t0.0\t1\t1\n",
      "16\t5\t0.0\t1\t5\n",
      "17\t6\t0.0\t1\t6\n",
      "18\t4\t0.0\t1\t4\n",
      "19\t3\t0.0\t1\t3\n",
      "20\t3\t0.0\t1\t3\n",
      "21\t8\t0.0\t1\t7 1\n",
      "22\t3\t0.0\t1\t2 1\n",
      "23\t16\t0.0\t1\t14 2\n",
      "24\t7\t0.0\t1\t6 1\n",
      "25\t4\t0.0\t1\t3 1\n",
      "26\t12\t0.0\t1\t12\n",
      "27\t11\t0.0\t1\t10 1\n",
      "28\t17\t0.0\t1\t17\n",
      "29\t31\t0.0\t1\t31\n",
      "30\t21\t0.0\t1\t21\n",
      "31\t123\t0.0\t1\t118 5\n",
      "32\t54\t0.0\t1\t52 2\n",
      "33\t52\t0.0\t1\t51 1\n",
      "34\t117\t0.0\t1\t113 4\n",
      "35\t14\t0.0\t1\t14\n",
      "36\t12\t0.0\t1\t10 2\n",
      "37\t51\t0.0\t1\t47 4\n",
      "38\t46\t0.0\t1\t44 2\n",
      "39\t26\t0.0\t1\t26\n",
      "40\t23\t0.0\t1\t22 1\n",
      "41\t22\t0.0\t1\t22\n",
      "42\t35\t0.0\t1\t35\n",
      "43\t136\t0.0\t1\t133 3\n",
      "44\t148\t0.0\t1\t145 3\n",
      "45\t163\t0.0\t1\t160 3\n",
      "46\t330\t0.0\t1\t326 4\n",
      "47\t217\t0.0\t1\t212 5\n",
      "48\t192\t0.0\t1\t191 1\n",
      "49\t253\t0.0\t1\t250 3\n",
      "50\t181\t0.0\t1\t179 2\n",
      "51\t122\t0.0\t1\t122\n",
      "52\t388\t0.0\t1\t383 5\n",
      "53\t125\t0.0\t1\t119 6\n",
      "54\t530\t0.0\t1\t525 5\n",
      "55\t148\t0.0\t1\t147 1\n",
      "56\t75\t0.0\t1\t71 4\n",
      "57\t331\t0.0\t1\t328 3\n",
      "58\t253\t0.0\t1\t251 2\n",
      "59\t107\t0.0\t1\t107\n",
      "60\t69\t0.0\t1\t69\n",
      "61\t73\t0.0\t1\t72 1\n",
      "62\t168\t0.0\t1\t167 1\n",
      "63\t225\t0.0\t1\t224 1\n",
      "64\t51\t0.0\t1\t51\n",
      "65\t23\t0.0\t1\t23\n",
      "66\t10\t0.0\t1\t10\n",
      "67\t10\t0.0\t1\t10\n",
      "68\t8\t0.0\t1\t7 1\n",
      "69\t14\t0.0\t1\t14\n",
      "70\t10\t0.0\t1\t10\n",
      "71\t5\t0.0\t1\t5\n",
      "72\t6\t0.0\t1\t6\n",
      "73\t4\t0.0\t1\t4\n",
      "74\t4\t0.0\t1\t4\n",
      "75\t54\t0.0\t1\t52 2\n",
      "76\t27\t0.0\t1\t27\n",
      "77\t89\t0.0\t1\t85 4\n",
      "78\t145\t0.0\t1\t143 2\n",
      "79\t230\t0.0\t1\t217 13\n",
      "80\t1908\t0.0\t1\t1896 12\n",
      "81\t811\t0.0\t1\t800 11\n",
      "82\t262\t0.0\t1\t258 4\n",
      "83\t51\t0.0\t1\t50 1\n",
      "84\t12\t0.0\t1\t12\n",
      "85\t2\t0.0\t1\t2\n",
      "92\t1\t0.0\t1\t1\n",
      "178\t1\t0.0\t1\t1\n",
      "231\t1\t0.0\t1\t1\n",
      "\n",
      "Read1 before filtering:\n",
      "total reads: 205077\n",
      "total bases: 56766278\n",
      "Q20 bases: 56447898(99.4391%)\n",
      "Q30 bases: 55398460(97.5904%)\n",
      "\n",
      "Read2 before filtering:\n",
      "total reads: 205077\n",
      "total bases: 52650236\n",
      "Q20 bases: 51859010(98.4972%)\n",
      "Q30 bases: 49762502(94.5152%)\n",
      "\n",
      "Merged and filtered:\n",
      "total reads: 198084\n",
      "total bases: 71701376\n",
      "Q20 bases: 71422752(99.6114%)\n",
      "Q30 bases: 70395364(98.1785%)\n",
      "\n",
      "Filtering result:\n",
      "reads passed filter: 406496\n",
      "reads failed due to low quality: 252\n",
      "reads failed due to too many N: 26\n",
      "reads failed due to too short: 3380\n",
      "reads with adapter trimmed: 17506\n",
      "bases trimmed due to adapters: 141471\n",
      "reads corrected by overlap analysis: 31846\n",
      "bases corrected by overlap analysis: 42327\n",
      "\n",
      "Duplication rate: 47.0968%\n",
      "\n",
      "Insert size peak (evaluated by paired-end reads): 377\n",
      "\n",
      "Read pairs merged: 198084\n",
      "% of original read pairs: 96.5901%\n",
      "% in reads after filtering: 100%\n",
      "\n",
      "\n",
      "JSON report: 2_fastp/Hrd1_DMS_Rep1_Region3_Input_report.json\n",
      "HTML report: 2_fastp/Hrd1_DMS_Rep1_Region3_Input_report.html\n",
      "\n",
      "fastp -i 1_cutadapt/Hrd1_DMS_Rep1_Region3_Input_R1_cut.fastq.gz -I 1_cutadapt/Hrd1_DMS_Rep1_Region3_Input_R2_cut.fastq.gz -m --merged_out 2_fastp/Hrd1_DMS_Rep1_Region3_Input_merge.fastq.gz -o 2_fastp/Hrd1_DMS_Rep1_Region3_Input_R1_cut.fastq.gz -O 2_fastp/Hrd1_DMS_Rep1_Region3_Input_R2_cut.fastq.gz -w 12 -h 2_fastp/Hrd1_DMS_Rep1_Region3_Input_report.html -j 2_fastp/Hrd1_DMS_Rep1_Region3_Input_report.json \n",
      "fastp v0.20.0, time used: 2 seconds\n",
      "198084 reads; of these:\n",
      "  198084 (100.00%) were unpaired; of these:\n",
      "    8727 (4.41%) aligned 0 times\n",
      "    189357 (95.59%) aligned exactly 1 time\n",
      "    0 (0.00%) aligned >1 times\n",
      "95.59% overall alignment rate\n",
      "[bam_sort_core] merging from 0 files and 12 in-memory blocks...\n",
      "This is cutadapt 1.18 with Python 3.7.16\n",
      "Command line parameters: -a CTGTCTCTTATACACATCT -A CTGTCTCTTATACACATCT -u 8 -U 8 -q 30 -o 1_cutadapt/Hrd1_DMS_Rep2_Region3_Input_R1_cut.fastq.gz -p 1_cutadapt/Hrd1_DMS_Rep2_Region3_Input_R2_cut.fastq.gz --cores=12 0_fastq_files/Hrd1_DMS_Rep2_Region3_Input_R1_001.fastq.gz 0_fastq_files/Hrd1_DMS_Rep2_Region3_Input_R2_001.fastq.gz\n",
      "Processing reads on 12 cores in paired-end mode ...\n",
      "Finished in 2.08 s (10 us/read; 5.94 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            206,186\n",
      "  Read 1 with adapter:                   9,640 (4.7%)\n",
      "  Read 2 with adapter:                   8,325 (4.0%)\n",
      "Pairs written (passing filters):       206,186 (100.0%)\n",
      "\n",
      "Total basepairs processed:   122,967,396 bp\n",
      "  Read 1:    61,294,713 bp\n",
      "  Read 2:    61,672,683 bp\n",
      "Quality-trimmed:               8,663,333 bp (7.0%)\n",
      "  Read 1:     2,017,582 bp\n",
      "  Read 2:     6,645,751 bp\n",
      "Total written (filtered):    109,871,156 bp (89.3%)\n",
      "  Read 1:    57,007,297 bp\n",
      "  Read 2:    52,863,859 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: CTGTCTCTTATACACATCT; Type: regular 3'; Length: 19; Trimmed: 9640 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 31.2%\n",
      "  C: 20.1%\n",
      "  G: 26.1%\n",
      "  T: 22.6%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1534\t3221.7\t0\t1534\n",
      "4\t17\t805.4\t0\t17\n",
      "5\t9\t201.4\t0\t9\n",
      "6\t3\t50.3\t0\t3\n",
      "8\t1\t3.1\t0\t1\n",
      "9\t2\t0.8\t0\t2\n",
      "10\t1\t0.2\t1\t0 1\n",
      "11\t6\t0.0\t1\t5 1\n",
      "12\t6\t0.0\t1\t6\n",
      "13\t2\t0.0\t1\t2\n",
      "14\t2\t0.0\t1\t2\n",
      "15\t2\t0.0\t1\t2\n",
      "16\t3\t0.0\t1\t2 1\n",
      "17\t7\t0.0\t1\t7\n",
      "18\t1\t0.0\t1\t1\n",
      "19\t3\t0.0\t1\t3\n",
      "20\t3\t0.0\t1\t3\n",
      "21\t2\t0.0\t1\t2\n",
      "22\t2\t0.0\t1\t1 1\n",
      "23\t3\t0.0\t1\t2 1\n",
      "24\t4\t0.0\t1\t2 2\n",
      "26\t1\t0.0\t1\t1\n",
      "27\t6\t0.0\t1\t6\n",
      "28\t7\t0.0\t1\t7\n",
      "29\t14\t0.0\t1\t13 1\n",
      "30\t9\t0.0\t1\t9\n",
      "31\t5\t0.0\t1\t5\n",
      "33\t2\t0.0\t1\t2\n",
      "34\t3\t0.0\t1\t2 1\n",
      "35\t1\t0.0\t1\t1\n",
      "36\t2\t0.0\t1\t2\n",
      "37\t1\t0.0\t1\t0 1\n",
      "38\t3\t0.0\t1\t2 1\n",
      "39\t13\t0.0\t1\t11 2\n",
      "40\t9\t0.0\t1\t8 1\n",
      "41\t7\t0.0\t1\t6 1\n",
      "42\t20\t0.0\t1\t18 2\n",
      "43\t19\t0.0\t1\t19\n",
      "44\t10\t0.0\t1\t10\n",
      "45\t2\t0.0\t1\t2\n",
      "48\t2\t0.0\t1\t2\n",
      "49\t5\t0.0\t1\t4 1\n",
      "51\t8\t0.0\t1\t8\n",
      "52\t4\t0.0\t1\t4\n",
      "53\t4\t0.0\t1\t3 1\n",
      "54\t7\t0.0\t1\t6 1\n",
      "55\t18\t0.0\t1\t16 2\n",
      "56\t15\t0.0\t1\t14 1\n",
      "57\t13\t0.0\t1\t12 1\n",
      "58\t12\t0.0\t1\t12\n",
      "59\t10\t0.0\t1\t10\n",
      "60\t9\t0.0\t1\t8 1\n",
      "61\t3\t0.0\t1\t3\n",
      "62\t9\t0.0\t1\t8 1\n",
      "63\t27\t0.0\t1\t27\n",
      "64\t20\t0.0\t1\t20\n",
      "65\t53\t0.0\t1\t51 2\n",
      "66\t51\t0.0\t1\t50 1\n",
      "67\t65\t0.0\t1\t60 5\n",
      "68\t30\t0.0\t1\t30\n",
      "69\t20\t0.0\t1\t20\n",
      "70\t15\t0.0\t1\t15\n",
      "71\t42\t0.0\t1\t39 3\n",
      "72\t48\t0.0\t1\t47 1\n",
      "73\t60\t0.0\t1\t58 2\n",
      "74\t167\t0.0\t1\t165 2\n",
      "75\t440\t0.0\t1\t406 34\n",
      "76\t4333\t0.0\t1\t4291 42\n",
      "77\t2099\t0.0\t1\t2080 19\n",
      "78\t138\t0.0\t1\t137 1\n",
      "79\t24\t0.0\t1\t24\n",
      "80\t7\t0.0\t1\t7\n",
      "81\t2\t0.0\t1\t2\n",
      "82\t3\t0.0\t1\t3\n",
      "83\t6\t0.0\t1\t6\n",
      "84\t1\t0.0\t1\t1\n",
      "85\t3\t0.0\t1\t3\n",
      "86\t2\t0.0\t1\t2\n",
      "87\t1\t0.0\t1\t1\n",
      "88\t1\t0.0\t1\t1\n",
      "89\t1\t0.0\t1\t1\n",
      "90\t3\t0.0\t1\t3\n",
      "92\t1\t0.0\t1\t1\n",
      "94\t1\t0.0\t1\t1\n",
      "98\t1\t0.0\t1\t1\n",
      "99\t1\t0.0\t1\t1\n",
      "102\t1\t0.0\t1\t1\n",
      "103\t1\t0.0\t1\t1\n",
      "109\t2\t0.0\t1\t2\n",
      "112\t1\t0.0\t1\t1\n",
      "113\t1\t0.0\t1\t1\n",
      "116\t1\t0.0\t1\t1\n",
      "124\t1\t0.0\t1\t1\n",
      "127\t1\t0.0\t1\t1\n",
      "128\t1\t0.0\t1\t1\n",
      "135\t1\t0.0\t1\t1\n",
      "139\t1\t0.0\t1\t1\n",
      "142\t1\t0.0\t1\t1\n",
      "154\t2\t0.0\t1\t1 1\n",
      "156\t1\t0.0\t1\t1\n",
      "159\t1\t0.0\t1\t1\n",
      "160\t2\t0.0\t1\t2\n",
      "162\t1\t0.0\t1\t1\n",
      "167\t1\t0.0\t1\t1\n",
      "169\t1\t0.0\t1\t1\n",
      "170\t1\t0.0\t1\t1\n",
      "173\t1\t0.0\t1\t1\n",
      "174\t1\t0.0\t1\t1\n",
      "175\t1\t0.0\t1\t0 1\n",
      "181\t1\t0.0\t1\t1\n",
      "186\t2\t0.0\t1\t2\n",
      "188\t1\t0.0\t1\t1\n",
      "189\t2\t0.0\t1\t2\n",
      "191\t1\t0.0\t1\t0 1\n",
      "193\t1\t0.0\t1\t1\n",
      "194\t1\t0.0\t1\t1\n",
      "199\t1\t0.0\t1\t1\n",
      "200\t1\t0.0\t1\t1\n",
      "203\t1\t0.0\t1\t0 1\n",
      "204\t1\t0.0\t1\t1\n",
      "208\t2\t0.0\t1\t2\n",
      "209\t2\t0.0\t1\t2\n",
      "211\t1\t0.0\t1\t1\n",
      "214\t1\t0.0\t1\t0 1\n",
      "215\t2\t0.0\t1\t2\n",
      "216\t1\t0.0\t1\t1\n",
      "221\t1\t0.0\t1\t1\n",
      "222\t2\t0.0\t1\t2\n",
      "226\t3\t0.0\t1\t2 1\n",
      "227\t2\t0.0\t1\t2\n",
      "228\t8\t0.0\t1\t6 2\n",
      "229\t4\t0.0\t1\t4\n",
      "230\t20\t0.0\t1\t19 1\n",
      "231\t20\t0.0\t1\t19 1\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CTGTCTCTTATACACATCT; Type: regular 3'; Length: 19; Trimmed: 8325 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 31.1%\n",
      "  C: 24.9%\n",
      "  G: 19.4%\n",
      "  T: 24.7%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t251\t3221.7\t0\t251\n",
      "4\t44\t805.4\t0\t44\n",
      "5\t24\t201.4\t0\t24\n",
      "6\t3\t50.3\t0\t3\n",
      "7\t2\t12.6\t0\t2\n",
      "8\t3\t3.1\t0\t3\n",
      "11\t3\t0.0\t1\t3\n",
      "12\t3\t0.0\t1\t3\n",
      "13\t2\t0.0\t1\t2\n",
      "14\t2\t0.0\t1\t0 2\n",
      "15\t1\t0.0\t1\t1\n",
      "16\t5\t0.0\t1\t4 1\n",
      "17\t5\t0.0\t1\t4 1\n",
      "18\t1\t0.0\t1\t1\n",
      "19\t6\t0.0\t1\t6\n",
      "20\t9\t0.0\t1\t9\n",
      "21\t6\t0.0\t1\t6\n",
      "22\t6\t0.0\t1\t6\n",
      "23\t3\t0.0\t1\t3\n",
      "24\t2\t0.0\t1\t2\n",
      "25\t9\t0.0\t1\t9\n",
      "26\t6\t0.0\t1\t6\n",
      "27\t4\t0.0\t1\t2 2\n",
      "28\t29\t0.0\t1\t26 3\n",
      "29\t36\t0.0\t1\t35 1\n",
      "30\t13\t0.0\t1\t13\n",
      "31\t98\t0.0\t1\t93 5\n",
      "32\t52\t0.0\t1\t50 2\n",
      "33\t64\t0.0\t1\t61 3\n",
      "34\t134\t0.0\t1\t126 8\n",
      "35\t8\t0.0\t1\t7 1\n",
      "36\t5\t0.0\t1\t5\n",
      "37\t49\t0.0\t1\t49\n",
      "38\t25\t0.0\t1\t25\n",
      "39\t22\t0.0\t1\t21 1\n",
      "40\t14\t0.0\t1\t11 3\n",
      "41\t25\t0.0\t1\t25\n",
      "42\t32\t0.0\t1\t30 2\n",
      "43\t102\t0.0\t1\t100 2\n",
      "44\t138\t0.0\t1\t135 3\n",
      "45\t144\t0.0\t1\t143 1\n",
      "46\t293\t0.0\t1\t289 4\n",
      "47\t174\t0.0\t1\t166 8\n",
      "48\t124\t0.0\t1\t120 4\n",
      "49\t200\t0.0\t1\t199 1\n",
      "50\t169\t0.0\t1\t169\n",
      "51\t74\t0.0\t1\t73 1\n",
      "52\t335\t0.0\t1\t333 2\n",
      "53\t108\t0.0\t1\t105 3\n",
      "54\t470\t0.0\t1\t467 3\n",
      "55\t130\t0.0\t1\t127 3\n",
      "56\t56\t0.0\t1\t54 2\n",
      "57\t290\t0.0\t1\t287 3\n",
      "58\t182\t0.0\t1\t181 1\n",
      "59\t116\t0.0\t1\t115 1\n",
      "60\t46\t0.0\t1\t46\n",
      "61\t31\t0.0\t1\t31\n",
      "62\t141\t0.0\t1\t141\n",
      "63\t159\t0.0\t1\t154 5\n",
      "64\t37\t0.0\t1\t35 2\n",
      "65\t11\t0.0\t1\t11\n",
      "66\t2\t0.0\t1\t2\n",
      "67\t11\t0.0\t1\t11\n",
      "68\t3\t0.0\t1\t3\n",
      "69\t3\t0.0\t1\t3\n",
      "70\t6\t0.0\t1\t6\n",
      "72\t2\t0.0\t1\t2\n",
      "73\t1\t0.0\t1\t1\n",
      "74\t4\t0.0\t1\t4\n",
      "75\t23\t0.0\t1\t23\n",
      "76\t16\t0.0\t1\t15 1\n",
      "77\t46\t0.0\t1\t44 2\n",
      "78\t98\t0.0\t1\t98\n",
      "79\t208\t0.0\t1\t201 7\n",
      "80\t1867\t0.0\t1\t1855 12\n",
      "81\t1066\t0.0\t1\t1058 8\n",
      "82\t321\t0.0\t1\t317 4\n",
      "83\t80\t0.0\t1\t79 1\n",
      "84\t25\t0.0\t1\t24 1\n",
      "85\t3\t0.0\t1\t3\n",
      "87\t1\t0.0\t1\t1\n",
      "230\t3\t0.0\t1\t3\n",
      "\n",
      "Read1 before filtering:\n",
      "total reads: 206186\n",
      "total bases: 57007297\n",
      "Q20 bases: 56708326(99.4756%)\n",
      "Q30 bases: 55807339(97.8951%)\n",
      "\n",
      "Read2 before filtering:\n",
      "total reads: 206186\n",
      "total bases: 52863859\n",
      "Q20 bases: 52241596(98.8229%)\n",
      "Q30 bases: 50344687(95.2346%)\n",
      "\n",
      "Merged and filtered:\n",
      "total reads: 198479\n",
      "total bases: 72069007\n",
      "Q20 bases: 71826117(99.663%)\n",
      "Q30 bases: 70964536(98.4675%)\n",
      "\n",
      "Filtering result:\n",
      "reads passed filter: 407978\n",
      "reads failed due to low quality: 200\n",
      "reads failed due to too many N: 28\n",
      "reads failed due to too short: 4166\n",
      "reads with adapter trimmed: 16070\n",
      "bases trimmed due to adapters: 130032\n",
      "reads corrected by overlap analysis: 30322\n",
      "bases corrected by overlap analysis: 37187\n",
      "\n",
      "Duplication rate: 70.4974%\n",
      "\n",
      "Insert size peak (evaluated by paired-end reads): 377\n",
      "\n",
      "Read pairs merged: 198479\n",
      "% of original read pairs: 96.2621%\n",
      "% in reads after filtering: 100%\n",
      "\n",
      "\n",
      "JSON report: 2_fastp/Hrd1_DMS_Rep2_Region3_Input_report.json\n",
      "HTML report: 2_fastp/Hrd1_DMS_Rep2_Region3_Input_report.html\n",
      "\n",
      "fastp -i 1_cutadapt/Hrd1_DMS_Rep2_Region3_Input_R1_cut.fastq.gz -I 1_cutadapt/Hrd1_DMS_Rep2_Region3_Input_R2_cut.fastq.gz -m --merged_out 2_fastp/Hrd1_DMS_Rep2_Region3_Input_merge.fastq.gz -o 2_fastp/Hrd1_DMS_Rep2_Region3_Input_R1_cut.fastq.gz -O 2_fastp/Hrd1_DMS_Rep2_Region3_Input_R2_cut.fastq.gz -w 12 -h 2_fastp/Hrd1_DMS_Rep2_Region3_Input_report.html -j 2_fastp/Hrd1_DMS_Rep2_Region3_Input_report.json \n",
      "fastp v0.20.0, time used: 2 seconds\n",
      "198479 reads; of these:\n",
      "  198479 (100.00%) were unpaired; of these:\n",
      "    8049 (4.06%) aligned 0 times\n",
      "    190430 (95.94%) aligned exactly 1 time\n",
      "    0 (0.00%) aligned >1 times\n",
      "95.94% overall alignment rate\n",
      "[bam_sort_core] merging from 0 files and 12 in-memory blocks...\n",
      "This is cutadapt 1.18 with Python 3.7.16\n",
      "Command line parameters: -a CTGTCTCTTATACACATCT -A CTGTCTCTTATACACATCT -u 8 -U 8 -q 30 -o 1_cutadapt/Hrd1_DMS_Rep3_Region3_Input_R1_cut.fastq.gz -p 1_cutadapt/Hrd1_DMS_Rep3_Region3_Input_R2_cut.fastq.gz --cores=12 0_fastq_files/Hrd1_DMS_Rep3_Region3_Input_R1_001.fastq.gz 0_fastq_files/Hrd1_DMS_Rep3_Region3_Input_R2_001.fastq.gz\n",
      "Processing reads on 12 cores in paired-end mode ...\n",
      "Finished in 2.21 s (10 us/read; 5.93 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            218,307\n",
      "  Read 1 with adapter:                   4,434 (2.0%)\n",
      "  Read 2 with adapter:                   9,340 (4.3%)\n",
      "Pairs written (passing filters):       218,307 (100.0%)\n",
      "\n",
      "Total basepairs processed:   130,968,756 bp\n",
      "  Read 1:    65,488,309 bp\n",
      "  Read 2:    65,480,447 bp\n",
      "Quality-trimmed:               5,913,980 bp (4.5%)\n",
      "  Read 1:     1,214,637 bp\n",
      "  Read 2:     4,699,343 bp\n",
      "Total written (filtered):    120,916,182 bp (92.3%)\n",
      "  Read 1:    62,204,028 bp\n",
      "  Read 2:    58,712,154 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: CTGTCTCTTATACACATCT; Type: regular 3'; Length: 19; Trimmed: 4434 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 34.0%\n",
      "  C: 23.8%\n",
      "  G: 17.4%\n",
      "  T: 24.7%\n",
      "  none/other: 0.1%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t136\t3411.0\t0\t136\n",
      "4\t28\t852.8\t0\t28\n",
      "5\t4\t213.2\t0\t4\n",
      "6\t3\t53.3\t0\t3\n",
      "8\t2\t3.3\t0\t2\n",
      "10\t1\t0.2\t1\t1\n",
      "12\t1\t0.0\t1\t1\n",
      "14\t1\t0.0\t1\t1\n",
      "15\t1\t0.0\t1\t1\n",
      "16\t1\t0.0\t1\t1\n",
      "17\t2\t0.0\t1\t2\n",
      "18\t2\t0.0\t1\t2\n",
      "20\t3\t0.0\t1\t3\n",
      "21\t12\t0.0\t1\t11 1\n",
      "22\t2\t0.0\t1\t2\n",
      "23\t6\t0.0\t1\t6\n",
      "24\t2\t0.0\t1\t1 1\n",
      "25\t3\t0.0\t1\t3\n",
      "28\t3\t0.0\t1\t3\n",
      "29\t4\t0.0\t1\t4\n",
      "30\t6\t0.0\t1\t6\n",
      "32\t1\t0.0\t1\t1\n",
      "33\t1\t0.0\t1\t1\n",
      "34\t4\t0.0\t1\t4\n",
      "35\t5\t0.0\t1\t5\n",
      "36\t7\t0.0\t1\t7\n",
      "37\t1\t0.0\t1\t1\n",
      "38\t2\t0.0\t1\t2\n",
      "39\t5\t0.0\t1\t4 1\n",
      "40\t2\t0.0\t1\t2\n",
      "41\t5\t0.0\t1\t5\n",
      "42\t6\t0.0\t1\t6\n",
      "43\t5\t0.0\t1\t4 1\n",
      "44\t6\t0.0\t1\t6\n",
      "45\t5\t0.0\t1\t5\n",
      "47\t2\t0.0\t1\t2\n",
      "48\t2\t0.0\t1\t2\n",
      "49\t4\t0.0\t1\t4\n",
      "50\t4\t0.0\t1\t4\n",
      "51\t1\t0.0\t1\t0 1\n",
      "52\t5\t0.0\t1\t4 1\n",
      "53\t3\t0.0\t1\t3\n",
      "54\t2\t0.0\t1\t2\n",
      "55\t6\t0.0\t1\t6\n",
      "56\t3\t0.0\t1\t3\n",
      "57\t6\t0.0\t1\t6\n",
      "58\t3\t0.0\t1\t2 1\n",
      "59\t6\t0.0\t1\t6\n",
      "60\t7\t0.0\t1\t7\n",
      "61\t8\t0.0\t1\t8\n",
      "62\t2\t0.0\t1\t2\n",
      "63\t8\t0.0\t1\t8\n",
      "64\t4\t0.0\t1\t3 1\n",
      "65\t36\t0.0\t1\t35 1\n",
      "66\t18\t0.0\t1\t18\n",
      "67\t5\t0.0\t1\t5\n",
      "68\t4\t0.0\t1\t4\n",
      "69\t12\t0.0\t1\t11 1\n",
      "70\t15\t0.0\t1\t15\n",
      "71\t18\t0.0\t1\t17 1\n",
      "72\t16\t0.0\t1\t14 2\n",
      "73\t34\t0.0\t1\t31 3\n",
      "74\t74\t0.0\t1\t71 3\n",
      "75\t158\t0.0\t1\t139 19\n",
      "76\t2433\t0.0\t1\t2409 24\n",
      "77\t1027\t0.0\t1\t1025 2\n",
      "78\t164\t0.0\t1\t161 3\n",
      "79\t15\t0.0\t1\t15\n",
      "80\t7\t0.0\t1\t6 1\n",
      "81\t1\t0.0\t1\t1\n",
      "82\t2\t0.0\t1\t2\n",
      "86\t1\t0.0\t1\t1\n",
      "93\t2\t0.0\t1\t2\n",
      "94\t1\t0.0\t1\t1\n",
      "97\t1\t0.0\t1\t1\n",
      "131\t1\t0.0\t1\t1\n",
      "134\t1\t0.0\t1\t1\n",
      "155\t1\t0.0\t1\t1\n",
      "160\t1\t0.0\t1\t1\n",
      "161\t1\t0.0\t1\t1\n",
      "177\t1\t0.0\t1\t1\n",
      "182\t1\t0.0\t1\t1\n",
      "198\t2\t0.0\t1\t2\n",
      "199\t1\t0.0\t1\t1\n",
      "203\t1\t0.0\t1\t1\n",
      "208\t1\t0.0\t1\t1\n",
      "214\t1\t0.0\t1\t1\n",
      "215\t1\t0.0\t1\t1\n",
      "223\t3\t0.0\t1\t3\n",
      "224\t5\t0.0\t1\t5\n",
      "225\t13\t0.0\t1\t11 2\n",
      "247\t1\t0.0\t1\t1\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CTGTCTCTTATACACATCT; Type: regular 3'; Length: 19; Trimmed: 9340 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 42.4%\n",
      "  C: 10.4%\n",
      "  G: 36.4%\n",
      "  T: 10.9%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t5026\t3411.0\t0\t5026\n",
      "4\t71\t852.8\t0\t71\n",
      "5\t11\t213.2\t0\t11\n",
      "6\t3\t53.3\t0\t3\n",
      "7\t1\t13.3\t0\t1\n",
      "8\t1\t3.3\t0\t1\n",
      "11\t1\t0.1\t1\t1\n",
      "12\t1\t0.0\t1\t1\n",
      "13\t2\t0.0\t1\t2\n",
      "14\t2\t0.0\t1\t2\n",
      "17\t1\t0.0\t1\t1\n",
      "18\t2\t0.0\t1\t2\n",
      "19\t1\t0.0\t1\t1\n",
      "20\t2\t0.0\t1\t2\n",
      "21\t4\t0.0\t1\t4\n",
      "22\t3\t0.0\t1\t3\n",
      "23\t3\t0.0\t1\t3\n",
      "24\t2\t0.0\t1\t2\n",
      "25\t8\t0.0\t1\t7 1\n",
      "26\t5\t0.0\t1\t5\n",
      "27\t2\t0.0\t1\t2\n",
      "28\t1\t0.0\t1\t1\n",
      "29\t6\t0.0\t1\t6\n",
      "30\t3\t0.0\t1\t2 1\n",
      "31\t3\t0.0\t1\t3\n",
      "32\t5\t0.0\t1\t5\n",
      "33\t21\t0.0\t1\t21\n",
      "34\t2\t0.0\t1\t2\n",
      "35\t38\t0.0\t1\t37 1\n",
      "36\t7\t0.0\t1\t7\n",
      "37\t8\t0.0\t1\t8\n",
      "38\t5\t0.0\t1\t5\n",
      "39\t1\t0.0\t1\t1\n",
      "40\t12\t0.0\t1\t12\n",
      "41\t21\t0.0\t1\t20 1\n",
      "42\t17\t0.0\t1\t17\n",
      "43\t71\t0.0\t1\t71\n",
      "44\t37\t0.0\t1\t36 1\n",
      "45\t42\t0.0\t1\t42\n",
      "46\t93\t0.0\t1\t91 2\n",
      "47\t30\t0.0\t1\t30\n",
      "48\t53\t0.0\t1\t52 1\n",
      "49\t20\t0.0\t1\t20\n",
      "50\t23\t0.0\t1\t23\n",
      "51\t11\t0.0\t1\t11\n",
      "52\t15\t0.0\t1\t15\n",
      "53\t14\t0.0\t1\t14\n",
      "54\t89\t0.0\t1\t89\n",
      "55\t50\t0.0\t1\t50\n",
      "56\t30\t0.0\t1\t30\n",
      "57\t63\t0.0\t1\t63\n",
      "58\t32\t0.0\t1\t32\n",
      "59\t29\t0.0\t1\t28 1\n",
      "60\t17\t0.0\t1\t17\n",
      "61\t13\t0.0\t1\t13\n",
      "62\t31\t0.0\t1\t31\n",
      "63\t61\t0.0\t1\t61\n",
      "64\t18\t0.0\t1\t18\n",
      "65\t14\t0.0\t1\t13 1\n",
      "66\t3\t0.0\t1\t3\n",
      "67\t13\t0.0\t1\t13\n",
      "68\t4\t0.0\t1\t3 1\n",
      "69\t2\t0.0\t1\t2\n",
      "70\t5\t0.0\t1\t5\n",
      "71\t1\t0.0\t1\t1\n",
      "72\t2\t0.0\t1\t2\n",
      "73\t4\t0.0\t1\t4\n",
      "74\t7\t0.0\t1\t7\n",
      "75\t12\t0.0\t1\t12\n",
      "76\t14\t0.0\t1\t14\n",
      "77\t29\t0.0\t1\t29\n",
      "78\t100\t0.0\t1\t98 2\n",
      "79\t299\t0.0\t1\t297 2\n",
      "80\t1188\t0.0\t1\t1174 14\n",
      "81\t1082\t0.0\t1\t1077 5\n",
      "82\t372\t0.0\t1\t364 8\n",
      "83\t34\t0.0\t1\t31 3\n",
      "84\t8\t0.0\t1\t7 1\n",
      "85\t1\t0.0\t1\t1\n",
      "224\t1\t0.0\t1\t1\n",
      "225\t1\t0.0\t1\t1\n",
      "\n",
      "Read1 before filtering:\n",
      "total reads: 218307\n",
      "total bases: 62204028\n",
      "Q20 bases: 61867049(99.4583%)\n",
      "Q30 bases: 60819845(97.7748%)\n",
      "\n",
      "Read2 before filtering:\n",
      "total reads: 218307\n",
      "total bases: 58712154\n",
      "Q20 bases: 57963951(98.7256%)\n",
      "Q30 bases: 55885349(95.1853%)\n",
      "\n",
      "Merged and filtered:\n",
      "total reads: 212070\n",
      "total bases: 79690131\n",
      "Q20 bases: 79395318(99.6301%)\n",
      "Q30 bases: 78350321(98.3187%)\n",
      "\n",
      "Filtering result:\n",
      "reads passed filter: 432192\n",
      "reads failed due to low quality: 376\n",
      "reads failed due to too many N: 20\n",
      "reads failed due to too short: 4026\n",
      "reads with adapter trimmed: 8478\n",
      "bases trimmed due to adapters: 68805\n",
      "reads corrected by overlap analysis: 28004\n",
      "bases corrected by overlap analysis: 38163\n",
      "\n",
      "Duplication rate: 55.5556%\n",
      "\n",
      "Insert size peak (evaluated by paired-end reads): 383\n",
      "\n",
      "Read pairs merged: 212070\n",
      "% of original read pairs: 97.143%\n",
      "% in reads after filtering: 100%\n",
      "\n",
      "\n",
      "JSON report: 2_fastp/Hrd1_DMS_Rep3_Region3_Input_report.json\n",
      "HTML report: 2_fastp/Hrd1_DMS_Rep3_Region3_Input_report.html\n",
      "\n",
      "fastp -i 1_cutadapt/Hrd1_DMS_Rep3_Region3_Input_R1_cut.fastq.gz -I 1_cutadapt/Hrd1_DMS_Rep3_Region3_Input_R2_cut.fastq.gz -m --merged_out 2_fastp/Hrd1_DMS_Rep3_Region3_Input_merge.fastq.gz -o 2_fastp/Hrd1_DMS_Rep3_Region3_Input_R1_cut.fastq.gz -O 2_fastp/Hrd1_DMS_Rep3_Region3_Input_R2_cut.fastq.gz -w 12 -h 2_fastp/Hrd1_DMS_Rep3_Region3_Input_report.html -j 2_fastp/Hrd1_DMS_Rep3_Region3_Input_report.json \n",
      "fastp v0.20.0, time used: 1 seconds\n",
      "212070 reads; of these:\n",
      "  212070 (100.00%) were unpaired; of these:\n",
      "    4246 (2.00%) aligned 0 times\n",
      "    207824 (98.00%) aligned exactly 1 time\n",
      "    0 (0.00%) aligned >1 times\n",
      "98.00% overall alignment rate\n",
      "[bam_sort_core] merging from 0 files and 12 in-memory blocks...\n",
      "This is cutadapt 1.18 with Python 3.7.16\n",
      "Command line parameters: -a CTGTCTCTTATACACATCT -A CTGTCTCTTATACACATCT -u 8 -U 8 -q 30 -o 1_cutadapt/Hrd1_DMS_Rep4_Region3_Input_R1_cut.fastq.gz -p 1_cutadapt/Hrd1_DMS_Rep4_Region3_Input_R2_cut.fastq.gz --cores=12 0_fastq_files/Hrd1_DMS_Rep4_Region3_Input_R1_001.fastq.gz 0_fastq_files/Hrd1_DMS_Rep4_Region3_Input_R2_001.fastq.gz\n",
      "Processing reads on 12 cores in paired-end mode ...\n",
      "Finished in 1.73 s (9 us/read; 6.36 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            183,761\n",
      "  Read 1 with adapter:                   3,620 (2.0%)\n",
      "  Read 2 with adapter:                   8,583 (4.7%)\n",
      "Pairs written (passing filters):       183,761 (100.0%)\n",
      "\n",
      "Total basepairs processed:   109,578,262 bp\n",
      "  Read 1:    54,952,660 bp\n",
      "  Read 2:    54,625,602 bp\n",
      "Quality-trimmed:               5,364,049 bp (4.9%)\n",
      "  Read 1:     1,288,610 bp\n",
      "  Read 2:     4,075,439 bp\n",
      "Total written (filtered):    100,744,693 bp (91.9%)\n",
      "  Read 1:    51,928,190 bp\n",
      "  Read 2:    48,816,503 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: CTGTCTCTTATACACATCT; Type: regular 3'; Length: 19; Trimmed: 3620 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 30.9%\n",
      "  C: 24.1%\n",
      "  G: 19.6%\n",
      "  T: 25.4%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t88\t2871.3\t0\t88\n",
      "4\t12\t717.8\t0\t12\n",
      "5\t3\t179.5\t0\t3\n",
      "6\t2\t44.9\t0\t2\n",
      "9\t2\t0.7\t0\t2\n",
      "11\t1\t0.0\t1\t1\n",
      "13\t1\t0.0\t1\t1\n",
      "14\t1\t0.0\t1\t1\n",
      "16\t1\t0.0\t1\t1\n",
      "17\t3\t0.0\t1\t3\n",
      "18\t1\t0.0\t1\t1\n",
      "22\t1\t0.0\t1\t1\n",
      "23\t1\t0.0\t1\t1\n",
      "24\t2\t0.0\t1\t2\n",
      "25\t2\t0.0\t1\t2\n",
      "28\t1\t0.0\t1\t1\n",
      "29\t10\t0.0\t1\t9 1\n",
      "30\t7\t0.0\t1\t7\n",
      "31\t5\t0.0\t1\t5\n",
      "35\t1\t0.0\t1\t1\n",
      "36\t4\t0.0\t1\t4\n",
      "37\t6\t0.0\t1\t6\n",
      "38\t3\t0.0\t1\t3\n",
      "39\t3\t0.0\t1\t3\n",
      "40\t3\t0.0\t1\t3\n",
      "41\t4\t0.0\t1\t4\n",
      "42\t8\t0.0\t1\t6 2\n",
      "43\t9\t0.0\t1\t9\n",
      "44\t6\t0.0\t1\t6\n",
      "46\t1\t0.0\t1\t1\n",
      "47\t1\t0.0\t1\t1\n",
      "49\t2\t0.0\t1\t1 1\n",
      "50\t3\t0.0\t1\t3\n",
      "51\t2\t0.0\t1\t2\n",
      "52\t2\t0.0\t1\t2\n",
      "53\t3\t0.0\t1\t2 1\n",
      "54\t5\t0.0\t1\t5\n",
      "55\t13\t0.0\t1\t13\n",
      "56\t7\t0.0\t1\t4 3\n",
      "57\t10\t0.0\t1\t10\n",
      "58\t6\t0.0\t1\t6\n",
      "59\t5\t0.0\t1\t4 1\n",
      "60\t7\t0.0\t1\t7\n",
      "61\t5\t0.0\t1\t4 1\n",
      "62\t4\t0.0\t1\t4\n",
      "63\t18\t0.0\t1\t18\n",
      "64\t9\t0.0\t1\t8 1\n",
      "65\t27\t0.0\t1\t25 2\n",
      "66\t20\t0.0\t1\t20\n",
      "67\t24\t0.0\t1\t23 1\n",
      "68\t24\t0.0\t1\t22 2\n",
      "69\t11\t0.0\t1\t9 2\n",
      "70\t13\t0.0\t1\t13\n",
      "71\t22\t0.0\t1\t21 1\n",
      "72\t23\t0.0\t1\t22 1\n",
      "73\t39\t0.0\t1\t38 1\n",
      "74\t77\t0.0\t1\t76 1\n",
      "75\t209\t0.0\t1\t192 17\n",
      "76\t1728\t0.0\t1\t1709 19\n",
      "77\t967\t0.0\t1\t958 9\n",
      "78\t79\t0.0\t1\t76 3\n",
      "79\t14\t0.0\t1\t14\n",
      "80\t3\t0.0\t1\t3\n",
      "81\t1\t0.0\t1\t1\n",
      "82\t2\t0.0\t1\t2\n",
      "83\t1\t0.0\t1\t1\n",
      "84\t3\t0.0\t1\t3\n",
      "86\t1\t0.0\t1\t1\n",
      "93\t1\t0.0\t1\t1\n",
      "96\t1\t0.0\t1\t1\n",
      "110\t1\t0.0\t1\t1\n",
      "115\t1\t0.0\t1\t1\n",
      "118\t1\t0.0\t1\t1\n",
      "120\t1\t0.0\t1\t1\n",
      "124\t1\t0.0\t1\t1\n",
      "128\t1\t0.0\t1\t1\n",
      "135\t1\t0.0\t1\t1\n",
      "141\t1\t0.0\t1\t1\n",
      "152\t1\t0.0\t1\t1\n",
      "167\t1\t0.0\t1\t1\n",
      "174\t1\t0.0\t1\t0 1\n",
      "180\t1\t0.0\t1\t1\n",
      "185\t1\t0.0\t1\t1\n",
      "193\t1\t0.0\t1\t1\n",
      "196\t1\t0.0\t1\t1\n",
      "205\t1\t0.0\t1\t1\n",
      "207\t1\t0.0\t1\t1\n",
      "216\t1\t0.0\t1\t1\n",
      "229\t2\t0.0\t1\t2\n",
      "230\t8\t0.0\t1\t7 1\n",
      "231\t8\t0.0\t1\t7 1\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CTGTCTCTTATACACATCT; Type: regular 3'; Length: 19; Trimmed: 8583 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 31.6%\n",
      "  C: 10.9%\n",
      "  G: 46.3%\n",
      "  T: 11.3%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t5052\t2871.3\t0\t5052\n",
      "4\t30\t717.8\t0\t30\n",
      "5\t5\t179.5\t0\t5\n",
      "6\t2\t44.9\t0\t2\n",
      "8\t2\t2.8\t0\t2\n",
      "9\t1\t0.7\t0\t1\n",
      "11\t2\t0.0\t1\t1 1\n",
      "12\t1\t0.0\t1\t1\n",
      "14\t1\t0.0\t1\t1\n",
      "17\t2\t0.0\t1\t2\n",
      "19\t1\t0.0\t1\t1\n",
      "21\t2\t0.0\t1\t2\n",
      "22\t3\t0.0\t1\t3\n",
      "23\t1\t0.0\t1\t0 1\n",
      "25\t1\t0.0\t1\t1\n",
      "26\t2\t0.0\t1\t2\n",
      "28\t4\t0.0\t1\t3 1\n",
      "29\t2\t0.0\t1\t2\n",
      "30\t3\t0.0\t1\t3\n",
      "31\t8\t0.0\t1\t7 1\n",
      "32\t7\t0.0\t1\t6 1\n",
      "33\t6\t0.0\t1\t6\n",
      "34\t5\t0.0\t1\t5\n",
      "35\t15\t0.0\t1\t12 3\n",
      "36\t12\t0.0\t1\t12\n",
      "37\t12\t0.0\t1\t12\n",
      "38\t5\t0.0\t1\t5\n",
      "39\t20\t0.0\t1\t19 1\n",
      "40\t15\t0.0\t1\t14 1\n",
      "41\t15\t0.0\t1\t14 1\n",
      "42\t15\t0.0\t1\t15\n",
      "43\t27\t0.0\t1\t27\n",
      "44\t41\t0.0\t1\t39 2\n",
      "45\t30\t0.0\t1\t30\n",
      "46\t75\t0.0\t1\t75\n",
      "47\t47\t0.0\t1\t46 1\n",
      "48\t36\t0.0\t1\t36\n",
      "49\t68\t0.0\t1\t67 1\n",
      "50\t40\t0.0\t1\t40\n",
      "51\t21\t0.0\t1\t19 2\n",
      "52\t65\t0.0\t1\t63 2\n",
      "53\t25\t0.0\t1\t24 1\n",
      "54\t86\t0.0\t1\t85 1\n",
      "55\t35\t0.0\t1\t33 2\n",
      "56\t16\t0.0\t1\t15 1\n",
      "57\t55\t0.0\t1\t53 2\n",
      "58\t46\t0.0\t1\t44 2\n",
      "59\t22\t0.0\t1\t21 1\n",
      "60\t25\t0.0\t1\t25\n",
      "61\t19\t0.0\t1\t19\n",
      "62\t37\t0.0\t1\t35 2\n",
      "63\t51\t0.0\t1\t50 1\n",
      "64\t25\t0.0\t1\t25\n",
      "65\t11\t0.0\t1\t11\n",
      "66\t2\t0.0\t1\t2\n",
      "67\t4\t0.0\t1\t4\n",
      "68\t3\t0.0\t1\t3\n",
      "69\t5\t0.0\t1\t5\n",
      "70\t5\t0.0\t1\t5\n",
      "73\t2\t0.0\t1\t2\n",
      "74\t2\t0.0\t1\t2\n",
      "75\t20\t0.0\t1\t20\n",
      "76\t11\t0.0\t1\t11\n",
      "77\t28\t0.0\t1\t28\n",
      "78\t58\t0.0\t1\t58\n",
      "79\t141\t0.0\t1\t130 11\n",
      "80\t1207\t0.0\t1\t1190 17\n",
      "81\t782\t0.0\t1\t774 8\n",
      "82\t144\t0.0\t1\t143 1\n",
      "83\t11\t0.0\t1\t11\n",
      "84\t1\t0.0\t1\t1\n",
      "86\t1\t0.0\t1\t1\n",
      "211\t1\t0.0\t1\t1\n",
      "\n",
      "Read1 before filtering:\n",
      "total reads: 183761\n",
      "total bases: 51928190\n",
      "Q20 bases: 51668558(99.5%)\n",
      "Q30 bases: 50802603(97.8324%)\n",
      "\n",
      "Read2 before filtering:\n",
      "total reads: 183761\n",
      "total bases: 48816503\n",
      "Q20 bases: 48186950(98.7104%)\n",
      "Q30 bases: 46468976(95.1911%)\n",
      "\n",
      "Merged and filtered:\n",
      "total reads: 177746\n",
      "total bases: 65737248\n",
      "Q20 bases: 65507383(99.6503%)\n",
      "Q30 bases: 64653295(98.3511%)\n",
      "\n",
      "Filtering result:\n",
      "reads passed filter: 363666\n",
      "reads failed due to low quality: 294\n",
      "reads failed due to too many N: 20\n",
      "reads failed due to too short: 3542\n",
      "reads with adapter trimmed: 7026\n",
      "bases trimmed due to adapters: 56993\n",
      "reads corrected by overlap analysis: 23555\n",
      "bases corrected by overlap analysis: 31218\n",
      "\n",
      "Duplication rate: 60.7843%\n",
      "\n",
      "Insert size peak (evaluated by paired-end reads): 377\n",
      "\n",
      "Read pairs merged: 177746\n",
      "% of original read pairs: 96.7267%\n",
      "% in reads after filtering: 100%\n",
      "\n",
      "\n",
      "JSON report: 2_fastp/Hrd1_DMS_Rep4_Region3_Input_report.json\n",
      "HTML report: 2_fastp/Hrd1_DMS_Rep4_Region3_Input_report.html\n",
      "\n",
      "fastp -i 1_cutadapt/Hrd1_DMS_Rep4_Region3_Input_R1_cut.fastq.gz -I 1_cutadapt/Hrd1_DMS_Rep4_Region3_Input_R2_cut.fastq.gz -m --merged_out 2_fastp/Hrd1_DMS_Rep4_Region3_Input_merge.fastq.gz -o 2_fastp/Hrd1_DMS_Rep4_Region3_Input_R1_cut.fastq.gz -O 2_fastp/Hrd1_DMS_Rep4_Region3_Input_R2_cut.fastq.gz -w 12 -h 2_fastp/Hrd1_DMS_Rep4_Region3_Input_report.html -j 2_fastp/Hrd1_DMS_Rep4_Region3_Input_report.json \n",
      "fastp v0.20.0, time used: 1 seconds\n",
      "177746 reads; of these:\n",
      "  177746 (100.00%) were unpaired; of these:\n",
      "    3513 (1.98%) aligned 0 times\n",
      "    174233 (98.02%) aligned exactly 1 time\n",
      "    0 (0.00%) aligned >1 times\n",
      "98.02% overall alignment rate\n",
      "[bam_sort_core] merging from 0 files and 12 in-memory blocks...\n",
      "This is cutadapt 1.18 with Python 3.7.16\n",
      "Command line parameters: -a CTGTCTCTTATACACATCT -A CTGTCTCTTATACACATCT -u 8 -U 8 -q 30 -o 1_cutadapt/Hrd1_DMS_Rep1_Region3_WT_R1_cut.fastq.gz -p 1_cutadapt/Hrd1_DMS_Rep1_Region3_WT_R2_cut.fastq.gz --cores=12 0_fastq_files/Hrd1_DMS_Rep1_Region3_WT_R1_001.fastq.gz 0_fastq_files/Hrd1_DMS_Rep1_Region3_WT_R2_001.fastq.gz\n",
      "Processing reads on 12 cores in paired-end mode ...\n",
      "Finished in 2.04 s (9 us/read; 6.33 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            215,061\n",
      "  Read 1 with adapter:                     232 (0.1%)\n",
      "  Read 2 with adapter:                   5,742 (2.7%)\n",
      "Pairs written (passing filters):       215,061 (100.0%)\n",
      "\n",
      "Total basepairs processed:   128,231,087 bp\n",
      "  Read 1:    64,311,899 bp\n",
      "  Read 2:    63,919,188 bp\n",
      "Quality-trimmed:               4,042,622 bp (3.2%)\n",
      "  Read 1:       734,044 bp\n",
      "  Read 2:     3,308,578 bp\n",
      "Total written (filtered):    120,726,365 bp (94.1%)\n",
      "  Read 1:    61,855,139 bp\n",
      "  Read 2:    58,871,226 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: CTGTCTCTTATACACATCT; Type: regular 3'; Length: 19; Trimmed: 232 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 3.4%\n",
      "  C: 15.1%\n",
      "  G: 65.9%\n",
      "  T: 15.5%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t199\t3360.3\t0\t199\n",
      "4\t7\t840.1\t0\t7\n",
      "5\t2\t210.0\t0\t2\n",
      "6\t3\t52.5\t0\t3\n",
      "49\t1\t0.0\t1\t1\n",
      "74\t1\t0.0\t1\t1\n",
      "75\t1\t0.0\t1\t1\n",
      "76\t9\t0.0\t1\t9\n",
      "77\t9\t0.0\t1\t9\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CTGTCTCTTATACACATCT; Type: regular 3'; Length: 19; Trimmed: 5742 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 28.3%\n",
      "  C: 1.5%\n",
      "  G: 66.0%\n",
      "  T: 4.3%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t5582\t3360.3\t0\t5582\n",
      "4\t123\t840.1\t0\t123\n",
      "5\t15\t210.0\t0\t15\n",
      "6\t1\t52.5\t0\t1\n",
      "47\t1\t0.0\t1\t1\n",
      "53\t1\t0.0\t1\t1\n",
      "55\t1\t0.0\t1\t1\n",
      "63\t1\t0.0\t1\t1\n",
      "64\t1\t0.0\t1\t1\n",
      "78\t1\t0.0\t1\t1\n",
      "80\t3\t0.0\t1\t3\n",
      "81\t7\t0.0\t1\t7\n",
      "82\t5\t0.0\t1\t5\n",
      "\n",
      "Read1 before filtering:\n",
      "total reads: 215061\n",
      "total bases: 61855139\n",
      "Q20 bases: 61592682(99.5757%)\n",
      "Q30 bases: 60707591(98.1448%)\n",
      "\n",
      "Read2 before filtering:\n",
      "total reads: 215061\n",
      "total bases: 58871226\n",
      "Q20 bases: 58222931(98.8988%)\n",
      "Q30 bases: 56434115(95.8603%)\n",
      "\n",
      "Merged and filtered:\n",
      "total reads: 209132\n",
      "total bases: 78684263\n",
      "Q20 bases: 78453523(99.7068%)\n",
      "Q30 bases: 77578742(98.595%)\n",
      "\n",
      "Filtering result:\n",
      "reads passed filter: 426396\n",
      "reads failed due to low quality: 434\n",
      "reads failed due to too many N: 18\n",
      "reads failed due to too short: 3274\n",
      "reads with adapter trimmed: 46\n",
      "bases trimmed due to adapters: 577\n",
      "reads corrected by overlap analysis: 22789\n",
      "bases corrected by overlap analysis: 30245\n",
      "\n",
      "Duplication rate: 62.3269%\n",
      "\n",
      "Insert size peak (evaluated by paired-end reads): 377\n",
      "\n",
      "Read pairs merged: 209132\n",
      "% of original read pairs: 97.2431%\n",
      "% in reads after filtering: 100%\n",
      "\n",
      "\n",
      "JSON report: 2_fastp/Hrd1_DMS_Rep1_Region3_WT_report.json\n",
      "HTML report: 2_fastp/Hrd1_DMS_Rep1_Region3_WT_report.html\n",
      "\n",
      "fastp -i 1_cutadapt/Hrd1_DMS_Rep1_Region3_WT_R1_cut.fastq.gz -I 1_cutadapt/Hrd1_DMS_Rep1_Region3_WT_R2_cut.fastq.gz -m --merged_out 2_fastp/Hrd1_DMS_Rep1_Region3_WT_merge.fastq.gz -o 2_fastp/Hrd1_DMS_Rep1_Region3_WT_R1_cut.fastq.gz -O 2_fastp/Hrd1_DMS_Rep1_Region3_WT_R2_cut.fastq.gz -w 12 -h 2_fastp/Hrd1_DMS_Rep1_Region3_WT_report.html -j 2_fastp/Hrd1_DMS_Rep1_Region3_WT_report.json \n",
      "fastp v0.20.0, time used: 1 seconds\n",
      "209132 reads; of these:\n",
      "  209132 (100.00%) were unpaired; of these:\n",
      "    33 (0.02%) aligned 0 times\n",
      "    209099 (99.98%) aligned exactly 1 time\n",
      "    0 (0.00%) aligned >1 times\n",
      "99.98% overall alignment rate\n",
      "[bam_sort_core] merging from 0 files and 12 in-memory blocks...\n",
      "This is cutadapt 1.18 with Python 3.7.16\n",
      "Command line parameters: -a CTGTCTCTTATACACATCT -A CTGTCTCTTATACACATCT -u 8 -U 8 -q 30 -o 1_cutadapt/Hrd1_DMS_Rep2_Region3_WT_R1_cut.fastq.gz -p 1_cutadapt/Hrd1_DMS_Rep2_Region3_WT_R2_cut.fastq.gz --cores=12 0_fastq_files/Hrd1_DMS_Rep2_Region3_WT_R1_001.fastq.gz 0_fastq_files/Hrd1_DMS_Rep2_Region3_WT_R2_001.fastq.gz\n",
      "Processing reads on 12 cores in paired-end mode ...\n",
      "Finished in 1.73 s (10 us/read; 5.81 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            167,229\n",
      "  Read 1 with adapter:                   1,371 (0.8%)\n",
      "  Read 2 with adapter:                     225 (0.1%)\n",
      "Pairs written (passing filters):       167,229 (100.0%)\n",
      "\n",
      "Total basepairs processed:    99,704,879 bp\n",
      "  Read 1:    49,691,004 bp\n",
      "  Read 2:    50,013,875 bp\n",
      "Quality-trimmed:               4,849,795 bp (4.9%)\n",
      "  Read 1:       687,546 bp\n",
      "  Read 2:     4,162,249 bp\n",
      "Total written (filtered):     92,172,940 bp (92.4%)\n",
      "  Read 1:    47,660,654 bp\n",
      "  Read 2:    44,512,286 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: CTGTCTCTTATACACATCT; Type: regular 3'; Length: 19; Trimmed: 1371 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 21.8%\n",
      "  C: 2.0%\n",
      "  G: 58.3%\n",
      "  T: 17.9%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1315\t2613.0\t0\t1315\n",
      "4\t39\t653.2\t0\t39\n",
      "5\t5\t163.3\t0\t5\n",
      "6\t1\t40.8\t0\t1\n",
      "76\t7\t0.0\t1\t7\n",
      "77\t4\t0.0\t1\t4\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CTGTCTCTTATACACATCT; Type: regular 3'; Length: 19; Trimmed: 225 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 7.1%\n",
      "  C: 32.4%\n",
      "  G: 26.2%\n",
      "  T: 34.2%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t178\t2613.0\t0\t178\n",
      "4\t15\t653.2\t0\t15\n",
      "5\t19\t163.3\t0\t19\n",
      "6\t1\t40.8\t0\t1\n",
      "19\t1\t0.0\t1\t0 1\n",
      "50\t1\t0.0\t1\t1\n",
      "57\t1\t0.0\t1\t1\n",
      "58\t1\t0.0\t1\t1\n",
      "63\t1\t0.0\t1\t1\n",
      "80\t2\t0.0\t1\t2\n",
      "81\t4\t0.0\t1\t4\n",
      "82\t1\t0.0\t1\t1\n",
      "\n",
      "Read1 before filtering:\n",
      "total reads: 167229\n",
      "total bases: 47660654\n",
      "Q20 bases: 47397908(99.4487%)\n",
      "Q30 bases: 46609488(97.7945%)\n",
      "\n",
      "Read2 before filtering:\n",
      "total reads: 167229\n",
      "total bases: 44512286\n",
      "Q20 bases: 44005749(98.862%)\n",
      "Q30 bases: 42451778(95.3709%)\n",
      "\n",
      "Merged and filtered:\n",
      "total reads: 160923\n",
      "total bases: 60563007\n",
      "Q20 bases: 60346552(99.6426%)\n",
      "Q30 bases: 59587583(98.3894%)\n",
      "\n",
      "Filtering result:\n",
      "reads passed filter: 331034\n",
      "reads failed due to low quality: 136\n",
      "reads failed due to too many N: 18\n",
      "reads failed due to too short: 3270\n",
      "reads with adapter trimmed: 36\n",
      "bases trimmed due to adapters: 1083\n",
      "reads corrected by overlap analysis: 25061\n",
      "bases corrected by overlap analysis: 30868\n",
      "\n",
      "Duplication rate: 76.2871%\n",
      "\n",
      "Insert size peak (evaluated by paired-end reads): 377\n",
      "\n",
      "Read pairs merged: 160923\n",
      "% of original read pairs: 96.2291%\n",
      "% in reads after filtering: 100%\n",
      "\n",
      "\n",
      "JSON report: 2_fastp/Hrd1_DMS_Rep2_Region3_WT_report.json\n",
      "HTML report: 2_fastp/Hrd1_DMS_Rep2_Region3_WT_report.html\n",
      "\n",
      "fastp -i 1_cutadapt/Hrd1_DMS_Rep2_Region3_WT_R1_cut.fastq.gz -I 1_cutadapt/Hrd1_DMS_Rep2_Region3_WT_R2_cut.fastq.gz -m --merged_out 2_fastp/Hrd1_DMS_Rep2_Region3_WT_merge.fastq.gz -o 2_fastp/Hrd1_DMS_Rep2_Region3_WT_R1_cut.fastq.gz -O 2_fastp/Hrd1_DMS_Rep2_Region3_WT_R2_cut.fastq.gz -w 12 -h 2_fastp/Hrd1_DMS_Rep2_Region3_WT_report.html -j 2_fastp/Hrd1_DMS_Rep2_Region3_WT_report.json \n",
      "fastp v0.20.0, time used: 2 seconds\n",
      "160923 reads; of these:\n",
      "  160923 (100.00%) were unpaired; of these:\n",
      "    21 (0.01%) aligned 0 times\n",
      "    160902 (99.99%) aligned exactly 1 time\n",
      "    0 (0.00%) aligned >1 times\n",
      "99.99% overall alignment rate\n",
      "[bam_sort_core] merging from 0 files and 12 in-memory blocks...\n",
      "This is cutadapt 1.18 with Python 3.7.16\n",
      "Command line parameters: -a CTGTCTCTTATACACATCT -A CTGTCTCTTATACACATCT -u 8 -U 8 -q 30 -o 1_cutadapt/Hrd1_DMS_Rep3_Region3_WT_R1_cut.fastq.gz -p 1_cutadapt/Hrd1_DMS_Rep3_Region3_WT_R2_cut.fastq.gz --cores=12 0_fastq_files/Hrd1_DMS_Rep3_Region3_WT_R1_001.fastq.gz 0_fastq_files/Hrd1_DMS_Rep3_Region3_WT_R2_001.fastq.gz\n",
      "Processing reads on 12 cores in paired-end mode ...\n",
      "Finished in 1.73 s (9 us/read; 6.45 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            185,808\n",
      "  Read 1 with adapter:                     156 (0.1%)\n",
      "  Read 2 with adapter:                   4,336 (2.3%)\n",
      "Pairs written (passing filters):       185,808 (100.0%)\n",
      "\n",
      "Total basepairs processed:   111,477,649 bp\n",
      "  Read 1:    55,741,352 bp\n",
      "  Read 2:    55,736,297 bp\n",
      "Quality-trimmed:               3,788,908 bp (3.4%)\n",
      "  Read 1:       631,745 bp\n",
      "  Read 2:     3,157,163 bp\n",
      "Total written (filtered):    104,699,784 bp (93.9%)\n",
      "  Read 1:    53,621,390 bp\n",
      "  Read 2:    51,078,394 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: CTGTCTCTTATACACATCT; Type: regular 3'; Length: 19; Trimmed: 156 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 3.8%\n",
      "  C: 49.4%\n",
      "  G: 8.3%\n",
      "  T: 36.5%\n",
      "  none/other: 1.9%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t102\t2903.2\t0\t102\n",
      "4\t31\t725.8\t0\t31\n",
      "5\t1\t181.5\t0\t1\n",
      "6\t5\t45.4\t0\t5\n",
      "71\t1\t0.0\t1\t1\n",
      "75\t3\t0.0\t1\t3\n",
      "76\t9\t0.0\t1\t9\n",
      "77\t4\t0.0\t1\t4\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CTGTCTCTTATACACATCT; Type: regular 3'; Length: 19; Trimmed: 4336 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 47.3%\n",
      "  C: 0.6%\n",
      "  G: 51.1%\n",
      "  T: 0.9%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t4272\t2903.2\t0\t4272\n",
      "4\t40\t725.8\t0\t40\n",
      "5\t7\t181.5\t0\t7\n",
      "42\t1\t0.0\t1\t1\n",
      "51\t1\t0.0\t1\t1\n",
      "54\t1\t0.0\t1\t1\n",
      "77\t1\t0.0\t1\t1\n",
      "78\t1\t0.0\t1\t1\n",
      "79\t1\t0.0\t1\t1\n",
      "80\t7\t0.0\t1\t7\n",
      "81\t4\t0.0\t1\t3 1\n",
      "\n",
      "Read1 before filtering:\n",
      "total reads: 185808\n",
      "total bases: 53621390\n",
      "Q20 bases: 53272604(99.3495%)\n",
      "Q30 bases: 52199391(97.3481%)\n",
      "\n",
      "Read2 before filtering:\n",
      "total reads: 185808\n",
      "total bases: 51078394\n",
      "Q20 bases: 50450530(98.7708%)\n",
      "Q30 bases: 48684066(95.3124%)\n",
      "\n",
      "Merged and filtered:\n",
      "total reads: 180954\n",
      "total bases: 69186825\n",
      "Q20 bases: 68870955(99.5435%)\n",
      "Q30 bases: 67782859(97.9708%)\n",
      "\n",
      "Filtering result:\n",
      "reads passed filter: 368540\n",
      "reads failed due to low quality: 266\n",
      "reads failed due to too many N: 28\n",
      "reads failed due to too short: 2782\n",
      "reads with adapter trimmed: 42\n",
      "bases trimmed due to adapters: 711\n",
      "reads corrected by overlap analysis: 25202\n",
      "bases corrected by overlap analysis: 34889\n",
      "\n",
      "Duplication rate: 60.2632%\n",
      "\n",
      "Insert size peak (evaluated by paired-end reads): 383\n",
      "\n",
      "Read pairs merged: 180954\n",
      "% of original read pairs: 97.3876%\n",
      "% in reads after filtering: 100%\n",
      "\n",
      "\n",
      "JSON report: 2_fastp/Hrd1_DMS_Rep3_Region3_WT_report.json\n",
      "HTML report: 2_fastp/Hrd1_DMS_Rep3_Region3_WT_report.html\n",
      "\n",
      "fastp -i 1_cutadapt/Hrd1_DMS_Rep3_Region3_WT_R1_cut.fastq.gz -I 1_cutadapt/Hrd1_DMS_Rep3_Region3_WT_R2_cut.fastq.gz -m --merged_out 2_fastp/Hrd1_DMS_Rep3_Region3_WT_merge.fastq.gz -o 2_fastp/Hrd1_DMS_Rep3_Region3_WT_R1_cut.fastq.gz -O 2_fastp/Hrd1_DMS_Rep3_Region3_WT_R2_cut.fastq.gz -w 12 -h 2_fastp/Hrd1_DMS_Rep3_Region3_WT_report.html -j 2_fastp/Hrd1_DMS_Rep3_Region3_WT_report.json \n",
      "fastp v0.20.0, time used: 2 seconds\n",
      "180954 reads; of these:\n",
      "  180954 (100.00%) were unpaired; of these:\n",
      "    31 (0.02%) aligned 0 times\n",
      "    180923 (99.98%) aligned exactly 1 time\n",
      "    0 (0.00%) aligned >1 times\n",
      "99.98% overall alignment rate\n",
      "[bam_sort_core] merging from 0 files and 12 in-memory blocks...\n",
      "This is cutadapt 1.18 with Python 3.7.16\n",
      "Command line parameters: -a CTGTCTCTTATACACATCT -A CTGTCTCTTATACACATCT -u 8 -U 8 -q 30 -o 1_cutadapt/Hrd1_DMS_Rep4_Region3_WT_R1_cut.fastq.gz -p 1_cutadapt/Hrd1_DMS_Rep4_Region3_WT_R2_cut.fastq.gz --cores=12 0_fastq_files/Hrd1_DMS_Rep4_Region3_WT_R1_001.fastq.gz 0_fastq_files/Hrd1_DMS_Rep4_Region3_WT_R2_001.fastq.gz\n",
      "Processing reads on 12 cores in paired-end mode ...\n",
      "Finished in 1.39 s (10 us/read; 5.96 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            138,609\n",
      "  Read 1 with adapter:                     107 (0.1%)\n",
      "  Read 2 with adapter:                   3,854 (2.8%)\n",
      "Pairs written (passing filters):       138,609 (100.0%)\n",
      "\n",
      "Total basepairs processed:    82,643,586 bp\n",
      "  Read 1:    41,447,890 bp\n",
      "  Read 2:    41,195,696 bp\n",
      "Quality-trimmed:               3,527,721 bp (4.3%)\n",
      "  Read 1:       970,799 bp\n",
      "  Read 2:     2,556,922 bp\n",
      "Total written (filtered):     76,885,060 bp (93.0%)\n",
      "  Read 1:    39,367,329 bp\n",
      "  Read 2:    37,517,731 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: CTGTCTCTTATACACATCT; Type: regular 3'; Length: 19; Trimmed: 107 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 2.8%\n",
      "  C: 23.4%\n",
      "  G: 43.0%\n",
      "  T: 30.8%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t86\t2165.8\t0\t86\n",
      "4\t10\t541.4\t0\t10\n",
      "5\t1\t135.4\t0\t1\n",
      "6\t2\t33.8\t0\t2\n",
      "43\t1\t0.0\t1\t1\n",
      "74\t1\t0.0\t1\t1\n",
      "75\t1\t0.0\t1\t1\n",
      "76\t3\t0.0\t1\t3\n",
      "77\t1\t0.0\t1\t1\n",
      "78\t1\t0.0\t1\t1\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CTGTCTCTTATACACATCT; Type: regular 3'; Length: 19; Trimmed: 3854 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 32.1%\n",
      "  C: 1.0%\n",
      "  G: 65.1%\n",
      "  T: 1.8%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3820\t2165.8\t0\t3820\n",
      "4\t20\t541.4\t0\t20\n",
      "5\t6\t135.4\t0\t6\n",
      "57\t1\t0.0\t1\t1\n",
      "64\t1\t0.0\t1\t1\n",
      "80\t6\t0.0\t1\t6\n",
      "\n",
      "Read1 before filtering:\n",
      "total reads: 138609\n",
      "total bases: 39367329\n",
      "Q20 bases: 39043928(99.1785%)\n",
      "Q30 bases: 38023413(96.5862%)\n",
      "\n",
      "Read2 before filtering:\n",
      "total reads: 138609\n",
      "total bases: 37517731\n",
      "Q20 bases: 37015715(98.6619%)\n",
      "Q30 bases: 35639616(94.9941%)\n",
      "\n",
      "Merged and filtered:\n",
      "total reads: 133847\n",
      "total bases: 50387944\n",
      "Q20 bases: 50094327(99.4173%)\n",
      "Q30 bases: 49074400(97.3931%)\n",
      "\n",
      "Filtering result:\n",
      "reads passed filter: 274948\n",
      "reads failed due to low quality: 214\n",
      "reads failed due to too many N: 10\n",
      "reads failed due to too short: 2046\n",
      "reads with adapter trimmed: 42\n",
      "bases trimmed due to adapters: 1654\n",
      "reads corrected by overlap analysis: 23233\n",
      "bases corrected by overlap analysis: 30999\n",
      "\n",
      "Duplication rate: 77.1639%\n",
      "\n",
      "Insert size peak (evaluated by paired-end reads): 377\n",
      "\n",
      "Read pairs merged: 133847\n",
      "% of original read pairs: 96.5644%\n",
      "% in reads after filtering: 100%\n",
      "\n",
      "\n",
      "JSON report: 2_fastp/Hrd1_DMS_Rep4_Region3_WT_report.json\n",
      "HTML report: 2_fastp/Hrd1_DMS_Rep4_Region3_WT_report.html\n",
      "\n",
      "fastp -i 1_cutadapt/Hrd1_DMS_Rep4_Region3_WT_R1_cut.fastq.gz -I 1_cutadapt/Hrd1_DMS_Rep4_Region3_WT_R2_cut.fastq.gz -m --merged_out 2_fastp/Hrd1_DMS_Rep4_Region3_WT_merge.fastq.gz -o 2_fastp/Hrd1_DMS_Rep4_Region3_WT_R1_cut.fastq.gz -O 2_fastp/Hrd1_DMS_Rep4_Region3_WT_R2_cut.fastq.gz -w 12 -h 2_fastp/Hrd1_DMS_Rep4_Region3_WT_report.html -j 2_fastp/Hrd1_DMS_Rep4_Region3_WT_report.json \n",
      "fastp v0.20.0, time used: 2 seconds\n",
      "133847 reads; of these:\n",
      "  133847 (100.00%) were unpaired; of these:\n",
      "    18 (0.01%) aligned 0 times\n",
      "    133829 (99.99%) aligned exactly 1 time\n",
      "    0 (0.00%) aligned >1 times\n",
      "99.99% overall alignment rate\n",
      "[bam_sort_core] merging from 0 files and 12 in-memory blocks...\n",
      "This is cutadapt 1.18 with Python 3.7.16\n",
      "Command line parameters: -a CTGTCTCTTATACACATCT -A CTGTCTCTTATACACATCT -u 8 -U 8 -q 30 -o 1_cutadapt/Hrd1_DMS_Rep1_Region3_LDead_R1_cut.fastq.gz -p 1_cutadapt/Hrd1_DMS_Rep1_Region3_LDead_R2_cut.fastq.gz --cores=12 0_fastq_files/Hrd1_DMS_Rep1_Region3_LDead_R1_001.fastq.gz 0_fastq_files/Hrd1_DMS_Rep1_Region3_LDead_R2_001.fastq.gz\n",
      "Processing reads on 12 cores in paired-end mode ...\n",
      "Finished in 0.35 s (16 us/read; 3.69 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             21,615\n",
      "  Read 1 with adapter:                      16 (0.1%)\n",
      "  Read 2 with adapter:                     373 (1.7%)\n",
      "Pairs written (passing filters):        21,615 (100.0%)\n",
      "\n",
      "Total basepairs processed:    12,886,670 bp\n",
      "  Read 1:     6,463,665 bp\n",
      "  Read 2:     6,423,005 bp\n",
      "Quality-trimmed:                 462,943 bp (3.6%)\n",
      "  Read 1:        88,009 bp\n",
      "  Read 2:       374,934 bp\n",
      "Total written (filtered):     12,076,405 bp (93.7%)\n",
      "  Read 1:     6,202,532 bp\n",
      "  Read 2:     5,873,873 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: CTGTCTCTTATACACATCT; Type: regular 3'; Length: 19; Trimmed: 16 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 18.8%\n",
      "  C: 18.8%\n",
      "  G: 25.0%\n",
      "  T: 37.5%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t7\t337.7\t0\t7\n",
      "4\t5\t84.4\t0\t5\n",
      "5\t1\t21.1\t0\t1\n",
      "6\t1\t5.3\t0\t1\n",
      "76\t2\t0.0\t1\t2\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CTGTCTCTTATACACATCT; Type: regular 3'; Length: 19; Trimmed: 373 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 16.6%\n",
      "  C: 0.8%\n",
      "  G: 33.2%\n",
      "  T: 49.3%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t370\t337.7\t0\t370\n",
      "4\t1\t84.4\t0\t1\n",
      "80\t1\t0.0\t1\t0 1\n",
      "84\t1\t0.0\t1\t1\n",
      "\n",
      "Read1 before filtering:\n",
      "total reads: 21615\n",
      "total bases: 6202532\n",
      "Q20 bases: 6169532(99.468%)\n",
      "Q30 bases: 6059553(97.6948%)\n",
      "\n",
      "Read2 before filtering:\n",
      "total reads: 21615\n",
      "total bases: 5873873\n",
      "Q20 bases: 5802995(98.7933%)\n",
      "Q30 bases: 5604282(95.4103%)\n",
      "\n",
      "Merged and filtered:\n",
      "total reads: 20959\n",
      "total bases: 7885264\n",
      "Q20 bases: 7856302(99.6327%)\n",
      "Q30 bases: 7748252(98.2624%)\n",
      "\n",
      "Filtering result:\n",
      "reads passed filter: 42824\n",
      "reads failed due to low quality: 48\n",
      "reads failed due to too many N: 4\n",
      "reads failed due to too short: 354\n",
      "reads with adapter trimmed: 4\n",
      "bases trimmed due to adapters: 32\n",
      "reads corrected by overlap analysis: 2701\n",
      "bases corrected by overlap analysis: 3513\n",
      "\n",
      "Duplication rate: 55.8442%\n",
      "\n",
      "Insert size peak (evaluated by paired-end reads): 377\n",
      "\n",
      "Read pairs merged: 20959\n",
      "% of original read pairs: 96.9651%\n",
      "% in reads after filtering: 100%\n",
      "\n",
      "\n",
      "JSON report: 2_fastp/Hrd1_DMS_Rep1_Region3_LDead_report.json\n",
      "HTML report: 2_fastp/Hrd1_DMS_Rep1_Region3_LDead_report.html\n",
      "\n",
      "fastp -i 1_cutadapt/Hrd1_DMS_Rep1_Region3_LDead_R1_cut.fastq.gz -I 1_cutadapt/Hrd1_DMS_Rep1_Region3_LDead_R2_cut.fastq.gz -m --merged_out 2_fastp/Hrd1_DMS_Rep1_Region3_LDead_merge.fastq.gz -o 2_fastp/Hrd1_DMS_Rep1_Region3_LDead_R1_cut.fastq.gz -O 2_fastp/Hrd1_DMS_Rep1_Region3_LDead_R2_cut.fastq.gz -w 12 -h 2_fastp/Hrd1_DMS_Rep1_Region3_LDead_report.html -j 2_fastp/Hrd1_DMS_Rep1_Region3_LDead_report.json \n",
      "fastp v0.20.0, time used: 0 seconds\n",
      "20959 reads; of these:\n",
      "  20959 (100.00%) were unpaired; of these:\n",
      "    4 (0.02%) aligned 0 times\n",
      "    20955 (99.98%) aligned exactly 1 time\n",
      "    0 (0.00%) aligned >1 times\n",
      "99.98% overall alignment rate\n",
      "[bam_sort_core] merging from 0 files and 12 in-memory blocks...\n",
      "This is cutadapt 1.18 with Python 3.7.16\n",
      "Command line parameters: -a CTGTCTCTTATACACATCT -A CTGTCTCTTATACACATCT -u 8 -U 8 -q 30 -o 1_cutadapt/Hrd1_DMS_Rep4_Region3_LDead_R1_cut.fastq.gz -p 1_cutadapt/Hrd1_DMS_Rep4_Region3_LDead_R2_cut.fastq.gz --cores=12 0_fastq_files/Hrd1_DMS_Rep4_Region3_LDead_R1_001.fastq.gz 0_fastq_files/Hrd1_DMS_Rep4_Region3_LDead_R2_001.fastq.gz\n",
      "Processing reads on 12 cores in paired-end mode ...\n",
      "Finished in 0.25 s (24 us/read; 2.53 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             10,384\n",
      "  Read 1 with adapter:                       6 (0.1%)\n",
      "  Read 2 with adapter:                     293 (2.8%)\n",
      "Pairs written (passing filters):        10,384 (100.0%)\n",
      "\n",
      "Total basepairs processed:     6,191,609 bp\n",
      "  Read 1:     3,104,989 bp\n",
      "  Read 2:     3,086,620 bp\n",
      "Quality-trimmed:                 359,761 bp (5.8%)\n",
      "  Read 1:        49,628 bp\n",
      "  Read 2:       310,133 bp\n",
      "Total written (filtered):      5,664,231 bp (91.5%)\n",
      "  Read 1:     2,971,906 bp\n",
      "  Read 2:     2,692,325 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: CTGTCTCTTATACACATCT; Type: regular 3'; Length: 19; Trimmed: 6 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 66.7%\n",
      "  C: 0.0%\n",
      "  G: 16.7%\n",
      "  T: 16.7%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t162.2\t0\t1\n",
      "74\t1\t0.0\t1\t1\n",
      "76\t3\t0.0\t1\t3\n",
      "78\t1\t0.0\t1\t1\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CTGTCTCTTATACACATCT; Type: regular 3'; Length: 19; Trimmed: 293 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 37.2%\n",
      "  C: 0.0%\n",
      "  G: 59.7%\n",
      "  T: 3.1%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t285\t162.2\t0\t285\n",
      "4\t2\t40.6\t0\t2\n",
      "6\t1\t2.5\t0\t1\n",
      "37\t2\t0.0\t1\t1 1\n",
      "44\t1\t0.0\t1\t1\n",
      "51\t1\t0.0\t1\t1\n",
      "52\t1\t0.0\t1\t1\n",
      "\n",
      "Read1 before filtering:\n",
      "total reads: 10384\n",
      "total bases: 2971906\n",
      "Q20 bases: 2953990(99.3972%)\n",
      "Q30 bases: 2897251(97.488%)\n",
      "\n",
      "Read2 before filtering:\n",
      "total reads: 10384\n",
      "total bases: 2692325\n",
      "Q20 bases: 2647557(98.3372%)\n",
      "Q30 bases: 2531109(94.012%)\n",
      "\n",
      "Merged and filtered:\n",
      "total reads: 9882\n",
      "total bases: 3708330\n",
      "Q20 bases: 3693928(99.6116%)\n",
      "Q30 bases: 3641764(98.205%)\n",
      "\n",
      "Filtering result:\n",
      "reads passed filter: 20400\n",
      "reads failed due to low quality: 14\n",
      "reads failed due to too many N: 6\n",
      "reads failed due to too short: 348\n",
      "reads with adapter trimmed: 10\n",
      "bases trimmed due to adapters: 80\n",
      "reads corrected by overlap analysis: 1655\n",
      "bases corrected by overlap analysis: 2232\n",
      "\n",
      "Duplication rate: 57.7982%\n",
      "\n",
      "Insert size peak (evaluated by paired-end reads): 377\n",
      "\n",
      "Read pairs merged: 9882\n",
      "% of original read pairs: 95.1656%\n",
      "% in reads after filtering: 100%\n",
      "\n",
      "\n",
      "JSON report: 2_fastp/Hrd1_DMS_Rep4_Region3_LDead_report.json\n",
      "HTML report: 2_fastp/Hrd1_DMS_Rep4_Region3_LDead_report.html\n",
      "\n",
      "fastp -i 1_cutadapt/Hrd1_DMS_Rep4_Region3_LDead_R1_cut.fastq.gz -I 1_cutadapt/Hrd1_DMS_Rep4_Region3_LDead_R2_cut.fastq.gz -m --merged_out 2_fastp/Hrd1_DMS_Rep4_Region3_LDead_merge.fastq.gz -o 2_fastp/Hrd1_DMS_Rep4_Region3_LDead_R1_cut.fastq.gz -O 2_fastp/Hrd1_DMS_Rep4_Region3_LDead_R2_cut.fastq.gz -w 12 -h 2_fastp/Hrd1_DMS_Rep4_Region3_LDead_report.html -j 2_fastp/Hrd1_DMS_Rep4_Region3_LDead_report.json \n",
      "fastp v0.20.0, time used: 0 seconds\n",
      "9882 reads; of these:\n",
      "  9882 (100.00%) were unpaired; of these:\n",
      "    6 (0.06%) aligned 0 times\n",
      "    9876 (99.94%) aligned exactly 1 time\n",
      "    0 (0.00%) aligned >1 times\n",
      "99.94% overall alignment rate\n",
      "[bam_sort_core] merging from 0 files and 12 in-memory blocks...\n",
      "This is cutadapt 1.18 with Python 3.7.16\n",
      "Command line parameters: -a CTGTCTCTTATACACATCT -A CTGTCTCTTATACACATCT -u 8 -U 8 -q 30 -o 1_cutadapt/Hrd1_DMS_Rep1_Region4_Input_R1_cut.fastq.gz -p 1_cutadapt/Hrd1_DMS_Rep1_Region4_Input_R2_cut.fastq.gz --cores=12 0_fastq_files/Hrd1_DMS_Rep1_Region4_Input_R1_001.fastq.gz 0_fastq_files/Hrd1_DMS_Rep1_Region4_Input_R2_001.fastq.gz\n",
      "Processing reads on 12 cores in paired-end mode ...\n",
      "Finished in 1.87 s (9 us/read; 6.72 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            209,145\n",
      "  Read 1 with adapter:                  51,998 (24.9%)\n",
      "  Read 2 with adapter:                  52,651 (25.2%)\n",
      "Pairs written (passing filters):       209,145 (100.0%)\n",
      "\n",
      "Total basepairs processed:   125,123,812 bp\n",
      "  Read 1:    62,560,775 bp\n",
      "  Read 2:    62,563,037 bp\n",
      "Quality-trimmed:              20,257,798 bp (16.2%)\n",
      "  Read 1:     8,419,958 bp\n",
      "  Read 2:    11,837,840 bp\n",
      "Total written (filtered):     94,279,462 bp (75.3%)\n",
      "  Read 1:    48,528,915 bp\n",
      "  Read 2:    45,750,547 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: CTGTCTCTTATACACATCT; Type: regular 3'; Length: 19; Trimmed: 51998 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 31.8%\n",
      "  C: 24.5%\n",
      "  G: 18.5%\n",
      "  T: 25.3%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t65\t3267.9\t0\t65\n",
      "4\t41\t817.0\t0\t41\n",
      "5\t24\t204.2\t0\t24\n",
      "6\t14\t51.1\t0\t14\n",
      "7\t4\t12.8\t0\t4\n",
      "8\t8\t3.2\t0\t8\n",
      "9\t17\t0.8\t0\t17\n",
      "10\t9\t0.2\t1\t7 2\n",
      "11\t6\t0.0\t1\t6\n",
      "12\t9\t0.0\t1\t9\n",
      "13\t3\t0.0\t1\t3\n",
      "14\t15\t0.0\t1\t14 1\n",
      "15\t15\t0.0\t1\t15\n",
      "16\t5\t0.0\t1\t5\n",
      "17\t17\t0.0\t1\t15 2\n",
      "18\t25\t0.0\t1\t23 2\n",
      "19\t18\t0.0\t1\t16 2\n",
      "20\t16\t0.0\t1\t12 4\n",
      "21\t42\t0.0\t1\t34 8\n",
      "22\t12\t0.0\t1\t11 1\n",
      "23\t14\t0.0\t1\t14\n",
      "24\t22\t0.0\t1\t19 3\n",
      "25\t18\t0.0\t1\t15 3\n",
      "26\t5\t0.0\t1\t5\n",
      "27\t17\t0.0\t1\t15 2\n",
      "28\t29\t0.0\t1\t25 4\n",
      "29\t42\t0.0\t1\t38 4\n",
      "30\t27\t0.0\t1\t24 3\n",
      "31\t22\t0.0\t1\t17 5\n",
      "32\t10\t0.0\t1\t7 3\n",
      "33\t19\t0.0\t1\t17 2\n",
      "34\t23\t0.0\t1\t20 3\n",
      "35\t4\t0.0\t1\t3 1\n",
      "36\t12\t0.0\t1\t10 2\n",
      "37\t9\t0.0\t1\t8 1\n",
      "38\t10\t0.0\t1\t9 1\n",
      "39\t11\t0.0\t1\t10 1\n",
      "40\t63\t0.0\t1\t54 9\n",
      "41\t37\t0.0\t1\t35 2\n",
      "42\t83\t0.0\t1\t73 10\n",
      "43\t233\t0.0\t1\t219 14\n",
      "44\t68\t0.0\t1\t61 7\n",
      "45\t21\t0.0\t1\t17 4\n",
      "46\t9\t0.0\t1\t8 1\n",
      "47\t39\t0.0\t1\t37 2\n",
      "48\t2\t0.0\t1\t1 1\n",
      "49\t24\t0.0\t1\t23 1\n",
      "50\t11\t0.0\t1\t10 1\n",
      "51\t47\t0.0\t1\t45 2\n",
      "52\t33\t0.0\t1\t30 3\n",
      "53\t78\t0.0\t1\t74 4\n",
      "54\t191\t0.0\t1\t183 8\n",
      "55\t116\t0.0\t1\t112 4\n",
      "56\t53\t0.0\t1\t50 3\n",
      "57\t115\t0.0\t1\t106 9\n",
      "58\t55\t0.0\t1\t52 3\n",
      "59\t36\t0.0\t1\t35 1\n",
      "60\t90\t0.0\t1\t85 5\n",
      "61\t81\t0.0\t1\t72 9\n",
      "62\t54\t0.0\t1\t52 2\n",
      "63\t110\t0.0\t1\t107 3\n",
      "64\t254\t0.0\t1\t247 7\n",
      "65\t423\t0.0\t1\t412 11\n",
      "66\t246\t0.0\t1\t239 7\n",
      "67\t169\t0.0\t1\t167 2\n",
      "68\t300\t0.0\t1\t292 8\n",
      "69\t266\t0.0\t1\t256 10\n",
      "70\t181\t0.0\t1\t175 6\n",
      "71\t170\t0.0\t1\t166 4\n",
      "72\t370\t0.0\t1\t362 8\n",
      "73\t399\t0.0\t1\t385 14\n",
      "74\t878\t0.0\t1\t857 21\n",
      "75\t2394\t0.0\t1\t2009 385\n",
      "76\t31881\t0.0\t1\t31544 337\n",
      "77\t9643\t0.0\t1\t9521 122\n",
      "78\t1162\t0.0\t1\t1135 27\n",
      "79\t186\t0.0\t1\t184 2\n",
      "80\t43\t0.0\t1\t42 1\n",
      "81\t8\t0.0\t1\t8\n",
      "82\t16\t0.0\t1\t16\n",
      "83\t9\t0.0\t1\t9\n",
      "84\t33\t0.0\t1\t33\n",
      "85\t9\t0.0\t1\t9\n",
      "86\t5\t0.0\t1\t5\n",
      "87\t12\t0.0\t1\t12\n",
      "88\t5\t0.0\t1\t5\n",
      "89\t3\t0.0\t1\t3\n",
      "90\t9\t0.0\t1\t9\n",
      "91\t12\t0.0\t1\t12\n",
      "92\t5\t0.0\t1\t4 1\n",
      "93\t4\t0.0\t1\t4\n",
      "94\t1\t0.0\t1\t1\n",
      "95\t2\t0.0\t1\t2\n",
      "96\t4\t0.0\t1\t4\n",
      "97\t5\t0.0\t1\t5\n",
      "98\t3\t0.0\t1\t2 1\n",
      "99\t3\t0.0\t1\t3\n",
      "100\t2\t0.0\t1\t1 1\n",
      "101\t4\t0.0\t1\t3 1\n",
      "102\t3\t0.0\t1\t3\n",
      "103\t2\t0.0\t1\t2\n",
      "104\t2\t0.0\t1\t2\n",
      "106\t2\t0.0\t1\t2\n",
      "107\t5\t0.0\t1\t5\n",
      "108\t1\t0.0\t1\t1\n",
      "109\t1\t0.0\t1\t1\n",
      "110\t3\t0.0\t1\t3\n",
      "112\t2\t0.0\t1\t2\n",
      "113\t2\t0.0\t1\t2\n",
      "114\t1\t0.0\t1\t1\n",
      "115\t2\t0.0\t1\t1 1\n",
      "116\t2\t0.0\t1\t2\n",
      "117\t1\t0.0\t1\t1\n",
      "118\t1\t0.0\t1\t1\n",
      "119\t1\t0.0\t1\t0 1\n",
      "120\t1\t0.0\t1\t1\n",
      "122\t3\t0.0\t1\t3\n",
      "123\t3\t0.0\t1\t3\n",
      "124\t2\t0.0\t1\t2\n",
      "125\t1\t0.0\t1\t1\n",
      "126\t3\t0.0\t1\t3\n",
      "127\t1\t0.0\t1\t1\n",
      "128\t2\t0.0\t1\t2\n",
      "129\t1\t0.0\t1\t1\n",
      "131\t1\t0.0\t1\t1\n",
      "132\t1\t0.0\t1\t1\n",
      "133\t2\t0.0\t1\t2\n",
      "134\t1\t0.0\t1\t1\n",
      "135\t1\t0.0\t1\t1\n",
      "136\t1\t0.0\t1\t1\n",
      "137\t1\t0.0\t1\t1\n",
      "138\t2\t0.0\t1\t2\n",
      "139\t3\t0.0\t1\t3\n",
      "140\t2\t0.0\t1\t2\n",
      "141\t3\t0.0\t1\t2 1\n",
      "142\t3\t0.0\t1\t3\n",
      "143\t2\t0.0\t1\t2\n",
      "144\t2\t0.0\t1\t2\n",
      "145\t1\t0.0\t1\t1\n",
      "146\t2\t0.0\t1\t2\n",
      "147\t2\t0.0\t1\t2\n",
      "148\t1\t0.0\t1\t1\n",
      "149\t3\t0.0\t1\t3\n",
      "150\t2\t0.0\t1\t2\n",
      "152\t1\t0.0\t1\t1\n",
      "153\t1\t0.0\t1\t1\n",
      "154\t1\t0.0\t1\t1\n",
      "155\t4\t0.0\t1\t4\n",
      "156\t4\t0.0\t1\t4\n",
      "157\t3\t0.0\t1\t3\n",
      "158\t1\t0.0\t1\t1\n",
      "160\t6\t0.0\t1\t5 1\n",
      "162\t3\t0.0\t1\t3\n",
      "163\t2\t0.0\t1\t1 1\n",
      "164\t4\t0.0\t1\t3 1\n",
      "165\t3\t0.0\t1\t3\n",
      "166\t4\t0.0\t1\t4\n",
      "167\t1\t0.0\t1\t1\n",
      "168\t1\t0.0\t1\t1\n",
      "169\t1\t0.0\t1\t1\n",
      "170\t3\t0.0\t1\t3\n",
      "171\t1\t0.0\t1\t1\n",
      "172\t1\t0.0\t1\t1\n",
      "173\t1\t0.0\t1\t1\n",
      "175\t2\t0.0\t1\t2\n",
      "176\t2\t0.0\t1\t2\n",
      "178\t1\t0.0\t1\t1\n",
      "179\t1\t0.0\t1\t1\n",
      "180\t1\t0.0\t1\t1\n",
      "181\t4\t0.0\t1\t4\n",
      "182\t4\t0.0\t1\t3 1\n",
      "183\t2\t0.0\t1\t1 1\n",
      "185\t3\t0.0\t1\t3\n",
      "187\t5\t0.0\t1\t5\n",
      "188\t1\t0.0\t1\t1\n",
      "189\t2\t0.0\t1\t1 1\n",
      "191\t4\t0.0\t1\t3 1\n",
      "192\t2\t0.0\t1\t1 1\n",
      "193\t1\t0.0\t1\t1\n",
      "195\t5\t0.0\t1\t4 1\n",
      "196\t3\t0.0\t1\t3\n",
      "197\t1\t0.0\t1\t1\n",
      "198\t1\t0.0\t1\t1\n",
      "199\t1\t0.0\t1\t1\n",
      "200\t4\t0.0\t1\t4\n",
      "201\t1\t0.0\t1\t1\n",
      "202\t4\t0.0\t1\t4\n",
      "203\t1\t0.0\t1\t1\n",
      "204\t3\t0.0\t1\t3\n",
      "205\t2\t0.0\t1\t2\n",
      "206\t3\t0.0\t1\t2 1\n",
      "208\t3\t0.0\t1\t2 1\n",
      "209\t1\t0.0\t1\t1\n",
      "210\t3\t0.0\t1\t2 1\n",
      "211\t2\t0.0\t1\t2\n",
      "212\t2\t0.0\t1\t1 1\n",
      "214\t1\t0.0\t1\t1\n",
      "215\t1\t0.0\t1\t1\n",
      "216\t1\t0.0\t1\t0 1\n",
      "217\t2\t0.0\t1\t2\n",
      "218\t1\t0.0\t1\t1\n",
      "219\t1\t0.0\t1\t1\n",
      "220\t3\t0.0\t1\t3\n",
      "221\t5\t0.0\t1\t4 1\n",
      "222\t4\t0.0\t1\t4\n",
      "223\t2\t0.0\t1\t1 1\n",
      "224\t3\t0.0\t1\t2 1\n",
      "225\t1\t0.0\t1\t1\n",
      "226\t4\t0.0\t1\t3 1\n",
      "227\t8\t0.0\t1\t8\n",
      "228\t5\t0.0\t1\t2 3\n",
      "229\t9\t0.0\t1\t9\n",
      "230\t28\t0.0\t1\t26 2\n",
      "231\t152\t0.0\t1\t131 21\n",
      "232\t133\t0.0\t1\t115 18\n",
      "233\t2\t0.0\t1\t2\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CTGTCTCTTATACACATCT; Type: regular 3'; Length: 19; Trimmed: 52651 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 30.3%\n",
      "  C: 27.3%\n",
      "  G: 18.0%\n",
      "  T: 24.4%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t947\t3267.9\t0\t947\n",
      "4\t104\t817.0\t0\t104\n",
      "5\t16\t204.2\t0\t16\n",
      "6\t4\t51.1\t0\t4\n",
      "7\t7\t12.8\t0\t7\n",
      "8\t6\t3.2\t0\t6\n",
      "9\t10\t0.8\t0\t10\n",
      "10\t11\t0.2\t1\t11\n",
      "11\t5\t0.0\t1\t5\n",
      "12\t12\t0.0\t1\t11 1\n",
      "13\t11\t0.0\t1\t9 2\n",
      "14\t10\t0.0\t1\t8 2\n",
      "15\t25\t0.0\t1\t24 1\n",
      "16\t13\t0.0\t1\t12 1\n",
      "17\t20\t0.0\t1\t18 2\n",
      "18\t23\t0.0\t1\t20 3\n",
      "19\t29\t0.0\t1\t29\n",
      "20\t39\t0.0\t1\t38 1\n",
      "21\t40\t0.0\t1\t38 2\n",
      "22\t24\t0.0\t1\t23 1\n",
      "23\t19\t0.0\t1\t18 1\n",
      "24\t50\t0.0\t1\t47 3\n",
      "25\t50\t0.0\t1\t48 2\n",
      "26\t62\t0.0\t1\t58 4\n",
      "27\t38\t0.0\t1\t38\n",
      "28\t74\t0.0\t1\t71 3\n",
      "29\t125\t0.0\t1\t118 7\n",
      "30\t72\t0.0\t1\t66 6\n",
      "31\t118\t0.0\t1\t113 5\n",
      "32\t318\t0.0\t1\t309 9\n",
      "33\t392\t0.0\t1\t375 17\n",
      "34\t841\t0.0\t1\t825 16\n",
      "35\t94\t0.0\t1\t90 4\n",
      "36\t153\t0.0\t1\t148 5\n",
      "37\t294\t0.0\t1\t287 7\n",
      "38\t337\t0.0\t1\t332 5\n",
      "39\t135\t0.0\t1\t132 3\n",
      "40\t121\t0.0\t1\t120 1\n",
      "41\t296\t0.0\t1\t293 3\n",
      "42\t439\t0.0\t1\t430 9\n",
      "43\t1150\t0.0\t1\t1137 13\n",
      "44\t578\t0.0\t1\t567 11\n",
      "45\t972\t0.0\t1\t950 22\n",
      "46\t1098\t0.0\t1\t1073 25\n",
      "47\t884\t0.0\t1\t865 19\n",
      "48\t1654\t0.0\t1\t1631 23\n",
      "49\t1435\t0.0\t1\t1419 16\n",
      "50\t735\t0.0\t1\t717 18\n",
      "51\t583\t0.0\t1\t572 11\n",
      "52\t1176\t0.0\t1\t1164 12\n",
      "53\t947\t0.0\t1\t924 23\n",
      "54\t2260\t0.0\t1\t2229 31\n",
      "55\t761\t0.0\t1\t752 9\n",
      "56\t1066\t0.0\t1\t1036 30\n",
      "57\t3292\t0.0\t1\t3257 35\n",
      "58\t1382\t0.0\t1\t1359 23\n",
      "59\t747\t0.0\t1\t737 10\n",
      "60\t629\t0.0\t1\t622 7\n",
      "61\t235\t0.0\t1\t233 2\n",
      "62\t714\t0.0\t1\t702 12\n",
      "63\t1094\t0.0\t1\t1081 13\n",
      "64\t278\t0.0\t1\t275 3\n",
      "65\t82\t0.0\t1\t82\n",
      "66\t51\t0.0\t1\t50 1\n",
      "67\t69\t0.0\t1\t69\n",
      "68\t25\t0.0\t1\t25\n",
      "69\t32\t0.0\t1\t32\n",
      "70\t61\t0.0\t1\t61\n",
      "71\t8\t0.0\t1\t8\n",
      "72\t11\t0.0\t1\t11\n",
      "73\t19\t0.0\t1\t19\n",
      "74\t36\t0.0\t1\t35 1\n",
      "75\t58\t0.0\t1\t57 1\n",
      "76\t242\t0.0\t1\t239 3\n",
      "77\t303\t0.0\t1\t299 4\n",
      "78\t1010\t0.0\t1\t997 13\n",
      "79\t1172\t0.0\t1\t1135 37\n",
      "80\t9035\t0.0\t1\t8955 80\n",
      "81\t8528\t0.0\t1\t8458 70\n",
      "82\t2299\t0.0\t1\t2275 24\n",
      "83\t428\t0.0\t1\t422 6\n",
      "84\t88\t0.0\t1\t86 2\n",
      "85\t20\t0.0\t1\t20\n",
      "86\t3\t0.0\t1\t3\n",
      "87\t2\t0.0\t1\t2\n",
      "88\t1\t0.0\t1\t1\n",
      "90\t1\t0.0\t1\t1\n",
      "230\t1\t0.0\t1\t1\n",
      "232\t12\t0.0\t1\t12\n",
      "\n",
      "Read1 before filtering:\n",
      "total reads: 209145\n",
      "total bases: 48528915\n",
      "Q20 bases: 48315938(99.5611%)\n",
      "Q30 bases: 47624932(98.1372%)\n",
      "\n",
      "Read2 before filtering:\n",
      "total reads: 209145\n",
      "total bases: 45750547\n",
      "Q20 bases: 45167635(98.7259%)\n",
      "Q30 bases: 43602992(95.3059%)\n",
      "\n",
      "Merged and filtered:\n",
      "total reads: 204292\n",
      "total bases: 60031521\n",
      "Q20 bases: 59862803(99.719%)\n",
      "Q30 bases: 59206863(98.6263%)\n",
      "\n",
      "Filtering result:\n",
      "reads passed filter: 414454\n",
      "reads failed due to low quality: 204\n",
      "reads failed due to too many N: 20\n",
      "reads failed due to too short: 3612\n",
      "reads with adapter trimmed: 103398\n",
      "bases trimmed due to adapters: 834906\n",
      "reads corrected by overlap analysis: 22936\n",
      "bases corrected by overlap analysis: 30592\n",
      "\n",
      "Duplication rate: 41.1765%\n",
      "\n",
      "Insert size peak (evaluated by paired-end reads): 376\n",
      "\n",
      "Read pairs merged: 204292\n",
      "% of original read pairs: 97.6796%\n",
      "% in reads after filtering: 100%\n",
      "\n",
      "\n",
      "JSON report: 2_fastp/Hrd1_DMS_Rep1_Region4_Input_report.json\n",
      "HTML report: 2_fastp/Hrd1_DMS_Rep1_Region4_Input_report.html\n",
      "\n",
      "fastp -i 1_cutadapt/Hrd1_DMS_Rep1_Region4_Input_R1_cut.fastq.gz -I 1_cutadapt/Hrd1_DMS_Rep1_Region4_Input_R2_cut.fastq.gz -m --merged_out 2_fastp/Hrd1_DMS_Rep1_Region4_Input_merge.fastq.gz -o 2_fastp/Hrd1_DMS_Rep1_Region4_Input_R1_cut.fastq.gz -O 2_fastp/Hrd1_DMS_Rep1_Region4_Input_R2_cut.fastq.gz -w 12 -h 2_fastp/Hrd1_DMS_Rep1_Region4_Input_report.html -j 2_fastp/Hrd1_DMS_Rep1_Region4_Input_report.json \n",
      "fastp v0.20.0, time used: 2 seconds\n",
      "204292 reads; of these:\n",
      "  204292 (100.00%) were unpaired; of these:\n",
      "    51579 (25.25%) aligned 0 times\n",
      "    152713 (74.75%) aligned exactly 1 time\n",
      "    0 (0.00%) aligned >1 times\n",
      "74.75% overall alignment rate\n",
      "[bam_sort_core] merging from 0 files and 12 in-memory blocks...\n",
      "This is cutadapt 1.18 with Python 3.7.16\n",
      "Command line parameters: -a CTGTCTCTTATACACATCT -A CTGTCTCTTATACACATCT -u 8 -U 8 -q 30 -o 1_cutadapt/Hrd1_DMS_Rep3_Region4_Input_R1_cut.fastq.gz -p 1_cutadapt/Hrd1_DMS_Rep3_Region4_Input_R2_cut.fastq.gz --cores=12 0_fastq_files/Hrd1_DMS_Rep3_Region4_Input_R1_001.fastq.gz 0_fastq_files/Hrd1_DMS_Rep3_Region4_Input_R2_001.fastq.gz\n",
      "Processing reads on 12 cores in paired-end mode ...\n",
      "Finished in 2.28 s (10 us/read; 6.03 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            228,815\n",
      "  Read 1 with adapter:                  14,099 (6.2%)\n",
      "  Read 2 with adapter:                  16,228 (7.1%)\n",
      "Pairs written (passing filters):       228,815 (100.0%)\n",
      "\n",
      "Total basepairs processed:   137,061,076 bp\n",
      "  Read 1:    68,630,443 bp\n",
      "  Read 2:    68,430,633 bp\n",
      "Quality-trimmed:               9,040,588 bp (6.6%)\n",
      "  Read 1:     2,566,646 bp\n",
      "  Read 2:     6,473,942 bp\n",
      "Total written (filtered):    122,342,231 bp (89.3%)\n",
      "  Read 1:    63,183,050 bp\n",
      "  Read 2:    59,159,181 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: CTGTCTCTTATACACATCT; Type: regular 3'; Length: 19; Trimmed: 14099 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 32.3%\n",
      "  C: 24.7%\n",
      "  G: 19.4%\n",
      "  T: 23.6%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t115\t3575.2\t0\t115\n",
      "4\t91\t893.8\t0\t91\n",
      "5\t44\t223.5\t0\t44\n",
      "6\t33\t55.9\t0\t33\n",
      "7\t1\t14.0\t0\t1\n",
      "8\t2\t3.5\t0\t2\n",
      "10\t1\t0.2\t1\t1\n",
      "11\t2\t0.1\t1\t2\n",
      "12\t12\t0.0\t1\t12\n",
      "13\t12\t0.0\t1\t12\n",
      "14\t1\t0.0\t1\t1\n",
      "15\t7\t0.0\t1\t7\n",
      "17\t4\t0.0\t1\t4\n",
      "18\t7\t0.0\t1\t6 1\n",
      "19\t1\t0.0\t1\t1\n",
      "20\t2\t0.0\t1\t2\n",
      "21\t17\t0.0\t1\t14 3\n",
      "23\t4\t0.0\t1\t3 1\n",
      "24\t2\t0.0\t1\t1 1\n",
      "25\t2\t0.0\t1\t2\n",
      "26\t3\t0.0\t1\t3\n",
      "28\t4\t0.0\t1\t3 1\n",
      "29\t8\t0.0\t1\t5 3\n",
      "30\t7\t0.0\t1\t5 2\n",
      "31\t4\t0.0\t1\t4\n",
      "32\t10\t0.0\t1\t10\n",
      "33\t8\t0.0\t1\t6 2\n",
      "34\t14\t0.0\t1\t11 3\n",
      "35\t4\t0.0\t1\t4\n",
      "36\t5\t0.0\t1\t4 1\n",
      "37\t10\t0.0\t1\t9 1\n",
      "38\t7\t0.0\t1\t6 1\n",
      "39\t13\t0.0\t1\t13\n",
      "40\t13\t0.0\t1\t10 3\n",
      "41\t10\t0.0\t1\t6 4\n",
      "42\t27\t0.0\t1\t22 5\n",
      "43\t74\t0.0\t1\t59 15\n",
      "44\t3\t0.0\t1\t1 2\n",
      "45\t11\t0.0\t1\t7 4\n",
      "46\t1\t0.0\t1\t1\n",
      "47\t3\t0.0\t1\t2 1\n",
      "48\t1\t0.0\t1\t1\n",
      "49\t17\t0.0\t1\t16 1\n",
      "50\t2\t0.0\t1\t2\n",
      "51\t10\t0.0\t1\t8 2\n",
      "52\t6\t0.0\t1\t4 2\n",
      "53\t16\t0.0\t1\t13 3\n",
      "54\t39\t0.0\t1\t33 6\n",
      "55\t16\t0.0\t1\t15 1\n",
      "56\t8\t0.0\t1\t6 2\n",
      "57\t37\t0.0\t1\t32 5\n",
      "58\t16\t0.0\t1\t15 1\n",
      "59\t13\t0.0\t1\t10 3\n",
      "60\t23\t0.0\t1\t19 4\n",
      "61\t62\t0.0\t1\t46 16\n",
      "62\t25\t0.0\t1\t23 2\n",
      "63\t24\t0.0\t1\t21 3\n",
      "64\t19\t0.0\t1\t14 5\n",
      "65\t65\t0.0\t1\t49 16\n",
      "66\t42\t0.0\t1\t37 5\n",
      "67\t37\t0.0\t1\t31 6\n",
      "68\t17\t0.0\t1\t16 1\n",
      "69\t26\t0.0\t1\t23 3\n",
      "70\t26\t0.0\t1\t19 7\n",
      "71\t63\t0.0\t1\t54 9\n",
      "72\t66\t0.0\t1\t59 7\n",
      "73\t185\t0.0\t1\t163 22\n",
      "74\t560\t0.0\t1\t468 92\n",
      "75\t887\t0.0\t1\t709 178\n",
      "76\t8253\t0.0\t1\t7091 1162\n",
      "77\t2368\t0.0\t1\t2008 360\n",
      "78\t265\t0.0\t1\t226 39\n",
      "79\t65\t0.0\t1\t55 10\n",
      "80\t55\t0.0\t1\t43 12\n",
      "81\t10\t0.0\t1\t7 3\n",
      "82\t2\t0.0\t1\t2\n",
      "84\t5\t0.0\t1\t4 1\n",
      "85\t1\t0.0\t1\t0 1\n",
      "86\t2\t0.0\t1\t2\n",
      "87\t5\t0.0\t1\t5\n",
      "88\t1\t0.0\t1\t1\n",
      "89\t1\t0.0\t1\t1\n",
      "95\t2\t0.0\t1\t2\n",
      "98\t1\t0.0\t1\t1\n",
      "99\t1\t0.0\t1\t1\n",
      "100\t1\t0.0\t1\t1\n",
      "106\t1\t0.0\t1\t1\n",
      "109\t1\t0.0\t1\t1\n",
      "116\t1\t0.0\t1\t1\n",
      "119\t2\t0.0\t1\t2\n",
      "120\t1\t0.0\t1\t1\n",
      "123\t1\t0.0\t1\t0 1\n",
      "129\t2\t0.0\t1\t2\n",
      "136\t1\t0.0\t1\t1\n",
      "142\t1\t0.0\t1\t1\n",
      "143\t2\t0.0\t1\t2\n",
      "154\t1\t0.0\t1\t1\n",
      "155\t1\t0.0\t1\t1\n",
      "164\t1\t0.0\t1\t1\n",
      "167\t1\t0.0\t1\t1\n",
      "168\t1\t0.0\t1\t1\n",
      "169\t3\t0.0\t1\t2 1\n",
      "170\t1\t0.0\t1\t0 1\n",
      "173\t1\t0.0\t1\t0 1\n",
      "175\t3\t0.0\t1\t2 1\n",
      "178\t1\t0.0\t1\t1\n",
      "179\t1\t0.0\t1\t1\n",
      "181\t1\t0.0\t1\t1\n",
      "185\t1\t0.0\t1\t1\n",
      "193\t1\t0.0\t1\t1\n",
      "198\t1\t0.0\t1\t1\n",
      "204\t1\t0.0\t1\t1\n",
      "205\t1\t0.0\t1\t1\n",
      "206\t1\t0.0\t1\t1\n",
      "207\t1\t0.0\t1\t1\n",
      "208\t2\t0.0\t1\t0 2\n",
      "209\t1\t0.0\t1\t1\n",
      "214\t1\t0.0\t1\t1\n",
      "216\t2\t0.0\t1\t2\n",
      "217\t3\t0.0\t1\t2 1\n",
      "218\t4\t0.0\t1\t3 1\n",
      "219\t11\t0.0\t1\t10 1\n",
      "220\t35\t0.0\t1\t28 7\n",
      "221\t58\t0.0\t1\t41 17\n",
      "224\t1\t0.0\t1\t1\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CTGTCTCTTATACACATCT; Type: regular 3'; Length: 19; Trimmed: 16228 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 29.5%\n",
      "  C: 21.2%\n",
      "  G: 23.9%\n",
      "  T: 25.4%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2276\t3575.2\t0\t2276\n",
      "4\t94\t893.8\t0\t94\n",
      "5\t27\t223.5\t0\t27\n",
      "6\t1\t55.9\t0\t1\n",
      "8\t1\t3.5\t0\t1\n",
      "9\t2\t0.9\t0\t1 1\n",
      "10\t2\t0.2\t1\t2\n",
      "11\t2\t0.1\t1\t2\n",
      "12\t4\t0.0\t1\t4\n",
      "13\t4\t0.0\t1\t4\n",
      "14\t4\t0.0\t1\t4\n",
      "15\t6\t0.0\t1\t6\n",
      "16\t4\t0.0\t1\t4\n",
      "17\t14\t0.0\t1\t13 1\n",
      "18\t13\t0.0\t1\t12 1\n",
      "19\t26\t0.0\t1\t25 1\n",
      "20\t6\t0.0\t1\t6\n",
      "21\t17\t0.0\t1\t16 1\n",
      "22\t15\t0.0\t1\t13 2\n",
      "23\t7\t0.0\t1\t7\n",
      "24\t6\t0.0\t1\t6\n",
      "25\t13\t0.0\t1\t11 2\n",
      "26\t10\t0.0\t1\t10\n",
      "27\t9\t0.0\t1\t9\n",
      "28\t9\t0.0\t1\t7 2\n",
      "29\t23\t0.0\t1\t23\n",
      "30\t19\t0.0\t1\t19\n",
      "31\t24\t0.0\t1\t24\n",
      "32\t23\t0.0\t1\t23\n",
      "33\t47\t0.0\t1\t45 2\n",
      "34\t33\t0.0\t1\t30 3\n",
      "35\t127\t0.0\t1\t124 3\n",
      "36\t57\t0.0\t1\t54 3\n",
      "37\t38\t0.0\t1\t37 1\n",
      "38\t56\t0.0\t1\t51 5\n",
      "39\t60\t0.0\t1\t60\n",
      "40\t53\t0.0\t1\t53\n",
      "41\t103\t0.0\t1\t103\n",
      "42\t122\t0.0\t1\t121 1\n",
      "43\t284\t0.0\t1\t282 2\n",
      "44\t209\t0.0\t1\t206 3\n",
      "45\t125\t0.0\t1\t122 3\n",
      "46\t252\t0.0\t1\t248 4\n",
      "47\t194\t0.0\t1\t192 2\n",
      "48\t176\t0.0\t1\t173 3\n",
      "49\t224\t0.0\t1\t221 3\n",
      "50\t136\t0.0\t1\t134 2\n",
      "51\t82\t0.0\t1\t81 1\n",
      "52\t88\t0.0\t1\t86 2\n",
      "53\t76\t0.0\t1\t74 2\n",
      "54\t226\t0.0\t1\t224 2\n",
      "55\t184\t0.0\t1\t184\n",
      "56\t73\t0.0\t1\t71 2\n",
      "57\t296\t0.0\t1\t292 4\n",
      "58\t229\t0.0\t1\t228 1\n",
      "59\t180\t0.0\t1\t178 2\n",
      "60\t141\t0.0\t1\t137 4\n",
      "61\t72\t0.0\t1\t72\n",
      "62\t178\t0.0\t1\t177 1\n",
      "63\t475\t0.0\t1\t470 5\n",
      "64\t45\t0.0\t1\t42 3\n",
      "65\t40\t0.0\t1\t40\n",
      "66\t21\t0.0\t1\t21\n",
      "67\t26\t0.0\t1\t26\n",
      "68\t11\t0.0\t1\t11\n",
      "69\t14\t0.0\t1\t14\n",
      "70\t13\t0.0\t1\t13\n",
      "71\t1\t0.0\t1\t1\n",
      "72\t2\t0.0\t1\t2\n",
      "73\t6\t0.0\t1\t6\n",
      "74\t8\t0.0\t1\t8\n",
      "75\t58\t0.0\t1\t55 3\n",
      "76\t46\t0.0\t1\t43 3\n",
      "77\t83\t0.0\t1\t83\n",
      "78\t431\t0.0\t1\t423 8\n",
      "79\t530\t0.0\t1\t512 18\n",
      "80\t3287\t0.0\t1\t3258 29\n",
      "81\t3411\t0.0\t1\t3380 31\n",
      "82\t767\t0.0\t1\t747 20\n",
      "83\t131\t0.0\t1\t131\n",
      "84\t29\t0.0\t1\t28 1\n",
      "85\t6\t0.0\t1\t6\n",
      "86\t1\t0.0\t1\t1\n",
      "215\t1\t0.0\t1\t1\n",
      "220\t1\t0.0\t1\t1\n",
      "221\t2\t0.0\t1\t2\n",
      "\n",
      "Read1 before filtering:\n",
      "total reads: 228815\n",
      "total bases: 63183050\n",
      "Q20 bases: 62885703(99.5294%)\n",
      "Q30 bases: 61961144(98.0661%)\n",
      "\n",
      "Read2 before filtering:\n",
      "total reads: 228815\n",
      "total bases: 59159181\n",
      "Q20 bases: 58324782(98.5896%)\n",
      "Q30 bases: 56093973(94.8187%)\n",
      "\n",
      "Merged and filtered:\n",
      "total reads: 221818\n",
      "total bases: 81294102\n",
      "Q20 bases: 81049070(99.6986%)\n",
      "Q30 bases: 80147958(98.5901%)\n",
      "\n",
      "Filtering result:\n",
      "reads passed filter: 452882\n",
      "reads failed due to low quality: 242\n",
      "reads failed due to too many N: 18\n",
      "reads failed due to too short: 4488\n",
      "reads with adapter trimmed: 27700\n",
      "bases trimmed due to adapters: 225489\n",
      "reads corrected by overlap analysis: 32103\n",
      "bases corrected by overlap analysis: 43142\n",
      "\n",
      "Duplication rate: 63.2735%\n",
      "\n",
      "Insert size peak (evaluated by paired-end reads): 387\n",
      "\n",
      "Read pairs merged: 221818\n",
      "% of original read pairs: 96.9421%\n",
      "% in reads after filtering: 100%\n",
      "\n",
      "\n",
      "JSON report: 2_fastp/Hrd1_DMS_Rep3_Region4_Input_report.json\n",
      "HTML report: 2_fastp/Hrd1_DMS_Rep3_Region4_Input_report.html\n",
      "\n",
      "fastp -i 1_cutadapt/Hrd1_DMS_Rep3_Region4_Input_R1_cut.fastq.gz -I 1_cutadapt/Hrd1_DMS_Rep3_Region4_Input_R2_cut.fastq.gz -m --merged_out 2_fastp/Hrd1_DMS_Rep3_Region4_Input_merge.fastq.gz -o 2_fastp/Hrd1_DMS_Rep3_Region4_Input_R1_cut.fastq.gz -O 2_fastp/Hrd1_DMS_Rep3_Region4_Input_R2_cut.fastq.gz -w 12 -h 2_fastp/Hrd1_DMS_Rep3_Region4_Input_report.html -j 2_fastp/Hrd1_DMS_Rep3_Region4_Input_report.json \n",
      "fastp v0.20.0, time used: 2 seconds\n",
      "221818 reads; of these:\n",
      "  221818 (100.00%) were unpaired; of these:\n",
      "    13813 (6.23%) aligned 0 times\n",
      "    208005 (93.77%) aligned exactly 1 time\n",
      "    0 (0.00%) aligned >1 times\n",
      "93.77% overall alignment rate\n",
      "[bam_sort_core] merging from 0 files and 12 in-memory blocks...\n",
      "This is cutadapt 1.18 with Python 3.7.16\n",
      "Command line parameters: -a CTGTCTCTTATACACATCT -A CTGTCTCTTATACACATCT -u 8 -U 8 -q 30 -o 1_cutadapt/Hrd1_DMS_Rep4_Region4_Input_R1_cut.fastq.gz -p 1_cutadapt/Hrd1_DMS_Rep4_Region4_Input_R2_cut.fastq.gz --cores=12 0_fastq_files/Hrd1_DMS_Rep4_Region4_Input_R1_001.fastq.gz 0_fastq_files/Hrd1_DMS_Rep4_Region4_Input_R2_001.fastq.gz\n",
      "Processing reads on 12 cores in paired-end mode ...\n",
      "Finished in 1.97 s (9 us/read; 6.49 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            213,579\n",
      "  Read 1 with adapter:                   7,419 (3.5%)\n",
      "  Read 2 with adapter:                   8,378 (3.9%)\n",
      "Pairs written (passing filters):       213,579 (100.0%)\n",
      "\n",
      "Total basepairs processed:   127,733,158 bp\n",
      "  Read 1:    63,864,899 bp\n",
      "  Read 2:    63,868,259 bp\n",
      "Quality-trimmed:               5,835,541 bp (4.6%)\n",
      "  Read 1:     1,740,177 bp\n",
      "  Read 2:     4,095,364 bp\n",
      "Total written (filtered):    117,413,382 bp (91.9%)\n",
      "  Read 1:    59,869,380 bp\n",
      "  Read 2:    57,544,002 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: CTGTCTCTTATACACATCT; Type: regular 3'; Length: 19; Trimmed: 7419 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 31.2%\n",
      "  C: 26.2%\n",
      "  G: 19.3%\n",
      "  T: 23.3%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t164\t3337.2\t0\t164\n",
      "4\t24\t834.3\t0\t24\n",
      "5\t2\t208.6\t0\t2\n",
      "8\t2\t3.3\t0\t2\n",
      "11\t1\t0.1\t1\t1\n",
      "12\t1\t0.0\t1\t1\n",
      "14\t2\t0.0\t1\t1 1\n",
      "15\t1\t0.0\t1\t1\n",
      "17\t3\t0.0\t1\t3\n",
      "18\t3\t0.0\t1\t3\n",
      "19\t1\t0.0\t1\t1\n",
      "20\t5\t0.0\t1\t5\n",
      "21\t7\t0.0\t1\t7\n",
      "22\t6\t0.0\t1\t6\n",
      "23\t4\t0.0\t1\t4\n",
      "24\t3\t0.0\t1\t2 1\n",
      "26\t3\t0.0\t1\t3\n",
      "27\t5\t0.0\t1\t5\n",
      "28\t1\t0.0\t1\t0 1\n",
      "29\t12\t0.0\t1\t9 3\n",
      "30\t4\t0.0\t1\t4\n",
      "31\t3\t0.0\t1\t3\n",
      "32\t1\t0.0\t1\t1\n",
      "33\t3\t0.0\t1\t2 1\n",
      "34\t6\t0.0\t1\t6\n",
      "35\t1\t0.0\t1\t1\n",
      "36\t6\t0.0\t1\t6\n",
      "37\t2\t0.0\t1\t2\n",
      "38\t6\t0.0\t1\t5 1\n",
      "39\t8\t0.0\t1\t5 3\n",
      "40\t10\t0.0\t1\t10\n",
      "41\t4\t0.0\t1\t4\n",
      "42\t7\t0.0\t1\t5 2\n",
      "43\t14\t0.0\t1\t11 3\n",
      "44\t4\t0.0\t1\t4\n",
      "45\t1\t0.0\t1\t1\n",
      "47\t2\t0.0\t1\t2\n",
      "50\t2\t0.0\t1\t2\n",
      "51\t1\t0.0\t1\t1\n",
      "52\t1\t0.0\t1\t1\n",
      "53\t8\t0.0\t1\t8\n",
      "54\t18\t0.0\t1\t18\n",
      "55\t16\t0.0\t1\t16\n",
      "56\t6\t0.0\t1\t5 1\n",
      "57\t15\t0.0\t1\t14 1\n",
      "58\t6\t0.0\t1\t6\n",
      "59\t3\t0.0\t1\t3\n",
      "60\t8\t0.0\t1\t8\n",
      "61\t19\t0.0\t1\t18 1\n",
      "62\t6\t0.0\t1\t6\n",
      "63\t15\t0.0\t1\t15\n",
      "64\t37\t0.0\t1\t34 3\n",
      "65\t56\t0.0\t1\t53 3\n",
      "66\t35\t0.0\t1\t35\n",
      "67\t19\t0.0\t1\t19\n",
      "68\t48\t0.0\t1\t48\n",
      "69\t41\t0.0\t1\t41\n",
      "70\t21\t0.0\t1\t20 1\n",
      "71\t24\t0.0\t1\t23 1\n",
      "72\t53\t0.0\t1\t49 4\n",
      "73\t54\t0.0\t1\t53 1\n",
      "74\t114\t0.0\t1\t111 3\n",
      "75\t296\t0.0\t1\t268 28\n",
      "76\t4667\t0.0\t1\t4627 40\n",
      "77\t1250\t0.0\t1\t1228 22\n",
      "78\t148\t0.0\t1\t146 2\n",
      "79\t19\t0.0\t1\t19\n",
      "80\t6\t0.0\t1\t6\n",
      "81\t2\t0.0\t1\t2\n",
      "82\t2\t0.0\t1\t2\n",
      "83\t2\t0.0\t1\t2\n",
      "84\t1\t0.0\t1\t1\n",
      "85\t2\t0.0\t1\t2\n",
      "86\t1\t0.0\t1\t1\n",
      "87\t2\t0.0\t1\t2\n",
      "93\t1\t0.0\t1\t0 1\n",
      "97\t2\t0.0\t1\t2\n",
      "99\t1\t0.0\t1\t1\n",
      "100\t1\t0.0\t1\t1\n",
      "103\t2\t0.0\t1\t1 1\n",
      "104\t1\t0.0\t1\t1\n",
      "105\t1\t0.0\t1\t1\n",
      "114\t1\t0.0\t1\t1\n",
      "115\t1\t0.0\t1\t1\n",
      "117\t2\t0.0\t1\t2\n",
      "118\t1\t0.0\t1\t1\n",
      "137\t1\t0.0\t1\t1\n",
      "138\t1\t0.0\t1\t1\n",
      "140\t1\t0.0\t1\t1\n",
      "142\t1\t0.0\t1\t1\n",
      "154\t1\t0.0\t1\t1\n",
      "157\t1\t0.0\t1\t1\n",
      "158\t1\t0.0\t1\t1\n",
      "160\t2\t0.0\t1\t2\n",
      "172\t1\t0.0\t1\t1\n",
      "175\t1\t0.0\t1\t1\n",
      "187\t2\t0.0\t1\t2\n",
      "192\t1\t0.0\t1\t1\n",
      "193\t1\t0.0\t1\t1\n",
      "194\t1\t0.0\t1\t0 1\n",
      "203\t1\t0.0\t1\t1\n",
      "220\t1\t0.0\t1\t0 1\n",
      "226\t1\t0.0\t1\t1\n",
      "228\t1\t0.0\t1\t1\n",
      "229\t2\t0.0\t1\t2\n",
      "230\t1\t0.0\t1\t0 1\n",
      "231\t15\t0.0\t1\t14 1\n",
      "232\t11\t0.0\t1\t9 2\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CTGTCTCTTATACACATCT; Type: regular 3'; Length: 19; Trimmed: 8378 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 29.5%\n",
      "  C: 32.9%\n",
      "  G: 16.3%\n",
      "  T: 21.3%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t277\t3337.2\t0\t277\n",
      "4\t400\t834.3\t0\t400\n",
      "5\t241\t208.6\t0\t241\n",
      "6\t258\t52.1\t0\t258\n",
      "9\t2\t0.8\t0\t1 1\n",
      "12\t1\t0.0\t1\t1\n",
      "13\t2\t0.0\t1\t2\n",
      "14\t1\t0.0\t1\t1\n",
      "15\t2\t0.0\t1\t1 1\n",
      "16\t3\t0.0\t1\t3\n",
      "17\t4\t0.0\t1\t4\n",
      "19\t2\t0.0\t1\t2\n",
      "20\t1\t0.0\t1\t1\n",
      "21\t7\t0.0\t1\t6 1\n",
      "22\t2\t0.0\t1\t2\n",
      "23\t1\t0.0\t1\t1\n",
      "24\t3\t0.0\t1\t3\n",
      "26\t1\t0.0\t1\t1\n",
      "27\t2\t0.0\t1\t2\n",
      "28\t5\t0.0\t1\t4 1\n",
      "29\t3\t0.0\t1\t2 1\n",
      "30\t2\t0.0\t1\t1 1\n",
      "31\t13\t0.0\t1\t13\n",
      "32\t12\t0.0\t1\t11 1\n",
      "33\t18\t0.0\t1\t15 3\n",
      "34\t14\t0.0\t1\t14\n",
      "35\t42\t0.0\t1\t39 3\n",
      "36\t23\t0.0\t1\t22 1\n",
      "37\t15\t0.0\t1\t15\n",
      "38\t21\t0.0\t1\t21\n",
      "39\t32\t0.0\t1\t30 2\n",
      "40\t37\t0.0\t1\t37\n",
      "41\t57\t0.0\t1\t56 1\n",
      "42\t50\t0.0\t1\t49 1\n",
      "43\t69\t0.0\t1\t68 1\n",
      "44\t44\t0.0\t1\t44\n",
      "45\t52\t0.0\t1\t50 2\n",
      "46\t121\t0.0\t1\t119 2\n",
      "47\t73\t0.0\t1\t70 3\n",
      "48\t163\t0.0\t1\t157 6\n",
      "49\t105\t0.0\t1\t103 2\n",
      "50\t67\t0.0\t1\t67\n",
      "51\t38\t0.0\t1\t37 1\n",
      "52\t56\t0.0\t1\t55 1\n",
      "53\t45\t0.0\t1\t44 1\n",
      "54\t128\t0.0\t1\t126 2\n",
      "55\t68\t0.0\t1\t67 1\n",
      "56\t74\t0.0\t1\t72 2\n",
      "57\t209\t0.0\t1\t207 2\n",
      "58\t86\t0.0\t1\t86\n",
      "59\t71\t0.0\t1\t70 1\n",
      "60\t54\t0.0\t1\t54\n",
      "61\t26\t0.0\t1\t26\n",
      "62\t59\t0.0\t1\t56 3\n",
      "63\t95\t0.0\t1\t95\n",
      "64\t23\t0.0\t1\t23\n",
      "65\t24\t0.0\t1\t24\n",
      "66\t9\t0.0\t1\t9\n",
      "67\t11\t0.0\t1\t11\n",
      "68\t3\t0.0\t1\t3\n",
      "69\t8\t0.0\t1\t8\n",
      "70\t10\t0.0\t1\t10\n",
      "72\t1\t0.0\t1\t1\n",
      "74\t3\t0.0\t1\t3\n",
      "75\t9\t0.0\t1\t9\n",
      "76\t25\t0.0\t1\t25\n",
      "77\t38\t0.0\t1\t37 1\n",
      "78\t129\t0.0\t1\t126 3\n",
      "79\t210\t0.0\t1\t204 6\n",
      "80\t1740\t0.0\t1\t1709 31\n",
      "81\t2366\t0.0\t1\t2335 31\n",
      "82\t445\t0.0\t1\t441 4\n",
      "83\t52\t0.0\t1\t52\n",
      "84\t12\t0.0\t1\t12\n",
      "85\t1\t0.0\t1\t1\n",
      "86\t1\t0.0\t1\t1\n",
      "231\t1\t0.0\t1\t1\n",
      "\n",
      "Read1 before filtering:\n",
      "total reads: 213579\n",
      "total bases: 59869380\n",
      "Q20 bases: 59584236(99.5237%)\n",
      "Q30 bases: 58719148(98.0788%)\n",
      "\n",
      "Read2 before filtering:\n",
      "total reads: 213579\n",
      "total bases: 57544002\n",
      "Q20 bases: 56846239(98.7874%)\n",
      "Q30 bases: 54943874(95.4815%)\n",
      "\n",
      "Merged and filtered:\n",
      "total reads: 208410\n",
      "total bases: 75818556\n",
      "Q20 bases: 75573294(99.6765%)\n",
      "Q30 bases: 74709685(98.5375%)\n",
      "\n",
      "Filtering result:\n",
      "reads passed filter: 423568\n",
      "reads failed due to low quality: 398\n",
      "reads failed due to too many N: 26\n",
      "reads failed due to too short: 3166\n",
      "reads with adapter trimmed: 14428\n",
      "bases trimmed due to adapters: 116954\n",
      "reads corrected by overlap analysis: 37037\n",
      "bases corrected by overlap analysis: 49982\n",
      "\n",
      "Duplication rate: 46.4497%\n",
      "\n",
      "Insert size peak (evaluated by paired-end reads): 376\n",
      "\n",
      "Read pairs merged: 208410\n",
      "% of original read pairs: 97.5798%\n",
      "% in reads after filtering: 100%\n",
      "\n",
      "\n",
      "JSON report: 2_fastp/Hrd1_DMS_Rep4_Region4_Input_report.json\n",
      "HTML report: 2_fastp/Hrd1_DMS_Rep4_Region4_Input_report.html\n",
      "\n",
      "fastp -i 1_cutadapt/Hrd1_DMS_Rep4_Region4_Input_R1_cut.fastq.gz -I 1_cutadapt/Hrd1_DMS_Rep4_Region4_Input_R2_cut.fastq.gz -m --merged_out 2_fastp/Hrd1_DMS_Rep4_Region4_Input_merge.fastq.gz -o 2_fastp/Hrd1_DMS_Rep4_Region4_Input_R1_cut.fastq.gz -O 2_fastp/Hrd1_DMS_Rep4_Region4_Input_R2_cut.fastq.gz -w 12 -h 2_fastp/Hrd1_DMS_Rep4_Region4_Input_report.html -j 2_fastp/Hrd1_DMS_Rep4_Region4_Input_report.json \n",
      "fastp v0.20.0, time used: 2 seconds\n",
      "208410 reads; of these:\n",
      "  208410 (100.00%) were unpaired; of these:\n",
      "    7220 (3.46%) aligned 0 times\n",
      "    201190 (96.54%) aligned exactly 1 time\n",
      "    0 (0.00%) aligned >1 times\n",
      "96.54% overall alignment rate\n",
      "[bam_sort_core] merging from 0 files and 12 in-memory blocks...\n",
      "This is cutadapt 1.18 with Python 3.7.16\n",
      "Command line parameters: -a CTGTCTCTTATACACATCT -A CTGTCTCTTATACACATCT -u 8 -U 8 -q 30 -o 1_cutadapt/Hrd1_DMS_Rep1_Region4_WT_R1_cut.fastq.gz -p 1_cutadapt/Hrd1_DMS_Rep1_Region4_WT_R2_cut.fastq.gz --cores=12 0_fastq_files/Hrd1_DMS_Rep1_Region4_WT_R1_001.fastq.gz 0_fastq_files/Hrd1_DMS_Rep1_Region4_WT_R2_001.fastq.gz\n",
      "Processing reads on 12 cores in paired-end mode ...\n",
      "Finished in 1.86 s (9 us/read; 6.73 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            208,928\n",
      "  Read 1 with adapter:                      96 (0.0%)\n",
      "  Read 2 with adapter:                   1,173 (0.6%)\n",
      "Pairs written (passing filters):       208,928 (100.0%)\n",
      "\n",
      "Total basepairs processed:   124,946,872 bp\n",
      "  Read 1:    62,473,792 bp\n",
      "  Read 2:    62,473,080 bp\n",
      "Quality-trimmed:               4,170,621 bp (3.3%)\n",
      "  Read 1:       392,831 bp\n",
      "  Read 2:     3,777,790 bp\n",
      "Total written (filtered):    117,427,794 bp (94.0%)\n",
      "  Read 1:    60,408,263 bp\n",
      "  Read 2:    57,019,531 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: CTGTCTCTTATACACATCT; Type: regular 3'; Length: 19; Trimmed: 96 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 8.3%\n",
      "  C: 75.0%\n",
      "  G: 2.1%\n",
      "  T: 14.6%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t26\t3264.5\t0\t26\n",
      "4\t25\t816.1\t0\t25\n",
      "5\t12\t204.0\t0\t12\n",
      "6\t12\t51.0\t0\t12\n",
      "7\t8\t12.8\t0\t8\n",
      "11\t1\t0.0\t1\t1\n",
      "62\t1\t0.0\t1\t1\n",
      "74\t1\t0.0\t1\t1\n",
      "75\t1\t0.0\t1\t1\n",
      "76\t7\t0.0\t1\t7\n",
      "77\t2\t0.0\t1\t2\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CTGTCTCTTATACACATCT; Type: regular 3'; Length: 19; Trimmed: 1173 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 0.4%\n",
      "  C: 0.5%\n",
      "  G: 50.6%\n",
      "  T: 48.4%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1088\t3264.5\t0\t1088\n",
      "4\t65\t816.1\t0\t65\n",
      "5\t8\t204.0\t0\t8\n",
      "11\t1\t0.0\t1\t1\n",
      "14\t1\t0.0\t1\t1\n",
      "43\t1\t0.0\t1\t1\n",
      "48\t1\t0.0\t1\t1\n",
      "78\t1\t0.0\t1\t1\n",
      "80\t1\t0.0\t1\t1\n",
      "81\t4\t0.0\t1\t4\n",
      "82\t1\t0.0\t1\t1\n",
      "91\t1\t0.0\t1\t1\n",
      "\n",
      "Read1 before filtering:\n",
      "total reads: 208928\n",
      "total bases: 60408263\n",
      "Q20 bases: 60128845(99.5375%)\n",
      "Q30 bases: 59229661(98.0489%)\n",
      "\n",
      "Read2 before filtering:\n",
      "total reads: 208928\n",
      "total bases: 57019531\n",
      "Q20 bases: 56276204(98.6964%)\n",
      "Q30 bases: 54278899(95.1935%)\n",
      "\n",
      "Merged and filtered:\n",
      "total reads: 203137\n",
      "total bases: 76322946\n",
      "Q20 bases: 76102161(99.7107%)\n",
      "Q30 bases: 75244612(98.5871%)\n",
      "\n",
      "Filtering result:\n",
      "reads passed filter: 413740\n",
      "reads failed due to low quality: 234\n",
      "reads failed due to too many N: 38\n",
      "reads failed due to too short: 3844\n",
      "reads with adapter trimmed: 32\n",
      "bases trimmed due to adapters: 1085\n",
      "reads corrected by overlap analysis: 28179\n",
      "bases corrected by overlap analysis: 38105\n",
      "\n",
      "Duplication rate: 39.7959%\n",
      "\n",
      "Insert size peak (evaluated by paired-end reads): 376\n",
      "\n",
      "Read pairs merged: 203137\n",
      "% of original read pairs: 97.2282%\n",
      "% in reads after filtering: 100%\n",
      "\n",
      "\n",
      "JSON report: 2_fastp/Hrd1_DMS_Rep1_Region4_WT_report.json\n",
      "HTML report: 2_fastp/Hrd1_DMS_Rep1_Region4_WT_report.html\n",
      "\n",
      "fastp -i 1_cutadapt/Hrd1_DMS_Rep1_Region4_WT_R1_cut.fastq.gz -I 1_cutadapt/Hrd1_DMS_Rep1_Region4_WT_R2_cut.fastq.gz -m --merged_out 2_fastp/Hrd1_DMS_Rep1_Region4_WT_merge.fastq.gz -o 2_fastp/Hrd1_DMS_Rep1_Region4_WT_R1_cut.fastq.gz -O 2_fastp/Hrd1_DMS_Rep1_Region4_WT_R2_cut.fastq.gz -w 12 -h 2_fastp/Hrd1_DMS_Rep1_Region4_WT_report.html -j 2_fastp/Hrd1_DMS_Rep1_Region4_WT_report.json \n",
      "fastp v0.20.0, time used: 2 seconds\n",
      "203137 reads; of these:\n",
      "  203137 (100.00%) were unpaired; of these:\n",
      "    21 (0.01%) aligned 0 times\n",
      "    203116 (99.99%) aligned exactly 1 time\n",
      "    0 (0.00%) aligned >1 times\n",
      "99.99% overall alignment rate\n",
      "[bam_sort_core] merging from 0 files and 12 in-memory blocks...\n",
      "This is cutadapt 1.18 with Python 3.7.16\n",
      "Command line parameters: -a CTGTCTCTTATACACATCT -A CTGTCTCTTATACACATCT -u 8 -U 8 -q 30 -o 1_cutadapt/Hrd1_DMS_Rep3_Region4_WT_R1_cut.fastq.gz -p 1_cutadapt/Hrd1_DMS_Rep3_Region4_WT_R2_cut.fastq.gz --cores=12 0_fastq_files/Hrd1_DMS_Rep3_Region4_WT_R1_001.fastq.gz 0_fastq_files/Hrd1_DMS_Rep3_Region4_WT_R2_001.fastq.gz\n",
      "Processing reads on 12 cores in paired-end mode ...\n",
      "Finished in 1.81 s (11 us/read; 5.59 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            168,146\n",
      "  Read 1 with adapter:                     302 (0.2%)\n",
      "  Read 2 with adapter:                   1,738 (1.0%)\n",
      "Pairs written (passing filters):       168,146 (100.0%)\n",
      "\n",
      "Total basepairs processed:   100,720,746 bp\n",
      "  Read 1:    50,439,847 bp\n",
      "  Read 2:    50,280,899 bp\n",
      "Quality-trimmed:               3,514,741 bp (3.5%)\n",
      "  Read 1:       670,580 bp\n",
      "  Read 2:     2,844,161 bp\n",
      "Total written (filtered):     94,507,387 bp (93.8%)\n",
      "  Read 1:    48,421,948 bp\n",
      "  Read 2:    46,085,439 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: CTGTCTCTTATACACATCT; Type: regular 3'; Length: 19; Trimmed: 302 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 9.6%\n",
      "  C: 88.7%\n",
      "  G: 0.7%\n",
      "  T: 1.0%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by \"C\" extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 3' end.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t73\t2627.3\t0\t73\n",
      "4\t103\t656.8\t0\t103\n",
      "5\t72\t164.2\t0\t72\n",
      "6\t42\t41.1\t0\t42\n",
      "70\t1\t0.0\t1\t1\n",
      "74\t1\t0.0\t1\t1\n",
      "76\t7\t0.0\t1\t7\n",
      "77\t2\t0.0\t1\t2\n",
      "78\t1\t0.0\t1\t1\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CTGTCTCTTATACACATCT; Type: regular 3'; Length: 19; Trimmed: 1738 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1670\t2627.3\t0\t1670\n",
      "4\t44\t656.8\t0\t44\n",
      "5\t11\t164.2\t0\t11\n",
      "10\t1\t0.2\t1\t1\n",
      "58\t2\t0.0\t1\t2\n",
      "62\t1\t0.0\t1\t1\n",
      "63\t1\t0.0\t1\t1\n",
      "78\t1\t0.0\t1\t1\n",
      "79\t1\t0.0\t1\t1\n",
      "80\t4\t0.0\t1\t4\n",
      "81\t2\t0.0\t1\t2\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "Read1 before filtering:\n",
      "total reads: 168146\n",
      "total bases: 48421948\n",
      "Q20 bases: 48067949(99.2689%)\n",
      "Q30 bases: 47020380(97.1055%)\n",
      "\n",
      "Read2 before filtering:\n",
      "total reads: 168146\n",
      "total bases: 46085439\n",
      "Q20 bases: 45469253(98.6629%)\n",
      "Q30 bases: 43814089(95.0714%)\n",
      "\n",
      "Merged and filtered:\n",
      "total reads: 163042\n",
      "total bases: 63049163\n",
      "Q20 bases: 62743020(99.5144%)\n",
      "Q30 bases: 61707982(97.8728%)\n",
      "\n",
      "Filtering result:\n",
      "reads passed filter: 333430\n",
      "reads failed due to low quality: 258\n",
      "reads failed due to too many N: 10\n",
      "reads failed due to too short: 2594\n",
      "reads with adapter trimmed: 32\n",
      "bases trimmed due to adapters: 1022\n",
      "reads corrected by overlap analysis: 28117\n",
      "bases corrected by overlap analysis: 38257\n",
      "\n",
      "Duplication rate: 67.6955%\n",
      "\n",
      "Insert size peak (evaluated by paired-end reads): 387\n",
      "\n",
      "Read pairs merged: 163042\n",
      "% of original read pairs: 96.9645%\n",
      "% in reads after filtering: 100%\n",
      "\n",
      "\n",
      "JSON report: 2_fastp/Hrd1_DMS_Rep3_Region4_WT_report.json\n",
      "HTML report: 2_fastp/Hrd1_DMS_Rep3_Region4_WT_report.html\n",
      "\n",
      "fastp -i 1_cutadapt/Hrd1_DMS_Rep3_Region4_WT_R1_cut.fastq.gz -I 1_cutadapt/Hrd1_DMS_Rep3_Region4_WT_R2_cut.fastq.gz -m --merged_out 2_fastp/Hrd1_DMS_Rep3_Region4_WT_merge.fastq.gz -o 2_fastp/Hrd1_DMS_Rep3_Region4_WT_R1_cut.fastq.gz -O 2_fastp/Hrd1_DMS_Rep3_Region4_WT_R2_cut.fastq.gz -w 12 -h 2_fastp/Hrd1_DMS_Rep3_Region4_WT_report.html -j 2_fastp/Hrd1_DMS_Rep3_Region4_WT_report.json \n",
      "fastp v0.20.0, time used: 2 seconds\n",
      "163042 reads; of these:\n",
      "  163042 (100.00%) were unpaired; of these:\n",
      "    27 (0.02%) aligned 0 times\n",
      "    163015 (99.98%) aligned exactly 1 time\n",
      "    0 (0.00%) aligned >1 times\n",
      "99.98% overall alignment rate\n",
      "[bam_sort_core] merging from 0 files and 12 in-memory blocks...\n",
      "This is cutadapt 1.18 with Python 3.7.16\n",
      "Command line parameters: -a CTGTCTCTTATACACATCT -A CTGTCTCTTATACACATCT -u 8 -U 8 -q 30 -o 1_cutadapt/Hrd1_DMS_Rep4_Region4_WT_R1_cut.fastq.gz -p 1_cutadapt/Hrd1_DMS_Rep4_Region4_WT_R2_cut.fastq.gz --cores=12 0_fastq_files/Hrd1_DMS_Rep4_Region4_WT_R1_001.fastq.gz 0_fastq_files/Hrd1_DMS_Rep4_Region4_WT_R2_001.fastq.gz\n",
      "Processing reads on 12 cores in paired-end mode ...\n",
      "Finished in 1.55 s (11 us/read; 5.28 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            136,787\n",
      "  Read 1 with adapter:                     237 (0.2%)\n",
      "  Read 2 with adapter:                     839 (0.6%)\n",
      "Pairs written (passing filters):       136,787 (100.0%)\n",
      "\n",
      "Total basepairs processed:    81,801,622 bp\n",
      "  Read 1:    40,900,133 bp\n",
      "  Read 2:    40,901,489 bp\n",
      "Quality-trimmed:               2,425,256 bp (3.0%)\n",
      "  Read 1:       571,142 bp\n",
      "  Read 2:     1,854,114 bp\n",
      "Total written (filtered):     77,177,928 bp (94.3%)\n",
      "  Read 1:    39,231,277 bp\n",
      "  Read 2:    37,946,651 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: CTGTCTCTTATACACATCT; Type: regular 3'; Length: 19; Trimmed: 237 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 6.3%\n",
      "  C: 3.8%\n",
      "  G: 40.5%\n",
      "  T: 49.4%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t174\t2137.3\t0\t174\n",
      "4\t23\t534.3\t0\t23\n",
      "5\t2\t133.6\t0\t2\n",
      "17\t1\t0.0\t1\t1\n",
      "54\t1\t0.0\t1\t1\n",
      "59\t1\t0.0\t1\t1\n",
      "74\t2\t0.0\t1\t2\n",
      "75\t2\t0.0\t1\t2\n",
      "76\t22\t0.0\t1\t22\n",
      "77\t8\t0.0\t1\t8\n",
      "78\t1\t0.0\t1\t1\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CTGTCTCTTATACACATCT; Type: regular 3'; Length: 19; Trimmed: 839 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 3.7%\n",
      "  C: 93.4%\n",
      "  G: 1.2%\n",
      "  T: 1.7%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by \"C\" extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 3' end.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t196\t2137.3\t0\t196\n",
      "4\t279\t534.3\t0\t279\n",
      "5\t159\t133.6\t0\t159\n",
      "6\t166\t33.4\t0\t166\n",
      "7\t1\t8.3\t0\t1\n",
      "38\t1\t0.0\t1\t1\n",
      "54\t1\t0.0\t1\t1\n",
      "56\t1\t0.0\t1\t1\n",
      "59\t1\t0.0\t1\t1\n",
      "76\t1\t0.0\t1\t1\n",
      "78\t3\t0.0\t1\t2 1\n",
      "79\t4\t0.0\t1\t4\n",
      "80\t18\t0.0\t1\t18\n",
      "81\t5\t0.0\t1\t5\n",
      "82\t2\t0.0\t1\t2\n",
      "84\t1\t0.0\t1\t1\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "Read1 before filtering:\n",
      "total reads: 136787\n",
      "total bases: 39231277\n",
      "Q20 bases: 38972780(99.3411%)\n",
      "Q30 bases: 38205274(97.3847%)\n",
      "\n",
      "Read2 before filtering:\n",
      "total reads: 136787\n",
      "total bases: 37946651\n",
      "Q20 bases: 37471693(98.7484%)\n",
      "Q30 bases: 36180536(95.3458%)\n",
      "\n",
      "Merged and filtered:\n",
      "total reads: 133277\n",
      "total bases: 49988946\n",
      "Q20 bases: 49758656(99.5393%)\n",
      "Q30 bases: 48985359(97.9924%)\n",
      "\n",
      "Filtering result:\n",
      "reads passed filter: 271400\n",
      "reads failed due to low quality: 300\n",
      "reads failed due to too many N: 20\n",
      "reads failed due to too short: 1854\n",
      "reads with adapter trimmed: 168\n",
      "bases trimmed due to adapters: 2628\n",
      "reads corrected by overlap analysis: 27907\n",
      "bases corrected by overlap analysis: 38611\n",
      "\n",
      "Duplication rate: 79.1885%\n",
      "\n",
      "Insert size peak (evaluated by paired-end reads): 376\n",
      "\n",
      "Read pairs merged: 133277\n",
      "% of original read pairs: 97.434%\n",
      "% in reads after filtering: 100%\n",
      "\n",
      "\n",
      "JSON report: 2_fastp/Hrd1_DMS_Rep4_Region4_WT_report.json\n",
      "HTML report: 2_fastp/Hrd1_DMS_Rep4_Region4_WT_report.html\n",
      "\n",
      "fastp -i 1_cutadapt/Hrd1_DMS_Rep4_Region4_WT_R1_cut.fastq.gz -I 1_cutadapt/Hrd1_DMS_Rep4_Region4_WT_R2_cut.fastq.gz -m --merged_out 2_fastp/Hrd1_DMS_Rep4_Region4_WT_merge.fastq.gz -o 2_fastp/Hrd1_DMS_Rep4_Region4_WT_R1_cut.fastq.gz -O 2_fastp/Hrd1_DMS_Rep4_Region4_WT_R2_cut.fastq.gz -w 12 -h 2_fastp/Hrd1_DMS_Rep4_Region4_WT_report.html -j 2_fastp/Hrd1_DMS_Rep4_Region4_WT_report.json \n",
      "fastp v0.20.0, time used: 2 seconds\n",
      "133277 reads; of these:\n",
      "  133277 (100.00%) were unpaired; of these:\n",
      "    49 (0.04%) aligned 0 times\n",
      "    133228 (99.96%) aligned exactly 1 time\n",
      "    0 (0.00%) aligned >1 times\n",
      "99.96% overall alignment rate\n",
      "[bam_sort_core] merging from 0 files and 12 in-memory blocks...\n",
      "This is cutadapt 1.18 with Python 3.7.16\n",
      "Command line parameters: -a CTGTCTCTTATACACATCT -A CTGTCTCTTATACACATCT -u 8 -U 8 -q 30 -o 1_cutadapt/Hrd1_DMS_Rep1_Region4_LDead_R1_cut.fastq.gz -p 1_cutadapt/Hrd1_DMS_Rep1_Region4_LDead_R2_cut.fastq.gz --cores=12 0_fastq_files/Hrd1_DMS_Rep1_Region4_LDead_R1_001.fastq.gz 0_fastq_files/Hrd1_DMS_Rep1_Region4_LDead_R2_001.fastq.gz\n",
      "Processing reads on 12 cores in paired-end mode ...\n",
      "Finished in 0.36 s (17 us/read; 3.61 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             21,973\n",
      "  Read 1 with adapter:                     319 (1.5%)\n",
      "  Read 2 with adapter:                     187 (0.9%)\n",
      "Pairs written (passing filters):        21,973 (100.0%)\n",
      "\n",
      "Total basepairs processed:    13,142,374 bp\n",
      "  Read 1:     6,570,915 bp\n",
      "  Read 2:     6,571,459 bp\n",
      "Quality-trimmed:                 452,673 bp (3.4%)\n",
      "  Read 1:        44,074 bp\n",
      "  Read 2:       408,599 bp\n",
      "Total written (filtered):     12,333,503 bp (93.8%)\n",
      "  Read 1:     6,348,419 bp\n",
      "  Read 2:     5,985,084 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: CTGTCTCTTATACACATCT; Type: regular 3'; Length: 19; Trimmed: 319 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 3.1%\n",
      "  C: 3.1%\n",
      "  G: 1.3%\n",
      "  T: 92.5%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by \"T\" extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 3' end.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t293\t343.3\t0\t293\n",
      "4\t1\t85.8\t0\t1\n",
      "5\t1\t21.5\t0\t1\n",
      "6\t2\t5.4\t0\t2\n",
      "37\t1\t0.0\t1\t1\n",
      "73\t1\t0.0\t1\t1\n",
      "75\t2\t0.0\t1\t2\n",
      "76\t13\t0.0\t1\t13\n",
      "77\t4\t0.0\t1\t4\n",
      "182\t1\t0.0\t1\t1\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CTGTCTCTTATACACATCT; Type: regular 3'; Length: 19; Trimmed: 187 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t155\t343.3\t0\t155\n",
      "4\t7\t85.8\t0\t7\n",
      "5\t1\t21.5\t0\t1\n",
      "6\t1\t5.4\t0\t1\n",
      "10\t1\t0.0\t1\t0 1\n",
      "29\t1\t0.0\t1\t1\n",
      "33\t1\t0.0\t1\t1\n",
      "34\t1\t0.0\t1\t1\n",
      "36\t1\t0.0\t1\t1\n",
      "56\t1\t0.0\t1\t1\n",
      "57\t1\t0.0\t1\t1\n",
      "60\t1\t0.0\t1\t1\n",
      "76\t1\t0.0\t1\t1\n",
      "77\t1\t0.0\t1\t1\n",
      "78\t7\t0.0\t1\t7\n",
      "79\t6\t0.0\t1\t6\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "Read1 before filtering:\n",
      "total reads: 21973\n",
      "total bases: 6348419\n",
      "Q20 bases: 6318965(99.536%)\n",
      "Q30 bases: 6223605(98.0339%)\n",
      "\n",
      "Read2 before filtering:\n",
      "total reads: 21973\n",
      "total bases: 5985084\n",
      "Q20 bases: 5906563(98.6881%)\n",
      "Q30 bases: 5691642(95.0971%)\n",
      "\n",
      "Merged and filtered:\n",
      "total reads: 21404\n",
      "total bases: 8022679\n",
      "Q20 bases: 7998206(99.695%)\n",
      "Q30 bases: 7905279(98.5366%)\n",
      "\n",
      "Filtering result:\n",
      "reads passed filter: 43534\n",
      "reads failed due to low quality: 24\n",
      "reads failed due to too many N: 2\n",
      "reads failed due to too short: 386\n",
      "reads with adapter trimmed: 270\n",
      "bases trimmed due to adapters: 1356\n",
      "reads corrected by overlap analysis: 3149\n",
      "bases corrected by overlap analysis: 4152\n",
      "\n",
      "Duplication rate: 14.1176%\n",
      "\n",
      "Insert size peak (evaluated by paired-end reads): 0\n",
      "\n",
      "Read pairs merged: 21404\n",
      "% of original read pairs: 97.4105%\n",
      "% in reads after filtering: 100%\n",
      "\n",
      "\n",
      "JSON report: 2_fastp/Hrd1_DMS_Rep1_Region4_LDead_report.json\n",
      "HTML report: 2_fastp/Hrd1_DMS_Rep1_Region4_LDead_report.html\n",
      "\n",
      "fastp -i 1_cutadapt/Hrd1_DMS_Rep1_Region4_LDead_R1_cut.fastq.gz -I 1_cutadapt/Hrd1_DMS_Rep1_Region4_LDead_R2_cut.fastq.gz -m --merged_out 2_fastp/Hrd1_DMS_Rep1_Region4_LDead_merge.fastq.gz -o 2_fastp/Hrd1_DMS_Rep1_Region4_LDead_R1_cut.fastq.gz -O 2_fastp/Hrd1_DMS_Rep1_Region4_LDead_R2_cut.fastq.gz -w 12 -h 2_fastp/Hrd1_DMS_Rep1_Region4_LDead_report.html -j 2_fastp/Hrd1_DMS_Rep1_Region4_LDead_report.json \n",
      "fastp v0.20.0, time used: 1 seconds\n",
      "21404 reads; of these:\n",
      "  21404 (100.00%) were unpaired; of these:\n",
      "    23 (0.11%) aligned 0 times\n",
      "    21381 (99.89%) aligned exactly 1 time\n",
      "    0 (0.00%) aligned >1 times\n",
      "99.89% overall alignment rate\n",
      "[bam_sort_core] merging from 0 files and 12 in-memory blocks...\n",
      "This is cutadapt 1.18 with Python 3.7.16\n",
      "Command line parameters: -a CTGTCTCTTATACACATCT -A CTGTCTCTTATACACATCT -u 8 -U 8 -q 30 -o 1_cutadapt/Hrd1_DMS_Rep3_Region4_LDead_R1_cut.fastq.gz -p 1_cutadapt/Hrd1_DMS_Rep3_Region4_LDead_R2_cut.fastq.gz --cores=12 0_fastq_files/Hrd1_DMS_Rep3_Region4_LDead_R1_001.fastq.gz 0_fastq_files/Hrd1_DMS_Rep3_Region4_LDead_R2_001.fastq.gz\n",
      "Processing reads on 12 cores in paired-end mode ...\n",
      "Finished in 0.25 s (22 us/read; 2.73 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             11,522\n",
      "  Read 1 with adapter:                     120 (1.0%)\n",
      "  Read 2 with adapter:                     145 (1.3%)\n",
      "Pairs written (passing filters):        11,522 (100.0%)\n",
      "\n",
      "Total basepairs processed:     6,902,219 bp\n",
      "  Read 1:     3,455,660 bp\n",
      "  Read 2:     3,446,559 bp\n",
      "Quality-trimmed:                 292,536 bp (4.2%)\n",
      "  Read 1:        33,558 bp\n",
      "  Read 2:       258,978 bp\n",
      "Total written (filtered):      6,423,850 bp (93.1%)\n",
      "  Read 1:     3,329,120 bp\n",
      "  Read 2:     3,094,730 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: CTGTCTCTTATACACATCT; Type: regular 3'; Length: 19; Trimmed: 120 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 0.8%\n",
      "  C: 90.0%\n",
      "  G: 1.7%\n",
      "  T: 7.5%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by \"C\" extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 3' end.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t103\t180.0\t0\t103\n",
      "4\t4\t45.0\t0\t4\n",
      "5\t2\t11.3\t0\t2\n",
      "6\t4\t2.8\t0\t4\n",
      "26\t1\t0.0\t1\t1\n",
      "52\t1\t0.0\t1\t0 1\n",
      "66\t1\t0.0\t1\t1\n",
      "73\t1\t0.0\t1\t1\n",
      "76\t1\t0.0\t1\t1\n",
      "77\t2\t0.0\t1\t2\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CTGTCTCTTATACACATCT; Type: regular 3'; Length: 19; Trimmed: 145 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t124\t180.0\t0\t124\n",
      "4\t15\t45.0\t0\t15\n",
      "5\t1\t11.3\t0\t1\n",
      "20\t1\t0.0\t1\t1\n",
      "38\t1\t0.0\t1\t1\n",
      "48\t1\t0.0\t1\t1\n",
      "52\t1\t0.0\t1\t1\n",
      "80\t1\t0.0\t1\t1\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "Read1 before filtering:\n",
      "total reads: 11522\n",
      "total bases: 3329120\n",
      "Q20 bases: 3312648(99.5052%)\n",
      "Q30 bases: 3262701(98.0049%)\n",
      "\n",
      "Read2 before filtering:\n",
      "total reads: 11522\n",
      "total bases: 3094730\n",
      "Q20 bases: 3049192(98.5285%)\n",
      "Q30 bases: 2928236(94.6201%)\n",
      "\n",
      "Merged and filtered:\n",
      "total reads: 11057\n",
      "total bases: 4261601\n",
      "Q20 bases: 4249078(99.7061%)\n",
      "Q30 bases: 4202320(98.609%)\n",
      "\n",
      "Filtering result:\n",
      "reads passed filter: 22754\n",
      "reads failed due to low quality: 8\n",
      "reads failed due to too many N: 0\n",
      "reads failed due to too short: 282\n",
      "reads with adapter trimmed: 10\n",
      "bases trimmed due to adapters: 80\n",
      "reads corrected by overlap analysis: 1755\n",
      "bases corrected by overlap analysis: 2454\n",
      "\n",
      "Duplication rate: 28.866%\n",
      "\n",
      "Insert size peak (evaluated by paired-end reads): 387\n",
      "\n",
      "Read pairs merged: 11057\n",
      "% of original read pairs: 95.9642%\n",
      "% in reads after filtering: 100%\n",
      "\n",
      "\n",
      "JSON report: 2_fastp/Hrd1_DMS_Rep3_Region4_LDead_report.json\n",
      "HTML report: 2_fastp/Hrd1_DMS_Rep3_Region4_LDead_report.html\n",
      "\n",
      "fastp -i 1_cutadapt/Hrd1_DMS_Rep3_Region4_LDead_R1_cut.fastq.gz -I 1_cutadapt/Hrd1_DMS_Rep3_Region4_LDead_R2_cut.fastq.gz -m --merged_out 2_fastp/Hrd1_DMS_Rep3_Region4_LDead_merge.fastq.gz -o 2_fastp/Hrd1_DMS_Rep3_Region4_LDead_R1_cut.fastq.gz -O 2_fastp/Hrd1_DMS_Rep3_Region4_LDead_R2_cut.fastq.gz -w 12 -h 2_fastp/Hrd1_DMS_Rep3_Region4_LDead_report.html -j 2_fastp/Hrd1_DMS_Rep3_Region4_LDead_report.json \n",
      "fastp v0.20.0, time used: 1 seconds\n",
      "11057 reads; of these:\n",
      "  11057 (100.00%) were unpaired; of these:\n",
      "    16 (0.14%) aligned 0 times\n",
      "    11041 (99.86%) aligned exactly 1 time\n",
      "    0 (0.00%) aligned >1 times\n",
      "99.86% overall alignment rate\n",
      "[bam_sort_core] merging from 0 files and 12 in-memory blocks...\n",
      "This is cutadapt 1.18 with Python 3.7.16\n",
      "Command line parameters: -a CTGTCTCTTATACACATCT -A CTGTCTCTTATACACATCT -u 8 -U 8 -q 30 -o 1_cutadapt/Hrd1_DMS_Rep4_Region4_LDead_R1_cut.fastq.gz -p 1_cutadapt/Hrd1_DMS_Rep4_Region4_LDead_R2_cut.fastq.gz --cores=12 0_fastq_files/Hrd1_DMS_Rep4_Region4_LDead_R1_001.fastq.gz 0_fastq_files/Hrd1_DMS_Rep4_Region4_LDead_R2_001.fastq.gz\n",
      "Processing reads on 12 cores in paired-end mode ...\n",
      "Finished in 0.24 s (23 us/read; 2.61 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             10,349\n",
      "  Read 1 with adapter:                      25 (0.2%)\n",
      "  Read 2 with adapter:                      53 (0.5%)\n",
      "Pairs written (passing filters):        10,349 (100.0%)\n",
      "\n",
      "Total basepairs processed:     6,189,376 bp\n",
      "  Read 1:     3,094,703 bp\n",
      "  Read 2:     3,094,673 bp\n",
      "Quality-trimmed:                 200,223 bp (3.2%)\n",
      "  Read 1:        39,625 bp\n",
      "  Read 2:       160,598 bp\n",
      "Total written (filtered):      5,822,722 bp (94.1%)\n",
      "  Read 1:     2,971,950 bp\n",
      "  Read 2:     2,850,772 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: CTGTCTCTTATACACATCT; Type: regular 3'; Length: 19; Trimmed: 25 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 16.0%\n",
      "  C: 4.0%\n",
      "  G: 36.0%\n",
      "  T: 44.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t16\t161.7\t0\t16\n",
      "4\t4\t40.4\t0\t4\n",
      "17\t1\t0.0\t1\t1\n",
      "41\t1\t0.0\t1\t1\n",
      "69\t1\t0.0\t1\t1\n",
      "70\t1\t0.0\t1\t1\n",
      "75\t1\t0.0\t1\t1\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CTGTCTCTTATACACATCT; Type: regular 3'; Length: 19; Trimmed: 53 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 3.8%\n",
      "  C: 88.7%\n",
      "  G: 3.8%\n",
      "  T: 3.8%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by \"C\" extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 3' end.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t12\t161.7\t0\t12\n",
      "4\t13\t40.4\t0\t13\n",
      "5\t10\t10.1\t0\t10\n",
      "6\t13\t2.5\t0\t13\n",
      "36\t1\t0.0\t1\t1\n",
      "59\t2\t0.0\t1\t2\n",
      "63\t1\t0.0\t1\t1\n",
      "78\t1\t0.0\t1\t1\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "Read1 before filtering:\n",
      "total reads: 10349\n",
      "total bases: 2971950\n",
      "Q20 bases: 2954633(99.4173%)\n",
      "Q30 bases: 2903205(97.6869%)\n",
      "\n",
      "Read2 before filtering:\n",
      "total reads: 10349\n",
      "total bases: 2850772\n",
      "Q20 bases: 2815202(98.7523%)\n",
      "Q30 bases: 2717300(95.318%)\n",
      "\n",
      "Merged and filtered:\n",
      "total reads: 10069\n",
      "total bases: 3769716\n",
      "Q20 bases: 3754611(99.5993%)\n",
      "Q30 bases: 3702981(98.2297%)\n",
      "\n",
      "Filtering result:\n",
      "reads passed filter: 20480\n",
      "reads failed due to low quality: 20\n",
      "reads failed due to too many N: 2\n",
      "reads failed due to too short: 196\n",
      "reads with adapter trimmed: 46\n",
      "bases trimmed due to adapters: 605\n",
      "reads corrected by overlap analysis: 2004\n",
      "bases corrected by overlap analysis: 2755\n",
      "\n",
      "Duplication rate: 47.0588%\n",
      "\n",
      "Insert size peak (evaluated by paired-end reads): 376\n",
      "\n",
      "Read pairs merged: 10069\n",
      "% of original read pairs: 97.2944%\n",
      "% in reads after filtering: 100%\n",
      "\n",
      "\n",
      "JSON report: 2_fastp/Hrd1_DMS_Rep4_Region4_LDead_report.json\n",
      "HTML report: 2_fastp/Hrd1_DMS_Rep4_Region4_LDead_report.html\n",
      "\n",
      "fastp -i 1_cutadapt/Hrd1_DMS_Rep4_Region4_LDead_R1_cut.fastq.gz -I 1_cutadapt/Hrd1_DMS_Rep4_Region4_LDead_R2_cut.fastq.gz -m --merged_out 2_fastp/Hrd1_DMS_Rep4_Region4_LDead_merge.fastq.gz -o 2_fastp/Hrd1_DMS_Rep4_Region4_LDead_R1_cut.fastq.gz -O 2_fastp/Hrd1_DMS_Rep4_Region4_LDead_R2_cut.fastq.gz -w 12 -h 2_fastp/Hrd1_DMS_Rep4_Region4_LDead_report.html -j 2_fastp/Hrd1_DMS_Rep4_Region4_LDead_report.json \n",
      "fastp v0.20.0, time used: 1 seconds\n",
      "10069 reads; of these:\n",
      "  10069 (100.00%) were unpaired; of these:\n",
      "    15 (0.15%) aligned 0 times\n",
      "    10054 (99.85%) aligned exactly 1 time\n",
      "    0 (0.00%) aligned >1 times\n",
      "99.85% overall alignment rate\n",
      "[bam_sort_core] merging from 0 files and 12 in-memory blocks...\n",
      "This is cutadapt 1.18 with Python 3.7.16\n",
      "Command line parameters: -a CTGTCTCTTATACACATCT -A CTGTCTCTTATACACATCT -u 8 -U 8 -q 30 -o 1_cutadapt/Hrd1_DMS_Rep1_Region5_Input_R1_cut.fastq.gz -p 1_cutadapt/Hrd1_DMS_Rep1_Region5_Input_R2_cut.fastq.gz --cores=12 0_fastq_files/Hrd1_DMS_Rep1_Region5_Input_R1_001.fastq.gz 0_fastq_files/Hrd1_DMS_Rep1_Region5_Input_R2_001.fastq.gz\n",
      "Processing reads on 12 cores in paired-end mode ...\n",
      "Finished in 1.81 s (9 us/read; 6.46 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            194,839\n",
      "  Read 1 with adapter:                 171,250 (87.9%)\n",
      "  Read 2 with adapter:                   5,107 (2.6%)\n",
      "Pairs written (passing filters):       194,839 (100.0%)\n",
      "\n",
      "Total basepairs processed:   116,889,784 bp\n",
      "  Read 1:    58,446,954 bp\n",
      "  Read 2:    58,442,830 bp\n",
      "Quality-trimmed:               5,290,765 bp (4.5%)\n",
      "  Read 1:     1,168,461 bp\n",
      "  Read 2:     4,122,304 bp\n",
      "Total written (filtered):    107,323,930 bp (91.8%)\n",
      "  Read 1:    54,851,469 bp\n",
      "  Read 2:    52,472,461 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: CTGTCTCTTATACACATCT; Type: regular 3'; Length: 19; Trimmed: 171250 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 1.3%\n",
      "  C: 1.3%\n",
      "  G: 1.1%\n",
      "  T: 96.4%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by \"T\" extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 3' end.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t166271\t3044.4\t0\t166271\n",
      "4\t84\t761.1\t0\t84\n",
      "5\t30\t190.3\t0\t30\n",
      "8\t1\t3.0\t0\t1\n",
      "9\t3\t0.7\t0\t3\n",
      "11\t1\t0.0\t1\t1\n",
      "12\t1\t0.0\t1\t1\n",
      "14\t1\t0.0\t1\t1\n",
      "17\t1\t0.0\t1\t1\n",
      "18\t2\t0.0\t1\t2\n",
      "19\t3\t0.0\t1\t3\n",
      "21\t7\t0.0\t1\t6 1\n",
      "22\t1\t0.0\t1\t1\n",
      "24\t2\t0.0\t1\t2\n",
      "26\t4\t0.0\t1\t3 1\n",
      "27\t1\t0.0\t1\t1\n",
      "28\t2\t0.0\t1\t1 1\n",
      "29\t4\t0.0\t1\t2 2\n",
      "30\t6\t0.0\t1\t6\n",
      "31\t2\t0.0\t1\t1 1\n",
      "33\t2\t0.0\t1\t2\n",
      "34\t9\t0.0\t1\t6 3\n",
      "35\t9\t0.0\t1\t6 3\n",
      "36\t5\t0.0\t1\t5\n",
      "37\t6\t0.0\t1\t4 2\n",
      "38\t5\t0.0\t1\t5\n",
      "39\t4\t0.0\t1\t3 1\n",
      "40\t9\t0.0\t1\t8 1\n",
      "41\t4\t0.0\t1\t4\n",
      "42\t5\t0.0\t1\t4 1\n",
      "43\t25\t0.0\t1\t24 1\n",
      "44\t1\t0.0\t1\t1\n",
      "49\t3\t0.0\t1\t2 1\n",
      "50\t1\t0.0\t1\t1\n",
      "51\t3\t0.0\t1\t2 1\n",
      "52\t2\t0.0\t1\t2\n",
      "53\t2\t0.0\t1\t2\n",
      "54\t9\t0.0\t1\t7 2\n",
      "55\t4\t0.0\t1\t3 1\n",
      "56\t4\t0.0\t1\t2 2\n",
      "57\t1\t0.0\t1\t1\n",
      "58\t4\t0.0\t1\t4\n",
      "59\t5\t0.0\t1\t5\n",
      "60\t6\t0.0\t1\t6\n",
      "61\t11\t0.0\t1\t9 2\n",
      "62\t6\t0.0\t1\t6\n",
      "63\t7\t0.0\t1\t7\n",
      "64\t7\t0.0\t1\t6 1\n",
      "65\t20\t0.0\t1\t18 2\n",
      "66\t18\t0.0\t1\t16 2\n",
      "67\t10\t0.0\t1\t5 5\n",
      "68\t8\t0.0\t1\t7 1\n",
      "69\t21\t0.0\t1\t19 2\n",
      "70\t21\t0.0\t1\t20 1\n",
      "71\t26\t0.0\t1\t26\n",
      "72\t40\t0.0\t1\t35 5\n",
      "73\t45\t0.0\t1\t39 6\n",
      "74\t70\t0.0\t1\t69 1\n",
      "75\t188\t0.0\t1\t151 37\n",
      "76\t2763\t0.0\t1\t2484 279\n",
      "77\t1224\t0.0\t1\t1077 147\n",
      "78\t141\t0.0\t1\t116 25\n",
      "79\t10\t0.0\t1\t8 2\n",
      "80\t7\t0.0\t1\t6 1\n",
      "81\t3\t0.0\t1\t1 2\n",
      "111\t1\t0.0\t1\t1\n",
      "131\t1\t0.0\t1\t1\n",
      "140\t1\t0.0\t1\t1\n",
      "144\t1\t0.0\t1\t0 1\n",
      "146\t1\t0.0\t1\t1\n",
      "154\t1\t0.0\t1\t1\n",
      "156\t2\t0.0\t1\t2\n",
      "157\t1\t0.0\t1\t1\n",
      "159\t1\t0.0\t1\t1\n",
      "167\t1\t0.0\t1\t1\n",
      "168\t1\t0.0\t1\t0 1\n",
      "169\t1\t0.0\t1\t1\n",
      "187\t1\t0.0\t1\t1\n",
      "191\t1\t0.0\t1\t0 1\n",
      "208\t1\t0.0\t1\t1\n",
      "212\t1\t0.0\t1\t1\n",
      "213\t1\t0.0\t1\t1\n",
      "214\t1\t0.0\t1\t1\n",
      "215\t4\t0.0\t1\t4\n",
      "216\t12\t0.0\t1\t11 1\n",
      "217\t11\t0.0\t1\t7 4\n",
      "218\t3\t0.0\t1\t3\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CTGTCTCTTATACACATCT; Type: regular 3'; Length: 19; Trimmed: 5107 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t239\t3044.4\t0\t239\n",
      "4\t36\t761.1\t0\t36\n",
      "5\t3\t190.3\t0\t3\n",
      "7\t1\t11.9\t0\t1\n",
      "11\t1\t0.0\t1\t1\n",
      "12\t3\t0.0\t1\t3\n",
      "13\t2\t0.0\t1\t2\n",
      "14\t2\t0.0\t1\t2\n",
      "15\t1\t0.0\t1\t1\n",
      "16\t3\t0.0\t1\t3\n",
      "17\t3\t0.0\t1\t3\n",
      "18\t4\t0.0\t1\t3 1\n",
      "19\t4\t0.0\t1\t4\n",
      "20\t2\t0.0\t1\t2\n",
      "21\t4\t0.0\t1\t4\n",
      "22\t15\t0.0\t1\t13 2\n",
      "23\t8\t0.0\t1\t8\n",
      "24\t6\t0.0\t1\t5 1\n",
      "25\t6\t0.0\t1\t6\n",
      "26\t8\t0.0\t1\t6 2\n",
      "27\t8\t0.0\t1\t8\n",
      "28\t7\t0.0\t1\t7\n",
      "29\t19\t0.0\t1\t18 1\n",
      "30\t14\t0.0\t1\t14\n",
      "31\t24\t0.0\t1\t22 2\n",
      "32\t40\t0.0\t1\t40\n",
      "33\t48\t0.0\t1\t47 1\n",
      "34\t92\t0.0\t1\t89 3\n",
      "35\t13\t0.0\t1\t13\n",
      "36\t2\t0.0\t1\t2\n",
      "37\t30\t0.0\t1\t30\n",
      "38\t39\t0.0\t1\t37 2\n",
      "39\t29\t0.0\t1\t28 1\n",
      "40\t14\t0.0\t1\t14\n",
      "41\t38\t0.0\t1\t37 1\n",
      "42\t56\t0.0\t1\t54 2\n",
      "43\t165\t0.0\t1\t164 1\n",
      "44\t45\t0.0\t1\t44 1\n",
      "45\t112\t0.0\t1\t110 2\n",
      "46\t189\t0.0\t1\t187 2\n",
      "47\t66\t0.0\t1\t63 3\n",
      "48\t88\t0.0\t1\t84 4\n",
      "49\t130\t0.0\t1\t129 1\n",
      "50\t83\t0.0\t1\t81 2\n",
      "51\t87\t0.0\t1\t84 3\n",
      "52\t134\t0.0\t1\t132 2\n",
      "53\t65\t0.0\t1\t64 1\n",
      "54\t208\t0.0\t1\t206 2\n",
      "55\t125\t0.0\t1\t124 1\n",
      "56\t54\t0.0\t1\t51 3\n",
      "57\t329\t0.0\t1\t325 4\n",
      "58\t177\t0.0\t1\t174 3\n",
      "59\t209\t0.0\t1\t207 2\n",
      "60\t105\t0.0\t1\t104 1\n",
      "61\t41\t0.0\t1\t40 1\n",
      "62\t158\t0.0\t1\t156 2\n",
      "63\t171\t0.0\t1\t169 2\n",
      "64\t30\t0.0\t1\t28 2\n",
      "65\t15\t0.0\t1\t15\n",
      "66\t7\t0.0\t1\t7\n",
      "67\t14\t0.0\t1\t14\n",
      "69\t1\t0.0\t1\t0 1\n",
      "70\t3\t0.0\t1\t3\n",
      "72\t1\t0.0\t1\t1\n",
      "73\t2\t0.0\t1\t2\n",
      "74\t4\t0.0\t1\t4\n",
      "75\t4\t0.0\t1\t4\n",
      "76\t3\t0.0\t1\t3\n",
      "77\t50\t0.0\t1\t49 1\n",
      "78\t93\t0.0\t1\t93\n",
      "79\t111\t0.0\t1\t107 4\n",
      "80\t613\t0.0\t1\t609 4\n",
      "81\t404\t0.0\t1\t399 5\n",
      "82\t146\t0.0\t1\t146\n",
      "83\t32\t0.0\t1\t32\n",
      "84\t4\t0.0\t1\t4\n",
      "85\t4\t0.0\t1\t4\n",
      "215\t1\t0.0\t1\t1\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "Read1 before filtering:\n",
      "total reads: 194839\n",
      "total bases: 54851469\n",
      "Q20 bases: 54610764(99.5612%)\n",
      "Q30 bases: 53842147(98.1599%)\n",
      "\n",
      "Read2 before filtering:\n",
      "total reads: 194839\n",
      "total bases: 52472461\n",
      "Q20 bases: 51738154(98.6006%)\n",
      "Q30 bases: 49916931(95.1298%)\n",
      "\n",
      "Merged and filtered:\n",
      "total reads: 188067\n",
      "total bases: 72391286\n",
      "Q20 bases: 72180671(99.7091%)\n",
      "Q30 bases: 71399851(98.6304%)\n",
      "\n",
      "Filtering result:\n",
      "reads passed filter: 384634\n",
      "reads failed due to low quality: 234\n",
      "reads failed due to too many N: 26\n",
      "reads failed due to too short: 4784\n",
      "reads with adapter trimmed: 9692\n",
      "bases trimmed due to adapters: 79237\n",
      "reads corrected by overlap analysis: 27301\n",
      "bases corrected by overlap analysis: 39170\n",
      "\n",
      "Duplication rate: 84.561%\n",
      "\n",
      "Insert size peak (evaluated by paired-end reads): 394\n",
      "\n",
      "Read pairs merged: 188067\n",
      "% of original read pairs: 96.5243%\n",
      "% in reads after filtering: 100%\n",
      "\n",
      "\n",
      "JSON report: 2_fastp/Hrd1_DMS_Rep1_Region5_Input_report.json\n",
      "HTML report: 2_fastp/Hrd1_DMS_Rep1_Region5_Input_report.html\n",
      "\n",
      "fastp -i 1_cutadapt/Hrd1_DMS_Rep1_Region5_Input_R1_cut.fastq.gz -I 1_cutadapt/Hrd1_DMS_Rep1_Region5_Input_R2_cut.fastq.gz -m --merged_out 2_fastp/Hrd1_DMS_Rep1_Region5_Input_merge.fastq.gz -o 2_fastp/Hrd1_DMS_Rep1_Region5_Input_R1_cut.fastq.gz -O 2_fastp/Hrd1_DMS_Rep1_Region5_Input_R2_cut.fastq.gz -w 12 -h 2_fastp/Hrd1_DMS_Rep1_Region5_Input_report.html -j 2_fastp/Hrd1_DMS_Rep1_Region5_Input_report.json \n",
      "fastp v0.20.0, time used: 2 seconds\n",
      "188067 reads; of these:\n",
      "  188067 (100.00%) were unpaired; of these:\n",
      "    4869 (2.59%) aligned 0 times\n",
      "    183198 (97.41%) aligned exactly 1 time\n",
      "    0 (0.00%) aligned >1 times\n",
      "97.41% overall alignment rate\n",
      "[bam_sort_core] merging from 0 files and 12 in-memory blocks...\n",
      "This is cutadapt 1.18 with Python 3.7.16\n",
      "Command line parameters: -a CTGTCTCTTATACACATCT -A CTGTCTCTTATACACATCT -u 8 -U 8 -q 30 -o 1_cutadapt/Hrd1_DMS_Rep2_Region5_Input_R1_cut.fastq.gz -p 1_cutadapt/Hrd1_DMS_Rep2_Region5_Input_R2_cut.fastq.gz --cores=12 0_fastq_files/Hrd1_DMS_Rep2_Region5_Input_R1_001.fastq.gz 0_fastq_files/Hrd1_DMS_Rep2_Region5_Input_R2_001.fastq.gz\n",
      "Processing reads on 12 cores in paired-end mode ...\n",
      "Finished in 1.57 s (10 us/read; 6.01 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            157,800\n",
      "  Read 1 with adapter:                   3,381 (2.1%)\n",
      "  Read 2 with adapter:                  73,026 (46.3%)\n",
      "Pairs written (passing filters):       157,800 (100.0%)\n",
      "\n",
      "Total basepairs processed:    94,669,266 bp\n",
      "  Read 1:    47,334,145 bp\n",
      "  Read 2:    47,335,121 bp\n",
      "Quality-trimmed:               4,483,463 bp (4.7%)\n",
      "  Read 1:     1,035,683 bp\n",
      "  Read 2:     3,447,780 bp\n",
      "Total written (filtered):     86,999,185 bp (91.9%)\n",
      "  Read 1:    44,789,963 bp\n",
      "  Read 2:    42,209,222 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: CTGTCTCTTATACACATCT; Type: regular 3'; Length: 19; Trimmed: 3381 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 29.7%\n",
      "  C: 22.6%\n",
      "  G: 19.9%\n",
      "  T: 27.8%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t103\t2465.6\t0\t103\n",
      "4\t25\t616.4\t0\t25\n",
      "6\t1\t38.5\t0\t1\n",
      "8\t1\t2.4\t0\t1\n",
      "9\t3\t0.6\t0\t3\n",
      "11\t2\t0.0\t1\t2\n",
      "15\t1\t0.0\t1\t1\n",
      "17\t3\t0.0\t1\t3\n",
      "18\t2\t0.0\t1\t2\n",
      "19\t1\t0.0\t1\t1\n",
      "20\t2\t0.0\t1\t2\n",
      "21\t2\t0.0\t1\t2\n",
      "22\t3\t0.0\t1\t3\n",
      "25\t1\t0.0\t1\t1\n",
      "27\t2\t0.0\t1\t1 1\n",
      "28\t1\t0.0\t1\t1\n",
      "29\t3\t0.0\t1\t3\n",
      "30\t1\t0.0\t1\t1\n",
      "31\t3\t0.0\t1\t3\n",
      "32\t3\t0.0\t1\t2 1\n",
      "34\t1\t0.0\t1\t1\n",
      "35\t1\t0.0\t1\t1\n",
      "36\t3\t0.0\t1\t2 1\n",
      "37\t4\t0.0\t1\t4\n",
      "38\t8\t0.0\t1\t6 2\n",
      "39\t5\t0.0\t1\t4 1\n",
      "40\t3\t0.0\t1\t3\n",
      "41\t3\t0.0\t1\t3\n",
      "42\t2\t0.0\t1\t1 1\n",
      "43\t19\t0.0\t1\t14 5\n",
      "44\t1\t0.0\t1\t1\n",
      "48\t1\t0.0\t1\t1\n",
      "49\t3\t0.0\t1\t3\n",
      "51\t4\t0.0\t1\t4\n",
      "52\t1\t0.0\t1\t0 1\n",
      "53\t2\t0.0\t1\t2\n",
      "54\t6\t0.0\t1\t5 1\n",
      "55\t3\t0.0\t1\t3\n",
      "56\t3\t0.0\t1\t3\n",
      "57\t3\t0.0\t1\t3\n",
      "58\t1\t0.0\t1\t1\n",
      "59\t4\t0.0\t1\t3 1\n",
      "60\t3\t0.0\t1\t2 1\n",
      "61\t8\t0.0\t1\t7 1\n",
      "62\t2\t0.0\t1\t2\n",
      "63\t11\t0.0\t1\t10 1\n",
      "64\t6\t0.0\t1\t6\n",
      "65\t18\t0.0\t1\t18\n",
      "66\t15\t0.0\t1\t15\n",
      "67\t12\t0.0\t1\t11 1\n",
      "68\t8\t0.0\t1\t6 2\n",
      "69\t13\t0.0\t1\t12 1\n",
      "70\t27\t0.0\t1\t21 6\n",
      "71\t32\t0.0\t1\t26 6\n",
      "72\t28\t0.0\t1\t25 3\n",
      "73\t32\t0.0\t1\t29 3\n",
      "74\t45\t0.0\t1\t40 5\n",
      "75\t137\t0.0\t1\t105 32\n",
      "76\t1649\t0.0\t1\t1479 170\n",
      "77\t898\t0.0\t1\t785 113\n",
      "78\t133\t0.0\t1\t111 22\n",
      "79\t18\t0.0\t1\t15 3\n",
      "80\t9\t0.0\t1\t8 1\n",
      "82\t4\t0.0\t1\t4\n",
      "90\t1\t0.0\t1\t0 1\n",
      "95\t1\t0.0\t1\t1\n",
      "127\t1\t0.0\t1\t1\n",
      "140\t1\t0.0\t1\t1\n",
      "148\t1\t0.0\t1\t0 1\n",
      "149\t1\t0.0\t1\t0 1\n",
      "169\t1\t0.0\t1\t1\n",
      "172\t1\t0.0\t1\t0 1\n",
      "181\t2\t0.0\t1\t2\n",
      "202\t1\t0.0\t1\t0 1\n",
      "213\t1\t0.0\t1\t1\n",
      "214\t1\t0.0\t1\t1\n",
      "216\t7\t0.0\t1\t5 2\n",
      "217\t8\t0.0\t1\t7 1\n",
      "218\t1\t0.0\t1\t1\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CTGTCTCTTATACACATCT; Type: regular 3'; Length: 19; Trimmed: 73026 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 2.2%\n",
      "  C: 3.4%\n",
      "  G: 1.1%\n",
      "  T: 93.3%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by \"T\" extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 3' end.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t69046\t2465.6\t0\t69046\n",
      "4\t472\t616.4\t0\t472\n",
      "5\t262\t154.1\t0\t262\n",
      "6\t1\t38.5\t0\t1\n",
      "7\t1\t9.6\t0\t1\n",
      "8\t2\t2.4\t0\t2\n",
      "9\t1\t0.6\t0\t1\n",
      "11\t4\t0.0\t1\t2 2\n",
      "13\t1\t0.0\t1\t1\n",
      "14\t3\t0.0\t1\t3\n",
      "17\t1\t0.0\t1\t1\n",
      "18\t1\t0.0\t1\t1\n",
      "19\t2\t0.0\t1\t1 1\n",
      "20\t2\t0.0\t1\t2\n",
      "21\t4\t0.0\t1\t4\n",
      "22\t6\t0.0\t1\t6\n",
      "23\t4\t0.0\t1\t4\n",
      "24\t4\t0.0\t1\t4\n",
      "25\t6\t0.0\t1\t6\n",
      "26\t5\t0.0\t1\t5\n",
      "27\t2\t0.0\t1\t2\n",
      "28\t4\t0.0\t1\t4\n",
      "29\t18\t0.0\t1\t18\n",
      "30\t13\t0.0\t1\t13\n",
      "31\t9\t0.0\t1\t9\n",
      "32\t15\t0.0\t1\t15\n",
      "33\t28\t0.0\t1\t28\n",
      "34\t48\t0.0\t1\t48\n",
      "35\t9\t0.0\t1\t8 1\n",
      "36\t5\t0.0\t1\t5\n",
      "37\t11\t0.0\t1\t11\n",
      "38\t26\t0.0\t1\t26\n",
      "39\t32\t0.0\t1\t32\n",
      "40\t7\t0.0\t1\t7\n",
      "41\t35\t0.0\t1\t34 1\n",
      "42\t26\t0.0\t1\t25 1\n",
      "43\t96\t0.0\t1\t93 3\n",
      "44\t24\t0.0\t1\t24\n",
      "45\t99\t0.0\t1\t96 3\n",
      "46\t102\t0.0\t1\t101 1\n",
      "47\t41\t0.0\t1\t41\n",
      "48\t59\t0.0\t1\t59\n",
      "49\t68\t0.0\t1\t66 2\n",
      "50\t39\t0.0\t1\t38 1\n",
      "51\t33\t0.0\t1\t33\n",
      "52\t68\t0.0\t1\t67 1\n",
      "53\t16\t0.0\t1\t16\n",
      "54\t123\t0.0\t1\t123\n",
      "55\t62\t0.0\t1\t61 1\n",
      "56\t32\t0.0\t1\t32\n",
      "57\t162\t0.0\t1\t161 1\n",
      "58\t110\t0.0\t1\t110\n",
      "59\t75\t0.0\t1\t75\n",
      "60\t84\t0.0\t1\t84\n",
      "61\t32\t0.0\t1\t32\n",
      "62\t58\t0.0\t1\t57 1\n",
      "63\t82\t0.0\t1\t80 2\n",
      "64\t15\t0.0\t1\t15\n",
      "65\t15\t0.0\t1\t15\n",
      "66\t9\t0.0\t1\t9\n",
      "67\t4\t0.0\t1\t4\n",
      "69\t1\t0.0\t1\t1\n",
      "70\t1\t0.0\t1\t1\n",
      "73\t1\t0.0\t1\t1\n",
      "74\t2\t0.0\t1\t2\n",
      "75\t2\t0.0\t1\t2\n",
      "76\t9\t0.0\t1\t9\n",
      "77\t17\t0.0\t1\t17\n",
      "78\t47\t0.0\t1\t47\n",
      "79\t65\t0.0\t1\t63 2\n",
      "80\t524\t0.0\t1\t522 2\n",
      "81\t455\t0.0\t1\t450 5\n",
      "82\t198\t0.0\t1\t196 2\n",
      "83\t65\t0.0\t1\t65\n",
      "84\t11\t0.0\t1\t11\n",
      "85\t2\t0.0\t1\t2\n",
      "217\t2\t0.0\t1\t2\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "Read1 before filtering:\n",
      "total reads: 157800\n",
      "total bases: 44789963\n",
      "Q20 bases: 44558635(99.4835%)\n",
      "Q30 bases: 43834722(97.8673%)\n",
      "\n",
      "Read2 before filtering:\n",
      "total reads: 157800\n",
      "total bases: 42209222\n",
      "Q20 bases: 41732489(98.8705%)\n",
      "Q30 bases: 40339132(95.5695%)\n",
      "\n",
      "Merged and filtered:\n",
      "total reads: 152207\n",
      "total bases: 58777163\n",
      "Q20 bases: 58559859(99.6303%)\n",
      "Q30 bases: 57809827(98.3542%)\n",
      "\n",
      "Filtering result:\n",
      "reads passed filter: 312090\n",
      "reads failed due to low quality: 220\n",
      "reads failed due to too many N: 30\n",
      "reads failed due to too short: 3260\n",
      "reads with adapter trimmed: 6536\n",
      "bases trimmed due to adapters: 55153\n",
      "reads corrected by overlap analysis: 16933\n",
      "bases corrected by overlap analysis: 21927\n",
      "\n",
      "Duplication rate: 27.8846%\n",
      "\n",
      "Insert size peak (evaluated by paired-end reads): 394\n",
      "\n",
      "Read pairs merged: 152207\n",
      "% of original read pairs: 96.4556%\n",
      "% in reads after filtering: 100%\n",
      "\n",
      "\n",
      "JSON report: 2_fastp/Hrd1_DMS_Rep2_Region5_Input_report.json\n",
      "HTML report: 2_fastp/Hrd1_DMS_Rep2_Region5_Input_report.html\n",
      "\n",
      "fastp -i 1_cutadapt/Hrd1_DMS_Rep2_Region5_Input_R1_cut.fastq.gz -I 1_cutadapt/Hrd1_DMS_Rep2_Region5_Input_R2_cut.fastq.gz -m --merged_out 2_fastp/Hrd1_DMS_Rep2_Region5_Input_merge.fastq.gz -o 2_fastp/Hrd1_DMS_Rep2_Region5_Input_R1_cut.fastq.gz -O 2_fastp/Hrd1_DMS_Rep2_Region5_Input_R2_cut.fastq.gz -w 12 -h 2_fastp/Hrd1_DMS_Rep2_Region5_Input_report.html -j 2_fastp/Hrd1_DMS_Rep2_Region5_Input_report.json \n",
      "fastp v0.20.0, time used: 1 seconds\n",
      "152207 reads; of these:\n",
      "  152207 (100.00%) were unpaired; of these:\n",
      "    3257 (2.14%) aligned 0 times\n",
      "    148950 (97.86%) aligned exactly 1 time\n",
      "    0 (0.00%) aligned >1 times\n",
      "97.86% overall alignment rate\n",
      "[bam_sort_core] merging from 0 files and 12 in-memory blocks...\n",
      "This is cutadapt 1.18 with Python 3.7.16\n",
      "Command line parameters: -a CTGTCTCTTATACACATCT -A CTGTCTCTTATACACATCT -u 8 -U 8 -q 30 -o 1_cutadapt/Hrd1_DMS_Rep3_Region5_Input_R1_cut.fastq.gz -p 1_cutadapt/Hrd1_DMS_Rep3_Region5_Input_R2_cut.fastq.gz --cores=12 0_fastq_files/Hrd1_DMS_Rep3_Region5_Input_R1_001.fastq.gz 0_fastq_files/Hrd1_DMS_Rep3_Region5_Input_R2_001.fastq.gz\n",
      "Processing reads on 12 cores in paired-end mode ...\n",
      "Finished in 1.40 s (9 us/read; 6.73 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            156,643\n",
      "  Read 1 with adapter:                   1,226 (0.8%)\n",
      "  Read 2 with adapter:                     823 (0.5%)\n",
      "Pairs written (passing filters):       156,643 (100.0%)\n",
      "\n",
      "Total basepairs processed:    93,979,319 bp\n",
      "  Read 1:    46,991,576 bp\n",
      "  Read 2:    46,987,743 bp\n",
      "Quality-trimmed:               3,449,489 bp (3.7%)\n",
      "  Read 1:       560,094 bp\n",
      "  Read 2:     2,889,395 bp\n",
      "Total written (filtered):     87,923,733 bp (93.6%)\n",
      "  Read 1:    45,125,382 bp\n",
      "  Read 2:    42,798,351 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: CTGTCTCTTATACACATCT; Type: regular 3'; Length: 19; Trimmed: 1226 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 21.1%\n",
      "  C: 25.4%\n",
      "  G: 11.7%\n",
      "  T: 41.8%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t429\t2447.5\t0\t429\n",
      "4\t86\t611.9\t0\t86\n",
      "5\t36\t153.0\t0\t36\n",
      "7\t3\t9.6\t0\t3\n",
      "9\t1\t0.6\t0\t1\n",
      "18\t1\t0.0\t1\t1\n",
      "20\t1\t0.0\t1\t1\n",
      "28\t2\t0.0\t1\t2\n",
      "33\t1\t0.0\t1\t1\n",
      "34\t1\t0.0\t1\t1\n",
      "36\t1\t0.0\t1\t1\n",
      "38\t1\t0.0\t1\t1\n",
      "41\t2\t0.0\t1\t2\n",
      "43\t4\t0.0\t1\t4\n",
      "45\t1\t0.0\t1\t1\n",
      "53\t2\t0.0\t1\t2\n",
      "54\t1\t0.0\t1\t1\n",
      "55\t1\t0.0\t1\t1\n",
      "56\t3\t0.0\t1\t3\n",
      "59\t2\t0.0\t1\t2\n",
      "60\t1\t0.0\t1\t1\n",
      "61\t2\t0.0\t1\t2\n",
      "62\t1\t0.0\t1\t1\n",
      "63\t1\t0.0\t1\t1\n",
      "64\t3\t0.0\t1\t3\n",
      "65\t2\t0.0\t1\t2\n",
      "66\t1\t0.0\t1\t1\n",
      "67\t2\t0.0\t1\t2\n",
      "68\t5\t0.0\t1\t5\n",
      "69\t4\t0.0\t1\t4\n",
      "70\t3\t0.0\t1\t3\n",
      "71\t1\t0.0\t1\t1\n",
      "72\t4\t0.0\t1\t4\n",
      "73\t6\t0.0\t1\t6\n",
      "74\t13\t0.0\t1\t13\n",
      "75\t27\t0.0\t1\t26 1\n",
      "76\t384\t0.0\t1\t382 2\n",
      "77\t146\t0.0\t1\t140 6\n",
      "78\t27\t0.0\t1\t27\n",
      "79\t3\t0.0\t1\t3\n",
      "80\t1\t0.0\t1\t1\n",
      "81\t1\t0.0\t1\t1\n",
      "130\t1\t0.0\t1\t1\n",
      "150\t1\t0.0\t1\t1\n",
      "167\t1\t0.0\t1\t1\n",
      "201\t1\t0.0\t1\t1\n",
      "204\t1\t0.0\t1\t1\n",
      "214\t1\t0.0\t1\t1\n",
      "215\t3\t0.0\t1\t3\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CTGTCTCTTATACACATCT; Type: regular 3'; Length: 19; Trimmed: 823 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 27.9%\n",
      "  C: 16.9%\n",
      "  G: 27.3%\n",
      "  T: 27.8%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t147\t2447.5\t0\t147\n",
      "4\t7\t611.9\t0\t7\n",
      "14\t1\t0.0\t1\t1\n",
      "18\t1\t0.0\t1\t1\n",
      "19\t3\t0.0\t1\t3\n",
      "20\t2\t0.0\t1\t2\n",
      "25\t1\t0.0\t1\t1\n",
      "28\t1\t0.0\t1\t1\n",
      "32\t2\t0.0\t1\t2\n",
      "33\t2\t0.0\t1\t2\n",
      "34\t3\t0.0\t1\t3\n",
      "35\t7\t0.0\t1\t7\n",
      "36\t3\t0.0\t1\t3\n",
      "37\t1\t0.0\t1\t1\n",
      "38\t3\t0.0\t1\t3\n",
      "39\t2\t0.0\t1\t2\n",
      "40\t6\t0.0\t1\t6\n",
      "41\t15\t0.0\t1\t15\n",
      "42\t4\t0.0\t1\t4\n",
      "43\t12\t0.0\t1\t12\n",
      "44\t7\t0.0\t1\t7\n",
      "45\t5\t0.0\t1\t5\n",
      "46\t18\t0.0\t1\t18\n",
      "47\t7\t0.0\t1\t7\n",
      "48\t5\t0.0\t1\t4 1\n",
      "49\t2\t0.0\t1\t2\n",
      "50\t2\t0.0\t1\t2\n",
      "51\t5\t0.0\t1\t5\n",
      "52\t4\t0.0\t1\t4\n",
      "53\t1\t0.0\t1\t1\n",
      "54\t15\t0.0\t1\t15\n",
      "55\t4\t0.0\t1\t4\n",
      "56\t11\t0.0\t1\t10 1\n",
      "57\t34\t0.0\t1\t34\n",
      "58\t7\t0.0\t1\t7\n",
      "59\t11\t0.0\t1\t11\n",
      "60\t4\t0.0\t1\t4\n",
      "61\t1\t0.0\t1\t1\n",
      "62\t2\t0.0\t1\t2\n",
      "63\t17\t0.0\t1\t17\n",
      "64\t6\t0.0\t1\t6\n",
      "66\t1\t0.0\t1\t1\n",
      "67\t1\t0.0\t1\t1\n",
      "68\t1\t0.0\t1\t1\n",
      "69\t1\t0.0\t1\t1\n",
      "73\t1\t0.0\t1\t1\n",
      "75\t2\t0.0\t1\t2\n",
      "76\t4\t0.0\t1\t4\n",
      "77\t9\t0.0\t1\t9\n",
      "78\t11\t0.0\t1\t11\n",
      "79\t23\t0.0\t1\t22 1\n",
      "80\t159\t0.0\t1\t159\n",
      "81\t164\t0.0\t1\t162 2\n",
      "82\t48\t0.0\t1\t47 1\n",
      "83\t7\t0.0\t1\t7\n",
      "\n",
      "Read1 before filtering:\n",
      "total reads: 156643\n",
      "total bases: 45125382\n",
      "Q20 bases: 44941006(99.5914%)\n",
      "Q30 bases: 44309595(98.1922%)\n",
      "\n",
      "Read2 before filtering:\n",
      "total reads: 156643\n",
      "total bases: 42798351\n",
      "Q20 bases: 42204174(98.6117%)\n",
      "Q30 bases: 40707496(95.1146%)\n",
      "\n",
      "Merged and filtered:\n",
      "total reads: 151123\n",
      "total bases: 59549361\n",
      "Q20 bases: 59377482(99.7114%)\n",
      "Q30 bases: 58722264(98.6111%)\n",
      "\n",
      "Filtering result:\n",
      "reads passed filter: 309376\n",
      "reads failed due to low quality: 146\n",
      "reads failed due to too many N: 24\n",
      "reads failed due to too short: 3740\n",
      "reads with adapter trimmed: 1340\n",
      "bases trimmed due to adapters: 11044\n",
      "reads corrected by overlap analysis: 19493\n",
      "bases corrected by overlap analysis: 28579\n",
      "\n",
      "Duplication rate: 85.7863%\n",
      "\n",
      "Insert size peak (evaluated by paired-end reads): 396\n",
      "\n",
      "Read pairs merged: 151123\n",
      "% of original read pairs: 96.4761%\n",
      "% in reads after filtering: 100%\n",
      "\n",
      "\n",
      "JSON report: 2_fastp/Hrd1_DMS_Rep3_Region5_Input_report.json\n",
      "HTML report: 2_fastp/Hrd1_DMS_Rep3_Region5_Input_report.html\n",
      "\n",
      "fastp -i 1_cutadapt/Hrd1_DMS_Rep3_Region5_Input_R1_cut.fastq.gz -I 1_cutadapt/Hrd1_DMS_Rep3_Region5_Input_R2_cut.fastq.gz -m --merged_out 2_fastp/Hrd1_DMS_Rep3_Region5_Input_merge.fastq.gz -o 2_fastp/Hrd1_DMS_Rep3_Region5_Input_R1_cut.fastq.gz -O 2_fastp/Hrd1_DMS_Rep3_Region5_Input_R2_cut.fastq.gz -w 12 -h 2_fastp/Hrd1_DMS_Rep3_Region5_Input_report.html -j 2_fastp/Hrd1_DMS_Rep3_Region5_Input_report.json \n",
      "fastp v0.20.0, time used: 1 seconds\n",
      "151123 reads; of these:\n",
      "  151123 (100.00%) were unpaired; of these:\n",
      "    681 (0.45%) aligned 0 times\n",
      "    150442 (99.55%) aligned exactly 1 time\n",
      "    0 (0.00%) aligned >1 times\n",
      "99.55% overall alignment rate\n",
      "[bam_sort_core] merging from 0 files and 12 in-memory blocks...\n",
      "This is cutadapt 1.18 with Python 3.7.16\n",
      "Command line parameters: -a CTGTCTCTTATACACATCT -A CTGTCTCTTATACACATCT -u 8 -U 8 -q 30 -o 1_cutadapt/Hrd1_DMS_Rep4_Region5_Input_R1_cut.fastq.gz -p 1_cutadapt/Hrd1_DMS_Rep4_Region5_Input_R2_cut.fastq.gz --cores=12 0_fastq_files/Hrd1_DMS_Rep4_Region5_Input_R1_001.fastq.gz 0_fastq_files/Hrd1_DMS_Rep4_Region5_Input_R2_001.fastq.gz\n",
      "Processing reads on 12 cores in paired-end mode ...\n",
      "Finished in 1.76 s (9 us/read; 6.90 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            202,634\n",
      "  Read 1 with adapter:                   1,195 (0.6%)\n",
      "  Read 2 with adapter:                     764 (0.4%)\n",
      "Pairs written (passing filters):       202,634 (100.0%)\n",
      "\n",
      "Total basepairs processed:   121,572,341 bp\n",
      "  Read 1:    60,788,569 bp\n",
      "  Read 2:    60,783,772 bp\n",
      "Quality-trimmed:               4,225,777 bp (3.5%)\n",
      "  Read 1:       717,903 bp\n",
      "  Read 2:     3,507,874 bp\n",
      "Total written (filtered):    114,023,856 bp (93.8%)\n",
      "  Read 1:    58,405,976 bp\n",
      "  Read 2:    55,617,880 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: CTGTCTCTTATACACATCT; Type: regular 3'; Length: 19; Trimmed: 1195 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 17.2%\n",
      "  C: 25.8%\n",
      "  G: 10.9%\n",
      "  T: 46.1%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t509\t3166.2\t0\t509\n",
      "4\t98\t791.5\t0\t98\n",
      "5\t40\t197.9\t0\t40\n",
      "7\t3\t12.4\t0\t3\n",
      "9\t1\t0.8\t0\t1\n",
      "10\t1\t0.2\t1\t1\n",
      "22\t1\t0.0\t1\t1\n",
      "29\t3\t0.0\t1\t3\n",
      "30\t1\t0.0\t1\t1\n",
      "32\t1\t0.0\t1\t1\n",
      "35\t2\t0.0\t1\t2\n",
      "37\t1\t0.0\t1\t1\n",
      "39\t1\t0.0\t1\t1\n",
      "43\t4\t0.0\t1\t4\n",
      "45\t1\t0.0\t1\t1\n",
      "47\t1\t0.0\t1\t1\n",
      "52\t1\t0.0\t1\t0 1\n",
      "56\t1\t0.0\t1\t1\n",
      "57\t1\t0.0\t1\t1\n",
      "60\t2\t0.0\t1\t2\n",
      "61\t1\t0.0\t1\t1\n",
      "65\t1\t0.0\t1\t1\n",
      "67\t1\t0.0\t1\t1\n",
      "68\t3\t0.0\t1\t3\n",
      "69\t5\t0.0\t1\t5\n",
      "70\t2\t0.0\t1\t2\n",
      "71\t1\t0.0\t1\t1\n",
      "72\t3\t0.0\t1\t3\n",
      "73\t15\t0.0\t1\t15\n",
      "74\t11\t0.0\t1\t11\n",
      "75\t24\t0.0\t1\t24\n",
      "76\t329\t0.0\t1\t326 3\n",
      "77\t90\t0.0\t1\t89 1\n",
      "78\t25\t0.0\t1\t23 2\n",
      "79\t3\t0.0\t1\t3\n",
      "119\t1\t0.0\t1\t1\n",
      "189\t1\t0.0\t1\t1\n",
      "211\t1\t0.0\t1\t1\n",
      "214\t1\t0.0\t1\t1\n",
      "215\t3\t0.0\t1\t2 1\n",
      "216\t1\t0.0\t1\t1\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CTGTCTCTTATACACATCT; Type: regular 3'; Length: 19; Trimmed: 764 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 26.0%\n",
      "  C: 14.9%\n",
      "  G: 31.3%\n",
      "  T: 27.7%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t209\t3166.2\t0\t209\n",
      "4\t10\t791.5\t0\t10\n",
      "5\t1\t197.9\t0\t1\n",
      "6\t1\t49.5\t0\t1\n",
      "15\t1\t0.0\t1\t1\n",
      "16\t1\t0.0\t1\t1\n",
      "17\t2\t0.0\t1\t1 1\n",
      "19\t1\t0.0\t1\t1\n",
      "20\t2\t0.0\t1\t2\n",
      "21\t1\t0.0\t1\t1\n",
      "22\t2\t0.0\t1\t1 1\n",
      "23\t1\t0.0\t1\t1\n",
      "25\t1\t0.0\t1\t1\n",
      "27\t1\t0.0\t1\t1\n",
      "30\t3\t0.0\t1\t3\n",
      "31\t3\t0.0\t1\t3\n",
      "32\t2\t0.0\t1\t2\n",
      "33\t1\t0.0\t1\t1\n",
      "34\t3\t0.0\t1\t3\n",
      "35\t4\t0.0\t1\t4\n",
      "36\t1\t0.0\t1\t1\n",
      "37\t1\t0.0\t1\t1\n",
      "38\t3\t0.0\t1\t3\n",
      "39\t6\t0.0\t1\t6\n",
      "40\t5\t0.0\t1\t5\n",
      "41\t10\t0.0\t1\t10\n",
      "42\t4\t0.0\t1\t4\n",
      "43\t5\t0.0\t1\t5\n",
      "44\t3\t0.0\t1\t3\n",
      "45\t7\t0.0\t1\t7\n",
      "46\t12\t0.0\t1\t12\n",
      "47\t7\t0.0\t1\t7\n",
      "48\t6\t0.0\t1\t5 1\n",
      "49\t9\t0.0\t1\t9\n",
      "50\t5\t0.0\t1\t5\n",
      "51\t10\t0.0\t1\t10\n",
      "52\t6\t0.0\t1\t5 1\n",
      "53\t4\t0.0\t1\t4\n",
      "54\t15\t0.0\t1\t15\n",
      "55\t3\t0.0\t1\t3\n",
      "56\t8\t0.0\t1\t7 1\n",
      "57\t26\t0.0\t1\t26\n",
      "58\t5\t0.0\t1\t5\n",
      "59\t6\t0.0\t1\t5 1\n",
      "60\t10\t0.0\t1\t10\n",
      "61\t1\t0.0\t1\t1\n",
      "62\t8\t0.0\t1\t8\n",
      "63\t22\t0.0\t1\t22\n",
      "75\t1\t0.0\t1\t1\n",
      "76\t1\t0.0\t1\t1\n",
      "77\t2\t0.0\t1\t2\n",
      "78\t12\t0.0\t1\t12\n",
      "79\t18\t0.0\t1\t18\n",
      "80\t100\t0.0\t1\t98 2\n",
      "81\t140\t0.0\t1\t140\n",
      "82\t25\t0.0\t1\t25\n",
      "83\t6\t0.0\t1\t5 1\n",
      "84\t1\t0.0\t1\t1\n",
      "\n",
      "Read1 before filtering:\n",
      "total reads: 202634\n",
      "total bases: 58405976\n",
      "Q20 bases: 58162290(99.5828%)\n",
      "Q30 bases: 57325404(98.1499%)\n",
      "\n",
      "Read2 before filtering:\n",
      "total reads: 202634\n",
      "total bases: 55617880\n",
      "Q20 bases: 54867208(98.6503%)\n",
      "Q30 bases: 52980762(95.2585%)\n",
      "\n",
      "Merged and filtered:\n",
      "total reads: 195712\n",
      "total bases: 77235288\n",
      "Q20 bases: 77004851(99.7016%)\n",
      "Q30 bases: 76131948(98.5715%)\n",
      "\n",
      "Filtering result:\n",
      "reads passed filter: 400290\n",
      "reads failed due to low quality: 222\n",
      "reads failed due to too many N: 30\n",
      "reads failed due to too short: 4726\n",
      "reads with adapter trimmed: 1102\n",
      "bases trimmed due to adapters: 10305\n",
      "reads corrected by overlap analysis: 24239\n",
      "bases corrected by overlap analysis: 36453\n",
      "\n",
      "Duplication rate: 83.5468%\n",
      "\n",
      "Insert size peak (evaluated by paired-end reads): 396\n",
      "\n",
      "Read pairs merged: 195712\n",
      "% of original read pairs: 96.584%\n",
      "% in reads after filtering: 100%\n",
      "\n",
      "\n",
      "JSON report: 2_fastp/Hrd1_DMS_Rep4_Region5_Input_report.json\n",
      "HTML report: 2_fastp/Hrd1_DMS_Rep4_Region5_Input_report.html\n",
      "\n",
      "fastp -i 1_cutadapt/Hrd1_DMS_Rep4_Region5_Input_R1_cut.fastq.gz -I 1_cutadapt/Hrd1_DMS_Rep4_Region5_Input_R2_cut.fastq.gz -m --merged_out 2_fastp/Hrd1_DMS_Rep4_Region5_Input_merge.fastq.gz -o 2_fastp/Hrd1_DMS_Rep4_Region5_Input_R1_cut.fastq.gz -O 2_fastp/Hrd1_DMS_Rep4_Region5_Input_R2_cut.fastq.gz -w 12 -h 2_fastp/Hrd1_DMS_Rep4_Region5_Input_report.html -j 2_fastp/Hrd1_DMS_Rep4_Region5_Input_report.json \n",
      "fastp v0.20.0, time used: 2 seconds\n",
      "195712 reads; of these:\n",
      "  195712 (100.00%) were unpaired; of these:\n",
      "    561 (0.29%) aligned 0 times\n",
      "    195151 (99.71%) aligned exactly 1 time\n",
      "    0 (0.00%) aligned >1 times\n",
      "99.71% overall alignment rate\n",
      "[bam_sort_core] merging from 0 files and 12 in-memory blocks...\n",
      "This is cutadapt 1.18 with Python 3.7.16\n",
      "Command line parameters: -a CTGTCTCTTATACACATCT -A CTGTCTCTTATACACATCT -u 8 -U 8 -q 30 -o 1_cutadapt/Hrd1_DMS_Rep1_Region5_WT_R1_cut.fastq.gz -p 1_cutadapt/Hrd1_DMS_Rep1_Region5_WT_R2_cut.fastq.gz --cores=12 0_fastq_files/Hrd1_DMS_Rep1_Region5_WT_R1_001.fastq.gz 0_fastq_files/Hrd1_DMS_Rep1_Region5_WT_R2_001.fastq.gz\n",
      "Processing reads on 12 cores in paired-end mode ...\n",
      "Finished in 1.48 s (10 us/read; 6.11 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            151,234\n",
      "  Read 1 with adapter:                 126,713 (83.8%)\n",
      "  Read 2 with adapter:                     150 (0.1%)\n",
      "Pairs written (passing filters):       151,234 (100.0%)\n",
      "\n",
      "Total basepairs processed:    90,734,403 bp\n",
      "  Read 1:    45,368,682 bp\n",
      "  Read 2:    45,365,721 bp\n",
      "Quality-trimmed:               2,925,740 bp (3.2%)\n",
      "  Read 1:       586,341 bp\n",
      "  Read 2:     2,339,399 bp\n",
      "Total written (filtered):     85,007,515 bp (93.7%)\n",
      "  Read 1:    43,191,814 bp\n",
      "  Read 2:    41,815,701 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: CTGTCTCTTATACACATCT; Type: regular 3'; Length: 19; Trimmed: 126713 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 0.5%\n",
      "  C: 0.7%\n",
      "  G: 0.5%\n",
      "  T: 98.3%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by \"T\" extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 3' end.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t126578\t2363.0\t0\t126578\n",
      "4\t108\t590.8\t0\t108\n",
      "5\t22\t147.7\t0\t22\n",
      "75\t1\t0.0\t1\t1\n",
      "76\t4\t0.0\t1\t4\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CTGTCTCTTATACACATCT; Type: regular 3'; Length: 19; Trimmed: 150 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t120\t2363.0\t0\t120\n",
      "4\t23\t590.8\t0\t23\n",
      "7\t1\t9.2\t0\t1\n",
      "8\t1\t2.3\t0\t1\n",
      "39\t1\t0.0\t1\t1\n",
      "47\t1\t0.0\t1\t1\n",
      "57\t1\t0.0\t1\t1\n",
      "59\t1\t0.0\t1\t1\n",
      "80\t1\t0.0\t1\t1\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "Read1 before filtering:\n",
      "total reads: 151234\n",
      "total bases: 43191814\n",
      "Q20 bases: 42933930(99.4029%)\n",
      "Q30 bases: 42109514(97.4942%)\n",
      "\n",
      "Read2 before filtering:\n",
      "total reads: 151234\n",
      "total bases: 41815701\n",
      "Q20 bases: 41272883(98.7019%)\n",
      "Q30 bases: 39896658(95.4107%)\n",
      "\n",
      "Merged and filtered:\n",
      "total reads: 146065\n",
      "total bases: 57485074\n",
      "Q20 bases: 57255038(99.5998%)\n",
      "Q30 bases: 56418638(98.1448%)\n",
      "\n",
      "Filtering result:\n",
      "reads passed filter: 299122\n",
      "reads failed due to low quality: 174\n",
      "reads failed due to too many N: 20\n",
      "reads failed due to too short: 3152\n",
      "reads with adapter trimmed: 32\n",
      "bases trimmed due to adapters: 2182\n",
      "reads corrected by overlap analysis: 22808\n",
      "bases corrected by overlap analysis: 31855\n",
      "\n",
      "Duplication rate: 87.4891%\n",
      "\n",
      "Insert size peak (evaluated by paired-end reads): 394\n",
      "\n",
      "Read pairs merged: 146065\n",
      "% of original read pairs: 96.5821%\n",
      "% in reads after filtering: 100%\n",
      "\n",
      "\n",
      "JSON report: 2_fastp/Hrd1_DMS_Rep1_Region5_WT_report.json\n",
      "HTML report: 2_fastp/Hrd1_DMS_Rep1_Region5_WT_report.html\n",
      "\n",
      "fastp -i 1_cutadapt/Hrd1_DMS_Rep1_Region5_WT_R1_cut.fastq.gz -I 1_cutadapt/Hrd1_DMS_Rep1_Region5_WT_R2_cut.fastq.gz -m --merged_out 2_fastp/Hrd1_DMS_Rep1_Region5_WT_merge.fastq.gz -o 2_fastp/Hrd1_DMS_Rep1_Region5_WT_R1_cut.fastq.gz -O 2_fastp/Hrd1_DMS_Rep1_Region5_WT_R2_cut.fastq.gz -w 12 -h 2_fastp/Hrd1_DMS_Rep1_Region5_WT_report.html -j 2_fastp/Hrd1_DMS_Rep1_Region5_WT_report.json \n",
      "fastp v0.20.0, time used: 1 seconds\n",
      "146065 reads; of these:\n",
      "  146065 (100.00%) were unpaired; of these:\n",
      "    28 (0.02%) aligned 0 times\n",
      "    146037 (99.98%) aligned exactly 1 time\n",
      "    0 (0.00%) aligned >1 times\n",
      "99.98% overall alignment rate\n",
      "[bam_sort_core] merging from 0 files and 12 in-memory blocks...\n",
      "This is cutadapt 1.18 with Python 3.7.16\n",
      "Command line parameters: -a CTGTCTCTTATACACATCT -A CTGTCTCTTATACACATCT -u 8 -U 8 -q 30 -o 1_cutadapt/Hrd1_DMS_Rep2_Region5_WT_R1_cut.fastq.gz -p 1_cutadapt/Hrd1_DMS_Rep2_Region5_WT_R2_cut.fastq.gz --cores=12 0_fastq_files/Hrd1_DMS_Rep2_Region5_WT_R1_001.fastq.gz 0_fastq_files/Hrd1_DMS_Rep2_Region5_WT_R2_001.fastq.gz\n",
      "Processing reads on 12 cores in paired-end mode ...\n",
      "Finished in 1.54 s (10 us/read; 6.08 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            156,107\n",
      "  Read 1 with adapter:                     248 (0.2%)\n",
      "  Read 2 with adapter:                  72,298 (46.3%)\n",
      "Pairs written (passing filters):       156,107 (100.0%)\n",
      "\n",
      "Total basepairs processed:    93,657,555 bp\n",
      "  Read 1:    46,828,605 bp\n",
      "  Read 2:    46,828,950 bp\n",
      "Quality-trimmed:               3,661,992 bp (3.9%)\n",
      "  Read 1:     1,060,194 bp\n",
      "  Read 2:     2,601,798 bp\n",
      "Total written (filtered):     87,276,274 bp (93.2%)\n",
      "  Read 1:    44,517,236 bp\n",
      "  Read 2:    42,759,038 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: CTGTCTCTTATACACATCT; Type: regular 3'; Length: 19; Trimmed: 248 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 8.1%\n",
      "  C: 8.9%\n",
      "  G: 35.1%\n",
      "  T: 48.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t189\t2439.2\t0\t189\n",
      "4\t14\t609.8\t0\t14\n",
      "6\t1\t38.1\t0\t1\n",
      "8\t1\t2.4\t0\t1\n",
      "9\t21\t0.6\t0\t21\n",
      "11\t2\t0.0\t1\t2\n",
      "41\t1\t0.0\t1\t1\n",
      "68\t1\t0.0\t1\t1\n",
      "71\t1\t0.0\t1\t1\n",
      "75\t3\t0.0\t1\t1 2\n",
      "76\t12\t0.0\t1\t12\n",
      "77\t2\t0.0\t1\t1 1\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CTGTCTCTTATACACATCT; Type: regular 3'; Length: 19; Trimmed: 72298 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 0.9%\n",
      "  C: 2.3%\n",
      "  G: 0.5%\n",
      "  T: 96.2%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by \"T\" extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 3' end.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t71510\t2439.2\t0\t71510\n",
      "4\t464\t609.8\t0\t464\n",
      "5\t284\t152.4\t0\t284\n",
      "6\t1\t38.1\t0\t1\n",
      "8\t2\t2.4\t0\t2\n",
      "9\t17\t0.6\t0\t17\n",
      "11\t1\t0.0\t1\t1\n",
      "44\t1\t0.0\t1\t1\n",
      "48\t1\t0.0\t1\t1\n",
      "51\t1\t0.0\t1\t1\n",
      "53\t1\t0.0\t1\t1\n",
      "54\t2\t0.0\t1\t2\n",
      "58\t2\t0.0\t1\t2\n",
      "59\t1\t0.0\t1\t1\n",
      "63\t1\t0.0\t1\t1\n",
      "78\t1\t0.0\t1\t1\n",
      "80\t3\t0.0\t1\t3\n",
      "81\t4\t0.0\t1\t4\n",
      "82\t1\t0.0\t1\t1\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "Read1 before filtering:\n",
      "total reads: 156107\n",
      "total bases: 44517236\n",
      "Q20 bases: 44109822(99.0848%)\n",
      "Q30 bases: 42895079(96.3561%)\n",
      "\n",
      "Read2 before filtering:\n",
      "total reads: 156107\n",
      "total bases: 42759038\n",
      "Q20 bases: 42285831(98.8933%)\n",
      "Q30 bases: 40892648(95.6351%)\n",
      "\n",
      "Merged and filtered:\n",
      "total reads: 150069\n",
      "total bases: 59042870\n",
      "Q20 bases: 58660719(99.3528%)\n",
      "Q30 bases: 57417176(97.2466%)\n",
      "\n",
      "Filtering result:\n",
      "reads passed filter: 309448\n",
      "reads failed due to low quality: 240\n",
      "reads failed due to too many N: 26\n",
      "reads failed due to too short: 2500\n",
      "reads with adapter trimmed: 90\n",
      "bases trimmed due to adapters: 972\n",
      "reads corrected by overlap analysis: 22949\n",
      "bases corrected by overlap analysis: 30547\n",
      "\n",
      "Duplication rate: 47.7663%\n",
      "\n",
      "Insert size peak (evaluated by paired-end reads): 394\n",
      "\n",
      "Read pairs merged: 150069\n",
      "% of original read pairs: 96.1321%\n",
      "% in reads after filtering: 100%\n",
      "\n",
      "\n",
      "JSON report: 2_fastp/Hrd1_DMS_Rep2_Region5_WT_report.json\n",
      "HTML report: 2_fastp/Hrd1_DMS_Rep2_Region5_WT_report.html\n",
      "\n",
      "fastp -i 1_cutadapt/Hrd1_DMS_Rep2_Region5_WT_R1_cut.fastq.gz -I 1_cutadapt/Hrd1_DMS_Rep2_Region5_WT_R2_cut.fastq.gz -m --merged_out 2_fastp/Hrd1_DMS_Rep2_Region5_WT_merge.fastq.gz -o 2_fastp/Hrd1_DMS_Rep2_Region5_WT_R1_cut.fastq.gz -O 2_fastp/Hrd1_DMS_Rep2_Region5_WT_R2_cut.fastq.gz -w 12 -h 2_fastp/Hrd1_DMS_Rep2_Region5_WT_report.html -j 2_fastp/Hrd1_DMS_Rep2_Region5_WT_report.json \n",
      "fastp v0.20.0, time used: 2 seconds\n",
      "150069 reads; of these:\n",
      "  150069 (100.00%) were unpaired; of these:\n",
      "    30 (0.02%) aligned 0 times\n",
      "    150039 (99.98%) aligned exactly 1 time\n",
      "    0 (0.00%) aligned >1 times\n",
      "99.98% overall alignment rate\n",
      "[bam_sort_core] merging from 0 files and 12 in-memory blocks...\n",
      "This is cutadapt 1.18 with Python 3.7.16\n",
      "Command line parameters: -a CTGTCTCTTATACACATCT -A CTGTCTCTTATACACATCT -u 8 -U 8 -q 30 -o 1_cutadapt/Hrd1_DMS_Rep4_Region5_WT_R1_cut.fastq.gz -p 1_cutadapt/Hrd1_DMS_Rep4_Region5_WT_R2_cut.fastq.gz --cores=12 0_fastq_files/Hrd1_DMS_Rep4_Region5_WT_R1_001.fastq.gz 0_fastq_files/Hrd1_DMS_Rep4_Region5_WT_R2_001.fastq.gz\n",
      "Processing reads on 12 cores in paired-end mode ...\n",
      "Finished in 1.35 s (9 us/read; 6.61 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            148,925\n",
      "  Read 1 with adapter:                     495 (0.3%)\n",
      "  Read 2 with adapter:                     166 (0.1%)\n",
      "Pairs written (passing filters):       148,925 (100.0%)\n",
      "\n",
      "Total basepairs processed:    89,349,616 bp\n",
      "  Read 1:    44,676,240 bp\n",
      "  Read 2:    44,673,376 bp\n",
      "Quality-trimmed:               2,978,451 bp (3.3%)\n",
      "  Read 1:       659,678 bp\n",
      "  Read 2:     2,318,773 bp\n",
      "Total written (filtered):     83,986,044 bp (94.0%)\n",
      "  Read 1:    42,823,410 bp\n",
      "  Read 2:    41,162,634 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: CTGTCTCTTATACACATCT; Type: regular 3'; Length: 19; Trimmed: 495 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 9.7%\n",
      "  C: 40.4%\n",
      "  G: 0.0%\n",
      "  T: 49.9%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t344\t2327.0\t0\t344\n",
      "4\t109\t581.7\t0\t109\n",
      "5\t40\t145.4\t0\t40\n",
      "7\t1\t9.1\t0\t1\n",
      "77\t1\t0.0\t1\t1\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CTGTCTCTTATACACATCT; Type: regular 3'; Length: 19; Trimmed: 166 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 3.0%\n",
      "  C: 4.2%\n",
      "  G: 62.0%\n",
      "  T: 30.7%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t155\t2327.0\t0\t155\n",
      "4\t9\t581.7\t0\t9\n",
      "7\t1\t9.1\t0\t1\n",
      "61\t1\t0.0\t1\t1\n",
      "\n",
      "Read1 before filtering:\n",
      "total reads: 148925\n",
      "total bases: 42823410\n",
      "Q20 bases: 42595751(99.4684%)\n",
      "Q30 bases: 41817216(97.6504%)\n",
      "\n",
      "Read2 before filtering:\n",
      "total reads: 148925\n",
      "total bases: 41162634\n",
      "Q20 bases: 40586823(98.6011%)\n",
      "Q30 bases: 39174175(95.1693%)\n",
      "\n",
      "Merged and filtered:\n",
      "total reads: 143913\n",
      "total bases: 56931983\n",
      "Q20 bases: 56719347(99.6265%)\n",
      "Q30 bases: 55917027(98.2172%)\n",
      "\n",
      "Filtering result:\n",
      "reads passed filter: 294816\n",
      "reads failed due to low quality: 206\n",
      "reads failed due to too many N: 22\n",
      "reads failed due to too short: 2806\n",
      "reads with adapter trimmed: 12\n",
      "bases trimmed due to adapters: 782\n",
      "reads corrected by overlap analysis: 20532\n",
      "bases corrected by overlap analysis: 29807\n",
      "\n",
      "Duplication rate: 85.6604%\n",
      "\n",
      "Insert size peak (evaluated by paired-end reads): 396\n",
      "\n",
      "Read pairs merged: 143913\n",
      "% of original read pairs: 96.6345%\n",
      "% in reads after filtering: 100%\n",
      "\n",
      "\n",
      "JSON report: 2_fastp/Hrd1_DMS_Rep4_Region5_WT_report.json\n",
      "HTML report: 2_fastp/Hrd1_DMS_Rep4_Region5_WT_report.html\n",
      "\n",
      "fastp -i 1_cutadapt/Hrd1_DMS_Rep4_Region5_WT_R1_cut.fastq.gz -I 1_cutadapt/Hrd1_DMS_Rep4_Region5_WT_R2_cut.fastq.gz -m --merged_out 2_fastp/Hrd1_DMS_Rep4_Region5_WT_merge.fastq.gz -o 2_fastp/Hrd1_DMS_Rep4_Region5_WT_R1_cut.fastq.gz -O 2_fastp/Hrd1_DMS_Rep4_Region5_WT_R2_cut.fastq.gz -w 12 -h 2_fastp/Hrd1_DMS_Rep4_Region5_WT_report.html -j 2_fastp/Hrd1_DMS_Rep4_Region5_WT_report.json \n",
      "fastp v0.20.0, time used: 1 seconds\n",
      "143913 reads; of these:\n",
      "  143913 (100.00%) were unpaired; of these:\n",
      "    13 (0.01%) aligned 0 times\n",
      "    143900 (99.99%) aligned exactly 1 time\n",
      "    0 (0.00%) aligned >1 times\n",
      "99.99% overall alignment rate\n",
      "[bam_sort_core] merging from 0 files and 12 in-memory blocks...\n",
      "This is cutadapt 1.18 with Python 3.7.16\n",
      "Command line parameters: -a CTGTCTCTTATACACATCT -A CTGTCTCTTATACACATCT -u 8 -U 8 -q 30 -o 1_cutadapt/Hrd1_DMS_Rep1_Region5_LDead_R1_cut.fastq.gz -p 1_cutadapt/Hrd1_DMS_Rep1_Region5_LDead_R2_cut.fastq.gz --cores=12 0_fastq_files/Hrd1_DMS_Rep1_Region5_LDead_R1_001.fastq.gz 0_fastq_files/Hrd1_DMS_Rep1_Region5_LDead_R2_001.fastq.gz\n",
      "Processing reads on 12 cores in paired-end mode ...\n",
      "Finished in 0.37 s (16 us/read; 3.69 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             22,681\n",
      "  Read 1 with adapter:                  17,920 (79.0%)\n",
      "  Read 2 with adapter:                   3,477 (15.3%)\n",
      "Pairs written (passing filters):        22,681 (100.0%)\n",
      "\n",
      "Total basepairs processed:    13,603,773 bp\n",
      "  Read 1:     6,802,009 bp\n",
      "  Read 2:     6,801,764 bp\n",
      "Quality-trimmed:               1,429,462 bp (10.5%)\n",
      "  Read 1:       549,062 bp\n",
      "  Read 2:       880,400 bp\n",
      "Total written (filtered):     11,270,493 bp (82.8%)\n",
      "  Read 1:     5,767,139 bp\n",
      "  Read 2:     5,503,354 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: CTGTCTCTTATACACATCT; Type: regular 3'; Length: 19; Trimmed: 17920 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 6.3%\n",
      "  C: 5.0%\n",
      "  G: 4.1%\n",
      "  T: 84.6%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by \"T\" extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 3' end.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t14445\t354.4\t0\t14445\n",
      "4\t20\t88.6\t0\t20\n",
      "5\t12\t22.1\t0\t12\n",
      "8\t1\t0.3\t0\t1\n",
      "11\t2\t0.0\t1\t2\n",
      "13\t2\t0.0\t1\t2\n",
      "15\t1\t0.0\t1\t1\n",
      "16\t1\t0.0\t1\t1\n",
      "17\t5\t0.0\t1\t5\n",
      "19\t2\t0.0\t1\t2\n",
      "20\t2\t0.0\t1\t1 1\n",
      "21\t5\t0.0\t1\t3 2\n",
      "22\t2\t0.0\t1\t2\n",
      "24\t1\t0.0\t1\t1\n",
      "25\t1\t0.0\t1\t1\n",
      "28\t1\t0.0\t1\t1\n",
      "29\t6\t0.0\t1\t6\n",
      "30\t4\t0.0\t1\t4\n",
      "32\t1\t0.0\t1\t1\n",
      "33\t1\t0.0\t1\t1\n",
      "34\t3\t0.0\t1\t3\n",
      "35\t6\t0.0\t1\t5 1\n",
      "36\t5\t0.0\t1\t5\n",
      "37\t2\t0.0\t1\t2\n",
      "38\t4\t0.0\t1\t4\n",
      "39\t3\t0.0\t1\t1 2\n",
      "40\t8\t0.0\t1\t6 2\n",
      "41\t2\t0.0\t1\t1 1\n",
      "42\t5\t0.0\t1\t5\n",
      "43\t13\t0.0\t1\t9 4\n",
      "44\t1\t0.0\t1\t1\n",
      "45\t1\t0.0\t1\t1\n",
      "49\t3\t0.0\t1\t2 1\n",
      "51\t3\t0.0\t1\t3\n",
      "52\t1\t0.0\t1\t1\n",
      "53\t1\t0.0\t1\t0 1\n",
      "54\t5\t0.0\t1\t4 1\n",
      "55\t1\t0.0\t1\t0 1\n",
      "57\t1\t0.0\t1\t1\n",
      "58\t6\t0.0\t1\t6\n",
      "59\t4\t0.0\t1\t4\n",
      "60\t3\t0.0\t1\t3\n",
      "61\t6\t0.0\t1\t5 1\n",
      "62\t7\t0.0\t1\t7\n",
      "63\t6\t0.0\t1\t6\n",
      "64\t5\t0.0\t1\t4 1\n",
      "65\t11\t0.0\t1\t10 1\n",
      "66\t8\t0.0\t1\t5 3\n",
      "67\t9\t0.0\t1\t8 1\n",
      "68\t5\t0.0\t1\t2 3\n",
      "69\t10\t0.0\t1\t10\n",
      "70\t11\t0.0\t1\t10 1\n",
      "71\t20\t0.0\t1\t20\n",
      "72\t23\t0.0\t1\t22 1\n",
      "73\t27\t0.0\t1\t24 3\n",
      "74\t45\t0.0\t1\t40 5\n",
      "75\t125\t0.0\t1\t106 19\n",
      "76\t1991\t0.0\t1\t1754 237\n",
      "77\t862\t0.0\t1\t752 110\n",
      "78\t106\t0.0\t1\t84 22\n",
      "79\t7\t0.0\t1\t7\n",
      "80\t5\t0.0\t1\t5\n",
      "81\t1\t0.0\t1\t1\n",
      "82\t1\t0.0\t1\t1\n",
      "94\t1\t0.0\t1\t1\n",
      "111\t1\t0.0\t1\t1\n",
      "134\t1\t0.0\t1\t1\n",
      "137\t2\t0.0\t1\t2\n",
      "171\t1\t0.0\t1\t1\n",
      "177\t1\t0.0\t1\t1\n",
      "180\t1\t0.0\t1\t0 1\n",
      "185\t1\t0.0\t1\t1\n",
      "189\t1\t0.0\t1\t1\n",
      "198\t1\t0.0\t1\t1\n",
      "199\t1\t0.0\t1\t1\n",
      "204\t1\t0.0\t1\t1\n",
      "212\t1\t0.0\t1\t0 1\n",
      "213\t2\t0.0\t1\t2\n",
      "215\t2\t0.0\t1\t1 1\n",
      "216\t5\t0.0\t1\t4 1\n",
      "217\t10\t0.0\t1\t8 2\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CTGTCTCTTATACACATCT; Type: regular 3'; Length: 19; Trimmed: 3477 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t36\t354.4\t0\t36\n",
      "4\t7\t88.6\t0\t7\n",
      "5\t1\t22.1\t0\t1\n",
      "9\t1\t0.1\t0\t1\n",
      "10\t1\t0.0\t1\t1\n",
      "12\t1\t0.0\t1\t1\n",
      "13\t1\t0.0\t1\t1\n",
      "14\t2\t0.0\t1\t2\n",
      "15\t3\t0.0\t1\t3\n",
      "16\t2\t0.0\t1\t1 1\n",
      "17\t1\t0.0\t1\t1\n",
      "18\t2\t0.0\t1\t2\n",
      "19\t6\t0.0\t1\t6\n",
      "21\t6\t0.0\t1\t6\n",
      "22\t6\t0.0\t1\t6\n",
      "23\t1\t0.0\t1\t1\n",
      "24\t4\t0.0\t1\t4\n",
      "25\t2\t0.0\t1\t2\n",
      "26\t10\t0.0\t1\t10\n",
      "27\t2\t0.0\t1\t2\n",
      "28\t5\t0.0\t1\t5\n",
      "29\t10\t0.0\t1\t10\n",
      "30\t9\t0.0\t1\t8 1\n",
      "31\t11\t0.0\t1\t11\n",
      "32\t15\t0.0\t1\t15\n",
      "33\t57\t0.0\t1\t54 3\n",
      "34\t43\t0.0\t1\t42 1\n",
      "35\t15\t0.0\t1\t13 2\n",
      "36\t53\t0.0\t1\t51 2\n",
      "37\t1\t0.0\t1\t1\n",
      "38\t8\t0.0\t1\t8\n",
      "39\t15\t0.0\t1\t15\n",
      "40\t4\t0.0\t1\t4\n",
      "41\t28\t0.0\t1\t28\n",
      "42\t13\t0.0\t1\t13\n",
      "43\t32\t0.0\t1\t31 1\n",
      "44\t5\t0.0\t1\t5\n",
      "45\t16\t0.0\t1\t15 1\n",
      "46\t54\t0.0\t1\t52 2\n",
      "47\t19\t0.0\t1\t18 1\n",
      "48\t18\t0.0\t1\t18\n",
      "49\t40\t0.0\t1\t40\n",
      "50\t18\t0.0\t1\t18\n",
      "51\t34\t0.0\t1\t33 1\n",
      "52\t19\t0.0\t1\t19\n",
      "53\t24\t0.0\t1\t23 1\n",
      "54\t46\t0.0\t1\t46\n",
      "55\t43\t0.0\t1\t43\n",
      "56\t41\t0.0\t1\t41\n",
      "57\t58\t0.0\t1\t58\n",
      "58\t71\t0.0\t1\t71\n",
      "59\t78\t0.0\t1\t76 2\n",
      "60\t36\t0.0\t1\t36\n",
      "61\t39\t0.0\t1\t38 1\n",
      "62\t49\t0.0\t1\t49\n",
      "63\t109\t0.0\t1\t108 1\n",
      "64\t28\t0.0\t1\t28\n",
      "65\t30\t0.0\t1\t29 1\n",
      "66\t5\t0.0\t1\t5\n",
      "67\t10\t0.0\t1\t10\n",
      "68\t5\t0.0\t1\t5\n",
      "69\t1\t0.0\t1\t1\n",
      "70\t8\t0.0\t1\t8\n",
      "71\t4\t0.0\t1\t4\n",
      "72\t5\t0.0\t1\t5\n",
      "73\t5\t0.0\t1\t5\n",
      "74\t5\t0.0\t1\t5\n",
      "75\t5\t0.0\t1\t5\n",
      "76\t11\t0.0\t1\t11\n",
      "77\t75\t0.0\t1\t73 2\n",
      "78\t116\t0.0\t1\t111 5\n",
      "79\t229\t0.0\t1\t224 5\n",
      "80\t1115\t0.0\t1\t1099 16\n",
      "81\t476\t0.0\t1\t473 3\n",
      "82\t92\t0.0\t1\t91 1\n",
      "83\t18\t0.0\t1\t16 2\n",
      "84\t3\t0.0\t1\t3\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "Read1 before filtering:\n",
      "total reads: 22681\n",
      "total bases: 5767139\n",
      "Q20 bases: 5740302(99.5347%)\n",
      "Q30 bases: 5654535(98.0475%)\n",
      "\n",
      "Read2 before filtering:\n",
      "total reads: 22681\n",
      "total bases: 5503354\n",
      "Q20 bases: 5426209(98.5982%)\n",
      "Q30 bases: 5235752(95.1375%)\n",
      "\n",
      "Merged and filtered:\n",
      "total reads: 21914\n",
      "total bases: 7492072\n",
      "Q20 bases: 7468584(99.6865%)\n",
      "Q30 bases: 7382293(98.5347%)\n",
      "\n",
      "Filtering result:\n",
      "reads passed filter: 44778\n",
      "reads failed due to low quality: 22\n",
      "reads failed due to too many N: 0\n",
      "reads failed due to too short: 562\n",
      "reads with adapter trimmed: 6872\n",
      "bases trimmed due to adapters: 55680\n",
      "reads corrected by overlap analysis: 2926\n",
      "bases corrected by overlap analysis: 4226\n",
      "\n",
      "Duplication rate: 65.0943%\n",
      "\n",
      "Insert size peak (evaluated by paired-end reads): 394\n",
      "\n",
      "Read pairs merged: 21914\n",
      "% of original read pairs: 96.6183%\n",
      "% in reads after filtering: 100%\n",
      "\n",
      "\n",
      "JSON report: 2_fastp/Hrd1_DMS_Rep1_Region5_LDead_report.json\n",
      "HTML report: 2_fastp/Hrd1_DMS_Rep1_Region5_LDead_report.html\n",
      "\n",
      "fastp -i 1_cutadapt/Hrd1_DMS_Rep1_Region5_LDead_R1_cut.fastq.gz -I 1_cutadapt/Hrd1_DMS_Rep1_Region5_LDead_R2_cut.fastq.gz -m --merged_out 2_fastp/Hrd1_DMS_Rep1_Region5_LDead_merge.fastq.gz -o 2_fastp/Hrd1_DMS_Rep1_Region5_LDead_R1_cut.fastq.gz -O 2_fastp/Hrd1_DMS_Rep1_Region5_LDead_R2_cut.fastq.gz -w 12 -h 2_fastp/Hrd1_DMS_Rep1_Region5_LDead_report.html -j 2_fastp/Hrd1_DMS_Rep1_Region5_LDead_report.json \n",
      "fastp v0.20.0, time used: 0 seconds\n",
      "21914 reads; of these:\n",
      "  21914 (100.00%) were unpaired; of these:\n",
      "    3448 (15.73%) aligned 0 times\n",
      "    18466 (84.27%) aligned exactly 1 time\n",
      "    0 (0.00%) aligned >1 times\n",
      "84.27% overall alignment rate\n",
      "[bam_sort_core] merging from 0 files and 12 in-memory blocks...\n",
      "This is cutadapt 1.18 with Python 3.7.16\n",
      "Command line parameters: -a CTGTCTCTTATACACATCT -A CTGTCTCTTATACACATCT -u 8 -U 8 -q 30 -o 1_cutadapt/Hrd1_DMS_Rep2_Region5_LDead_R1_cut.fastq.gz -p 1_cutadapt/Hrd1_DMS_Rep2_Region5_LDead_R2_cut.fastq.gz --cores=12 0_fastq_files/Hrd1_DMS_Rep2_Region5_LDead_R1_001.fastq.gz 0_fastq_files/Hrd1_DMS_Rep2_Region5_LDead_R2_001.fastq.gz\n",
      "Processing reads on 12 cores in paired-end mode ...\n",
      "Finished in 0.43 s (20 us/read; 3.07 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             21,837\n",
      "  Read 1 with adapter:                   3,494 (16.0%)\n",
      "  Read 2 with adapter:                  10,766 (49.3%)\n",
      "Pairs written (passing filters):        21,837 (100.0%)\n",
      "\n",
      "Total basepairs processed:    13,096,910 bp\n",
      "  Read 1:     6,548,113 bp\n",
      "  Read 2:     6,548,797 bp\n",
      "Quality-trimmed:               1,441,858 bp (11.0%)\n",
      "  Read 1:       589,821 bp\n",
      "  Read 2:       852,037 bp\n",
      "Total written (filtered):     10,776,716 bp (82.3%)\n",
      "  Read 1:     5,522,587 bp\n",
      "  Read 2:     5,254,129 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: CTGTCTCTTATACACATCT; Type: regular 3'; Length: 19; Trimmed: 3494 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 31.1%\n",
      "  C: 24.5%\n",
      "  G: 18.6%\n",
      "  T: 25.8%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t24\t341.2\t0\t24\n",
      "4\t1\t85.3\t0\t1\n",
      "5\t2\t21.3\t0\t2\n",
      "6\t1\t5.3\t0\t1\n",
      "8\t3\t0.3\t0\t3\n",
      "9\t3\t0.1\t0\t3\n",
      "13\t1\t0.0\t1\t1\n",
      "15\t1\t0.0\t1\t1\n",
      "17\t2\t0.0\t1\t2\n",
      "18\t1\t0.0\t1\t1\n",
      "19\t2\t0.0\t1\t2\n",
      "20\t5\t0.0\t1\t5\n",
      "21\t2\t0.0\t1\t2\n",
      "22\t2\t0.0\t1\t2\n",
      "23\t2\t0.0\t1\t2\n",
      "26\t1\t0.0\t1\t1\n",
      "27\t2\t0.0\t1\t2\n",
      "28\t1\t0.0\t1\t1\n",
      "29\t2\t0.0\t1\t1 1\n",
      "30\t3\t0.0\t1\t2 1\n",
      "31\t3\t0.0\t1\t2 1\n",
      "32\t2\t0.0\t1\t2\n",
      "34\t1\t0.0\t1\t1\n",
      "35\t5\t0.0\t1\t3 2\n",
      "36\t7\t0.0\t1\t7\n",
      "37\t9\t0.0\t1\t9\n",
      "38\t6\t0.0\t1\t6\n",
      "39\t10\t0.0\t1\t7 3\n",
      "40\t3\t0.0\t1\t3\n",
      "41\t5\t0.0\t1\t4 1\n",
      "42\t8\t0.0\t1\t6 2\n",
      "43\t30\t0.0\t1\t26 4\n",
      "44\t1\t0.0\t1\t1\n",
      "45\t2\t0.0\t1\t1 1\n",
      "47\t1\t0.0\t1\t1\n",
      "49\t2\t0.0\t1\t2\n",
      "51\t2\t0.0\t1\t2\n",
      "52\t2\t0.0\t1\t1 1\n",
      "53\t1\t0.0\t1\t1\n",
      "54\t10\t0.0\t1\t9 1\n",
      "55\t2\t0.0\t1\t1 1\n",
      "57\t4\t0.0\t1\t4\n",
      "58\t2\t0.0\t1\t2\n",
      "59\t5\t0.0\t1\t5\n",
      "60\t4\t0.0\t1\t2 2\n",
      "61\t9\t0.0\t1\t8 1\n",
      "62\t4\t0.0\t1\t3 1\n",
      "63\t7\t0.0\t1\t6 1\n",
      "64\t7\t0.0\t1\t6 1\n",
      "65\t16\t0.0\t1\t14 2\n",
      "66\t19\t0.0\t1\t17 2\n",
      "67\t9\t0.0\t1\t8 1\n",
      "68\t5\t0.0\t1\t5\n",
      "69\t15\t0.0\t1\t14 1\n",
      "70\t26\t0.0\t1\t23 3\n",
      "71\t31\t0.0\t1\t25 6\n",
      "72\t28\t0.0\t1\t26 2\n",
      "73\t54\t0.0\t1\t48 6\n",
      "74\t56\t0.0\t1\t49 7\n",
      "75\t208\t0.0\t1\t171 37\n",
      "76\t1702\t0.0\t1\t1500 202\n",
      "77\t916\t0.0\t1\t796 120\n",
      "78\t127\t0.0\t1\t108 19\n",
      "79\t19\t0.0\t1\t16 3\n",
      "80\t8\t0.0\t1\t7 1\n",
      "81\t6\t0.0\t1\t5 1\n",
      "84\t2\t0.0\t1\t2\n",
      "113\t1\t0.0\t1\t1\n",
      "126\t2\t0.0\t1\t1 1\n",
      "143\t1\t0.0\t1\t1\n",
      "150\t1\t0.0\t1\t1\n",
      "174\t1\t0.0\t1\t0 1\n",
      "177\t1\t0.0\t1\t1\n",
      "212\t1\t0.0\t1\t1\n",
      "213\t2\t0.0\t1\t2\n",
      "215\t2\t0.0\t1\t2\n",
      "216\t11\t0.0\t1\t7 4\n",
      "217\t8\t0.0\t1\t5 3\n",
      "218\t1\t0.0\t1\t1\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CTGTCTCTTATACACATCT; Type: regular 3'; Length: 19; Trimmed: 10766 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 11.3%\n",
      "  C: 10.6%\n",
      "  G: 5.6%\n",
      "  T: 72.5%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t7203\t341.2\t0\t7203\n",
      "4\t64\t85.3\t0\t64\n",
      "5\t41\t21.3\t0\t41\n",
      "9\t2\t0.1\t0\t2\n",
      "12\t3\t0.0\t1\t3\n",
      "13\t1\t0.0\t1\t1\n",
      "14\t2\t0.0\t1\t2\n",
      "15\t2\t0.0\t1\t2\n",
      "16\t1\t0.0\t1\t1\n",
      "17\t1\t0.0\t1\t1\n",
      "18\t2\t0.0\t1\t2\n",
      "19\t5\t0.0\t1\t4 1\n",
      "20\t2\t0.0\t1\t2\n",
      "21\t3\t0.0\t1\t3\n",
      "22\t3\t0.0\t1\t2 1\n",
      "23\t3\t0.0\t1\t2 1\n",
      "24\t5\t0.0\t1\t5\n",
      "25\t4\t0.0\t1\t4\n",
      "26\t7\t0.0\t1\t7\n",
      "27\t3\t0.0\t1\t3\n",
      "28\t4\t0.0\t1\t3 1\n",
      "29\t7\t0.0\t1\t6 1\n",
      "30\t14\t0.0\t1\t14\n",
      "31\t7\t0.0\t1\t7\n",
      "32\t10\t0.0\t1\t9 1\n",
      "33\t51\t0.0\t1\t49 2\n",
      "34\t44\t0.0\t1\t43 1\n",
      "35\t16\t0.0\t1\t15 1\n",
      "36\t35\t0.0\t1\t35\n",
      "37\t4\t0.0\t1\t4\n",
      "38\t8\t0.0\t1\t8\n",
      "39\t6\t0.0\t1\t6\n",
      "40\t9\t0.0\t1\t9\n",
      "41\t21\t0.0\t1\t21\n",
      "42\t5\t0.0\t1\t4 1\n",
      "43\t19\t0.0\t1\t19\n",
      "44\t7\t0.0\t1\t7\n",
      "45\t13\t0.0\t1\t13\n",
      "46\t48\t0.0\t1\t46 2\n",
      "47\t12\t0.0\t1\t12\n",
      "48\t15\t0.0\t1\t14 1\n",
      "49\t39\t0.0\t1\t39\n",
      "50\t21\t0.0\t1\t21\n",
      "51\t27\t0.0\t1\t26 1\n",
      "52\t26\t0.0\t1\t25 1\n",
      "53\t20\t0.0\t1\t19 1\n",
      "54\t34\t0.0\t1\t33 1\n",
      "55\t43\t0.0\t1\t43\n",
      "56\t23\t0.0\t1\t23\n",
      "57\t47\t0.0\t1\t47\n",
      "58\t57\t0.0\t1\t57\n",
      "59\t49\t0.0\t1\t49\n",
      "60\t39\t0.0\t1\t38 1\n",
      "61\t25\t0.0\t1\t25\n",
      "62\t31\t0.0\t1\t31\n",
      "63\t78\t0.0\t1\t77 1\n",
      "64\t17\t0.0\t1\t16 1\n",
      "65\t26\t0.0\t1\t26\n",
      "66\t4\t0.0\t1\t4\n",
      "67\t8\t0.0\t1\t8\n",
      "69\t2\t0.0\t1\t2\n",
      "70\t12\t0.0\t1\t12\n",
      "71\t1\t0.0\t1\t1\n",
      "72\t3\t0.0\t1\t3\n",
      "73\t6\t0.0\t1\t6\n",
      "74\t5\t0.0\t1\t5\n",
      "75\t8\t0.0\t1\t8\n",
      "76\t13\t0.0\t1\t12 1\n",
      "77\t40\t0.0\t1\t40\n",
      "78\t80\t0.0\t1\t77 3\n",
      "79\t165\t0.0\t1\t159 6\n",
      "80\t1138\t0.0\t1\t1130 8\n",
      "81\t694\t0.0\t1\t687 7\n",
      "82\t205\t0.0\t1\t198 7\n",
      "83\t55\t0.0\t1\t54 1\n",
      "84\t11\t0.0\t1\t11\n",
      "85\t1\t0.0\t1\t1\n",
      "217\t1\t0.0\t1\t1\n",
      "\n",
      "Read1 before filtering:\n",
      "total reads: 21837\n",
      "total bases: 5522587\n",
      "Q20 bases: 5489146(99.3945%)\n",
      "Q30 bases: 5385680(97.521%)\n",
      "\n",
      "Read2 before filtering:\n",
      "total reads: 21837\n",
      "total bases: 5254129\n",
      "Q20 bases: 5196892(98.9106%)\n",
      "Q30 bases: 5028538(95.7064%)\n",
      "\n",
      "Merged and filtered:\n",
      "total reads: 21147\n",
      "total bases: 7183077\n",
      "Q20 bases: 7150815(99.5509%)\n",
      "Q30 bases: 7042315(98.0404%)\n",
      "\n",
      "Filtering result:\n",
      "reads passed filter: 43226\n",
      "reads failed due to low quality: 24\n",
      "reads failed due to too many N: 0\n",
      "reads failed due to too short: 424\n",
      "reads with adapter trimmed: 6958\n",
      "bases trimmed due to adapters: 57913\n",
      "reads corrected by overlap analysis: 2429\n",
      "bases corrected by overlap analysis: 3117\n",
      "\n",
      "Duplication rate: 13.6364%\n",
      "\n",
      "Insert size peak (evaluated by paired-end reads): 394\n",
      "\n",
      "Read pairs merged: 21147\n",
      "% of original read pairs: 96.8402%\n",
      "% in reads after filtering: 100%\n",
      "\n",
      "\n",
      "JSON report: 2_fastp/Hrd1_DMS_Rep2_Region5_LDead_report.json\n",
      "HTML report: 2_fastp/Hrd1_DMS_Rep2_Region5_LDead_report.html\n",
      "\n",
      "fastp -i 1_cutadapt/Hrd1_DMS_Rep2_Region5_LDead_R1_cut.fastq.gz -I 1_cutadapt/Hrd1_DMS_Rep2_Region5_LDead_R2_cut.fastq.gz -m --merged_out 2_fastp/Hrd1_DMS_Rep2_Region5_LDead_merge.fastq.gz -o 2_fastp/Hrd1_DMS_Rep2_Region5_LDead_R1_cut.fastq.gz -O 2_fastp/Hrd1_DMS_Rep2_Region5_LDead_R2_cut.fastq.gz -w 12 -h 2_fastp/Hrd1_DMS_Rep2_Region5_LDead_report.html -j 2_fastp/Hrd1_DMS_Rep2_Region5_LDead_report.json \n",
      "fastp v0.20.0, time used: 0 seconds\n",
      "21147 reads; of these:\n",
      "  21147 (100.00%) were unpaired; of these:\n",
      "    3460 (16.36%) aligned 0 times\n",
      "    17687 (83.64%) aligned exactly 1 time\n",
      "    0 (0.00%) aligned >1 times\n",
      "83.64% overall alignment rate\n",
      "[bam_sort_core] merging from 0 files and 12 in-memory blocks...\n",
      "This is cutadapt 1.18 with Python 3.7.16\n",
      "Command line parameters: -a CTGTCTCTTATACACATCT -A CTGTCTCTTATACACATCT -u 8 -U 8 -q 30 -o 1_cutadapt/Hrd1_DMS_Rep3_Region5_LDead_R1_cut.fastq.gz -p 1_cutadapt/Hrd1_DMS_Rep3_Region5_LDead_R2_cut.fastq.gz --cores=12 0_fastq_files/Hrd1_DMS_Rep3_Region5_LDead_R1_001.fastq.gz 0_fastq_files/Hrd1_DMS_Rep3_Region5_LDead_R2_001.fastq.gz\n",
      "Processing reads on 12 cores in paired-end mode ...\n",
      "Finished in 0.66 s (11 us/read; 5.48 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             60,230\n",
      "  Read 1 with adapter:                  59,434 (98.7%)\n",
      "  Read 2 with adapter:                  59,055 (98.0%)\n",
      "Pairs written (passing filters):        60,230 (100.0%)\n",
      "\n",
      "Total basepairs processed:    36,070,864 bp\n",
      "  Read 1:    18,034,951 bp\n",
      "  Read 2:    18,035,913 bp\n",
      "Quality-trimmed:              17,275,632 bp (47.9%)\n",
      "  Read 1:     8,469,007 bp\n",
      "  Read 2:     8,806,625 bp\n",
      "Total written (filtered):      9,213,567 bp (25.5%)\n",
      "  Read 1:     4,630,594 bp\n",
      "  Read 2:     4,582,973 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: CTGTCTCTTATACACATCT; Type: regular 3'; Length: 19; Trimmed: 59434 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 30.6%\n",
      "  C: 26.1%\n",
      "  G: 19.8%\n",
      "  T: 23.5%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t14\t941.1\t0\t14\n",
      "4\t17\t235.3\t0\t17\n",
      "5\t12\t58.8\t0\t12\n",
      "6\t8\t14.7\t0\t8\n",
      "7\t2\t3.7\t0\t2\n",
      "8\t12\t0.9\t0\t12\n",
      "9\t23\t0.2\t0\t23\n",
      "10\t4\t0.1\t1\t4\n",
      "11\t12\t0.0\t1\t12\n",
      "12\t20\t0.0\t1\t19 1\n",
      "13\t10\t0.0\t1\t10\n",
      "14\t11\t0.0\t1\t7 4\n",
      "15\t8\t0.0\t1\t7 1\n",
      "16\t8\t0.0\t1\t8\n",
      "17\t37\t0.0\t1\t36 1\n",
      "18\t13\t0.0\t1\t12 1\n",
      "19\t26\t0.0\t1\t25 1\n",
      "20\t23\t0.0\t1\t22 1\n",
      "21\t47\t0.0\t1\t42 5\n",
      "22\t28\t0.0\t1\t26 2\n",
      "23\t26\t0.0\t1\t26\n",
      "24\t37\t0.0\t1\t33 4\n",
      "25\t28\t0.0\t1\t25 3\n",
      "26\t16\t0.0\t1\t16\n",
      "27\t26\t0.0\t1\t23 3\n",
      "28\t49\t0.0\t1\t49\n",
      "29\t56\t0.0\t1\t51 5\n",
      "30\t93\t0.0\t1\t84 9\n",
      "31\t45\t0.0\t1\t41 4\n",
      "32\t27\t0.0\t1\t23 4\n",
      "33\t24\t0.0\t1\t24\n",
      "34\t57\t0.0\t1\t56 1\n",
      "35\t85\t0.0\t1\t78 7\n",
      "36\t164\t0.0\t1\t155 9\n",
      "37\t17\t0.0\t1\t15 2\n",
      "38\t72\t0.0\t1\t65 7\n",
      "39\t112\t0.0\t1\t108 4\n",
      "40\t115\t0.0\t1\t110 5\n",
      "41\t85\t0.0\t1\t82 3\n",
      "42\t131\t0.0\t1\t119 12\n",
      "43\t463\t0.0\t1\t440 23\n",
      "44\t33\t0.0\t1\t32 1\n",
      "45\t24\t0.0\t1\t24\n",
      "46\t15\t0.0\t1\t14 1\n",
      "47\t24\t0.0\t1\t22 2\n",
      "48\t28\t0.0\t1\t27 1\n",
      "49\t32\t0.0\t1\t32\n",
      "50\t32\t0.0\t1\t30 2\n",
      "51\t77\t0.0\t1\t76 1\n",
      "52\t53\t0.0\t1\t51 2\n",
      "53\t63\t0.0\t1\t61 2\n",
      "54\t94\t0.0\t1\t91 3\n",
      "55\t85\t0.0\t1\t83 2\n",
      "56\t153\t0.0\t1\t149 4\n",
      "57\t94\t0.0\t1\t94\n",
      "58\t51\t0.0\t1\t51\n",
      "59\t52\t0.0\t1\t50 2\n",
      "60\t124\t0.0\t1\t123 1\n",
      "61\t130\t0.0\t1\t128 2\n",
      "62\t57\t0.0\t1\t54 3\n",
      "63\t107\t0.0\t1\t105 2\n",
      "64\t119\t0.0\t1\t114 5\n",
      "65\t186\t0.0\t1\t182 4\n",
      "66\t239\t0.0\t1\t238 1\n",
      "67\t238\t0.0\t1\t234 4\n",
      "68\t202\t0.0\t1\t199 3\n",
      "69\t247\t0.0\t1\t241 6\n",
      "70\t174\t0.0\t1\t170 4\n",
      "71\t238\t0.0\t1\t234 4\n",
      "72\t262\t0.0\t1\t255 7\n",
      "73\t560\t0.0\t1\t554 6\n",
      "74\t1196\t0.0\t1\t1186 10\n",
      "75\t2549\t0.0\t1\t2446 103\n",
      "76\t35043\t0.0\t1\t34763 280\n",
      "77\t11201\t0.0\t1\t11069 132\n",
      "78\t2458\t0.0\t1\t2390 68\n",
      "79\t544\t0.0\t1\t535 9\n",
      "80\t86\t0.0\t1\t83 3\n",
      "81\t39\t0.0\t1\t39\n",
      "82\t16\t0.0\t1\t14 2\n",
      "83\t9\t0.0\t1\t9\n",
      "84\t2\t0.0\t1\t2\n",
      "85\t3\t0.0\t1\t2 1\n",
      "87\t2\t0.0\t1\t2\n",
      "92\t2\t0.0\t1\t2\n",
      "93\t1\t0.0\t1\t1\n",
      "94\t1\t0.0\t1\t1\n",
      "97\t2\t0.0\t1\t2\n",
      "98\t1\t0.0\t1\t1\n",
      "104\t1\t0.0\t1\t1\n",
      "105\t1\t0.0\t1\t1\n",
      "111\t1\t0.0\t1\t1\n",
      "114\t1\t0.0\t1\t1\n",
      "115\t1\t0.0\t1\t1\n",
      "116\t1\t0.0\t1\t1\n",
      "117\t4\t0.0\t1\t4\n",
      "119\t1\t0.0\t1\t1\n",
      "123\t1\t0.0\t1\t1\n",
      "126\t2\t0.0\t1\t2\n",
      "127\t1\t0.0\t1\t1\n",
      "128\t1\t0.0\t1\t1\n",
      "130\t2\t0.0\t1\t2\n",
      "133\t1\t0.0\t1\t1\n",
      "136\t1\t0.0\t1\t1\n",
      "143\t1\t0.0\t1\t1\n",
      "144\t2\t0.0\t1\t2\n",
      "145\t1\t0.0\t1\t1\n",
      "146\t1\t0.0\t1\t1\n",
      "147\t3\t0.0\t1\t1 2\n",
      "148\t2\t0.0\t1\t2\n",
      "150\t4\t0.0\t1\t4\n",
      "151\t1\t0.0\t1\t1\n",
      "152\t1\t0.0\t1\t1\n",
      "153\t2\t0.0\t1\t2\n",
      "154\t1\t0.0\t1\t1\n",
      "155\t1\t0.0\t1\t1\n",
      "156\t1\t0.0\t1\t1\n",
      "157\t2\t0.0\t1\t2\n",
      "158\t1\t0.0\t1\t1\n",
      "159\t1\t0.0\t1\t1\n",
      "161\t2\t0.0\t1\t1 1\n",
      "162\t2\t0.0\t1\t2\n",
      "163\t1\t0.0\t1\t1\n",
      "164\t1\t0.0\t1\t1\n",
      "165\t1\t0.0\t1\t1\n",
      "166\t1\t0.0\t1\t1\n",
      "168\t2\t0.0\t1\t2\n",
      "169\t1\t0.0\t1\t1\n",
      "171\t1\t0.0\t1\t1\n",
      "174\t1\t0.0\t1\t1\n",
      "175\t2\t0.0\t1\t2\n",
      "176\t4\t0.0\t1\t4\n",
      "177\t2\t0.0\t1\t2\n",
      "178\t1\t0.0\t1\t1\n",
      "179\t1\t0.0\t1\t1\n",
      "180\t1\t0.0\t1\t0 1\n",
      "181\t2\t0.0\t1\t2\n",
      "183\t1\t0.0\t1\t1\n",
      "187\t2\t0.0\t1\t2\n",
      "188\t3\t0.0\t1\t3\n",
      "189\t1\t0.0\t1\t1\n",
      "190\t2\t0.0\t1\t2\n",
      "191\t1\t0.0\t1\t1\n",
      "192\t1\t0.0\t1\t1\n",
      "196\t1\t0.0\t1\t1\n",
      "198\t1\t0.0\t1\t1\n",
      "199\t2\t0.0\t1\t2\n",
      "202\t1\t0.0\t1\t1\n",
      "206\t2\t0.0\t1\t2\n",
      "207\t2\t0.0\t1\t1 1\n",
      "208\t1\t0.0\t1\t1\n",
      "209\t1\t0.0\t1\t0 1\n",
      "210\t6\t0.0\t1\t5 1\n",
      "211\t10\t0.0\t1\t9 1\n",
      "212\t3\t0.0\t1\t3\n",
      "213\t14\t0.0\t1\t14\n",
      "214\t144\t0.0\t1\t140 4\n",
      "215\t143\t0.0\t1\t136 7\n",
      "216\t5\t0.0\t1\t4 1\n",
      "217\t2\t0.0\t1\t2\n",
      "221\t1\t0.0\t1\t1\n",
      "225\t1\t0.0\t1\t0 1\n",
      "228\t1\t0.0\t1\t1\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CTGTCTCTTATACACATCT; Type: regular 3'; Length: 19; Trimmed: 59055 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 34.3%\n",
      "  C: 23.0%\n",
      "  G: 17.4%\n",
      "  T: 25.3%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t17\t941.1\t0\t17\n",
      "4\t43\t235.3\t0\t43\n",
      "5\t21\t58.8\t0\t21\n",
      "6\t1\t14.7\t0\t1\n",
      "7\t2\t3.7\t0\t2\n",
      "8\t5\t0.9\t0\t5\n",
      "9\t7\t0.2\t0\t7\n",
      "10\t15\t0.1\t1\t15\n",
      "11\t12\t0.0\t1\t12\n",
      "12\t27\t0.0\t1\t26 1\n",
      "13\t28\t0.0\t1\t26 2\n",
      "14\t19\t0.0\t1\t17 2\n",
      "15\t47\t0.0\t1\t45 2\n",
      "16\t69\t0.0\t1\t66 3\n",
      "17\t83\t0.0\t1\t76 7\n",
      "18\t35\t0.0\t1\t32 3\n",
      "19\t80\t0.0\t1\t76 4\n",
      "20\t39\t0.0\t1\t36 3\n",
      "21\t27\t0.0\t1\t25 2\n",
      "22\t50\t0.0\t1\t47 3\n",
      "23\t68\t0.0\t1\t67 1\n",
      "24\t86\t0.0\t1\t83 3\n",
      "25\t69\t0.0\t1\t66 3\n",
      "26\t49\t0.0\t1\t43 6\n",
      "27\t93\t0.0\t1\t87 6\n",
      "28\t77\t0.0\t1\t70 7\n",
      "29\t100\t0.0\t1\t95 5\n",
      "30\t101\t0.0\t1\t95 6\n",
      "31\t314\t0.0\t1\t303 11\n",
      "32\t386\t0.0\t1\t371 15\n",
      "33\t85\t0.0\t1\t81 4\n",
      "34\t127\t0.0\t1\t125 2\n",
      "35\t241\t0.0\t1\t238 3\n",
      "36\t429\t0.0\t1\t412 17\n",
      "37\t546\t0.0\t1\t537 9\n",
      "38\t194\t0.0\t1\t187 7\n",
      "39\t270\t0.0\t1\t265 5\n",
      "40\t206\t0.0\t1\t202 4\n",
      "41\t177\t0.0\t1\t172 5\n",
      "42\t213\t0.0\t1\t212 1\n",
      "43\t215\t0.0\t1\t212 3\n",
      "44\t259\t0.0\t1\t256 3\n",
      "45\t223\t0.0\t1\t210 13\n",
      "46\t686\t0.0\t1\t673 13\n",
      "47\t591\t0.0\t1\t577 14\n",
      "48\t473\t0.0\t1\t467 6\n",
      "49\t585\t0.0\t1\t569 16\n",
      "50\t408\t0.0\t1\t403 5\n",
      "51\t454\t0.0\t1\t451 3\n",
      "52\t501\t0.0\t1\t493 8\n",
      "53\t580\t0.0\t1\t572 8\n",
      "54\t615\t0.0\t1\t608 7\n",
      "55\t706\t0.0\t1\t694 12\n",
      "56\t700\t0.0\t1\t693 7\n",
      "57\t1549\t0.0\t1\t1526 23\n",
      "58\t489\t0.0\t1\t482 7\n",
      "59\t296\t0.0\t1\t284 12\n",
      "60\t499\t0.0\t1\t491 8\n",
      "61\t268\t0.0\t1\t265 3\n",
      "62\t313\t0.0\t1\t303 10\n",
      "63\t1751\t0.0\t1\t1731 20\n",
      "64\t438\t0.0\t1\t428 10\n",
      "65\t343\t0.0\t1\t339 4\n",
      "66\t66\t0.0\t1\t66\n",
      "67\t169\t0.0\t1\t166 3\n",
      "68\t166\t0.0\t1\t163 3\n",
      "69\t233\t0.0\t1\t229 4\n",
      "70\t161\t0.0\t1\t160 1\n",
      "71\t100\t0.0\t1\t100\n",
      "72\t231\t0.0\t1\t228 3\n",
      "73\t165\t0.0\t1\t165\n",
      "74\t196\t0.0\t1\t191 5\n",
      "75\t771\t0.0\t1\t763 8\n",
      "76\t899\t0.0\t1\t884 15\n",
      "77\t1510\t0.0\t1\t1485 25\n",
      "78\t2653\t0.0\t1\t2628 25\n",
      "79\t4574\t0.0\t1\t4447 127\n",
      "80\t20367\t0.0\t1\t20215 152\n",
      "81\t8573\t0.0\t1\t8502 71\n",
      "82\t679\t0.0\t1\t660 19\n",
      "83\t109\t0.0\t1\t108 1\n",
      "84\t10\t0.0\t1\t10\n",
      "85\t3\t0.0\t1\t3\n",
      "86\t2\t0.0\t1\t2\n",
      "101\t1\t0.0\t1\t1\n",
      "106\t1\t0.0\t1\t1\n",
      "209\t1\t0.0\t1\t1\n",
      "213\t1\t0.0\t1\t1\n",
      "214\t3\t0.0\t1\t3\n",
      "215\t11\t0.0\t1\t10 1\n",
      "\n",
      "Read1 before filtering:\n",
      "total reads: 60230\n",
      "total bases: 4630594\n",
      "Q20 bases: 4617079(99.7081%)\n",
      "Q30 bases: 4573243(98.7615%)\n",
      "\n",
      "Read2 before filtering:\n",
      "total reads: 60230\n",
      "total bases: 4582973\n",
      "Q20 bases: 4573527(99.7939%)\n",
      "Q30 bases: 4539517(99.0518%)\n",
      "\n",
      "Merged and filtered:\n",
      "total reads: 59389\n",
      "total bases: 4099741\n",
      "Q20 bases: 4090549(99.7758%)\n",
      "Q30 bases: 4056288(98.9401%)\n",
      "\n",
      "Filtering result:\n",
      "reads passed filter: 119324\n",
      "reads failed due to low quality: 2\n",
      "reads failed due to too many N: 2\n",
      "reads failed due to too short: 1132\n",
      "reads with adapter trimmed: 118190\n",
      "bases trimmed due to adapters: 953841\n",
      "reads corrected by overlap analysis: 1360\n",
      "bases corrected by overlap analysis: 1515\n",
      "\n",
      "Duplication rate: 78.0435%\n",
      "\n",
      "Insert size peak (evaluated by paired-end reads): 69\n",
      "\n",
      "Read pairs merged: 59389\n",
      "% of original read pairs: 98.6037%\n",
      "% in reads after filtering: 100%\n",
      "\n",
      "\n",
      "JSON report: 2_fastp/Hrd1_DMS_Rep3_Region5_LDead_report.json\n",
      "HTML report: 2_fastp/Hrd1_DMS_Rep3_Region5_LDead_report.html\n",
      "\n",
      "fastp -i 1_cutadapt/Hrd1_DMS_Rep3_Region5_LDead_R1_cut.fastq.gz -I 1_cutadapt/Hrd1_DMS_Rep3_Region5_LDead_R2_cut.fastq.gz -m --merged_out 2_fastp/Hrd1_DMS_Rep3_Region5_LDead_merge.fastq.gz -o 2_fastp/Hrd1_DMS_Rep3_Region5_LDead_R1_cut.fastq.gz -O 2_fastp/Hrd1_DMS_Rep3_Region5_LDead_R2_cut.fastq.gz -w 12 -h 2_fastp/Hrd1_DMS_Rep3_Region5_LDead_report.html -j 2_fastp/Hrd1_DMS_Rep3_Region5_LDead_report.json \n",
      "fastp v0.20.0, time used: 1 seconds\n",
      "59389 reads; of these:\n",
      "  59389 (100.00%) were unpaired; of these:\n",
      "    59343 (99.92%) aligned 0 times\n",
      "    46 (0.08%) aligned exactly 1 time\n",
      "    0 (0.00%) aligned >1 times\n",
      "0.08% overall alignment rate\n",
      "[bam_sort_core] merging from 0 files and 12 in-memory blocks...\n",
      "This is cutadapt 1.18 with Python 3.7.16\n",
      "Command line parameters: -a CTGTCTCTTATACACATCT -A CTGTCTCTTATACACATCT -u 8 -U 8 -q 30 -o 1_cutadapt/Hrd1_DMS_Rep4_Region5_LDead_R1_cut.fastq.gz -p 1_cutadapt/Hrd1_DMS_Rep4_Region5_LDead_R2_cut.fastq.gz --cores=12 0_fastq_files/Hrd1_DMS_Rep4_Region5_LDead_R1_001.fastq.gz 0_fastq_files/Hrd1_DMS_Rep4_Region5_LDead_R2_001.fastq.gz\n",
      "Processing reads on 12 cores in paired-end mode ...\n",
      "Finished in 0.57 s (12 us/read; 5.14 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             48,536\n",
      "  Read 1 with adapter:                  46,512 (95.8%)\n",
      "  Read 2 with adapter:                  47,148 (97.1%)\n",
      "Pairs written (passing filters):        48,536 (100.0%)\n",
      "\n",
      "Total basepairs processed:    29,068,209 bp\n",
      "  Read 1:    14,534,163 bp\n",
      "  Read 2:    14,534,046 bp\n",
      "Quality-trimmed:              14,177,333 bp (48.8%)\n",
      "  Read 1:     7,015,296 bp\n",
      "  Read 2:     7,162,037 bp\n",
      "Total written (filtered):      7,649,750 bp (26.3%)\n",
      "  Read 1:     3,833,297 bp\n",
      "  Read 2:     3,816,453 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: CTGTCTCTTATACACATCT; Type: regular 3'; Length: 19; Trimmed: 46512 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 31.0%\n",
      "  C: 25.2%\n",
      "  G: 20.4%\n",
      "  T: 23.5%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t31\t758.4\t0\t31\n",
      "4\t24\t189.6\t0\t24\n",
      "5\t22\t47.4\t0\t22\n",
      "6\t19\t11.8\t0\t19\n",
      "7\t9\t3.0\t0\t9\n",
      "8\t14\t0.7\t0\t14\n",
      "9\t28\t0.2\t0\t28\n",
      "10\t17\t0.0\t1\t14 3\n",
      "11\t19\t0.0\t1\t18 1\n",
      "12\t29\t0.0\t1\t27 2\n",
      "13\t23\t0.0\t1\t20 3\n",
      "14\t40\t0.0\t1\t36 4\n",
      "15\t24\t0.0\t1\t22 2\n",
      "16\t21\t0.0\t1\t19 2\n",
      "17\t83\t0.0\t1\t80 3\n",
      "18\t19\t0.0\t1\t17 2\n",
      "19\t60\t0.0\t1\t53 7\n",
      "20\t41\t0.0\t1\t38 3\n",
      "21\t74\t0.0\t1\t70 4\n",
      "22\t69\t0.0\t1\t60 9\n",
      "23\t75\t0.0\t1\t72 3\n",
      "24\t60\t0.0\t1\t52 8\n",
      "25\t48\t0.0\t1\t42 6\n",
      "26\t52\t0.0\t1\t50 2\n",
      "27\t38\t0.0\t1\t33 5\n",
      "28\t110\t0.0\t1\t103 7\n",
      "29\t125\t0.0\t1\t109 16\n",
      "30\t123\t0.0\t1\t115 8\n",
      "31\t98\t0.0\t1\t82 16\n",
      "32\t54\t0.0\t1\t51 3\n",
      "33\t70\t0.0\t1\t64 6\n",
      "34\t92\t0.0\t1\t83 9\n",
      "35\t111\t0.0\t1\t101 10\n",
      "36\t203\t0.0\t1\t186 17\n",
      "37\t386\t0.0\t1\t360 26\n",
      "38\t96\t0.0\t1\t88 8\n",
      "39\t117\t0.0\t1\t103 14\n",
      "40\t80\t0.0\t1\t70 10\n",
      "41\t202\t0.0\t1\t188 14\n",
      "42\t226\t0.0\t1\t206 20\n",
      "43\t773\t0.0\t1\t714 59\n",
      "44\t94\t0.0\t1\t85 9\n",
      "45\t39\t0.0\t1\t34 5\n",
      "46\t33\t0.0\t1\t31 2\n",
      "47\t52\t0.0\t1\t48 4\n",
      "48\t46\t0.0\t1\t41 5\n",
      "49\t76\t0.0\t1\t74 2\n",
      "50\t55\t0.0\t1\t52 3\n",
      "51\t163\t0.0\t1\t156 7\n",
      "52\t118\t0.0\t1\t113 5\n",
      "53\t127\t0.0\t1\t117 10\n",
      "54\t203\t0.0\t1\t193 10\n",
      "55\t195\t0.0\t1\t185 10\n",
      "56\t291\t0.0\t1\t281 10\n",
      "57\t158\t0.0\t1\t152 6\n",
      "58\t123\t0.0\t1\t117 6\n",
      "59\t170\t0.0\t1\t164 6\n",
      "60\t233\t0.0\t1\t229 4\n",
      "61\t236\t0.0\t1\t229 7\n",
      "62\t133\t0.0\t1\t130 3\n",
      "63\t257\t0.0\t1\t243 14\n",
      "64\t327\t0.0\t1\t319 8\n",
      "65\t373\t0.0\t1\t359 14\n",
      "66\t464\t0.0\t1\t452 12\n",
      "67\t335\t0.0\t1\t329 6\n",
      "68\t305\t0.0\t1\t297 8\n",
      "69\t486\t0.0\t1\t472 14\n",
      "70\t341\t0.0\t1\t335 6\n",
      "71\t443\t0.0\t1\t431 12\n",
      "72\t591\t0.0\t1\t577 14\n",
      "73\t1189\t0.0\t1\t1161 28\n",
      "74\t1870\t0.0\t1\t1830 40\n",
      "75\t3081\t0.0\t1\t2973 108\n",
      "76\t24318\t0.0\t1\t24070 248\n",
      "77\t4552\t0.0\t1\t4447 105\n",
      "78\t898\t0.0\t1\t874 24\n",
      "79\t165\t0.0\t1\t159 6\n",
      "80\t43\t0.0\t1\t42 1\n",
      "81\t11\t0.0\t1\t11\n",
      "82\t1\t0.0\t1\t1\n",
      "83\t2\t0.0\t1\t2\n",
      "84\t1\t0.0\t1\t1\n",
      "86\t1\t0.0\t1\t1\n",
      "88\t1\t0.0\t1\t1\n",
      "92\t1\t0.0\t1\t1\n",
      "93\t1\t0.0\t1\t0 1\n",
      "121\t1\t0.0\t1\t0 1\n",
      "122\t1\t0.0\t1\t1\n",
      "142\t1\t0.0\t1\t1\n",
      "161\t1\t0.0\t1\t1\n",
      "165\t1\t0.0\t1\t1\n",
      "172\t1\t0.0\t1\t1\n",
      "184\t1\t0.0\t1\t1\n",
      "196\t1\t0.0\t1\t0 1\n",
      "205\t2\t0.0\t1\t2\n",
      "209\t2\t0.0\t1\t2\n",
      "211\t2\t0.0\t1\t2\n",
      "212\t1\t0.0\t1\t1\n",
      "213\t6\t0.0\t1\t6\n",
      "214\t37\t0.0\t1\t35 2\n",
      "215\t41\t0.0\t1\t37 4\n",
      "216\t5\t0.0\t1\t5\n",
      "221\t1\t0.0\t1\t1\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CTGTCTCTTATACACATCT; Type: regular 3'; Length: 19; Trimmed: 47148 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 35.1%\n",
      "  C: 22.7%\n",
      "  G: 17.4%\n",
      "  T: 24.8%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t9\t758.4\t0\t9\n",
      "4\t28\t189.6\t0\t28\n",
      "5\t20\t47.4\t0\t20\n",
      "7\t1\t3.0\t0\t1\n",
      "8\t8\t0.7\t0\t8\n",
      "9\t3\t0.2\t0\t3\n",
      "10\t10\t0.0\t1\t8 2\n",
      "11\t14\t0.0\t1\t13 1\n",
      "12\t12\t0.0\t1\t12\n",
      "13\t23\t0.0\t1\t20 3\n",
      "14\t10\t0.0\t1\t9 1\n",
      "15\t18\t0.0\t1\t18\n",
      "16\t38\t0.0\t1\t34 4\n",
      "17\t51\t0.0\t1\t46 5\n",
      "18\t10\t0.0\t1\t10\n",
      "19\t57\t0.0\t1\t54 3\n",
      "20\t37\t0.0\t1\t37\n",
      "21\t16\t0.0\t1\t14 2\n",
      "22\t32\t0.0\t1\t28 4\n",
      "23\t31\t0.0\t1\t31\n",
      "24\t31\t0.0\t1\t29 2\n",
      "25\t60\t0.0\t1\t57 3\n",
      "26\t26\t0.0\t1\t24 2\n",
      "27\t45\t0.0\t1\t42 3\n",
      "28\t73\t0.0\t1\t70 3\n",
      "29\t96\t0.0\t1\t89 7\n",
      "30\t99\t0.0\t1\t97 2\n",
      "31\t164\t0.0\t1\t155 9\n",
      "32\t283\t0.0\t1\t273 10\n",
      "33\t62\t0.0\t1\t57 5\n",
      "34\t123\t0.0\t1\t120 3\n",
      "35\t159\t0.0\t1\t153 6\n",
      "36\t382\t0.0\t1\t368 14\n",
      "37\t431\t0.0\t1\t419 12\n",
      "38\t210\t0.0\t1\t204 6\n",
      "39\t224\t0.0\t1\t221 3\n",
      "40\t177\t0.0\t1\t173 4\n",
      "41\t205\t0.0\t1\t198 7\n",
      "42\t212\t0.0\t1\t207 5\n",
      "43\t167\t0.0\t1\t165 2\n",
      "44\t245\t0.0\t1\t242 3\n",
      "45\t211\t0.0\t1\t198 13\n",
      "46\t615\t0.0\t1\t608 7\n",
      "47\t688\t0.0\t1\t676 12\n",
      "48\t449\t0.0\t1\t443 6\n",
      "49\t630\t0.0\t1\t620 10\n",
      "50\t385\t0.0\t1\t376 9\n",
      "51\t579\t0.0\t1\t569 10\n",
      "52\t619\t0.0\t1\t613 6\n",
      "53\t731\t0.0\t1\t722 9\n",
      "54\t773\t0.0\t1\t766 7\n",
      "55\t835\t0.0\t1\t822 13\n",
      "56\t984\t0.0\t1\t976 8\n",
      "57\t2109\t0.0\t1\t2091 18\n",
      "58\t723\t0.0\t1\t715 8\n",
      "59\t539\t0.0\t1\t533 6\n",
      "60\t591\t0.0\t1\t584 7\n",
      "61\t415\t0.0\t1\t408 7\n",
      "62\t469\t0.0\t1\t458 11\n",
      "63\t2703\t0.0\t1\t2680 23\n",
      "64\t827\t0.0\t1\t817 10\n",
      "65\t520\t0.0\t1\t510 10\n",
      "66\t166\t0.0\t1\t161 5\n",
      "67\t320\t0.0\t1\t319 1\n",
      "68\t314\t0.0\t1\t306 8\n",
      "69\t367\t0.0\t1\t364 3\n",
      "70\t288\t0.0\t1\t286 2\n",
      "71\t119\t0.0\t1\t116 3\n",
      "72\t364\t0.0\t1\t360 4\n",
      "73\t254\t0.0\t1\t253 1\n",
      "74\t272\t0.0\t1\t264 8\n",
      "75\t917\t0.0\t1\t910 7\n",
      "76\t1204\t0.0\t1\t1193 11\n",
      "77\t1607\t0.0\t1\t1580 27\n",
      "78\t2069\t0.0\t1\t2039 30\n",
      "79\t3410\t0.0\t1\t3321 89\n",
      "80\t10442\t0.0\t1\t10340 102\n",
      "81\t4311\t0.0\t1\t4258 53\n",
      "82\t331\t0.0\t1\t315 16\n",
      "83\t74\t0.0\t1\t73 1\n",
      "84\t11\t0.0\t1\t11\n",
      "85\t1\t0.0\t1\t1\n",
      "86\t1\t0.0\t1\t1\n",
      "123\t1\t0.0\t1\t1\n",
      "214\t1\t0.0\t1\t1\n",
      "215\t7\t0.0\t1\t5 2\n",
      "\n",
      "Read1 before filtering:\n",
      "total reads: 48536\n",
      "total bases: 3833297\n",
      "Q20 bases: 3807311(99.3221%)\n",
      "Q30 bases: 3730151(97.3092%)\n",
      "\n",
      "Read2 before filtering:\n",
      "total reads: 48536\n",
      "total bases: 3816453\n",
      "Q20 bases: 3807328(99.7609%)\n",
      "Q30 bases: 3774796(98.9085%)\n",
      "\n",
      "Merged and filtered:\n",
      "total reads: 47417\n",
      "total bases: 3453612\n",
      "Q20 bases: 3434915(99.4586%)\n",
      "Q30 bases: 3372449(97.6499%)\n",
      "\n",
      "Filtering result:\n",
      "reads passed filter: 96022\n",
      "reads failed due to low quality: 0\n",
      "reads failed due to too many N: 4\n",
      "reads failed due to too short: 1046\n",
      "reads with adapter trimmed: 94392\n",
      "bases trimmed due to adapters: 777384\n",
      "reads corrected by overlap analysis: 2108\n",
      "bases corrected by overlap analysis: 2439\n",
      "\n",
      "Duplication rate: 73.3189%\n",
      "\n",
      "Insert size peak (evaluated by paired-end reads): 69\n",
      "\n",
      "Read pairs merged: 47417\n",
      "% of original read pairs: 97.6945%\n",
      "% in reads after filtering: 100%\n",
      "\n",
      "\n",
      "JSON report: 2_fastp/Hrd1_DMS_Rep4_Region5_LDead_report.json\n",
      "HTML report: 2_fastp/Hrd1_DMS_Rep4_Region5_LDead_report.html\n",
      "\n",
      "fastp -i 1_cutadapt/Hrd1_DMS_Rep4_Region5_LDead_R1_cut.fastq.gz -I 1_cutadapt/Hrd1_DMS_Rep4_Region5_LDead_R2_cut.fastq.gz -m --merged_out 2_fastp/Hrd1_DMS_Rep4_Region5_LDead_merge.fastq.gz -o 2_fastp/Hrd1_DMS_Rep4_Region5_LDead_R1_cut.fastq.gz -O 2_fastp/Hrd1_DMS_Rep4_Region5_LDead_R2_cut.fastq.gz -w 12 -h 2_fastp/Hrd1_DMS_Rep4_Region5_LDead_report.html -j 2_fastp/Hrd1_DMS_Rep4_Region5_LDead_report.json \n",
      "fastp v0.20.0, time used: 0 seconds\n",
      "47417 reads; of these:\n",
      "  47417 (100.00%) were unpaired; of these:\n",
      "    46859 (98.82%) aligned 0 times\n",
      "    558 (1.18%) aligned exactly 1 time\n",
      "    0 (0.00%) aligned >1 times\n",
      "1.18% overall alignment rate\n",
      "[bam_sort_core] merging from 0 files and 12 in-memory blocks...\n"
     ]
    }
   ],
   "source": [
    "import os\n",
    "\n",
    "##cores dedicated for multiprocessing steps\n",
    "num_core= 12\n",
    "\n",
    "##base names of the files. Used for input and for output base name; 1 name per line\n",
    "basename_file_input =  'Hrd1_DMS_Input_Name.txt' \n",
    "\n",
    "## fastq input location\n",
    "in_dir = '0_fastq_files/' \n",
    "##output directories\n",
    "cutadapt_dir = '1_cutadapt/'\n",
    "fastp_dir = '2_fastp/'\n",
    "bowtie_dir = '3_bowtie/'\n",
    "\n",
    "## make directories if they don't exist\n",
    "if not os.path.isdir(cutadapt_dir):\n",
    "    os.mkdir(cutadapt_dir)\n",
    "if not os.path.isdir(fastp_dir):\n",
    "    os.mkdir(fastp_dir)\n",
    "if not os.path.isdir(bowtie_dir):\n",
    "    os.mkdir(bowtie_dir)\n",
    "\n",
    "\n",
    "##fastq suffix for read1 and read2 (input)\n",
    "R1_suffix = '_R1_001.fastq.gz'\n",
    "R2_suffix = '_R2_001.fastq.gz'\n",
    "\n",
    "##cutadapt suffix (output)\n",
    "cut_R1_suffix = '_R1_cut.fastq.gz'\n",
    "cut_R2_suffix = '_R2_cut.fastq.gz'\n",
    "\n",
    "##fastp suffix (output)\n",
    "merge_suffix = '_merge.fastq.gz'\n",
    "unpair_R1_suffix = '_R1_unpaired.fastq.gz'\n",
    "unpair_R2_suffix = '_R2_unpaired.fastq.gz'\n",
    "\n",
    "#bowtie/ samtools suffix (output)\n",
    "bowtie_suffix = '_align.sam'\n",
    "##reference is hrd1 coding sequence with 90 bp of promoter and terminator\n",
    "bt_ref = 'hrd1_ref/hrd1_90-prom_90-term'\n",
    "bamfile_suffix ='_align.bam'\n",
    "\n",
    "\n",
    "##open and process 1 paired-end read at a time\n",
    "with open(basename_file_input,'r') as file:\n",
    "    header = file.readline()\n",
    "    for line in file:\n",
    "        basename= line.strip('\\n')\n",
    "        ## cutadapt\n",
    "        cut_r1_in=str(in_dir+basename+R1_suffix)\n",
    "        cut_r2_in=str(in_dir+basename+R2_suffix)\n",
    "        \n",
    "        cut_r1_out = str(cutadapt_dir+basename+cut_R1_suffix)\n",
    "        cut_r2_out = str(cutadapt_dir+basename+cut_R2_suffix)\n",
    "        ## cutadapt trim adaptor and remove 8 N nucleotides directly 3' of \n",
    "        !cutadapt -a CTGTCTCTTATACACATCT -A CTGTCTCTTATACACATCT -u 8 -U 8 -q 30 -o $cut_r1_out -p $cut_r2_out --cores=$num_core $cut_r1_in $cut_r2_in\n",
    "\n",
    "        ##fastp\n",
    "        fastp_merge=str(fastp_dir+basename+merge_suffix)\n",
    "        fastp_unpair_R1= str(fastp_dir+basename+cut_R1_suffix)\n",
    "        fastp_unpair_R2= str(fastp_dir+basename+cut_R2_suffix)\n",
    "        fastp_report_html= str(fastp_dir+basename+'_report.html')\n",
    "        fastp_report_json = str(fastp_dir+basename+'_report.json')\n",
    "        ##fastp merge with default settings\n",
    "        !fastp -i $cut_r1_out -I $cut_r2_out -m --merged_out $fastp_merge -o $fastp_unpair_R1 -O $fastp_unpair_R2 -w $num_core -h $fastp_report_html -j $fastp_report_json\n",
    "\n",
    "        ##bowtie2 align very sensitive\n",
    "        bt_out = str(bowtie_dir+basename+bowtie_suffix)\n",
    "        !bowtie2 -x $bt_ref --very-sensitive --threads $num_core -U $fastp_merge -S $bt_out\n",
    "        \n",
    "        ##samtools sort on name and convert sam to bam file\n",
    "        samtool_out = str(bowtie_dir+basename+bamfile_suffix)\n",
    "        !samtools sort -n -@ $num_core -o $samtool_out $bt_out\n",
    "        ##remove sam file\n",
    "        !rm $bt_out"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "d341777a-41f4-47c6-9caa-2af83e09fba6",
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3 (ipykernel)",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.9.15"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
